var data =
[
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake version of purge_tips",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp40els14h/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "yoshihikosuzuki/purge_tips_smk",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp40els14h/workflow/Snakefile:\n    * Absolute path \"/^S/ \u0026\u0026 int(substr($5,6)) \u003c= thres_cov {{print $2}}\" in line 79:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp40els14h/workflow/common.smk:\n    * Absolute path \"/path/to\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmp40els14h/workflow/hifiasm/../common.smk:\n    * Absolute path \"/path/to\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmp40els14h/workflow/winnowmap/../common.smk:\n    * Absolute path \"/path/to\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule ln_input (line 35, /tmp/tmp40els14h/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hifiasm (line 29, /tmp/tmp40els14h/workflow/hifiasm/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule winnowmap (line 31, /tmp/tmp40els14h/workflow/winnowmap/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule purge_tips (line 97, /tmp/tmp40els14h/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640320026.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/09_virus_clustering.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/02_read_assembly.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/06_virus_taxonomy.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/05_virus_quality.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/04_virus_identification.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/03_virus_enrichment.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppug30_vv/workflow/rules/01_read_preprocessing.smk\":  Formatted content is different from original\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "CarsonJM/phide_piper",
    "latest_release": null,
    "linting": "KeyError in line 8 of /tmp/tmppug30_vv/workflow/Snakefile:\n\u0027sample_info\u0027\n  File \"/tmp/tmppug30_vv/workflow/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640308835.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-configuration-settings\" class=\"anchor\" href=\"#configuration-settings\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eConfiguration Settings\u003c/h1\u003e\n",
    "data_format": 2,
    "description": "kGWASflow is a Snakemake workflow for performing k-mers-based GWAS.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/filter_kmers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/align_reads.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/trimming.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/combine_and_filter_kmers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/fetch_kmers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/fetch_reads.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/kmers_gwas.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/count_kmers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/summarize_results.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/generate_kinship_matrix.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/align_kmers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/generate_kmers_table.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/ref.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/build_kmers_gwas.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmppzyt07sz/workflow/rules/common.smk\":  Formatted content is different from original\n[INFO] 17 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "akcorut/kGWASflow",
    "latest_release": null,
    "linting": "WorkflowError in line 13 of /tmp/tmppzyt07sz/workflow/Snakefile:\nWorkflow defines configfile ../.test/config_arabidopsis/config.yaml but it is not present or accessible.\n  File \"/tmp/tmppzyt07sz/workflow/Snakefile\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": null,
    "report": false,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "genomics",
      "gwas",
      "gwas-pipeline",
      "kmers",
      "bioinformatics-pipeline",
      "ngs",
      "sciworkflows",
      "bioinformatics",
      "conda",
      "pipeline"
    ],
    "updated_at": 1640273793.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Collection of rules used to create references, panel of normals (PoN), and background filters.",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/create_artifact_file.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/gatk_cnv_panel_of_normal.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/create_background_file.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/msi_sensor_pro_panel_of_normal.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdnm_11o7/workflow/rules/cnvkit_panel_of_normal.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "hydra-genetics/references",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 13 of /tmp/tmpdnm_11o7/workflow/rules/common.smk:\nNo module named \u0027hydra_genetics\u0027\n  File \"/tmp/tmpdnm_11o7/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmpdnm_11o7/workflow/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1640253948.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-configuration-options\" class=\"anchor\" href=\"#configuration-options\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eConfiguration options\u003c/h1\u003e\n\u003cp\u003eA description of every valid option in \u003ccode\u003econfig.yaml\u003c/code\u003e.\u003c/p\u003e\n\u003cp\u003ee.g.\u003c/p\u003e\n\u003cp\u003eA prefix for outputs.\u003c/p\u003e\n\u003cdiv class=\"highlight highlight-source-yaml\"\u003e\u003cpre\u003e\u003cspan class=\"pl-ent\"\u003emanifest\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Manifest file location (OPTIONAL)\u003c/span\u003e\n\u003cspan class=\"pl-ent\"\u003eREADS\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Dictionary for indexed alignments of raw reads\u003c/span\u003e\n  \u003cspan class=\"pl-ent\"\u003eONT\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Example name, can be anything\u003c/span\u003e\n\u003cspan class=\"pl-ent\"\u003eASM\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Dictionary for indexed alignments of assemblies\u003c/span\u003e\n  \u003cspan class=\"pl-ent\"\u003eHIFIASM\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Example name, could be anything\u003c/span\u003e\n\u003cspan class=\"pl-ent\"\u003eSVPOP\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Dictionary for intersect.tab produced by svpop (https://github.com/EichlerLab/svpop)\u003c/span\u003e\n  \u003cspan class=\"pl-ent\"\u003ePBSV\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Example name, could be anything\u003c/span\u003e\n\u003cspan class=\"pl-ent\"\u003eCALLABLE\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Location of callable regions\u003c/span\u003e\n  \u003cspan class=\"pl-ent\"\u003ePAV\u003c/span\u003e: \u003cspan class=\"pl-c\"\u003e\u003cspan class=\"pl-c\"\u003e#\u003c/span\u003e Bed file locations of valid regions per haplotype in the parents\u003c/span\u003e\n\u003c/pre\u003e\u003c/div\u003e\n",
    "data_format": 2,
    "description": "Pipeline for validation of de novo SVs",
    "formatting": "[DEBUG] \n[ERROR] In file \"/tmp/tmpk7c8nwz6/workflow/rules/callable.smk\":  \u0027NoParametersError: L16: In input definition.\u0027\n[DEBUG] In file \"/tmp/tmpk7c8nwz6/workflow/rules/callable.smk\":  \n[DEBUG] In file \"/tmp/tmpk7c8nwz6/workflow/rules/msa.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpk7c8nwz6/workflow/rules/subseq.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpk7c8nwz6/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpk7c8nwz6/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpk7c8nwz6/workflow/rules/combine.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 4:0: elif val_method == \u0027CALLABLE\u0027:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpk7c8nwz6/workflow/rules/combine.smk\":  2 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpk7c8nwz6/workflow/rules/combine.smk\":  4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "wharvey31/denovo_sv_validation",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 8 of /tmp/tmpk7c8nwz6/workflow/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpk7c8nwz6/workflow/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "mandatory_flags": null,
    "report": true,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640221029.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-workflow-configuration\" class=\"anchor\" href=\"#workflow-configuration\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eWorkflow configuration\u003c/h1\u003e\n\u003ch2\u003e\n\u003ca id=\"user-content-configuration\" class=\"anchor\" href=\"#configuration\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eConfiguration\u003c/h2\u003e\n\u003cp\u003eTo configure the workflow, modifiy \u003ccode\u003econfig/config.yaml\u003c/code\u003e according\nto your needs.\u003c/p\u003e\n\u003cp\u003eFor specific details about the parameters\ncheck the main README.\u003c/p\u003e\n\u003ch2\u003e\n\u003ca id=\"user-content-sample-sheet\" class=\"anchor\" href=\"#sample-sheet\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eSample sheet\u003c/h2\u003e\n\u003cp\u003eAdd your samples to \u003ccode\u003econfig/samples.tsv\u003c/code\u003e or use the\nthe helper script from Bundes Insitut f\u00fcr Risikobewertung in\n\u003ccode\u003eressources/creat_sampleSheet.sh\u003c/code\u003e.\u003c/p\u003e\n",
    "data_format": 2,
    "description": "A pipeline for Food DNA Metabarcoding analysis",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/dada2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/vsearch.smk\":  Formatted content is different from original\n[DEBUG] \n[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 2 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "CVUA-RRW/FooDMe",
    "latest_release": "1.4.2",
    "linting": "Lints for snakefile /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk:\n    * Absolute path \"/\" -f1)\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule get_primer_revcomp (line 11, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule primer_trimming_stats (line 166, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule parse_fastp (line 304, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule trimming_stats (line 345, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_trimming_stats (line 374, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule unpack_fastq (line 8, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/dada2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule denoise (line 40, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/dada2.smk):\n    * Param sample_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule collect_denoising_stats (line 93, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/dada2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prep_taxonomy (line 10, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_taxid_from_db (line 36, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_blast_mask (line 75, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule apply_blocklist (line 108, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule no_masking (line 133, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule no_blocklist (line 157, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_blast (line 274, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule find_consensus (line 321, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule blast_stats (line 353, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_blast_stats (line 452, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tax_stats (line 488, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_tax_stats (line 558, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule summarize_results (line 597, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule collect_results (line 644, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule krona_table (line 8, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule krona (line 33, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule krona_all (line 64, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summary_report (line 120, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_summaries (line 226, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_sample (line 265, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param sample is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_all (line 316, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule software_versions (line 370, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule database_version (line 463, /tmp/tmpou1ubevj/CVUA-RRW-FooDMe-f638487/workflow/rules/reports.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": null,
    "report": true,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "ngs",
      "ngs-pipeline",
      "metabarcoding",
      "targeted-sequencing",
      "snakemake",
      "food-monitoring",
      "public-health",
      "food-authenticity",
      "workflow",
      "pipeline"
    ],
    "updated_at": 1640171673.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Data and code associated with the upcoming manuscript on killifish microbiota",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp_552w5fs/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "microlei/killifish_microbiome",
    "latest_release": null,
    "linting": "WorkflowError in line 40 of /tmp/tmp_552w5fs/Snakefile:\nDuplicate output file pattern in rule plotQP. First two duplicate for entries R1 and R2\n  File \"/tmp/tmp_552w5fs/Snakefile\", line 40, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640148266.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp89a1l0p2/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "BeeCSI-Microbiome/taxonomic_profiling_pipeline",
    "latest_release": null,
    "linting": "KeyError in line 13 of /tmp/tmp89a1l0p2/Snakefile:\n\"None of [\u0027sample\u0027] are in the columns\"\n  File \"/tmp/tmp89a1l0p2/Snakefile\", line 13, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/core/frame.py\", line 5451, in set_index\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1640127261.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "General rules could be used for specific purpose",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/coverage.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/fastp.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/bowtie2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/merge.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/bwa.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/filter.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/download.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/trim.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpv3ate1ws/rules/global.smk\":  Formatted content is different from original\n[INFO] 11 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "yztxwd/snakemake-pipeline-general",
    "latest_release": null,
    "linting": "TypeError in line 9 of /tmp/tmpv3ate1ws/Snakefile:\nunhashable type: \u0027collections.OrderedDict\u0027\n  File \"/tmp/tmpv3ate1ws/Snakefile\", line 9, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/__init__.py\", line 81, in load\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/constructor.py\", line 51, in get_single_data\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/constructor.py\", line 55, in construct_document\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/constructor.py\", line 100, in construct_object\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/constructor.py\", line 155, in construct_pairs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/yaml/constructor.py\", line 100, in construct_object\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640115255.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow to analyze bulk RNA-seq ",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp6rj1u34i/workflow/rules/mapping.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp6rj1u34i/workflow/rules/splicing.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp6rj1u34i/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp6rj1u34i/workflow/rules/common.smk\":  Formatted content is different from original\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "khayer/rna_seq_standard_pipeline",
    "latest_release": null,
    "linting": "KeyError in line 4 of /tmp/tmp6rj1u34i/workflow/rules/common.smk:\n\u0027samples\u0027\n  File \"/tmp/tmp6rj1u34i/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmp6rj1u34i/workflow/rules/common.smk\", line 4, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1640104575.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/add_multi_snv_in_codon.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/bgzip_vcf.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/background_filter.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/filter_vcf_on_format.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/tabix_vcf.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/artifact_filter.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpubllfuol/workflow/rules/sort_vcf.smk\":  Formatted content is different from original\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "hydra-genetics/filtering",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 13 of /tmp/tmpubllfuol/workflow/rules/common.smk:\nNo module named \u0027hydra_genetics\u0027\n  File \"/tmp/tmpubllfuol/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmpubllfuol/workflow/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1640087793.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Collections of rules to annotate vcf files.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpfn14ydkj/workflow/rules/calculate_seqrun_background.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfn14ydkj/workflow/rules/vep.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfn14ydkj/workflow/rules/common.smk\":  Formatted content is different from original\n[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "hydra-genetics/annotation",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 13 of /tmp/tmpfn14ydkj/workflow/rules/common.smk:\nNo module named \u0027hydra_genetics\u0027\n  File \"/tmp/tmpfn14ydkj/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmpfn14ydkj/workflow/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1640087596.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "A Snakemake workflow for visualising UK Covid-19 data",
    "formatting": null,
    "full_name": "rbpatt2019/uk_covid_data",
    "latest_release": "v0.1.0",
    "linting": null,
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640076249.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Pipeline to process duplex sequencing data.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/variants.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/umi_correct.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/umi.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/consensus.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/preprocess.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/align.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpdajsj7tg/workflow/rules/common.smk\":  Formatted content is different from original\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "WEHIGenomicsRnD/duplex-seq-pipeline",
    "latest_release": null,
    "linting": "Lints for rule align (line 51, /tmp/tmpdajsj7tg/workflow/rules/align.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge_bam_alignment (line 95, /tmp/tmpdajsj7tg/workflow/rules/align.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remap_align (line 208, /tmp/tmpdajsj7tg/workflow/rules/align.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remap_merge_bam_alignment (line 252, /tmp/tmpdajsj7tg/workflow/rules/align.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule call_duplex_consensus (line 1, /tmp/tmpdajsj7tg/workflow/rules/consensus.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule filter_consensus_reads (line 53, /tmp/tmpdajsj7tg/workflow/rules/consensus.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clip_bam (line 1, /tmp/tmpdajsj7tg/workflow/rules/variants.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule varscan (line 53, /tmp/tmpdajsj7tg/workflow/rules/variants.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_umis (line 1, /tmp/tmpdajsj7tg/workflow/rules/umi_correct.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule correct_umis (line 51, /tmp/tmpdajsj7tg/workflow/rules/umi_correct.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule group_reads_by_umi (line 102, /tmp/tmpdajsj7tg/workflow/rules/umi_correct.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640045410.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "genome tRNA annotation",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp5s1xnbxt/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ibmp-bip/tRNAflow",
    "latest_release": null,
    "linting": "FileNotFoundError in line 24 of /tmp/tmp5s1xnbxt/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027Genome_description.txt\u0027\n  File \"/tmp/tmp5s1xnbxt/workflow/Snakefile\", line 24, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 683, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 482, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 811, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 1040, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 51, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/base_parser.py\", line 222, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/common.py\", line 702, in get_handle\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640004486.0
  },
  {
    "config_readme": "\u003cp\u003ePlease follow the instructions in the template config.yaml that is copied to your deployment via snakedeploy.\u003c/p\u003e\n",
    "data_format": 2,
    "description": "A snakemake workflow for benchmarking variant calling approaches with Genome in a Bottle (GIAB)",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpf62tc8yv/snakemake-workflows-benchmark-giab-97c0b1b/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 3 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "snakemake-workflows/benchmark-giab",
    "latest_release": "v1.0.1",
    "linting": null,
    "mandatory_flags": {
      "desc": null,
      "flags": null
    },
    "report": true,
    "software_stack_deployment": {
      "conda": true,
      "singularity": false,
      "singularity+conda": false
    },
    "standardized": true,
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1640001126.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow for generating read count data from CRISPR screens.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpqp2k9dpg/workflow/rules/gene_essentiality.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpqp2k9dpg/workflow/rules/guide_mapping.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpqp2k9dpg/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpqp2k9dpg/workflow/rules/read_counts.smk\":  Formatted content is different from original\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "david-a-parry/perturbseq_workflow",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpqp2k9dpg/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqp2k9dpg/workflow/rules/guide_mapping.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqp2k9dpg/workflow/rules/read_counts.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpqp2k9dpg/workflow/rules/gene_essentiality.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule bt_index_reference (line 71, /tmp/tmpqp2k9dpg/workflow/rules/guide_mapping.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bowtie_map (line 161, /tmp/tmpqp2k9dpg/workflow/rules/guide_mapping.smk):\n    * Param ref_idx is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule combine_read_counts (line 37, /tmp/tmpqp2k9dpg/workflow/rules/read_counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_guide_counts (line 63, /tmp/tmpqp2k9dpg/workflow/rules/read_counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_bagel_fc (line 59, /tmp/tmpqp2k9dpg/workflow/rules/gene_essentiality.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule run_bagel_bf (line 95, /tmp/tmpqp2k9dpg/workflow/rules/gene_essentiality.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule regression_analysis (line 306, /tmp/tmpqp2k9dpg/workflow/rules/gene_essentiality.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639958104.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Phylogeography of SARS-CoV-2",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpcu9p671k/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ktmeaton/ncov-phylogeography",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpcu9p671k/workflow/Snakefile:\n    * Absolute path \"/workflow.rst\" in line 47:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 105:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.{ext}\" in line 129:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.fna\" in line 167:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/reference/{sample}/{sample}.gbff\" in line 168:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.tab\" in line 174:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.txt\" in line 175:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.aligned.fa\" in line 176:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.subs.vcf\" in line 177:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.raw.vcf\" in line 178:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.log\" in line 179:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/\" + \" in line 213:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/reference/{sample}/{sample}.fna\" in line 218:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.full.aln\" in line 222:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.log\" in line 223:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.constant_sites.txt\" in line 224:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.filter.aln\" in line 225:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.filter.log\" in line 226:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.filter.aln\" in line 270:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.constant_sites.txt\" in line 271:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree.nwk\" in line 274:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree.iqtree\" in line 275:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree.log\" in line 276:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree\" in line 283:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree.nwk\" in line 320:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.full.aln\" in line 321:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/reference/{sample}/{sample}.gff\" in line 322:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/{reads_origin}/augur.nwk\" in line 327:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/{reads_origin}/nt_muts.json\" in line 328:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/{reads_origin}/aa_muts.json\" in line 329:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/iqtree.nwk\" in line 364:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 365:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/{reads_origin}/nt_muts.json\" in line 366:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/{reads_origin}/aa_muts.json\" in line 367:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/config/colors.tsv\" in line 368:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/{reads_origin}/iqtree.json\" in line 371:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/nucleotide/metadata.tsv\" in line 394:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/nucleotide/snippy-multi.log\" in line 438:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/nucleotide/iqtree.nwk\" in line 445:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd2/nucleotide/lsd.log\" in line 447:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/augur/nucleotide/nt_muts.json\" in line 452:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/nucleotide/iqtree.json\" in line 457:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Environment variable CONDA_CACHEDIR used but not asserted with envvars directive in line 60.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_CONDA_CACHEDIR used but not asserted with envvars directive in line 61.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable CONDA_CACHEDIR used but not asserted with envvars directive in line 63.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_CONDA_CACHEDIR used but not asserted with envvars directive in line 64.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LANGUAGE used but not asserted with envvars directive in line 68.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LANG used but not asserted with envvars directive in line 70.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LC_ALL used but not asserted with envvars directive in line 72.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 47:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 129:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 167:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 173:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 175:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 176:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 177:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 178:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 179:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 213:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 218:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 222:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 223:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 224:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 225:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 226:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 270:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 271:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 274:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 275:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 276:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 282:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 320:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 321:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 322:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 327:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 328:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 329:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 364:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 365:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 366:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 367:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 368:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 371:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 373:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 394:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 438:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 445:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 447:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 452:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 457:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule metadata (line 101, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_nucleotide (line 140, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_pairwise (line 203, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_multi (line 281, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule iqtree (line 377, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule augur (line 461, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule auspice (line 535, /tmp/tmpcu9p671k/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639941631.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/00_common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/01_ref.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk\":  \u0027NoParametersError: L129: In output definition.\u0027\n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk\":  \n[ERROR] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:7: params:\n```\nwhile formatting code with black.\u0027\n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk\":  \n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk\":  Formatted content is different from original\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "brettellebi/somites",
    "latest_release": "v0.0.1",
    "linting": "Lints for snakefile /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk:\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk:\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk:\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk:\n    * Absolute path \"/|\\|\" in line 40:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/hps/nobackup/birney/users/ian/somites/dp4s/batch_01/bwamem2/11.dp4.txt\" in line 93:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/hps/nobackup/birney/users/ian/somites/data/sites_files/F0_Cab_Kaga/final/all.txt\" in line 97:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_genome (line 1, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/01_ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genome_faidx (line 23, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/01_ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genome_dict (line 42, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/01_ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_mem2_index (line 77, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/01_ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_reads (line 1, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule replace_rg (line 28, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_sam (line 61, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_duplicates (line 95, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_bams (line 132, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 167, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/02_F0_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule haplotype_caller (line 1, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combine_calls (line 32, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genotype_variants (line 55, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_variants (line 75, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/03_F0_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_mem2_mem (line 66, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_sam_f2 (line 112, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_duplicates_f2 (line 158, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index_f2 (line 204, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/04_F2_mapping.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_homozygous_sites (line 2, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_divergent_sites (line 51, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule bind_divergent_sites (line 81, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule bam_readcount (line 105, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_dp_AB (line 144, /tmp/tmpnvhqc5fx/brettellebi-somites-f9896e9/workflow/rules/05_F2_recomb_blocks.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639915479.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-general-settings\" class=\"anchor\" href=\"#general-settings\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eGeneral settings\u003c/h1\u003e\n\u003cp\u003eTo configure this workflow, modify \u003ccode\u003econfig/config.yaml\u003c/code\u003e according to your needs, following the explanations provided in the file.\u003c/p\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-sample-and-unit-sheet\" class=\"anchor\" href=\"#sample-and-unit-sheet\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eSample and unit sheet\u003c/h1\u003e\n\u003cul\u003e\n\u003cli\u003eAdd samples to \u003ccode\u003econfig/samples.tsv\u003c/code\u003e. For each sample, the columns \u003ccode\u003esample_name\u003c/code\u003e, and \u003ccode\u003econdition\u003c/code\u003e have to be defined. The \u003ccode\u003econdition\u003c/code\u003e (healthy/tumor, before Treatment / after Treatment) will be used as contrast for the DEG analysis in DESeq2. To include other relevant variables such as batches, add a new column to the sheet.\u003c/li\u003e\n\u003cli\u003eFor each sample, add one or more sequencing units (runs, lanes or replicates) to the unit sheet \u003ccode\u003econfig/units.tsv\u003c/code\u003e. By activating or deactivating \u003ccode\u003emergeReads\u003c/code\u003e in the \u003ccode\u003econfig/config.yaml\u003c/code\u003e, you can decide wether to merge replicates or run them individually. For each unit, define adapters, and either one (column \u003ccode\u003efq1\u003c/code\u003e) or two (columns \u003ccode\u003efq1\u003c/code\u003e, \u003ccode\u003efq2\u003c/code\u003e) FASTQ files (these can point to anywhere in your system). Alternatively, you can define an SRA (sequence read archive) accession (starting with e.g. ERR or SRR) by using a column \u003ccode\u003esra\u003c/code\u003e. In the latter case, the pipeline will automatically download the corresponding paired end reads from SRA. If both local files and SRA accession are available, the local files will be preferred.\nTo choose the correct geneCounts produced by STAR, you can define the strandedness of a unit. STAR produces counts for unstranded (\u0027None\u0027 - default), forward oriented (\u0027yes\u0027) and reverse oriented (\u0027reverse\u0027) protocols.\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003eMissing values can be specified by empty columns or by writing \u003ccode\u003eNA\u003c/code\u003e.\u003c/p\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-deseq-scenario\" class=\"anchor\" href=\"#deseq-scenario\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eDESeq scenario\u003c/h1\u003e\n\u003cp\u003eTo initialize the DEG analysis, you need to define a model in the \u003ccode\u003econfig/config.yaml\u003c/code\u003e. The model can include all variables introduced as columns in \u003ccode\u003econfig/samples.tsv\u003c/code\u003e.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eThe standard model is \u003ccode\u003e~condition\u003c/code\u003e - to include a batch variable, write \u003ccode\u003e~batch + condition\u003c/code\u003e.\u003c/li\u003e\n\u003c/ul\u003e\n",
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/haplogatk.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/Strelka.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/SequenzaFromVarscan.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/mutect2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/vcftoMAFindel.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpwg31tlld/workflow/rules/mutect1.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:6: shell:\n```\nwhile formatting code with black.\u0027\n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/mutect1.smk\":  \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/varscan.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpwg31tlld/workflow/rules/align.smk\":  \u0027NoParametersError: L8: In params definition.\u0027\n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/align.smk\":  \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/Sequenza.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/VCFIntersect.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpwg31tlld/workflow/rules/vcftoMAFsnv.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:7: output:\n```\nwhile formatting code with black.\u0027\n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/rules/vcftoMAFsnv.smk\":  \n[DEBUG] In file \"/tmp/tmpwg31tlld/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 3 file(s) raised parsing errors \ud83e\udd15\n[INFO] 10 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "elsamah/ExomeSeq_wNorm",
    "latest_release": null,
    "linting": "PermissionError in line 5 of /tmp/tmpwg31tlld/workflow/Snakefile:\n[Errno 13] Permission denied: \u0027/cluster\u0027\n  File \"/tmp/tmpwg31tlld/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/os.py\", line 225, in makedirs\n",
    "mandatory_flags": null,
    "report": true,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639769220.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Build resitance gene catelogue based the RGI output from CARD, and do alignements (packed in the Snakemake workflow) ",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpyz1btz8w/Snakefile\":  Inline-formatted keyword \"threads\" at line 122 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpyz1btz8w/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "yanhui09/RGIcat",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpyz1btz8w/Snakefile:\n    * Absolute path \"/{dir}/{file}.fastq.gz\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/count_matrix.tsv\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.fasta\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/kept.tsv\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/extract_AMG.log\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.fasta\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.cdhit.fasta\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/cdhit.log\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.cdhit.fasta\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.cdhit\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/bowtie2_build/build.log\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}\" in line 74:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}/{sample}_R1.fastq.gz\" in line 76:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}/{sample}_R2.fastq.gz\" in line 77:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}/{sample}_R1.fastq.gz\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}/{sample}_R2.fastq.gz\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.bam\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/bowtie2/{sample}.log\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/AMG/AMG.cdhit\" in line 95:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.bam\" in line 103:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam\" in line 105:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/samtools/sorted/{sample}.log\" in line 109:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam\" in line 115:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam.bai\" in line 117:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/samtools_index/{sample}.log\" in line 119:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam\" in line 129:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam.bai\" in line 130:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/gene_names\" in line 132:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/gene_names.log\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam\" in line 145:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped/{sample}.sorted.bam.bai\" in line 146:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/counts/{sample}.count\" in line 148:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/counts/{sample}.log\" in line 152:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/gene_names\" in line 161:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/counts/{sample}.count\" in line 162:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/count_matrix.tsv\" in line 164:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/count_matrix.log\" in line 166:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 74:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 77:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 115:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 117:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 129:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 132:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 145:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 146:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 148:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 152:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 161:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 164:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 15.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cleanup_fq_names (line 134, /tmp/tmpyz1btz8w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bwt2_map (line 160, /tmp/tmpyz1btz8w/Snakefile):\n    * Param index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule count_matrix (line 322, /tmp/tmpyz1btz8w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639760540.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow to run pangolin on all SARS-CoV-2 sequences in ENA",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/join_json.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/split_embl_file.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/make_json_header_footer.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/copy_embl_file.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/run_pangolin.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpkszo5p3c/workflow/rules/convert_to_json.smk\":  Formatted content is different from original\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "carlacummins/cv19-lineages-workflow",
    "latest_release": null,
    "linting": "KeyError in line 2 of /tmp/tmpkszo5p3c/workflow/Snakefile:\n\u0027embl_input\u0027\n  File \"/tmp/tmpkszo5p3c/workflow/Snakefile\", line 2, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639754114.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/ReadFunction.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/QC.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/BinQA.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/BamProcessFromFastq.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/CoAssembly.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/ReadClassification.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/Assembly.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/BamProcessRaw.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/MapCoverageBin.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfaoo07vr/workflow/rules/Functions.smk\":  Formatted content is different from original\n[INFO] 11 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "aidanfoo96/MINUUR",
    "latest_release": null,
    "linting": "WorkflowError in line 10 of /tmp/tmpfaoo07vr/workflow/Snakefile:\nWorkflow defines configfile ../config/config.yaml but it is not present or accessible.\n  File \"/tmp/tmpfaoo07vr/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639753032.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Multiple sequence alignment pipeline",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp40t2xl0b/workflow/rules/chain_and_net.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp40t2xl0b/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp40t2xl0b/workflow/rules/roast.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp40t2xl0b/workflow/rules/align.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmp40t2xl0b/workflow/rules/call_conservation.smk\":  \u0027SyntaxError: L164: Colon (not \u0027maf_split\u0027) expected after \u0027checkpoint rule\u0027\u0027\n[INFO] In file \"/tmp/tmp40t2xl0b/workflow/rules/call_conservation.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmp40t2xl0b/workflow/rules/call_conservation.smk\":  4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ascheben/msa_pipeline",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk:\n    * Absolute path \"/^\u003e/! s/N/-/g\" in line 101:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule lastdb_index (line 62, /tmp/tmp40t2xl0b/workflow/./rules/align.smk):\n    * Param indexBase is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule build_index (line 123, /tmp/tmp40t2xl0b/workflow/./rules/align.smk):\n    * Param speciesSizeFile is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule align_single (line 232, /tmp/tmp40t2xl0b/workflow/./rules/align.smk):\n    * Param speciesPath is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param speciesSizeFile is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule align_split (line 307, /tmp/tmp40t2xl0b/workflow/./rules/align.smk):\n    * Param splitDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param speciesPath is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fa_to_2bit (line 25, /tmp/tmp40t2xl0b/workflow/./rules/chain_and_net.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule axt_chain_prenet (line 53, /tmp/tmp40t2xl0b/workflow/./rules/chain_and_net.smk):\n    * Param pslFiles is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule write_tree (line 35, /tmp/tmp40t2xl0b/workflow/./rules/roast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_roast (line 137, /tmp/tmp40t2xl0b/workflow/./rules/roast.smk):\n    * Param inputDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param outputDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule clean_maf (line 123, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule raxml (line 258, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * Param workDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule phylofit (line 305, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule wig2bed (line 468, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_phylop_header (line 498, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gerp2bed (line 524, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule maf2cov (line 558, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_cov (line 584, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_gerp_header (line 615, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gerp_clean (line 655, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule phylop_clean (line 685, /tmp/tmp40t2xl0b/workflow/./rules/call_conservation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639751355.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Collection of scripts for the Bayesian GGM project",
    "formatting": "[DEBUG] \n[ERROR] In file \"/tmp/tmpgfi0o7xy/Snakefile\":  \u0027SyntaxError: L594: Unrecognised keyword \u0027snp_summary\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmpgfi0o7xy/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "jhawe/bggm",
    "latest_release": null,
    "linting": "ValueError in line 85 of /tmp/tmpgfi0o7xy/workflows/2_2_simulation.sm:\nlist.remove(x): x not in list\n  File \"/tmp/tmpgfi0o7xy/Snakefile\", line 64, in \u003cmodule\u003e\n  File \"/tmp/tmpgfi0o7xy/workflows/2_2_simulation.sm\", line 85, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1639745132.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpk_xsmvek/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "nioo-knaw/orochi",
    "latest_release": null,
    "linting": "KeyError in line 3 of /tmp/tmpk_xsmvek/src/rules/core/taxonomic-annotation/cat.smk:\n\u0027assembler\u0027\n  File \"/tmp/tmpk_xsmvek/Snakefile\", line 15, in \u003cmodule\u003e\n  File \"/tmp/tmpk_xsmvek/src/rules/core/taxonomic-annotation/cat.smk\", line 3, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639734772.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "miRNA analysis pipeline",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/umi_and_trimming.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/samtools.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/delivery.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/mapping.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp75tnn_ye/rules/count_mirna.smk\":  Formatted content is different from original\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "solida-core/miRNA",
    "latest_release": null,
    "linting": "KeyError in line 4 of /tmp/tmp75tnn_ye/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmp75tnn_ye/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1639730787.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Analysis code for Olink CVD1 - HF analysis",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpzo7uupdx/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "alhenry/cvd1-hf",
    "latest_release": null,
    "linting": "Lints for rule obs_analysis (line 21, /tmp/tmpzo7uupdx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MR_analysis (line 44, /tmp/tmpzo7uupdx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_MR (line 69, /tmp/tmpzo7uupdx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639711032.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Mapping mutations to the SARS-CoV-2 B.1.351 RBD that reduce binding of polyclonal plasmas from B.1.351-convalescent individuals",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpg5g43752/Snakefile\":  Keyword \"input\" at line 265 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpg5g43752/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:0: elif config[\u0027seqdata_source\u0027] == \u0027SRA\u0027:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpg5g43752/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_B.1.351",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 10 of /tmp/tmpg5g43752/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpg5g43752/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1639709375.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "A Snakemake pipeline for variant calling of genomic FASTQ data using GATK",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpmf4gru71/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpmf4gru71/workflow/rules/phasing.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpmf4gru71/workflow/rules/relations.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmpmf4gru71/workflow/rules/variant_calling.smk\":  Keyword \"shell\" at line 156 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpmf4gru71/workflow/rules/variant_calling.smk\":  Formatted content is different from original\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "allytrope/variant-analysis",
    "latest_release": null,
    "linting": "KeyError in line 130 of /tmp/tmpmf4gru71/workflow/rules/variant_calling.smk:\n\u0027map\u0027\n  File \"/tmp/tmpmf4gru71/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmpmf4gru71/workflow/rules/variant_calling.smk\", line 130, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639690550.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "A RNA-seq workflow.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/rules/hisat2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/rules/cutadapt.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/rules/fastqc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/rules/trim.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp70j4fnss/workflow/rules/featureCounts.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "zerostwo/rna-seq",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp70j4fnss/workflow/rules/featureCounts.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "snakemake-workflows",
      "rna-seq"
    ],
    "updated_at": 1639685188.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Create metamers using models of the ventral stream and run experiments to validate them",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpt8zwal0n/Snakefile\":  Inline-formatted keyword \"configfile\" at line 14 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpt8zwal0n/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "billbrod/foveated-metamers",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 5 of /tmp/tmpt8zwal0n/Snakefile:\nNo module named \u0027imageio\u0027\n  File \"/tmp/tmpt8zwal0n/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1639678496.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Workflow that processes and annotates ParlaCLARIN corpus files",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/rules/extract_speeches.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/rules/help.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/rules/compute_frequency.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/rules/update_repository.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpazja7v88/workflow/rules/tag_protocols.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "welfare-state-analytics/pyriksprot_tagger",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 15 of /tmp/tmpazja7v88/workflow/Snakefile:\nNo module named \u0027loguru\u0027\n  File \"/tmp/tmpazja7v88/workflow/Snakefile\", line 15, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1639667899.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Workflow for functional annotation of a set of bacterial genomes",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp_jnmgt3n/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "alexmsalmeida/genome-funcs",
    "latest_release": null,
    "linting": "FileNotFoundError in line 14 of /tmp/tmp_jnmgt3n/Snakefile:\n[Errno 2] No such file or directory: \u0027/hps/research/finn/aalmeida/scripts/snakemake/genome-funcs_test/test/input_files.txt\u0027\n  File \"/tmp/tmp_jnmgt3n/Snakefile\", line 14, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639667784.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmp7e4y4kmv/workflow/rules/original.smk\":  Formatted content is different from original\n[DEBUG] \n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 2 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "maarten-k/realignment",
    "latest_release": null,
    "linting": "FileNotFoundError in line 21 of /tmp/tmp7e4y4kmv/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027resources/minitest.list\u0027\n  File \"/tmp/tmp7e4y4kmv/workflow/Snakefile\", line 21, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639599810.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Internal imputation pipeline using snakemake",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpa4jzzins/rules/impute_auto.smk\":  Keyword \"input\" at line 11 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/impute_auto.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmpa4jzzins/rules/impute_x.smk\":  \u0027NameError: name \u0027output_folder\u0027 is not defined\u0027\n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/impute_x.smk\":  \n[WARNING] In file \"/tmp/tmpa4jzzins/rules/chunker.smk\":  Keyword \"output\" at line 9 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/chunker.smk\":  Keyword \"params\" at line 20 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/chunker.smk\":  Keyword \"output\" at line 35 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/chunker.smk\":  Keyword \"input\" at line 39 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/chunker.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"output\" at line 15 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"output\" at line 269 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"output\" at line 364 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"output\" at line 386 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"input\" at line 393 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Keyword \"input\" at line 459 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/preproc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/stats.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/postproc.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmpa4jzzins/rules/phasing.smk\":  Keyword \"input\" at line 61 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpa4jzzins/rules/phasing.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmpa4jzzins/Snakefile\":  Keyword \"input\" at line 94 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpa4jzzins/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "macsx82/ImputationPipeline-snakemake",
    "latest_release": null,
    "linting": "KeyError in line 12 of /tmp/tmpa4jzzins/Snakefile:\n\u0027paths\u0027\n  File \"/tmp/tmpa4jzzins/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639590073.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "circularRNA Detection Annotation Quantification",
    "formatting": "[DEBUG] \n[ERROR] In file \"/tmp/tmp3b1lmlvt/CCBR-CCBR_circRNA_DAQ-875bd23/workflow/Snakefile\":  \u0027EmptyContextError: L608: rule has no keywords attached to it.\u0027\n[INFO] In file \"/tmp/tmp3b1lmlvt/CCBR-CCBR_circRNA_DAQ-875bd23/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "CCBR/CCBR_circRNA_DAQ",
    "latest_release": "v0.6.5",
    "linting": "KeyError in line 86 of /tmp/tmp3b1lmlvt/CCBR-CCBR_circRNA_DAQ-875bd23/workflow/Snakefile:\n\u0027workdir\u0027\n  File \"/tmp/tmp3b1lmlvt/CCBR-CCBR_circRNA_DAQ-875bd23/workflow/Snakefile\", line 86, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "circrna",
      "circularrna-detection",
      "ngs-analysis",
      "ngs-pipeline"
    ],
    "updated_at": 1639579568.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Collection of rules used to calculate biomarkers like MSI, TMB and HRD.",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmprlm0263n/workflow/rules/tmb.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmprlm0263n/workflow/rules/msisensor_pro.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmprlm0263n/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmprlm0263n/workflow/rules/hrd.smk\":  Formatted content is different from original\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "hydra-genetics/biomarker",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 13 of /tmp/tmprlm0263n/workflow/rules/common.smk:\nNo module named \u0027hydra_genetics\u0027\n  File \"/tmp/tmprlm0263n/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmprlm0263n/workflow/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1639568657.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/star.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/picard.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/jobs.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/featurecounts.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/r.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/multiqc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/trimmomatic.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpl82bit9w/workflow/rules/fastqc.smk\":  Formatted content is different from original\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "cyouh95/mRNA_seq_pipeline",
    "latest_release": null,
    "linting": "WorkflowError in line 3 of /tmp/tmpl82bit9w/workflow/Snakefile:\nWorkflow defines configfile config/config.yaml but it is not present or accessible.\n  File \"/tmp/tmpl82bit9w/workflow/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639565995.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/rules/dge.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/rules/salmon.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/rules/build_site.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/rules/process_raw.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmps_x8t9d4/workflow/rules/reference.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "steveped/snakemake-salmon",
    "latest_release": null,
    "linting": "Viable git repository detected. Files will be added when created\nKeyError in line 85 of /tmp/tmps_x8t9d4/workflow/Snakefile:\n\u0027tag\u0027\n  File \"/tmp/tmps_x8t9d4/workflow/Snakefile\", line 85, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639532487.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "snakemake workflow using R scripts to pull precipitation/climate data for Davis",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpl9zkr7cy/Snakefile\":  Keyword \"input\" at line 18 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpl9zkr7cy/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ryanpeek/r_with_snakemake",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpl9zkr7cy/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_metadata (line 28, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_clim_data (line 48, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_clim_data (line 71, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule viz_data (line 98, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean_zips (line 118, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ZIPS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_all (line 132, /tmp/tmpl9zkr7cy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ZIPS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DATA_CLEAN from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639522289.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "General snakemake pipeline to generate cell matrices for single-cell analysis and .loom files to assess cell development",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp14kny26a/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "nasiegel88/10x-snake",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp14kny26a/Snakefile:\n    * Absolute path \"/{sample}_S{num1}_{seq}_{num2}\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_tar (line 40, /tmp/tmp14kny26a/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule transcriptome (line 54, /tmp/tmp14kny26a/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mask_off (line 84, /tmp/tmp14kny26a/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_cellranger (line 105, /tmp/tmp14kny26a/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CELL_NUMBER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639509699.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Pipeline implementing the MERIDA logical modelling method using Snakemake to dispatch hyperparmeter tuning on a Slurm based HPC cluster.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmppc_go6ai/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "bhklab/MERIDA_snakemake_pipeline",
    "latest_release": null,
    "linting": "FileNotFoundError in line 58 of /tmp/tmppc_go6ai/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/ceeles/development/MERIDA_snakemake_pipeline/procdata/CCLE_bimodal_genes.txt\u0027\n  File \"/tmp/tmppc_go6ai/Snakefile\", line 58, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 586, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 482, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 811, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 1040, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 51, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/base_parser.py\", line 222, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/common.py\", line 702, in get_handle\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639504379.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp2drtqn45/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "jbloomlab/SARSr-CoV_RBD_MAP",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 10 of /tmp/tmp2drtqn45/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmp2drtqn45/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639437174.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp37ncgn70/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_MAP_S2K146",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 10 of /tmp/tmp37ncgn70/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmp37ncgn70/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639437136.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Single cell data from Germinal Center B cell paper",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/rules/star_index_gdc38_gencode38.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/rules/starsolo_alignment_gdc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/rules/stellarscope.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/rules/fasterq_dump.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp718qgx1n/workflow/rules/download_remote.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "nixonlab/HERV_scGCB",
    "latest_release": null,
    "linting": "Lints for rule fasterq_dump (line 8, /tmp/tmp718qgx1n/workflow/rules/fasterq_dump.smk):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule gzip (line 46, /tmp/tmp718qgx1n/workflow/rules/fasterq_dump.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_remote (line 6, /tmp/tmp718qgx1n/workflow/rules/download_remote.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_genome (line 40, /tmp/tmp718qgx1n/workflow/rules/download_remote.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_transcriptome (line 76, /tmp/tmp718qgx1n/workflow/rules/download_remote.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule id_mapping (line 116, /tmp/tmp718qgx1n/workflow/rules/download_remote.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule telescope_annotation (line 163, /tmp/tmp718qgx1n/workflow/rules/download_remote.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_index_gdc38_gencode38 (line 4, /tmp/tmp718qgx1n/workflow/rules/star_index_gdc38_gencode38.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable rule from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule starsolo_alignment (line 10, /tmp/tmp718qgx1n/workflow/rules/starsolo_alignment_gdc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_collate (line 83, /tmp/tmp718qgx1n/workflow/rules/starsolo_alignment_gdc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule stellarscope (line 4, /tmp/tmp718qgx1n/workflow/rules/stellarscope.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639428035.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpdfpktdin/Snakefile\":  Keyword \"input\" at line 31 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpdfpktdin/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "Nevada-Bioinformatics-Center/snakemake_gbs_phasing_workflow",
    "latest_release": null,
    "linting": "Lints for rule samtools_index_merged (line 94, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats (line 192, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats_unobs (line 222, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mawk_missind (line 579, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mawk_missind90 (line 605, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mawk_missind70 (line 631, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fix_missind50_noindel (line 726, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fix_missind50 (line 748, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fix_missind90 (line 844, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule beagle_phase_missind90 (line 866, /tmp/tmpdfpktdin/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fix_missind70 (line 896, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule beagle_phase_missind70 (line 918, /tmp/tmpdfpktdin/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule beagle_phase_missind50 (line 984, /tmp/tmpdfpktdin/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mawk_allind90 (line 1075, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mawk_allind50 (line 1101, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats_g70mac3 (line 1201, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats_g70mac3lm (line 1230, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats_g70mac3lmphased (line 1259, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_stats_g70mac3lmphased_noindels (line 1288, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_mkdir (line 1353, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plink_mkdir_noindels (line 1375, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plink_dst70_noindels (line 1428, /tmp/tmpdfpktdin/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule plink_converto_linkage70 (line 1489, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_converto_linkage70_noindels (line 1519, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_indeppairwise70 (line 1549, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_ld70 (line 1580, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_blocks70 (line 1610, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_indeppairwise70_noindels (line 1641, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_ld70_noindels (line 1672, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink_blocks70_noindels (line 1702, /tmp/tmpdfpktdin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639416456.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Collection of rules performing QC and generating reports.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/fastqc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/picard_hs_metrics.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/picard_insert_size.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/samtools_stats.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/hotspot_info.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/mosdepth.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/picard_alignment_summary_metrics.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/multiqc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpc6h278j1/workflow/rules/picard_duplication_metrics.smk\":  Formatted content is different from original\n[INFO] 10 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "hydra-genetics/qc",
    "latest_release": null,
    "linting": "ModuleNotFoundError in line 13 of /tmp/tmpc6h278j1/workflow/rules/common.smk:\nNo module named \u0027hydra_genetics\u0027\n  File \"/tmp/tmpc6h278j1/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmpc6h278j1/workflow/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1639400807.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp19rc_2u1/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "andersonbrito/ncovbr",
    "latest_release": null,
    "linting": "Lints for rule add_sequences (line 70, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_metadata (line 115, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_metadata (line 158, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule geoscheme (line 222, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule coordinates (line 260, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule colours (line 308, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter (line 373, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule align (line 415, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable files from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule mask (line 465, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tree (line 516, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule refine (line 555, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ancestral (line 631, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule translate (line 672, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule traits (line 711, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clades (line 755, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tip_frequencies (line 792, /tmp/tmp19rc_2u1/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule export (line 851, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 904, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule reset (line 930, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sanitize (line 960, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule delete (line 985, /tmp/tmp19rc_2u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639362101.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmptdgc68l9/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "paulaidt/CFMH",
    "latest_release": null,
    "linting": "KeyError in line 20 of /tmp/tmptdgc68l9/workflow/Snakefile:\n\u0027output_dir\u0027\n  File \"/tmp/tmptdgc68l9/workflow/Snakefile\", line 20, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639307087.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "snakemake pipeline for DNA mapping and coverage stats",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/rules/SamtoolsCoverageStats.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/rules/FASTQ.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/rules/Bowtie2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/rules/Bam2Cram.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp_iuxv080/rules/ExonGeneCoverage.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "FrancescoFerrari88/ams_umc",
    "latest_release": null,
    "linting": "IndexError in line 8 of /tmp/tmp_iuxv080/Snakefile:\nlist index out of range\n  File \"/tmp/tmp_iuxv080/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639250162.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "A light weight pipeline for RNA-seq analysis. Orignally design for cell free RNA-seq  data analysis, but can in principle applied to any paired end long RNA sequencing data ",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp4j2_buhr/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "uaauaguga/cfRNA-pipe",
    "latest_release": null,
    "linting": "KeyError in line 11 of /tmp/tmp4j2_buhr/Snakefile:\n\u0027sample_ids\u0027\n  File \"/tmp/tmp4j2_buhr/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639220776.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow for the recalculations steps of data release and new experiments processing.",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpiksy99en/Snakefile\":  Keyword \"input\" at line 212 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpiksy99en/Snakefile\":  \u0027NoParametersError: L498: In params definition.\u0027\n[INFO] In file \"/tmp/tmpiksy99en/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "ebi-gene-expression-group/bulk-recalculations",
    "latest_release": null,
    "linting": "KeyError in line 14 of /tmp/tmpiksy99en/Snakefile:\n\u0027metadata_summary\u0027\n  File \"/tmp/tmpiksy99en/Snakefile\", line 17, in \u003cmodule\u003e\n  File \"/tmp/tmpiksy99en/Snakefile\", line 14, in read_metadata_summary\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1639147122.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "pipeline to process foursu timecourse data",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/trim.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/genome_plots.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/rate_modeling.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmpx1ys5xog/Snakefile\":  Keyword \"input\" at line 29 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpx1ys5xog/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpx1ys5xog/rules/align.smk\":  Formatted content is different from original\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "marykthompson/foursu_timecourse",
    "latest_release": null,
    "linting": "WorkflowError in line 12 of /tmp/tmpx1ys5xog/Snakefile:\nWorkflow defines configfile config.yaml but it is not present or accessible.\n  File \"/tmp/tmpx1ys5xog/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1639145339.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "APA target caller - derived from the exar2020 paper",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpbsyuui4x/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "hilgers-lab/apa_target_caller",
    "latest_release": null,
    "linting": "KeyError in line 9 of /tmp/tmpbsyuui4x/Snakefile:\n\u0027samplesheets_dir\u0027\n  File \"/tmp/tmpbsyuui4x/Snakefile\", line 9, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639074683.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "CHROME ChIPseq analysis associated with GEO: GSE190413",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp9t4j8lpr/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "mgildea87/CHROME_ChIPseq",
    "latest_release": null,
    "linting": "Lints for rule fastqc (line 48, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_post_trim (line 71, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim (line 94, /tmp/tmp9t4j8lpr/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule align (line 128, /tmp/tmp9t4j8lpr/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort (line 163, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index (line 185, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MACS2 (line 207, /tmp/tmp9t4j8lpr/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fragment_size (line 235, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule FRP (line 258, /tmp/tmp9t4j8lpr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639061155.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp9nkeyirp/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ys-lim/cryptex-snakemake",
    "latest_release": null,
    "linting": "WorkflowError in line 7 of /tmp/tmp9nkeyirp/Snakefile:\nWorkflow defines configfile /mnt/cbis/home/yongshan/20211201_cnp_hippocampus_p60_snakemake/config/config.yaml but it is not present or accessible.\n  File \"/tmp/tmp9nkeyirp/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640101315.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "A meta-repository to organize my public projects.",
    "formatting": "Usage: snakefmt [OPTIONS] [SRC]...\n\nError: Invalid value for \u0027[SRC]...\u0027: Path \u0027/tmp/tmpboozdrfx/kelly-sovacool-meta-repo-06e35f6/Snakefile\u0027 does not exist.\n\nsnakefmt version: 0.4.4",
    "full_name": "kelly-sovacool/meta-repo",
    "latest_release": "v0.1",
    "linting": "",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639978272.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Annotates sequences with Eggnog-mapper and hhblits against PDB70",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp4icc5ajn/rules/eggmapper.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp4icc5ajn/rules/clustering.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp4icc5ajn/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp4icc5ajn/rules/get_prot_seqs.smk\":  Formatted content is different from original\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "matrs/Annotating_from_PDB",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule build_index (line 8, /tmp/tmp4icc5ajn/rules/get_prot_seqs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule write_multi_fasta (line 26, /tmp/tmp4icc5ajn/rules/get_prot_seqs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule mmseqs_db (line 1, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mmseqs_clustering (line 33, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mmseqs_createsubdb (line 81, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mmseqs_repr_seqs (line 117, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mmseqs_createtsv (line 149, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_seqs (line 184, /tmp/tmp4icc5ajn/rules/clustering.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule eggnog_mapper (line 3, /tmp/tmp4icc5ajn/rules/eggmapper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule select_no_kos (line 51, /tmp/tmp4icc5ajn/rules/eggmapper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule write_no_ko_seqs (line 74, /tmp/tmp4icc5ajn/rules/eggmapper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule write_subset_seqs (line 98, /tmp/tmp4icc5ajn/rules/eggmapper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule select_by_size (line 146, /tmp/tmp4icc5ajn/rules/eggmapper.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hhblits_msa (line 9, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hhblits_pdb (line 48, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule select_pdbs (line 88, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pdb_to_ko (line 117, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule propagate_annotations (line 148, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_ko_list (line 177, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule ko_completness (line 209, /tmp/tmp4icc5ajn/rules/hhblits_to_ko.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable py_komapper from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639779762.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Phylogeography of Yersinia pestis.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/metadata.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/alignment.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/plot.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/download.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/targets.smk\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7fb050868040\u003eNewline expected after keyword \u0027rule locus_coverage_collect_all\u0027\u0027\n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/targets.smk\":  \n[DEBUG] In file \"/tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ktmeaton/plague-phylogeography",
    "latest_release": "v0.3.0",
    "linting": "Multiple includes of /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk ignored\nMultiple includes of /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk ignored\nMultiple includes of /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk ignored\nMultiple includes of /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk ignored\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/Snakefile:\n    * Absolute path \"/download.smk\" in line 46:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/alignment.smk\" in line 47:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/phylogeny.smk\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qc.smk\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/filter_mask.smk\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/plot.smk\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata.smk\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/functions.smk\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/targets.smk\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/workflow.rst\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Environment variable SINGULARITY_CACHEDIR used but not asserted with envvars directive in line 70.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_SINGULARITY_CACHEDIR used but not asserted with envvars directive in line 71.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable SINGULARITY_CACHEDIR used but not asserted with envvars directive in line 73.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_SINGULARITY_CACHEDIR used but not asserted with envvars directive in line 74.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable CONDA_CACHEDIR used but not asserted with envvars directive in line 77.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_CONDA_CACHEDIR used but not asserted with envvars directive in line 78.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable CONDA_CACHEDIR used but not asserted with envvars directive in line 80.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable NXF_CONDA_CACHEDIR used but not asserted with envvars directive in line 81.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LANGUAGE used but not asserted with envvars directive in line 84.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LANG used but not asserted with envvars directive in line 86.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Environment variable LC_ALL used but not asserted with envvars directive in line 88.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Path composition with \u0027+\u0027 in line 46:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/download.smk:\n    * Absolute path \"/data/sra/{sample_dir}/{sample}_1.fastq.gz\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/sra/{sample_dir}/{sample}.log\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.{ext}\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.gbff\" in line 61:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.bed\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/functions.smk:\n    * Absolute path \"/\")[9] + \" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\")[9] + \" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\")[9] + \" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\")[9] + \" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\"+ url.split(\" in line 70:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\"+ url.split(\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample_dir}/{sample}\" in line 146:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample_dir}/{sample}\" in line 162:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 162:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/alignment.smk:\n    * Absolute path \"/eager/{reads_origin}/{sample}/final_bams/{sample}.bam\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/eager/{reads_origin}/{sample}/metadata_{sample}.tsv\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/eager/{reads_origin}/{sample}/{sample}.html\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/eager/{reads_origin}/{sample}/{sample}.log\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{dir}/{{reads_origin}}/{{sample}}/{filename}\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/reference/{reference}/{reference}.gbff\" in line 74:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.tab\" in line 77:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.txt\" in line 78:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.aligned.fa\" in line 79:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.subs.vcf\" in line 80:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.raw.vcf\" in line 81:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.log\" in line 82:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/{{reads_origin}}/snpden{density}.bed\" in line 131:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.txt\" in line 134:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.aln\" in line 135:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.full.aln\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.log\" in line 137:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 74:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 79:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 128:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 134:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 135:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk:\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.constant_sites.txt\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/snippy-multi.snps.aln\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.treefile\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.nex\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.iqtree\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.log\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.filter-taxa.txt\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/snippy-multi.snps.aln\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.treefile\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.filter-taxa.txt\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/snippy-multi.snps.aln\" in line 60:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/metadata.tsv\" in line 61:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/iqtree.treefile\" in line 62:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-sites/snippy-multi.snps.aln\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-sites/snippy-multi.snps.log\" in line 64:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/iqtree.treefile\" in line 90:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/metadata.tsv\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/variant_qc/{reads_origin}/{locus_name}/variant_qc.txt\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/temporal_constraints.txt\" in line 94:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/processed_iqtree_stats.py.ipynb\" in line 96:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 105:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/iqtree.treefile\" in line 106:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/snippy-multi.snps.aln\" in line 107:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.constant_sites.txt\" in line 108:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree_stats/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/temporal_constraints.txt\" in line 109:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.timetree.nex\" in line 111:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.timetree.nwk\" in line 112:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.dates.txt\" in line 113:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.outgroups.txt\" in line 114:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.log\" in line 115:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.filter-taxa.txt\" in line 116:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd\" in line 119:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/metadata.tsv\" in line 163:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-sites/snippy-multi.snps.aln\" in line 164:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/beast.latlon.txt\" in line 166:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/beast.dates.txt\" in line 167:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/beast.fasta\" in line 168:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/beast.nex\" in line 169:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree_stats/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/metadata.tsv\" in line 188:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/iqtree.treefile\" in line 189:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mugration/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/metadata.tsv\" in line 191:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/^$/d\" in line 214:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/lsd.timetree.nex\" in line 228:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/filter-taxa/iqtree.treefile\" in line 229:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mugration/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/metadata.tsv\" in line 230:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/all.json\" in line 232:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/processed_auspice.ipynb\" in line 234:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 60:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 62:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 64:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 94:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 106:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 107:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 111:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 112:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 113:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 114:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 115:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 163:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 164:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 167:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 169:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 188:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 189:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 191:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 194:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 228:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 229:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 230:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 232:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 234:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk:\n    * Absolute path \"/qualimap/{reads_origin}/{sample}/{sample}.bam\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/{reads_origin}/{sample}/qualimapReport.html\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/{reads_origin}/{sample}/{sample}.log\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc.yaml\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.txt\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc_report.html\" in line 47:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc_plots/pdf/mqc_snippy_variants_1.pdf\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc_plots/pdf/mqc_qualimap_gc_content_1.pdf\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc_plots/pdf/mqc_qualimap_genome_fraction_1.pdf\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc_plots/pdf/mqc_qualimap_coverage_histogram_1.pdf\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/{reads_origin}/multiqc.log\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/{reads_origin}/{sample}/{sample}.bam\" in line 81:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage/{reads_origin}/{sample}/locus_coverage_full.txt\" in line 84:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage/{reads_origin}/{sample}/locus_coverage.txt\" in line 85:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage/{reads_origin}/{sample}/locus_depth_full.txt\" in line 86:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage/{reads_origin}/{sample}/locus_depth.txt\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage.txt\" in line 104:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_depth.txt\" in line 106:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage_collect/{reads_origin}/locus_coverage.txt\" in line 109:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/locus_coverage_collect/{reads_origin}/locus_depth.txt\" in line 110:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.tab\" in line 124:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dnds/{reads_origin}/{locus_name}/{sample}.txt\" in line 126:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.subs.vcf\" in line 139:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tstv/{reads_origin}/{locus_name}/{sample}.vcf\" in line 141:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tstv/{reads_origin}/{locus_name}/{sample}.txt\" in line 142:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.subs.vcf\" in line 162:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.raw.vcf\" in line 163:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/heterozygosity/{reads_origin}/{locus_name}/{sample}.txt\" in line 165:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/variant_qc/{reads_origin}/{locus_name}/singletons.txt\" in line 185:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/variant_qc/{reads_origin}/{locus_name}/variant_qc.txt\" in line 218:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 47:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 85:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 86:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 124:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 139:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 141:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 142:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 162:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 163:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 165:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 185:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 218:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk:\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.fna\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_repeats/{reads_origin}/{sample}.inexact.repeats.bed\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_repeats/{reads_origin}/{sample}.log\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/{reads_origin}/{sample}/{sample}.fna\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_low_complexity/{reads_origin}/{sample}.dustmasker.bed\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_low_complexity/{reads_origin}/{sample}.dustmasker.intervals\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_pairwise/{reads_origin}/{sample}/{sample}.subs.vcf\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density/{{reads_origin}}/{{sample}}.subs.snpden{density}\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density/{reads_origin}/{sample}.log\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/{{reads_origin}}/snpden{density}.bed\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/snippy-multi.full.aln\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.full.aln\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.snps.aln\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.constant_sites.txt\" in line 93:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/metadata.tsv\" in line 94:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.snps.aln\" in line 118:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 119:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/full/snippy-multi.snps.dist\" in line 120:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/prune/snippy-multi.snps.aln\" in line 122:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/prune/taxa-keep.txt\" in line 123:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/prune/metadata.tsv\" in line 124:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/prune/snippy-multi.snps.log\" in line 125:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/snippy-multi.snps.aln\" in line 152:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/metadata.tsv\" in line 153:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/snippy-multi.snps.aln\" in line 155:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/filter{missing_data}/snippy-multi.snps.log\" in line 156:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 93:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 94:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 98:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 122:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 124:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 125:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 129:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 152:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 153:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 155:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 156:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/plot.smk:\n    * Absolute path \"/snippy_multi/{{reads_origin}}/{{locus_name}}/{{prune}}/filter{missing_data}/snippy-multi.snps.log\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/snippy-multi.snps.missing-data.html\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/processed_{locus_name}_plot_missing_data.py.ipynb\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/snippy-multi.snps.aln\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/snippy-multi.snps.dist\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/{reads_origin}/{locus_name}/{prune}/snippy-multi.snps.dist.heatmap.html\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/metadata.smk:\n    * Absolute path \"/metadata/{reads_origin}/metadata.tsv\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/targets.smk:\n    * Absolute path \"/locus_coverage_collect/all/locus_coverage.txt\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/variant_qc/all/{locus_name}/singletons.txt\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/variant_qc/all/{locus_name}/variant_qc.txt\" in line 107:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/assembly/snpden\" in line 160:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/sra/snpden\" in line 164:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/local/snpden\" in line 168:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detect_snp_density_collect/all/snpden\" in line 172:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/full/snippy-multi.full.aln\" in line 175:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/prune/snippy-multi.snps.aln\" in line 195:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/full/filter{missing_data}/snippy-multi.snps.aln\" in line 203:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/prune/filter{missing_data}/snippy-multi.snps.aln\" in line 226:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/assembly/{sample}/qualimapReport.html\" in line 237:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/sra/{sample}/qualimapReport.html\" in line 242:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/qualimap/local/{sample}/qualimapReport.html\" in line 247:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/all/multiqc_report.html\" in line 253:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/assembly/snippy-multi.full.aln\" in line 280:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/sra/snippy-multi.full.aln\" in line 284:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/local/snippy-multi.full.aln\" in line 288:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/snippy-multi.full.aln\" in line 292:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/full/filter{missing_data}/iqtree.treefile\" in line 297:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/prune/filter{missing_data}/iqtree.treefile\" in line 320:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/full/filter{missing_data}/filter-sites/snippy-multi.snps.aln\" in line 329:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/prune/filter{missing_data}/filter-sites/snippy-multi.snps.aln\" in line 336:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/full/filter{missing_data}/filter-taxa/temporal_constraints.txt\" in line 344:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/iqtree/all/{locus_name}/prune/filter{missing_data}/filter-taxa/temporal_constraints.txt\" in line 351:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/all/{locus_name}/full/filter{missing_data}/lsd.timetree.nex\" in line 359:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lsd/all/{locus_name}/prune/filter{missing_data}/lsd.timetree.nex\" in line 366:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/all/{locus_name}/full/filter{missing_data}/beast.dates.txt\" in line 375:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/beast/all/{locus_name}/prune/filter{missing_data}/beast.dates.txt\" in line 382:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mugration/all/{locus_name}/full/filter{missing_data}/metadata.tsv\" in line 391:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mugration/all/{locus_name}/prune/filter{missing_data}/metadata.tsv\" in line 398:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/all/{locus_name}/full/filter{missing_data}/all.json\" in line 406:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/auspice/all/{locus_name}/prune/filter{missing_data}/all.json\" in line 413:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/full/snippy-multi.snps.missing-data.html\" in line 423:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/snippy_multi/all/{locus_name}/full/filter{missing_data}/snippy-multi.snps.dist.heatmap.html\" in line 448:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/metadata/all/metadata.tsv\" in line 477:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 160:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 164:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 172:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 175:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 195:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 203:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 226:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 237:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 242:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 247:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 253:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 280:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 284:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 288:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 292:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 297:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 320:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 329:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 336:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 344:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 351:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 359:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 366:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 375:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 382:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 391:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 398:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 406:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 413:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 423:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 448:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 477:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 144:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 160:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 164:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_sra (line 10, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable project_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_assembly (line 50, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule locus_bed (line 102, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule eager (line 11, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_pairwise (line 115, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_multi (line 212, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule iqtree (line 6, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule iqtree_filter (line 81, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule iqtree_stats (line 144, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule lsd (line 185, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule beast (line 291, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule mugration (line 337, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule auspice (line 416, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/phylogeny.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qualimap (line 5, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule multiqc (line 53, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule locus_coverage (line 125, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule locus_coverage_collect (line 168, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dnds (line 205, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tstv (line 236, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule heterozygosity (line 282, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule singletons (line 314, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule variant_qc (line 351, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule detect_repeats (line 7, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable results_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule detect_low_complexity (line 48, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule detect_snp_density (line 83, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule detect_snp_density_collect (line 117, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snippy_multi_extract (line 152, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_multi_prune (line 208, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule snippy_multi_filter (line 271, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/filter_mask.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_missing_data (line 5, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/plot.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_snp_matrix (line 42, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/plot.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule metadata (line 5, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable scripts_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule help (line 156, /tmp/tmptdpexrzu/ktmeaton-plague-phylogeography-3b8f53b/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "phylogeography",
      "plagues"
    ],
    "updated_at": 1639760861.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "TimeAdapt makes a joint inference of demography and selection from longitudinal population genomic data. The inference is based on approximate Bayesian computation via random forest. It takes whole genome data and information about the ages of samples to perform simulations (using SLiM + pyslim/msprime).",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp3irrxr9b/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "mnavascues/ABC_TimeAdapt",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp3irrxr9b/Snakefile:\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule reftable_batch (line 89, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mutsim (line 129, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable python_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule forwsim (line 158, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule coalsim (line 187, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable python_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule getparams_batch (line 229, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule setproject (line 259, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_project (line 285, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 298, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule test (line 309, /tmp/tmp3irrxr9b/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639661381.0
  },
  {
    "config_readme": "\u003cp\u003eDescribe how to configure the workflow (using config.yaml and maybe additional files).\nAll of them need to be present with example entries inside of the config folder.\u003c/p\u003e\n",
    "data_format": 2,
    "description": "Snakemake workflow for converting FASTQ files to self-contained CRAM files with maximum lossless compression.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpv7_wa_hb/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "koesterlab/ngs-fastq-to-cram",
    "latest_release": null,
    "linting": null,
    "mandatory_flags": {
      "desc": null,
      "flags": null
    },
    "report": true,
    "software_stack_deployment": {
      "conda": true,
      "singularity": false,
      "singularity+conda": false
    },
    "standardized": true,
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1639684216.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": null,
    "full_name": "williamrowell/hifi-deepvariant-snakemake",
    "latest_release": null,
    "linting": "WorkflowError in line 5 of /tmp/tmpycoq8fge/Snakefile:\nWorkflow defines configfile workflow/config.yaml but it is not present or accessible.\n  File \"/tmp/tmpycoq8fge/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639629251.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpywe915q2/Snakefile\":  Keyword \"input\" at line 87 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpywe915q2/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "AlthafSinghawansaUHN/cfmedipseq_pipeline",
    "latest_release": null,
    "linting": "FileNotFoundError in line 16 of /tmp/tmpywe915q2/Snakefile:\n[Errno 2] No such file or directory: \u0027/cluster/projects/scottgroup/people/althaf/pipelines/cfmedipseq_pipeline/input/cohorts/samplesheet_Justin_HN_Norm_cfDNA_PBL.csv\u0027\n  File \"/tmp/tmpywe915q2/Snakefile\", line 53, in \u003cmodule\u003e\n  File \"/tmp/tmpywe915q2/Snakefile\", line 40, in get_all_samples_list\n  File \"/tmp/tmpywe915q2/Snakefile\", line 31, in get_all_samples\n  File \"/tmp/tmpywe915q2/Snakefile\", line 32, in \u003clistcomp\u003e\n  File \"/tmp/tmpywe915q2/Snakefile\", line 16, in get_cohort_data\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 586, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 482, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 811, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 1040, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 51, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/base_parser.py\", line 222, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/common.py\", line 702, in get_handle\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639592497.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "demo of Papermill running on NeSI",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmppchwoldf/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "nesi/papermill_demo",
    "latest_release": null,
    "linting": "Lints for rule preprocessing (line 16, /tmp/tmppchwoldf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable kernel from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule model_fitting (line 39, /tmp/tmppchwoldf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable kernel from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 6,
    "topics": [],
    "updated_at": 1639514687.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Mtb variant calling pipeline (bwa/GATK)",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp6ur0z5t4/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "ksw9/mtb_pipeline",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp6ur0z5t4/workflow/Snakefile:\n    * Absolute path \"/labs/jandr/walter/tb/data/Stanford/{run}/{samp}_L001_R1_001.fastq.gz\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/Stanford/{run}/{samp}_L001_R1_001.fastq.gz\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/Stanford/{run}/{samp}_L001_R2_001.fastq.gz\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 83:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 99:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 166:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 180:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/tb/data/refs/{ref}.fa\" in line 236:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/labs/jandr/walter/varcal/data/refs/ppe_gagneux_0based_fmt.bed\" in line 240:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule trim_reads (line 61, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule taxonomic_filter (line 87, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule map_reads (line 117, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule per_samp_stats (line 150, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule all_stats (line 182, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_bams (line 202, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cov_stats (line 229, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule predict_amr (line 268, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gatk_call (line 292, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vqsr (line 321, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vqsr_filter (line 346, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hard_filter (line 392, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vcf_to_fasta (line 438, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tb_profiler (line 472, /tmp/tmp6ur0z5t4/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639520735.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/bbsketch.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/alignments.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/pyani.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/mummer.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/filter.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/sketch.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/checkm.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/cluster.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/fastani.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[ERROR] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/download.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:16: download_genome:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/download.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/download.smk\":  9 file(s) would be changed \ud83d\ude2c\n[INFO] In file \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/rules/download.smk\":  1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "SilasK/FastDrep",
    "latest_release": "0.1",
    "linting": "KeyError in line 9 of /tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/compare.smk:\n\u0027genome_folder\u0027\n  File \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmpyufc5jq_/SilasK-FastDrep-92b1f56/compare.smk\", line 9, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1639409502.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Downloads TCGA data and puts it into a nice HDF file",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpzlqjsq57/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "dpmerrell/tcga-pipeline",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpzlqjsq57/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 196:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 196:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule visualize_data (line 71, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_all_omic_data (line 95, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_default_omic_data (line 124, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_restore_barcode_data (line 150, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule restore_barcodes (line 188, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_mutation_data (line 219, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule featurize_mutations (line 249, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_clinical_data (line 271, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CANCER_TYPES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule unzip_data (line 292, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TIMESTAMP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TIMESTAMP_ABBR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_data (line 316, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TIMESTAMP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_analyses (line 338, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TIMESTAMP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule unzip_firehose_get (line 364, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_firehose_get (line 384, /tmp/tmpzlqjsq57/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable FIREHOSE_GET_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639165876.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Extract set of tandem poly(A) sites from the PolyASite atlas.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpaem54qbq/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "zavolanlab/tandem-pas",
    "latest_release": null,
    "linting": "KeyError in line 28 of /tmp/tmpaem54qbq/Snakefile:\n\u0027logdir\u0027\n  File \"/tmp/tmpaem54qbq/Snakefile\", line 28, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 1,
    "subscribers_count": 4,
    "topics": [
      "bioinformatics",
      "snakemake",
      "polyadenylation"
    ],
    "updated_at": 1639145775.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Having fun plotting my activity data from Strava!",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp297xhypk/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "kelly-sovacool/strava",
    "latest_release": null,
    "linting": "Lints for rule download (line 27, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule process (line 46, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_activities (line 67, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_summary (line 96, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_report (line 117, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cat_figures_readme (line 139, /tmp/tmp297xhypk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640281158.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "An experiment to determine the relationship between spatial frequency and eccentricity in the human early visual cortex.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmprkith6n0/billbrod-spatial-frequency-preferences-4a83fe2/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "billbrod/spatial-frequency-preferences",
    "latest_release": "v1.0.3",
    "linting": "ModuleNotFoundError in line 6 of /tmp/tmprkith6n0/billbrod-spatial-frequency-preferences-4a83fe2/Snakefile:\nNo module named \u0027neuropythy\u0027\n  File \"/tmp/tmprkith6n0/billbrod-spatial-frequency-preferences-4a83fe2/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1640106242.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "a snakemake workflow for reproducible metabarcoding",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/lca.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/kraken2.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/vsearch.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/blast.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpw001fzvm/workflow/rules/reports.smk\":  Formatted content is different from original\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "EvoHull/Tapirs",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmpw001fzvm/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpw001fzvm/workflow/rules/qc.smk:\n    * Absolute path \"/{LIBRARIES}/{SAMPLES}.R1.fastq.gz\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{LIBRARIES}/{SAMPLES}.R2.fastq.gz\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpw001fzvm/workflow/rules/lca.smk:\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpw001fzvm/workflow/rules/blast.smk:\n    * Absolute path \"/names.dmp\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpw001fzvm/workflow/rules/kraken2.smk:\n    * Absolute path \"/names.dmp\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule fastp_trim_reads (line 9, /tmp/tmpw001fzvm/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fastp_merge_reads (line 91, /tmp/tmpw001fzvm/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge_forward_reads (line 151, /tmp/tmpw001fzvm/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule seqkit_fq2fa (line 176, /tmp/tmpw001fzvm/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mlca (line 9, /tmp/tmpw001fzvm/workflow/rules/lca.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mlca_to_tsv (line 40, /tmp/tmpw001fzvm/workflow/rules/lca.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vsearch_dereplicate (line 10, /tmp/tmpw001fzvm/workflow/rules/vsearch.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule vsearch_cluster (line 46, /tmp/tmpw001fzvm/workflow/rules/vsearch.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule vsearch_denoise (line 86, /tmp/tmpw001fzvm/workflow/rules/vsearch.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule vsearch_uchime3_denovo (line 166, /tmp/tmpw001fzvm/workflow/rules/vsearch.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule vsearch_rereplicate (line 202, /tmp/tmpw001fzvm/workflow/rules/vsearch.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule blast (line 9, /tmp/tmpw001fzvm/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule taxonomy_to_blast (line 53, /tmp/tmpw001fzvm/workflow/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param taxdump is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kraken2 (line 9, /tmp/tmpw001fzvm/workflow/rules/kraken2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule taxonomy_to_kraken2 (line 37, /tmp/tmpw001fzvm/workflow/rules/kraken2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param taxdump is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kraken2_to_tsv (line 65, /tmp/tmpw001fzvm/workflow/rules/kraken2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1639748612.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "The manuscript accompanying the OptiFit algorithm in mothur",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp5xt3212m/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "SchlossLab/Sovacool_OptiFit_mSphere_2022",
    "latest_release": null,
    "linting": "Lints for rule summarize_results (line 84, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calc_results_stats (line 109, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_algorithm (line 130, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_workflow (line 153, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_results_sum (line 187, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_results_split (line 211, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_markdown (line 235, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_pdf (line 268, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_words_abstract (line 293, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_test_data (line 313, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule test_R_code (line 334, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule test_Python_code (line 350, /tmp/tmp5xt3212m/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [
      "mothur",
      "snakemake",
      "microbiome",
      "reproducible-paper"
    ],
    "updated_at": 1639691798.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Method to detect 3\u0027UTR changes from scRNA-seq",
    "formatting": "Usage: snakefmt [OPTIONS] [SRC]...\n\nError: Invalid value for \u0027[SRC]...\u0027: Path \u0027/tmp/tmpu0ysfrwy/zavolanlab-SCUREL-5671e4d/workflow/Snakefile\u0027 does not exist.\n\nsnakefmt version: 0.4.4",
    "full_name": "zavolanlab/SCUREL",
    "latest_release": "v1.0.0",
    "linting": "KeyError in line 20 of /tmp/tmpu0ysfrwy/zavolanlab-SCUREL-5671e4d/Snakefile:\n\u0027defsamples\u0027\n  File \"/tmp/tmpu0ysfrwy/zavolanlab-SCUREL-5671e4d/Snakefile\", line 20, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 5,
    "topics": [
      "scrnaseq",
      "bioinformatics",
      "snakemake-workflow"
    ],
    "updated_at": 1639673687.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Bioinformatics pipeline for the analysis of amplicon sequencing data of eDNA samples from the PacMAN project",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"shell\" at line 135 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"output\" at line 182 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"input\" at line 240 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"output\" at line 248 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"input\" at line 291 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Keyword \"input\" at line 460 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp6axtdug3/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "iobis/PacMAN-pipeline",
    "latest_release": null,
    "linting": "ValueError in line 13 of /tmp/tmp6axtdug3/workflow/Snakefile:\n\u0027--configfiles\u0027 is not in list\n  File \"/tmp/tmp6axtdug3/workflow/Snakefile\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1639657930.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow for Illumina RNA-sequencing experiments, including variant calling and more - relating to population genetics and insecticide resistance",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/rules/diff.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/rules/lncrna.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/rules/variantAnalysis.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/rules/variantCalling.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/rules/common.smk\":  Formatted content is different from original\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "sanjaynagi/rna-seq-pop",
    "latest_release": "v0.7.0",
    "linting": "WorkflowError in line 13 of /tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/Snakefile:\nWorkflow defines configfile config/config.yaml but it is not present or accessible.\n  File \"/tmp/tmpiv0ib_jv/sanjaynagi-rna-seq-pop-c64f36e/workflow/Snakefile\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflow",
      "rna-seq",
      "variant-calling",
      "differential-expression",
      "population-genomics",
      "snakemake",
      "transcriptomics",
      "workflow",
      "population-genetics",
      "selection",
      "karyotype",
      "ancestry"
    ],
    "updated_at": 1639399941.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Updated version of PAQR, which was previously available in the PAQR_KAPAC joint repository.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp5ohfrjfb/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "zavolanlab/PAQR2",
    "latest_release": null,
    "linting": "KeyError in line 24 of /tmp/tmp5ohfrjfb/Snakefile:\n\u0027PAQ_logdir\u0027\n  File \"/tmp/tmp5ohfrjfb/Snakefile\", line 24, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 2,
    "subscribers_count": 4,
    "topics": [
      "bioinformatics",
      "snakemake",
      "rna-seq-pipeline",
      "polyadenylation"
    ],
    "updated_at": 1639145777.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "RNA Biology Pipeline to Characterize protein-RNA Interactions",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmp15_g4i1h/RBL-NCI-iCLIP-a6df4e3/workflow/Snakefile\":  Keyword \"input\" at line 473 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp15_g4i1h/RBL-NCI-iCLIP-a6df4e3/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp15_g4i1h/RBL-NCI-iCLIP-a6df4e3/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "RBL-NCI/iCLIP",
    "latest_release": "v1.0",
    "linting": "KeyError in line 30 of /tmp/tmp15_g4i1h/RBL-NCI-iCLIP-a6df4e3/workflow/Snakefile:\n\u0027sourceDir\u0027\n  File \"/tmp/tmp15_g4i1h/RBL-NCI-iCLIP-a6df4e3/workflow/Snakefile\", line 30, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1640288461.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Bitextor Neural generates translation memories from multilingual websites using state-of-the-art Machine Learning tools",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpbumgmmz5/bitextor-bitextor-neural-f52f99b/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "bitextor/bitextor-neural",
    "latest_release": "v1.0",
    "linting": "ModuleNotFoundError in line 6 of /tmp/tmpbumgmmz5/bitextor-bitextor-neural-f52f99b/workflow/Snakefile:\nNo module named \u0027bitextor\u0027\n  File \"/tmp/tmpbumgmmz5/bitextor-bitextor-neural-f52f99b/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 3,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1639475451.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": null,
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 4 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "euronion/trace",
    "latest_release": "2021.08.03",
    "linting": "Lints for snakefile /tmp/tmpswzh04bk/euronion-trace-4495310/Snakefile:\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk:\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpswzh04bk/euronion-trace-4495310/rules/esc_construction.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpswzh04bk/euronion-trace-4495310/rules/solving.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpswzh04bk/euronion-trace-4495310/rules/results.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule create_region (line 67, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_synthetic_demand (line 89, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_renewables (line 119, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_GEGIS_supply (line 150, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_GEGIS_demand (line 183, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/gegis.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_additional_components (line 6, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/esc_construction.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_network (line 32, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/esc_construction.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule attach_supply (line 82, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/esc_construction.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 29, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/solving.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule backup_scenario (line 67, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/solving.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_scenario_results (line 18, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/results.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_result (line 50, /tmp/tmpswzh04bk/euronion-trace-4495310/rules/results.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639466012.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-general-settings\" class=\"anchor\" href=\"#general-settings\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eGeneral settings\u003c/h1\u003e\n\u003cp\u003eTo configure this workflow, modify \u003ccode\u003econfig/config.yaml\u003c/code\u003e according to your\nneeds, following the explanations provided in the file.\u003c/p\u003e\n\u003ch2\u003e\n\u003ca id=\"user-content-sample-sheet\" class=\"anchor\" href=\"#sample-sheet\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eSample sheet\u003c/h2\u003e\n\u003cp\u003eThe sample sheet contains all samples to be analyzed by UnCoVar.\u003c/p\u003e\n\u003ch2\u003e\n\u003ca id=\"user-content-auto-filling\" class=\"anchor\" href=\"#auto-filling\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eAuto filling\u003c/h2\u003e\n\u003cp\u003eUnCoVar offers the possibility to automatically append samples to the sample\nsheet. To load your data into the workflow execute\u003c/p\u003e\n\u003cpre\u003e\u003ccode\u003esnakemake --cores all --use-conda update_sample\n\u003c/code\u003e\u003c/pre\u003e\n\u003cp\u003ewith the root of the UnCoVar as working directory. It is recommended to use\nthe following structure to when adding data automatically:\u003c/p\u003e\n\u003cpre\u003e\u003ccode\u003e\u251c\u2500\u2500 archive\n\u251c\u2500\u2500 incoming\n\u2514\u2500\u2500 snakemake-workflow-sars-cov2\n    \u251c\u2500\u2500 data\n    \u2514\u2500\u2500 ...\n\u003c/code\u003e\u003c/pre\u003e\n\u003cp\u003eHowever, this structure is not set in stone and can be adjusted via the\n\u003ccode\u003econfig/config.yaml\u003c/code\u003e file under \u003ccode\u003edata-handling\u003c/code\u003e. Only the following path to the\ncorresponding folders, relative to the directory of UnCoVar are needed:\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003e\n\u003cstrong\u003eincoming\u003c/strong\u003e: path of incoming data, which is moved to the data directory by\nthe preprocessing script. Defaults to \u003ccode\u003e../incoming/\u003c/code\u003e.\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003edata\u003c/strong\u003e: path to store data within the workflow. defaults to \u003ccode\u003edata/\u003c/code\u003e.\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003earchive\u003c/strong\u003e: path to archive data from the results from the analysis to.\nDefaults to \u003ccode\u003e../archive/\u003c/code\u003e.\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003eThe incoming directory should contain paired end reads in (compressed) FASTQ\nformat. UnCoVar automatically copies your data into the data directory and moves\nall files from incoming directory to the archive. After the analysis, all results\nare compressed and saved alongside the reads.\u003c/p\u003e\n\u003cp\u003eMoreover, the sample sheet is automatically updated with the new files. Please\nnote, that only the part of the filename before the first \u0027_\u0027 character is used\nas the sample name within the workflow.\u003c/p\u003e\n\u003ch2\u003e\n\u003ca id=\"user-content-manual-filling\" class=\"anchor\" href=\"#manual-filling\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eManual filling\u003c/h2\u003e\n\u003cp\u003eOf course, samples to be analyzed can also be added manually to the sample sheet.\nFor each sample, the a new line in \u003ccode\u003econfig/pep/samples.csv\u003c/code\u003e with the following\ncontent has to be defined:\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003e\n\u003cstrong\u003esample_name\u003c/strong\u003e: name or identifier of sample\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003efq1\u003c/strong\u003e: path to read 1 in FASTQ format\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003efq2\u003c/strong\u003e: path to read 2 in FASTQ format\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003edate\u003c/strong\u003e: sampling date of the sample\u003c/li\u003e\n\u003cli\u003e\n\u003cstrong\u003eis_amplicon_data\u003c/strong\u003e: indicates whether the data was generated with a\nshotgun (0) or amplicon (1) sequencing\u003c/li\u003e\n\u003c/ul\u003e\n",
    "data_format": 2,
    "description": "Transparent and robust SARS-CoV-2 variant calling and lineage assignment with comprehensive reporting.",
    "formatting": null,
    "full_name": "IKIM-Essen/uncovar",
    "latest_release": "v0.11.0",
    "linting": null,
    "mandatory_flags": null,
    "report": false,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 5,
    "subscribers_count": 3,
    "topics": [
      "sars-cov-2",
      "variant-calling",
      "lineage-assignment"
    ],
    "updated_at": 1640277105.0
  },
  {
    "config_readme": "\u003cp\u003eIn the config.yaml file you can find the parameters and options specified for each of the rules of mapache. These can be changed depending on specific needs.\u003c/p\u003e\n",
    "data_format": 2,
    "description": "mapping pipeline for ancient DNA",
    "formatting": null,
    "full_name": "sneuensc/mapache",
    "latest_release": "v0.1.0",
    "linting": "",
    "mandatory_flags": null,
    "report": true,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 5,
    "subscribers_count": 4,
    "topics": [
      "adna",
      "ancient-dna",
      "genomics",
      "mapping",
      "mapper",
      "pipeline",
      "workflow",
      "snakemake",
      "fastq",
      "bam"
    ],
    "updated_at": 1640069255.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "snakemake pipeline for Hi-C post-processing",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmphzcalkkk/open2c-quaich-f49263e/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 2 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "open2c/quaich",
    "latest_release": "v0.1.2",
    "linting": "ModuleNotFoundError in line 5 of /tmp/tmphzcalkkk/open2c-quaich-f49263e/workflow/Snakefile:\nNo module named \u0027cooler\u0027\n  File \"/tmp/tmphzcalkkk/open2c-quaich-f49263e/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 5,
    "subscribers_count": 2,
    "topics": [
      "workflow",
      "snakemake",
      "analysis",
      "hi-c"
    ],
    "updated_at": 1639588861.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow for highly parallel variant calling designed for ease-of-use in non-model organisms.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/fastq2bam.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/intervals.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/bam2vcf_gatk.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/bam2vcf_fb.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/qc.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmph55acnbh/workflow/rules/callable_sites.smk\":  Formatted content is different from original\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "harvardinformatics/snpArcher",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmph55acnbh/workflow/Snakefile:\n    * Absolute path \"/\" + idx[0] + \" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + idx[0] + \" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + idx[0] + \" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + idx[0] + \" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/common.smk:\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 101:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk:\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 74:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 75:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 100:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 121:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 122:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 135:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 135:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 151:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 153:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 154:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/intervals.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/bam2vcf_gatk.smk:\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 64:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 65:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 67:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 101:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 127:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/qc.smk:\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 47:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 77:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 85:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 102:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 107:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 121:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 131:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 132:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 133:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 134:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmph55acnbh/workflow/rules/callable_sites.smk:\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule get_fastq_pe (line 7, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule gzip_fastq (line 44, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bams (line 235, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bam_sumstats (line 301, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_fastp_stats (line 336, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_sumstats (line 356, /tmp/tmph55acnbh/workflow/rules/fastq2bam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule genmap_map (line 32, /tmp/tmph55acnbh/workflow/rules/intervals.smk):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule sort_genmap (line 71, /tmp/tmph55acnbh/workflow/rules/intervals.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_intervals (line 131, /tmp/tmph55acnbh/workflow/rules/intervals.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule mkDBmapfile (line 71, /tmp/tmph55acnbh/workflow/rules/bam2vcf_gatk.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule check_fai (line 1, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule snp_filters_qc (line 22, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule vcftools_individuals (line 49, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule subsample_snps (line 85, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plink (line 144, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule admixture (line 190, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param tmpbim is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule qc_plots (line 245, /tmp/tmph55acnbh/workflow/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule genome_prep (line 6, /tmp/tmph55acnbh/workflow/rules/callable_sites.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bedgraphs (line 32, /tmp/tmph55acnbh/workflow/rules/callable_sites.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gzip_bedgraph (line 90, /tmp/tmph55acnbh/workflow/rules/callable_sites.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 5,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1639520652.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Template for running mikRopML with Snakemake",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "SchlossLab/mikropml-snakemake-workflow",
    "latest_release": "v1.0.0",
    "linting": "Lints for rule preprocess_data (line 23, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_ml (line 59, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_results (line 100, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_hp_performance (line 128, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_benchmarks (line 156, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_performance (line 180, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_hp_performance (line 204, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_benchmarks (line 228, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_report (line 252, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 286, /tmp/tmpkoofmooc/SchlossLab-mikropml-snakemake-workflow-7eec776/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 6,
    "subscribers_count": 5,
    "topics": [
      "snakemake",
      "machine-learning",
      "rstats"
    ],
    "updated_at": 1639503183.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Education4Climate is a tool for crawling and analyzing data from higher education schools.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp1x16ni12/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "Education4Climate/Education4Climate",
    "latest_release": null,
    "linting": "Lints for rule crawl_programs (line 74, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CRAWLERS_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule crawl_programs_with_duplicates (line 88, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CRAWLERS_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge_duplicates (line 103, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SRC_CRAWL_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule crawl_courses (line 121, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CRAWLERS_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule score_courses (line 138, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SRC_SCORE_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule score_programs (line 157, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SRC_SCORE_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule prepare_to_web (line 177, /tmp/tmp1x16ni12/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SRC_WEB_FOLDER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 7,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1639940757.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "We present HRIBO (High-throughput annotation by Ribo-seq), a workflow to enable reproducible and high-throughput analysis of bacterial Ribo-seq data. The workflow performs all required pre-processing steps and quality control.  Importantly, HRIBO outputs annotation-independent ORF predictions based on two complementary prokaryotic-focused tools, and integrates them with additional computed features. This facilitates both the rapid discovery of ORFs and their prioritization for functional characterization.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/auxiliary.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/xtail.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/merge.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/readcounting.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/preprocessing.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/trimming.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/mapping.smk\":  Keyword \"shell\" at line 41 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/mapping.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/reparation.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/visualization.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/rrnafiltering.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/deepribo.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcauxiliary.smk\":  Keyword \"output\" at line 6 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcauxiliary.smk\":  Keyword \"output\" at line 21 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcauxiliary.smk\":  Keyword \"output\" at line 36 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcauxiliary.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/xtailclassic.smk\":  Formatted content is different from original\n[DEBUG] \n[WARNING] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/report.smk\":  Keyword \"output\" at line 16 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/report.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/trimmingmerged.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcmerged.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/conditionals.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/metageneprofiling.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/rules/qcsingleend.smk\":  Formatted content is different from original\n[INFO] 20 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "RickGelhausen/HRIBO",
    "latest_release": "1.5.1",
    "linting": "KeyError in line 8 of /tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/Snakefile:\n\u0027adapter\u0027\n  File \"/tmp/tmp973gvrdl/RickGelhausen-HRIBO-9d2fb50/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 7,
    "subscribers_count": 3,
    "topics": [
      "ribosome-profiling",
      "prokaryotes",
      "snakemake",
      "bioinformatics"
    ],
    "updated_at": 1639414042.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Deep learning model for predicting environmental variables on river systems",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp32bonr7m/USGS-R-river-dl-3825747/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "USGS-R/river-dl",
    "latest_release": "paper_v0",
    "linting": "ModuleNotFoundError in line 6 of /tmp/tmp32bonr7m/USGS-R-river-dl-3825747/Snakefile:\nNo module named \u0027xarray\u0027\n  File \"/tmp/tmp32bonr7m/USGS-R-river-dl-3825747/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmp32bonr7m/USGS-R-river-dl-3825747/river_dl/preproc_utils.py\", line 4, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 8,
    "subscribers_count": 8,
    "topics": [],
    "updated_at": 1639406702.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Benchpress is a snakemake workflow for reproducible and scalable execution and benchmarks of structure learning algorithms for probabilistic graphical models.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/algorithm_rules.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/pattern_strings.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/helper_functions.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/algorithm_strings.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/evaluation.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:30: f(traj=\"{output_dir}/adjvecs/\"\\ \"adjmat=/{adjmat_string}/\"\\ \"parameters=/{param_string}/\"\"data=/{data_string}/\"\\ \"algorithm=/{alg_string}/\"\\ \"seed={seed}/\" \"adjvecs.csv\")\n```\nwhile formatting code with black.\u0027\n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/evaluation.smk\":  \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/sample_parameters.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/docker_images.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/algorithm_shell_commands.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/sample_data.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/algorithm_summary_shell_command.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/sample_adjmat.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/rules/validate.smk\":  Formatted content is different from original\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 12 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "felixleopoldo/benchpress",
    "latest_release": "v1.0.1",
    "linting": "WorkflowError in line 23 of /tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/Snakefile:\nWorkflow defines configfile config/config.json but it is not present or accessible.\n  File \"/tmp/tmp0yp4uln5/felixleopoldo-benchpress-2e442e8/workflow/Snakefile\", line 23, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 9,
    "subscribers_count": 2,
    "topics": [
      "graphical-models",
      "bayesian-networks",
      "markov-networks",
      "benchmarking",
      "reproducible-research",
      "snakemake",
      "machine-learning",
      "snakemake-workflow",
      "structure-learning"
    ],
    "updated_at": 1640084025.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Zavolan-Lab Automated RNA-Seq Pipeline",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmprmfmd59o/zavolanlab-zarp-d98d7ed/workflow/rules/paired_end.snakefile.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmprmfmd59o/zavolanlab-zarp-d98d7ed/workflow/rules/single_end.snakefile.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmprmfmd59o/zavolanlab-zarp-d98d7ed/workflow/Snakefile\":  Formatted content is different from original\n[INFO] 3 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "zavolanlab/zarp",
    "latest_release": "v0.3.1",
    "linting": "KeyError in line 13 of /tmp/tmprmfmd59o/zavolanlab-zarp-d98d7ed/workflow/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmprmfmd59o/zavolanlab-zarp-d98d7ed/workflow/Snakefile\", line 13, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 12,
    "subscribers_count": 6,
    "topics": [
      "bioinformatics",
      "rna-seq",
      "rna-seq-pipeline",
      "snakemake"
    ],
    "updated_at": 1640047124.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Files and methodology pertaining to the sequencing and analysis of SARS-CoV-2, causative agent of COVID-19.",
    "formatting": "[DEBUG] \n[ERROR] In file \"/tmp/tmp761b414m/jaleezyy-covid-19-signal-b708440/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     else:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmp761b414m/jaleezyy-covid-19-signal-b708440/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "jaleezyy/covid-19-signal",
    "latest_release": "v1.4.4",
    "linting": "FileNotFoundError in line 45 of /tmp/tmp761b414m/jaleezyy-covid-19-signal-b708440/Snakefile:\n[Errno 2] No such file or directory: \u0027sample_table.csv\u0027\n  File \"/tmp/tmp761b414m/jaleezyy-covid-19-signal-b708440/Snakefile\", line 45, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/util/_decorators.py\", line 311, in wrapper\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 683, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 482, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 811, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/readers.py\", line 1040, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/c_parser_wrapper.py\", line 51, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/parsers/base_parser.py\", line 222, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.10/site-packages/pandas/io/common.py\", line 702, in get_handle\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 28,
    "subscribers_count": 14,
    "topics": [],
    "updated_at": 1640079438.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "PyPSA meets Africa: An Open Source Optimisation Model of the African Energy System. Our website: https://pypsa-meets-africa.github.io/",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmp9lba67xn/Snakefile\":  Keyword \"input\" at line 124 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp9lba67xn/Snakefile\":  Keyword \"input\" at line 226 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmp9lba67xn/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "pypsa-meets-africa/pypsa-africa",
    "latest_release": null,
    "linting": "Lints for snakefile /tmp/tmp9lba67xn/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule retrieve_databundle_light (line 58, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_osm_data (line 77, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_osm_data (line 99, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_osm_network (line 130, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_shapes (line 156, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_network (line 192, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_bus_regions (line 241, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_renewable_profiles (line 328, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_powerplants (line 367, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_electricity (line 396, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule simplify_network (line 435, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cluster_network (line 473, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule augmented_line_connections (line 514, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_extra_components (line 591, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_network (line 622, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 669, /tmp/tmp9lba67xn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 30,
    "subscribers_count": 11,
    "topics": [
      "pypsa-africa",
      "pypsa-meets-africa"
    ],
    "updated_at": 1640195249.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "PyPSA-Eur-Sec: A Sector-Coupled Open Optimisation Model of the European Energy System",
    "formatting": "[DEBUG] \n[ERROR] In file \"/tmp/tmp4xm27rve/PyPSA-pypsa-eur-sec-52b7caf/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0: else:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmp4xm27rve/PyPSA-pypsa-eur-sec-52b7caf/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n\nsnakefmt version: 0.4.4",
    "full_name": "PyPSA/pypsa-eur-sec",
    "latest_release": "v0.6.0",
    "linting": "WorkflowError in line 5 of /tmp/tmp4xm27rve/PyPSA-pypsa-eur-sec-52b7caf/Snakefile:\nWorkflow defines configfile config.yaml but it is not present or accessible.\n  File \"/tmp/tmp4xm27rve/PyPSA-pypsa-eur-sec-52b7caf/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 37,
    "subscribers_count": 11,
    "topics": [
      "pypsa",
      "energy",
      "energy-system",
      "energy-model",
      "energy-system-model"
    ],
    "updated_at": 1639840659.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Snakemake workflow for metagenomic classification with Kraken2",
    "formatting": "[DEBUG] \n[WARNING] In file \"/tmp/tmpe0dgil27/bhattlab-kraken2_classification-300ac80/Snakefile\":  Keyword \"input\" at line 131 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[DEBUG] In file \"/tmp/tmpe0dgil27/bhattlab-kraken2_classification-300ac80/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "bhattlab/kraken2_classification",
    "latest_release": "v1.0",
    "linting": "KeyError in line 8 of /tmp/tmpe0dgil27/bhattlab-kraken2_classification-300ac80/Snakefile:\n\u0027outdir\u0027\n  File \"/tmp/tmpe0dgil27/bhattlab-kraken2_classification-300ac80/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 37,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1639657738.0
  },
  {
    "config_readme": "\u003ch1\u003e\n\u003ca id=\"user-content-general-settings\" class=\"anchor\" href=\"#general-settings\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eGeneral settings\u003c/h1\u003e\n\u003cp\u003eTo configure this workflow, modify \u003ccode\u003econfig/config.yaml\u003c/code\u003e according to your needs, following the explanations provided in the file.\u003c/p\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-sample-sheet\" class=\"anchor\" href=\"#sample-sheet\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eSample sheet\u003c/h1\u003e\n\u003cp\u003eAdd samples to \u003ccode\u003econfig/samples.tsv\u003c/code\u003e. For each sample, the columns \u003ccode\u003esample_name\u003c/code\u003e, \u003ccode\u003ealias\u003c/code\u003e, \u003ccode\u003eplatform\u003c/code\u003e, and \u003ccode\u003egroup\u003c/code\u003e have to be defined.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eSamples within the same \u003ccode\u003egroup\u003c/code\u003e will be called jointly.\u003c/li\u003e\n\u003cli\u003eAliases represent the name of the sample within its group (they can be the same as the sample name, or something simpler, e.g. tumor or normal).\u003c/li\u003e\n\u003cli\u003eThe \u003ccode\u003eplatform\u003c/code\u003e column needs to contain the used sequencing plaform (one of \u0027CAPILLARY\u0027, \u0027LS454\u0027, \u0027ILLUMINA\u0027, \u0027SOLID\u0027, \u0027HELICOS\u0027, \u0027IONTORRENT\u0027, \u0027ONT\u0027, \u0027PACBIO\u2019).\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003eMissing values can be specified by empty columns or by writing \u003ccode\u003eNA\u003c/code\u003e. Lines can be commented out with \u003ccode\u003e#\u003c/code\u003e.\u003c/p\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-unit-sheet\" class=\"anchor\" href=\"#unit-sheet\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eUnit sheet\u003c/h1\u003e\n\u003cp\u003eFor each sample, add one or more sequencing units (runs, lanes or replicates) to the unit sheet \u003ccode\u003econfig/units.tsv\u003c/code\u003e.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eEach unit has a \u003ccode\u003eunit_name\u003c/code\u003e, which can be e.g. a running number, or an actual run, lane or replicate id.\u003c/li\u003e\n\u003cli\u003eEach unit has a \u003ccode\u003esample_name\u003c/code\u003e, which associates it with the biological sample it comes from.\u003c/li\u003e\n\u003cli\u003eFor each unit, define either one (column \u003ccode\u003efq1\u003c/code\u003e) or two (columns \u003ccode\u003efq1\u003c/code\u003e, \u003ccode\u003efq2\u003c/code\u003e) FASTQ files (these can point to anywhere in your system).\u003c/li\u003e\n\u003cli\u003eAlternatively, you can define an SRA (sequence read archive) accession (starting with e.g. ERR or SRR) by using a column \u003ccode\u003esra\u003c/code\u003e. In the latter case, the pipeline will automatically download the corresponding paired end reads from SRA. If both local files and SRA accession are available, the local files will be preferred.\u003c/li\u003e\n\u003cli\u003eDefine adapters in the \u003ccode\u003eadapters\u003c/code\u003e column, by putting \u003ca href=\"https://cutadapt.readthedocs.org\" rel=\"nofollow\"\u003ecutadapt arguments\u003c/a\u003e in quotation marks (e.g. \u003ccode\u003e\"-a ACGCGATCG -A GCTAGCGTACT\"\u003c/code\u003e).\u003c/li\u003e\n\u003c/ul\u003e\n\u003cp\u003eMissing values can be specified by empty columns or by writing \u003ccode\u003eNA\u003c/code\u003e. Lines can be commented out with \u003ccode\u003e#\u003c/code\u003e.\u003c/p\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-calling-scenario\" class=\"anchor\" href=\"#calling-scenario\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003eCalling scenario\u003c/h1\u003e\n\u003cp\u003eVarlociraptor supports integrated uncertainty aware calling and filtering of variants for arbitrary scenarios. These are defined as so-called scenarios, via a \u003ca href=\"https://varlociraptor.github.io/docs/calling#generic-variant-calling\" rel=\"nofollow\"\u003evariant calling grammar\u003c/a\u003e.\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003eFor each group, a scenario is rendered via \u003ca href=\"https://jinja.palletsprojects.com\" rel=\"nofollow\"\u003eJinja\u003c/a\u003e.\u003c/li\u003e\n\u003cli\u003eTherefore, edit the template scenario (\u003ccode\u003escenario.yaml\u003c/code\u003e) according to your needs. The sample sheet is available for jinja rendering as a pandas data frame in the variable \u003ccode\u003esamples\u003c/code\u003e. This allows to customize the scenario according to the contents of the sample sheet. You can therefore add additional columns to the sample sheet (e.g. purity) and access them in the scenario template, in order to pass the information to Varlociraptor.\u003c/li\u003e\n\u003c/ul\u003e\n\u003ch1\u003e\n\u003ca id=\"user-content-primer-trimming\" class=\"anchor\" href=\"#primer-trimming\" aria-hidden=\"true\"\u003e\u003cspan aria-hidden=\"true\" class=\"octicon octicon-link\"\u003e\u003c/span\u003e\u003c/a\u003ePrimer trimming\u003c/h1\u003e\n\u003cp\u003eFor panel data the pipeline allows trimming of amplicon primers on both ends of a fragment but also on a single end only.\nIn case of single end primers these are supposed to be located at the left end of a read.\nWhen primer trimming is enabled, primers have to be defined either directly in the \u003ccode\u003econfig.yaml\u003c/code\u003e or in a seperate tsv-file.\nDefining primers directly in the config file is prefered when all samples come from the same primer set.\nIn case of different panels, primers have to be set panel-wise in a seperate tsv-file.\nFor each panel the following columns need to be set: \u003ccode\u003epanel\u003c/code\u003e, \u003ccode\u003efa1\u003c/code\u003e and \u003ccode\u003efa2\u003c/code\u003e (optional).\nAdditionally, for each sample the corresponding panel must be defined in \u003ccode\u003esamples.tsv\u003c/code\u003e (column \u003ccode\u003epanel\u003c/code\u003e).\nFor single primer trimming only, the first entry in the config (respective in the tsv file) needs to be defined.\u003c/p\u003e\n",
    "data_format": 2,
    "description": "A Snakemake workflow for calling small and structural variants under any kind of scenario (tumor/normal, tumor/normal/relapse, germline, pedigree, populations) via the unified statistical model of Varlociraptor.",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpthva698s/snakemake-workflows-dna-seq-varlociraptor-a8f5167/workflow/rules/annotation.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpthva698s/snakemake-workflows-dna-seq-varlociraptor-a8f5167/workflow/rules/primers.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 15 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "snakemake-workflows/dna-seq-varlociraptor",
    "latest_release": "v3.4.0",
    "linting": null,
    "mandatory_flags": null,
    "report": true,
    "software_stack_deployment": {
      "conda": true
    },
    "standardized": true,
    "stargazers_count": 46,
    "subscribers_count": 7,
    "topics": [
      "varlociraptor",
      "sciworkflows",
      "snakemake",
      "reproducibility",
      "genomic-variant-calling"
    ],
    "updated_at": 1640009574.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "V-pipe is a pipeline designed for analysing NGS data of short viral genomes",
    "formatting": "[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow/rules/haplotypes.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] \n[DEBUG] In file \"/tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow/rules/visualization.smk\":  Formatted content is different from original\n[DEBUG] \n[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 14 file(s) would be left unchanged \ud83c\udf89\n\nsnakefmt version: 0.4.4",
    "full_name": "cbg-ethz/V-pipe",
    "latest_release": "v2.99.1",
    "linting": "VPIPE_BASEDIR = /tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow\nNo virus base configuration, using defaults\nWARNING: No samples found in samples/. Not generating config/samples.tsv.\nWARNING: Sample list file config/samples.tsv not found.\nWARNING: Reference file results/cohort_consensus.fasta not found. Trying references/results/cohort_consensus.fasta.\nValueError in line 452 of /tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow/rules/common.smk:\nERROR: Reference file references/results/cohort_consensus.fasta not found.\n  File \"/tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow/Snakefile\", line 12, in \u003cmodule\u003e\n  File \"/tmp/tmpfwefae9c/cbg-ethz-V-pipe-ffdec68/workflow/rules/common.smk\", line 452, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 73,
    "subscribers_count": 8,
    "topics": [
      "ngs",
      "snakemake",
      "conda",
      "biohackeu20",
      "virus",
      "sequencing",
      "bioinformatics",
      "bioinformatics-pipeline",
      "biohackcovid20",
      "sars-cov-2",
      "sarscov2",
      "hiv",
      "genomics",
      "biohackeu21"
    ],
    "updated_at": 1639840883.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "PyPSA-Eur: An Open Optimisation Model of the European Transmission System",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile\":  Formatted content is different from original\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "PyPSA/pypsa-eur",
    "latest_release": "v0.4.0",
    "linting": "Lints for snakefile /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 191:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 205:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule retrieve_databundle (line 105, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_load_data (line 122, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_powerplants (line 139, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_network (line 167, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_shapes (line 207, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_bus_regions (line 244, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule retrieve_cutout (line 307, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule retrieve_natura_raster (line 348, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_renewable_profiles (line 365, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_hydro_profile (line 407, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_electricity (line 433, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule simplify_network (line 472, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cluster_network (line 510, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_extra_components (line 551, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_network (line 582, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 629, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_operations_network (line 664, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_network (line 701, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_summary (line 740, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_summary (line 760, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_p_nom_max (line 786, /tmp/tmpkp7866kp/PyPSA-pypsa-eur-7b6c1ec/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 111,
    "subscribers_count": 12,
    "topics": [
      "snakemake",
      "energy",
      "energy-system",
      "power-systems",
      "energy-model",
      "pypsa",
      "energy-system-model",
      "energy-systems"
    ],
    "updated_at": 1640311817.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Fully reproducible, open source scientific articles in LaTeX.",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/download_manual.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/class_name.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/generate.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/upload.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/tectonic.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/download_auto.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/class_file.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/tmptexfile.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/stylesheet.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/script_info.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/repo.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/docstrings.smk\":  Formatted content is different from original\n[DEBUG] \n[ERROR] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 5:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/Snakefile\":  \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/pdf.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/texfile.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/xml.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/arxiv.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/figure.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/metadata.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/aux_file.smk\":  Formatted content is different from original\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 20 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "rodluger/showyourwork",
    "latest_release": "v0.2.1",
    "linting": "Lints for snakefile /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/zenodo.py:\n    * Absolute path \"/\".join(get_repo_url().split(\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dev/null\" in line 392:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk:\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule repo (line 8, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/repo.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule aux_file (line 8, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/aux_file.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tectonic (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/tectonic.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule class_name (line 9, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/class_name.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule class_file (line 5, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/class_file.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tmptexfile (line 5, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/tmptexfile.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule xml (line 5, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/xml.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule script_info (line 11, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/script_info.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule metadata (line 9, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule figure (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/figure.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule texfile (line 5, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/texfile.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule stylesheet (line 5, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/stylesheet.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dotgraph (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 14 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 15 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 16 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 17 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 18 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 19 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 20 (line 40, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dag (line 73, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/dag.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pdf (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/pdf.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule arxiv (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/arxiv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule docstrings (line 1, /tmp/tmp23zb4grk/rodluger-showyourwork-8096e6d/workflow/rules/docstrings.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 193,
    "subscribers_count": 5,
    "topics": [
      "latex",
      "open-source",
      "reproducible-science",
      "continuous-integration",
      "reproducible-research",
      "scientific-papers",
      "scientific-workflows"
    ],
    "updated_at": 1640222285.0
  },
  {
    "config_readme": null,
    "data_format": 2,
    "description": "Bitextor generates translation memories from multilingual websites",
    "formatting": "[DEBUG] \n[DEBUG] In file \"/tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/rules/common.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/Snakefile\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/rules/dicgeneration.smk\":  Formatted content is different from original\n[DEBUG] \n[DEBUG] In file \"/tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/rules/dictdocsegalign.smk\":  Formatted content is different from original\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n\nsnakefmt version: 0.4.4",
    "full_name": "bitextor/bitextor",
    "latest_release": "v8.1.1",
    "linting": "ModuleNotFoundError in line 6 of /tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/Snakefile:\nNo module named \u0027bitextor\u0027\n  File \"/tmp/tmptqb3irqj/bitextor-bitextor-987c0df/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "mandatory_flags": [],
    "report": false,
    "software_stack_deployment": null,
    "standardized": false,
    "stargazers_count": 196,
    "subscribers_count": 28,
    "topics": [
      "document-aligner",
      "httrack",
      "apertium",
      "dictionaries",
      "crawler",
      "wget",
      "creepy",
      "hunalign",
      "sentence-segmentation",
      "tokenizer",
      "bicleaner",
      "tmx",
      "warc",
      "corpus-tools",
      "corpus-processing",
      "corpus-generator",
      "parallel-corpora",
      "machine-translation",
      "neural-machine-translation",
      "statistical-machine-translation"
    ],
    "updated_at": 1639475502.0
  }
]
