var data =
[
  {
    "data_format": 1,
    "description": "Reproducibility and package management : workflow languages (CWL, Snakemake, Conda).",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "eanbit-rt/Workflows_and_package_management",
    "linting": "Lints for rule complement (line 1, /tmp/tmpicmwszc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule reverse (line 20, /tmp/tmpicmwszc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1598452249.0
  },
  {
    "data_format": 1,
    "description": "General-purpose Snakemake workflow for diffusion-based subcortical parcellation",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "akhanf/diffparc-smk",
    "linting": "Lints for snakefile /tmp/tmpmxtr5xwf/workflow/rules/common.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule run_probtrack (line 347, /tmp/tmpmxtr5xwf/workflow/rules/diffparc.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule track_from_clusters (line 157, /tmp/tmpmxtr5xwf/workflow/rules/tractmap.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule avg_tractmaps_template (line 341, /tmp/tmpmxtr5xwf/workflow/rules/tractmap.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1602172025.0
  },
  {
    "data_format": 1,
    "description": "Additional Training materials for workflows and package management",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "eanbit-rt/ReproducibleResearch",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1563259609.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tanaes/Moeller_lab_Snakemake_tutorial",
    "linting": "Lints for snakefile /tmp/tmp0sie8deo/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613581054.0
  },
  {
    "data_format": 1,
    "description": "DiVA (DNA Variant Analysis) is a pipeline for Next-Generation Sequencing Exome data anlysis",
    "formatting": "[WARNING] In file \"/tmp/tmpro88ahmq/rules/joint_call.smk\":  Keyword \"shell\" at line 47 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpro88ahmq/rules/call_variants.smk\":  Keyword \"shell\" at line 19 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpro88ahmq/rules/annotation.smk\":  Keyword \"input\" at line 2 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpro88ahmq/rules/annotation.smk\":  \u0027SyntaxError: L12: Unrecognised keyword \u0027samples\u0027 in rule definition\u0027\n[ERROR] In file \"/tmp/tmpro88ahmq/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 8:0:     \"delivery.completed\"\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpro88ahmq/Snakefile\":  2 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpro88ahmq/Snakefile\":  9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "solida-core/diva",
    "linting": "KeyError in line 11 of /tmp/tmpro88ahmq/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpro88ahmq/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [
      "bioinformatics",
      "snakemake",
      "genomics",
      "workflow",
      "variant-calling",
      "variant-annotations"
    ],
    "updated_at": 1611906269.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Magiduck/EindopdrachtOWE11",
    "linting": "Lints for snakefile /tmp/tmpo5hhd76e/Snakefile:\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 40:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule preprocessing (line 4, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule convertGeneIDs (line 27, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule getSeq (line 57, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule getFunction (line 85, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule getRecords (line 116, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule getPubMed (line 159, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule sortPubMed (line 184, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule getPathway (line 210, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule results (line 241, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule visualize (line 311, /tmp/tmpo5hhd76e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1529093905.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "stan-fmrg/Snakemake_haps",
    "linting": "FileNotFoundError in line 6 of /tmp/tmpa3184wqy/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpa3184wqy/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1538492907.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "TomHarrop/seedling-trt",
    "linting": "Lints for snakefile /tmp/tmpbw6onfp2/Snakefile:\n    * Absolute path \"/adapters.fa\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sequencing_artifacts.fa.gz\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Deprecated singularity directive used for container definition in line 92.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 121.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 150.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 179.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 218.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_second_pass (line 122, /tmp/tmpbw6onfp2/Snakefile):\n    * Param genome_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule star_first_pass (line 190, /tmp/tmpbw6onfp2/Snakefile):\n    * Param genome_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule star_reference (line 326, /tmp/tmpbw6onfp2/Snakefile):\n    * Param genome_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1538274235.0
  },
  {
    "data_format": 1,
    "description": "BMX-Seq: Brain Metastasis Xenograft Sequencing Pipeline",
    "formatting": "[WARNING] In file \"/tmp/tmpcvfob_p9/Snakefile\":  Keyword \"input\" at line 74 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpcvfob_p9/Snakefile\":  Keyword \"input\" at line 84 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpcvfob_p9/Snakefile\":  Keyword \"script\" at line 165 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zz2liu/bmx_seq",
    "linting": "FileNotFoundError in line 25 of /tmp/tmpcvfob_p9/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpcvfob_p9/Snakefile\", line 25, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1541166876.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for phigaro (https://github.com/lpenguin/phigaro)",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lpenguin/phigaro-pipeline",
    "linting": "FileNotFoundError in line 3 of /tmp/tmp7of7z174/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yml\u0027\n  File \"/tmp/tmp7of7z174/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1518166274.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp5bfb07wi/Snakefile\":  Keyword \"script\" at line 136 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "komax/BioTIME-fastText-classification",
    "linting": "Lints for rule all (line 38, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gen_parameters (line 50, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule chunk_parameters (line 62, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule select_columns_biotime (line 85, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule data_prep_fasttext (line 106, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule normalize_fasttext (line 132, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_data (line 153, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param cross_val_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cross_validate_chunk (line 180, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_chunks (line 210, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_f1_scores (line 229, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule train_model (line 249, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule test_model (line 273, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 289, /tmp/tmp5bfb07wi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1554973744.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ivyduk/identiclust-train",
    "linting": "Lints for rule tag_samples (line 24, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gene_annotation (line 52, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_file_creation (line 72, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule intergenic_extraction_bed (line 97, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule intergenic_extraction_fasta (line 117, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule concatenate_gene_sequences (line 137, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule concatenate_intergenic_sequences (line 157, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clustering_cd_hit_iterative_Gene (line 177, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clustering_cd_hit_iterative_Intergenic (line 206, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clustering_mcl__Genic (line 234, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clustering_mcl_Intergenic (line 264, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule train_model_random_forest_gene (line 294, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule train_model_random_forest_inter (line 318, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule join_gene_and_intergenic_matrix (line 341, /tmp/tmpgf0t5xnd/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule train_model_random_forest_both (line 359, /tmp/tmpgf0t5xnd/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1563055564.0
  },
  {
    "data_format": 1,
    "description": "Processor - MinION Run Processing Pipeline",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ares-lab/processor",
    "linting": "KeyError in line 12 of /tmp/tmpy3vsw8zp/Snakefile:\n\u0027samplesheet_fp\u0027\n  File \"/tmp/tmpy3vsw8zp/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1551206467.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for running kneaddata",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tanaes/snakemake_kneaddata",
    "linting": "KeyError in line 7 of /tmp/tmp_4cyuji0/Snakefile:\n\u0027TMP_DIR_ROOT\u0027\n  File \"/tmp/tmp_4cyuji0/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1477422258.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "chop-dbhi/ceu_trio_for_varify",
    "linting": "Lints for snakefile /tmp/tmpugn29yla/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule get_a_bam (line 19, /tmp/tmpugn29yla/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 55, /tmp/tmpugn29yla/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_ranges (line 77, /tmp/tmpugn29yla/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gatk_hard_filtration (line 208, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule select_passing (line 253, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gatk_variant_recalibration (line 299, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vqsr (line 364, /tmp/tmpugn29yla/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gatk_apply_variant_recalibration (line 385, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule phase_by_transmission (line 447, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule annotate_dbsnp (line 494, /tmp/tmpugn29yla/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule varify_manifest (line 539, /tmp/tmpugn29yla/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 9,
    "topics": [],
    "updated_at": 1453487677.0
  },
  {
    "data_format": 1,
    "description": "A collection of data sets for benchmarking de novo variant discovery tools",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "standage/noble",
    "linting": "Lints for rule simulategenome (line 15, /tmp/tmp_xkvmj5a/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule simulatetriovariants (line 46, /tmp/tmp_xkvmj5a/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sequencing (line 82, /tmp/tmp_xkvmj5a/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trustedkmers (line 115, /tmp/tmp_xkvmj5a/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule errorcorrect (line 150, /tmp/tmp_xkvmj5a/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule interleave_compress (line 186, /tmp/tmp_xkvmj5a/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [
      "variant-calling",
      "trio",
      "benchmarking",
      "genomics",
      "bioinformatics"
    ],
    "updated_at": 1521221913.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for analysis and assembly of viral genomes from IonTorrent AmpliSeq data.",
    "formatting": "[INFO] 13 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "peterk87/viral-ampliseq-assembly",
    "linting": "Lints for snakefile /tmp/tmpf9zlfito/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpf9zlfito/rules/mash.smk:\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpf9zlfito/rules/download_references.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpf9zlfito/rules/spades.smk:\n    * Absolute path \"/tmp/viral-ampliseq-assembly-spades\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpf9zlfito/rules/variant_calling.smk:\n    * Absolute path \"/^#/{{f=1;next}} f{{ print $1 }}\" in line 90:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpf9zlfito/rules/msa.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpf9zlfito/rules/phylogeny.smk:\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule samtools_index_bam_initial (line 35, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_flagstat_bam_initial (line 68, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_idxstats_bam_initial (line 91, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule process_samtools_idxstats_bam_initial (line 117, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_depth_bam_initial (line 138, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule process_samtools_depth_bam_initial (line 167, /tmp/tmpf9zlfito/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_mash_screen (line 72, /tmp/tmpf9zlfito/rules/mash.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_faidx (line 68, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 151, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_flagstat (line 180, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_idxstats (line 203, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule process_samtools_idxstats (line 229, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_depth (line 250, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule process_samtools_depth (line 275, /tmp/tmpf9zlfito/rules/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rename_denovo_assembly (line 53, /tmp/tmpf9zlfito/rules/spades.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter_vcf (line 41, /tmp/tmpf9zlfito/rules/variant_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule snpeff (line 215, /tmp/tmpf9zlfito/rules/variant_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule consensus (line 2, /tmp/tmpf9zlfito/rules/consensus.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prepare_msa_input (line 1, /tmp/tmpf9zlfito/rules/msa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clearcut (line 3, /tmp/tmpf9zlfito/rules/phylogeny.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1566573045.0
  },
  {
    "data_format": 1,
    "description": "ARES Lab Bacterial Genome Assembly Pipeline",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ares-lab/assembler",
    "linting": "KeyError in line 14 of /tmp/tmpxaizl37_/Snakefile:\n\u0027processor_config\u0027\n  File \"/tmp/tmpxaizl37_/Snakefile\", line 14, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1539524195.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for partitioning heritability with ldsc",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nrachman/ldsc_snakemake",
    "linting": "PermissionError in line 2 of /tmp/tmpy5cfommu/Snakefile:\n[Errno 13] Permission denied: \u0027/hpcdata\u0027\n  File \"/tmp/tmpy5cfommu/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1582834925.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline to validate exfi\u0027s performance",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/exfi_validation",
    "linting": "/tmp/tmp8nvdd_bc/Snakefile:11: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  params = yaml.load(open(\"params.yml\", \"r\"))\n/tmp/tmp8nvdd_bc/Snakefile:12: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  features = yaml.load(open(\"features.yml\", \"r\"))\nAttributeError in line 31 of /tmp/tmp8nvdd_bc/src/snakefiles/raw.py:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmp8nvdd_bc/Snakefile\", line 25, in \u003cmodule\u003e\n  File \"/tmp/tmp8nvdd_bc/src/snakefiles/raw.py\", line 31, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "rna-seq",
      "genome",
      "exome",
      "transcriptome"
    ],
    "updated_at": 1544535397.0
  },
  {
    "data_format": 1,
    "description": "snakemake pipeline to get the results from the exfi paper",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/smsk_exfi_paper",
    "linting": "/tmp/tmp4094ggxa/Snakefile:11: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  params = yaml.load(open(\"params.yml\", \"r\"))\n/tmp/tmp4094ggxa/Snakefile:12: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  features = yaml.load(open(\"features.yml\", \"r\"))\nAttributeError in line 32 of /tmp/tmp4094ggxa/src/snakefiles/raw.py:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmp4094ggxa/Snakefile\", line 26, in \u003cmodule\u003e\n  File \"/tmp/tmp4094ggxa/src/snakefiles/raw.py\", line 32, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1563799580.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ALuesink/Workflow_Course11",
    "linting": "Lints for snakefile /tmp/tmporlboduc/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule open_file (line 10, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_PMIDs (line 29, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gene_info (line 48, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule seq_gc (line 67, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gen_ids (line 86, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule uniprot_info (line 106, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule visualise (line 125, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report (line 144, /tmp/tmporlboduc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1528455779.0
  },
  {
    "data_format": 1,
    "description": "Call broad ChIP peaks from paired Illumina sequencing data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "johnstonmj/chip_broad_pipeline",
    "linting": "Lints for snakefile /tmp/tmpyke_f21w/Snakefile:\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 69:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 104:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 172:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 174:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 200:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 69:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 104:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 172:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 192:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule fastqc (line 44, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_align (line 95, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule macs2_call_broadpeaks (line 170, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule foldEnrich (line 243, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule coverage_normalize (line 301, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multiBigwig_norm_counts (line 357, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pearson (line 419, /tmp/tmpyke_f21w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1552580657.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "EnricoASchmitz/Snakemake",
    "linting": "Lints for snakefile /tmp/tmp0yr95wn7/Snakefile:\n    * Absolute path \"/home/enrico/Desktop/Snakemake/GeneIDList.txt\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/allSequence_GCcontent.fasta\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Kegg.txt\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Blast.txt\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/MergedFile.txt\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/InfoFile.txt\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/PubmedIDs.txt\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/InfoOutFile.txt\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Info-File.txt\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Report.ods\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/RNA-Seq-counts.txt\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/GeneIDList.txt\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/GENEID.py\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/GeneIDList.txt\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/allSequence_GCcontent.fasta\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Sequentie_GCpercentage.py\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/RNA-Seq-counts.txt\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Kegg.txt\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/KEGG.py\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/allSequence_GCcontent.fasta\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Blast.txt\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Blast.py\" in line 46:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/allSequence_GCcontent.fasta\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/GeneIDList.txt\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/PubmedIDs.txt\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/FindPubmedIDs.py\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/GeneIDList.txt\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/RNA-Seq-counts.txt\" in line 60:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/MergedFile.txt\" in line 62:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/MergeID.py\" in line 64:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/MergedFile.txt\" in line 68:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Kegg.txt\" in line 69:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/PubmedIDs.txt\" in line 70:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/InfoFile.txt\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/MergeInfo.py\" in line 74:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/InfoFile.txt\" in line 78:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/allSequence_GCcontent.fasta\" in line 79:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/InfoOutFile.txt\" in line 81:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Info-File.txt\" in line 82:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/addSeq.py\" in line 84:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Blast.txt\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Info-File.txt\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/Snakemake/Report.ods\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/enrico/Desktop/MergeBlast.py\" in line 93:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule GENEID (line 21, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biopython (line 41, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule KEGG (line 60, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule BLAST (line 79, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule FindPubmedIDs (line 98, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MergeID (line 118, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MergeInfo (line 138, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule AddSeq (line 159, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MergeBlast (line 180, /tmp/tmp0yr95wn7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1528888855.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zehnlab/microarray_ngs_scripts",
    "linting": "Lints for rule DGE_RNAseq (line 12, /tmp/tmp8k1efixq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_heatmaps (line 37, /tmp/tmp8k1efixq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1558454097.0
  },
  {
    "data_format": 1,
    "description": "Reproduce PROBer paper analysis",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pachterlab/PROBer_paper_analysis",
    "linting": "Wildcard constraints in inputs are ignored. (rule: extract_info_from_bpseq)\nLints for snakefile /tmp/tmpvn832a0x/Snakefile:\n    * Absolute path \"/bowtie-1.1.2\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bowtie2-2.2.9\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/RSEM-1.2.31\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/scatters.pdf\" in line 97:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/comparing_with_BUMHMM_table.txt\" in line 98:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 41:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvn832a0x/tools/Snakefile:\n    * Absolute path \"/PROBer-0.3.0.tar.gz\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PROBer_full_model.tar.gz\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvn832a0x/ground_truth/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpvn832a0x/references/Snakefile:\n    * Absolute path \"/SHAPE_MaP.t2g\" in line 77:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 69:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvn832a0x/data/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 187:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvn832a0x/exp/Snakefile:\n    * Absolute path \"/hnRNPC_HeLa_UleLab_rep1_reprm.fq\" in line 703:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/SRR3147675_reprm.fq\" in line 705:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/countMotifRate\" in line 910:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/detectMotif\" in line 921:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calcWeightedMotifRate\" in line 932:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 231:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 364:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 601:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 703:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 886:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 910:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 921:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 932:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvn832a0x/simulation/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule compile_cpp (line 1, /tmp/tmpvn832a0x/scripts/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_fastq_dump (line 5, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_cutadapt (line 24, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_samtools (line 43, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_bowtie (line 62, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_bowtie2 (line 81, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_rsem (line 100, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_PROBer (line 119, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_PROBer_full_model (line 138, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_RNAstructure (line 157, /tmp/tmpvn832a0x/tools/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable tool_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_bpseq (line 1, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_info_from_bpseq (line 16, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule download_pdb (line 39, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_arabidopsis_bpseq (line 55, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_arabidopsis_rRNAs_fa (line 76, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_yeast_bpseq (line 95, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_yeast_solvent_accessible_surface_areas (line 125, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_ecoli_bpseq (line 151, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_ecoli_solvent_accessible_surface_areas (line 170, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_mouse_bpseq (line 193, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_mouse_rRNAs_fa (line 212, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_ct_file_from_bpseq (line 232, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_yeast_valid (line 249, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule unzip_SHAPE_MaP (line 268, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule process_SHAPE_MaP (line 287, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule generate_pseudoU_ground_truth (line 313, /tmp/tmpvn832a0x/ground_truth/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_PROBer_and_Bowtie_indices (line 5, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_RSEM_indices (line 25, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable RSEM_prepare from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_arabidopsis_genome (line 51, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_arabidopsis from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_arabidopsis from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_arabidopsis_GTF (line 71, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_arabidopsis from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_arabidopsis from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_arabidopsis_remove_list (line 91, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_arabidopsis_transcripts (line 103, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable arabidopsis_filt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_mappability_for_arabidopsis_filt (line 125, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable arabidopsis_filt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule add_spike_in_sequences_to_arabidopsis_transcripts (line 145, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_yeast_remove_list (line 169, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_yeast_transcripts (line 181, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_yeast from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_yeast from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_yeast from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_yeast from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable yeast_filt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_yeast_rRNAs_fa (line 213, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_PROBer_and_Bowtie2_indices_for_yeast_rRNAs_fa (line 232, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie2_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_assembled_ecoli_16S_fa (line 253, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_PROBer_and_Bowtie2_indices_for_ecoli_16S_fa (line 267, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie2_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_mouse_genome (line 288, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_mouse from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_mouse from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_mouse_GTF (line 308, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_mouse from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_mouse from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_mouse_remove_list (line 328, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_mouse_transcripts (line 347, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable mouse_filt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_human_genome (line 371, /tmp/tmpvn832a0x/references/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ref_human from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ref_human from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_Bowtie_and_Bowtie2_indices_for_iCLIP_and_eCLIP_analyses (line 391, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie2_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable human_ref from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_rep_ref (line 411, /tmp/tmpvn832a0x/references/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie2_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable rep_ref from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_sra (line 1, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dump_to_FASTQ (line 12, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable fastq_dump from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule dump_to_FASTQ_PE (line 28, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable fastq_dump from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_adapter_AssmannLab (line 48, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_3_prime_adapter_McmanusLab (line 73, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_5_prime_adapter_McmanusLab (line 97, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_reads_for_trimming_mismatches_McmanusLab (line 121, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable yeast_filt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_mismatches_McmanusLab (line 148, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_adapter_ChangLab (line 168, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_PCR_duplicate_ChangLab (line 192, /tmp/tmpvn832a0x/data/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_adapter_GilbertLab (line 216, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_info_GrannemanLab (line 241, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GrannemanLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable GrannemanLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable GrannemanLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule integrate_umi_and_trim_first_nucleotide_GrannemanLab (line 258, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable GrannemanLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable GrannemanLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_adapter_GrannemanLab (line 291, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_PCR_duplicate_GrannemanLab (line 317, /tmp/tmpvn832a0x/data/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_gz_file_VintherLab (line 341, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_adapter_VintherLab (line 356, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_UleLab_hnRNPC_iCLIP_data (line 384, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UleLab from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_adapter_iCLIP (line 408, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_PCR_duplicate_iCLIP (line 432, /tmp/tmpvn832a0x/data/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_reads_mapped_to_rep_iCLIP (line 455, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sra_to_ENCODE_eCLIP (line 542, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule cutadapt_RBFOX2_eCLIP (line 566, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_ENCODE_eCLIP (line 613, /tmp/tmpvn832a0x/data/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cutadapt_ENCODE_eCLIP (line 630, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Cutadapt from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_PCR_duplicate_eCLIP (line 658, /tmp/tmpvn832a0x/data/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule remove_reads_mapped_to_rep_eCLIP (line 680, /tmp/tmpvn832a0x/data/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable YeoLab_eCLIP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Bowtie2 from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable YeoLab_eCLIP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable YeoLab_eCLIP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable YeoLab_eCLIP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable YeoLab_eCLIP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie_alignment_all (line 40, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie_alignment_best_strata (line 68, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie_alignment_best (line 96, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie2_alignment_all (line 137, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie2_alignment_best_strata (line 165, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bowtie2_alignment_best (line 193, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer (line 246, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_no_control (line 275, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_count_vector_StructureFold_or_Pseudoseq (line 322, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_StructureFold_score (line 341, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_count_vector_Modseeker (line 359, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_Modseq_score (line 378, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_icSHAPE_score (line 421, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_Pseudoseq_score (line 439, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sort_bam_by_name (line 463, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable rsem_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_structure_seq_mapping_statistics (line 489, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_RBFOX2_eCLIP_mapping_statistics (line 509, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_mapping_statistics_table (line 528, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_for_time_and_memory (line 554, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_time_and_memory_table (line 584, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_full_model (line 607, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer_full_model from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_RSEM_PROBer_star_pipeline (line 635, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer_single from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable rsem_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule produce_simulation_results (line 663, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule produce_boxplot_pdf (line 701, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule produce_digital_spike_in_results (line 723, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule produce_boxplot_pdf_for_spike_in_experiments (line 746, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_RSEM_for_scatter_plots (line 767, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable RSEM_calculate from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_expression_scatter_plots (line 794, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_mod_seq_PR_ROC_sasa (line 819, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable result_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_ChemModSeq_PR_ROC_sasa (line 840, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable result_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extractReactivity (line 885, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule runRNAstructure (line 908, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable RNAstructure_data_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Fold from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule runRNAstructure_baseline (line 929, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable RNAstructure_data_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Fold from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule evaluate_structure (line 949, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable scorer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_structure_prediction_result (line 969, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_structure_seq_ROC_AUC (line 993, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_mod_seq_ROC_AUC (line 1015, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_icSHAPE_ROC_AUC (line 1037, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_roc_auc_table (line 1058, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_reads_SHAPES (line 1079, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Bowtie2 from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_terminals_SHAPES (line 1117, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule collect_data_for_RNAprobR_SHAPES (line 1140, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_bias_SHAPES (line 1159, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_ROC_SHAPES (line 1178, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable result_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_doc_cov_for_BUMHMM (line 1202, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_BUMHMM_scores (line 1221, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calc_PR_ROC_BUMHMM (line 1240, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_comparing_with_BUMHMM_table (line 1260, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_pseudoU_PR_ROC_plots (line 1283, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gt_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exp_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable result_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule visualize_18S_for_pseudoU_experiment (line 1305, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_iCLIP (line 1337, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_naive_iCLIP (line 1365, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sample_multireads_iCLIP (line 1389, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer_sampler from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param sample_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule run_PROBer_eCLIP (line 1456, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable bowtie2_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_PROBer_naive_eCLIP (line 1488, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sample_multireads_eCLIP (line 1512, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer_sampler from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param sample_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule extract_unique_CLIP (line 1539, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_all_CLIP (line 1558, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule samtools_sort_CLIP (line 1577, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule samtools_index_CLIP (line 1597, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Samtools from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_CLIPper_CLIP (line 1614, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_peaks_iCLIP (line 1636, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule normalize_peaks_eCLIP (line 1658, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n    * Param sample is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule count_rate_for_peaks_CLIP (line 1697, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_context_for_sites_CLIP (line 1721, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calculate_weighted_rate_for_sites_CLIP (line 1745, /tmp/tmpvn832a0x/exp/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_iCLIP_site_result_table (line 1770, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_eCLIP_site_result_table_eCLIP (line 1790, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_iCLIP_peak_result_table (line 1810, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_eCLIP_peak_result_table (line 1830, /tmp/tmpvn832a0x/exp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_simulation (line 3, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule simulate (line 34, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_input_list_for_boxplot (line 61, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_spike_in_ground_truth (line 82, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_spike_in_simulation (line 102, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule simulate_spike_in (line 128, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PROBer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule build_RSEM_simulation (line 156, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable script_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule simulate_RSEM (line 178, /tmp/tmpvn832a0x/simulation/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable RSEM_simulate from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1571995290.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "enezermjema/Mini-project-group-06",
    "linting": "Lints for rule nanoqc (line 18, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule porechop (line 39, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule minimap2_illum_nanop (line 62, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sam_sorting1 (line 94, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule freebayes_vc (line 133, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ngmlr_svc (line 169, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sam_sorting2 (line 201, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sniffles_svc (line 239, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule variants_stats (line 268, /tmp/tmpfp2rcamn/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1598603620.0
  },
  {
    "data_format": 1,
    "description": "machine learning project for Cyber Carpentry 2019",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cyber-carpentry/group2-machine-learning",
    "linting": "Lints for rule fashion (line 10, /tmp/tmprjd5lr01/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mnist (line 29, /tmp/tmprjd5lr01/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule copy (line 45, /tmp/tmprjd5lr01/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1564155144.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow implementing Accel Amplicon (Swift) trimming guidelines.",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "clinical-genomics-uppsala/accel_amplicon_trimming",
    "linting": "ModuleNotFoundError in line 12 of /tmp/tmppu_7yajd/rules/accel_amplicon.smk:\nNo module named \u0027pytools\u0027\n  File \"/tmp/tmppu_7yajd/Snakefile\", line 18, in \u003cmodule\u003e\n  File \"/tmp/tmppu_7yajd/rules/accel_amplicon.smk\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1542986831.0
  },
  {
    "data_format": 1,
    "description": "An example use of multiQC within snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jakevc/snakemake_multiqc",
    "linting": "Lints for rule fastqc (line 15, /tmp/tmp2ic0x7cb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multiqc (line 42, /tmp/tmp2ic0x7cb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614617650.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpwhb2fzwk/Snakefile\":  Keyword \"input\" at line 36 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Teichlab/sc_snakemake",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmpwhb2fzwk/Snakefile:\nNo module named \u0027tqdm\u0027\n  File \"/tmp/tmpwhb2fzwk/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmpwhb2fzwk/utils/parse.py\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1536609501.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpjvhlx4ji/Snakefile\":  Keyword \"input\" at line 32 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "solida-core/diva.wgs",
    "linting": "KeyError in line 10 of /tmp/tmpjvhlx4ji/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpjvhlx4ji/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1611907296.0
  },
  {
    "data_format": 1,
    "description": "This is a pipeline to run basic RNA-seq analysis for single-end data.",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ohsu-cedar-comp-hub/Bulk-RNA-seq-pipeline-SE",
    "linting": "Lints for snakefile /tmp/tmp66_cxqj3/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule trimming (line 1, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 33, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqscreen (line 62, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule STAR (line 92, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index (line 130, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule star_statistics (line 154, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compile_star_counts (line 174, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_counts (line 198, /tmp/tmp66_cxqj3/rules/align_rmdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule insertion_profile (line 2, /tmp/tmp66_cxqj3/rules/omic_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clipping_profile (line 28, /tmp/tmp66_cxqj3/rules/omic_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule read_distribution (line 54, /tmp/tmp66_cxqj3/rules/omic_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule read_GC (line 82, /tmp/tmp66_cxqj3/rules/omic_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_init (line 4, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_pairwise (line 39, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_group (line 80, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_QC (line 118, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_qplot (line 156, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deseq2_density (line 186, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule GO (line 217, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule volcano (line 250, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule permutation (line 280, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_glimma (line 313, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_glimma_mds (line 343, /tmp/tmp66_cxqj3/rules/deseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1582765139.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ngs-docs/2020-ggg-201b-rnaseq",
    "linting": "Lints for snakefile /tmp/tmpjw60yvhu/Snakefile:\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule knit_actual (line 35, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_reads (line 58, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_raw (line 80, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule download_trimmomatic_adapter_file (line 109, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule quality_trim (line 127, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc_trimmed (line 153, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule multiqc (line 181, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param raw_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param trimmed_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule download_yeast_transcriptome (line 213, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule salmon_index (line 231, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_quantify (line 256, /tmp/tmpjw60yvhu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1614964694.0
  },
  {
    "data_format": 1,
    "description": "bioinformatics tutorial demonstrating a simple workflow for cDNA based differential gene expression using minimap2, Rsubread and DESeq2",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "callumparr/ont_tutorial_cdna_deseq2",
    "linting": "Lints for snakefile /tmp/tmpe7o_49hh/Snakefile:\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_reference_genome (line 53, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_genome_annotation (line 69, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule minimap_ref_idx (line 84, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule minimap (line 107, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule flagstat (line 131, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1608045959.0
  },
  {
    "data_format": 1,
    "description": "bioinformatics tutorial demonstrating a simple workflow for cDNA based differential gene expression using minimap2, Rsubread and DESeq2",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "callumparr/ont_tutorial_cdna_deseq2",
    "linting": "Lints for snakefile /tmp/tmpe7o_49hh/Snakefile:\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_reference_genome (line 53, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_genome_annotation (line 69, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule minimap_ref_idx (line 84, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule minimap (line 107, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule flagstat (line 131, /tmp/tmpe7o_49hh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1608045959.0
  },
  {
    "data_format": 1,
    "description": "Snakemake based analysis pipeline to identify m6As from eCLIP data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ajlabuc/meCLIP",
    "linting": "Lints for rule ip_fastqc_preAdapters (line 15, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_umitools_extract (line 45, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_cutadapt_round1 (line 75, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_cutadapt_round2 (line 106, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_fastqc_postAdapters (line 134, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_STAR_repbase (line 164, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_STAR_genome (line 202, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_umitools_dedup (line 238, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_samtools_mpileup (line 278, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_meCLIP (line 304, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ip_formatOutput (line 341, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_fastqc_preAdapters (line 371, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_umitools_extract (line 402, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_cutadapt_round1 (line 432, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_cutadapt_round2 (line 463, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_fastqc_postAdapters (line 491, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_STAR_repbase (line 521, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_STAR_genome (line 559, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_umitools_dedup (line 595, /tmp/tmpp355swaa/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_samtools_mpileup (line 635, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_meCLIP (line 661, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule input_formatOutput (line 698, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule meCLIP_inputParser (line 728, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule meCLIP_bedGenerator (line 751, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bedtools (line 777, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule meCLIP_m6aAnnotator (line 803, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule metaPlotR_perl (line 825, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule metaPlotR_R (line 855, /tmp/tmpp355swaa/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607966123.0
  },
  {
    "data_format": 1,
    "description": "Infer network from count data within Snakemake using a sparse Poisson lognormal model (PLNnetwork)",
    "formatting": "[ERROR] In file \"/tmp/tmpkh_673d5/workflow/Snakefile\":  \u0027EmptyContextError: L51: rule has no keywords attached to it.\u0027\n[INFO] In file \"/tmp/tmpkh_673d5/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpkh_673d5/workflow/Snakefile\":  1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cpauvert/PLNnetwork-workflow",
    "linting": "FileNotFoundError in line 6 of /tmp/tmpkh_673d5/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpkh_673d5/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmpkh_673d5/workflow/rules/common.smk\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "snakemake-workflows",
      "microbiome-analysis",
      "network-inference",
      "count-data"
    ],
    "updated_at": 1605801399.0
  },
  {
    "data_format": 1,
    "description": "TIGER Snakemake pipeline",
    "formatting": "[WARNING] In file \"/tmp/tmp4blo6wgc/workflow/rules/01_mapping.smk\":  Keyword \"params\" at line 47 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmp4blo6wgc/workflow/rules/07_make_tiger_output.smk\":  \u0027NoParametersError: L18: In resources definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ibebio/tiger-pipeline",
    "linting": "/tmp/tmp4blo6wgc/workflow/rules/common.smk:16: ParserWarning: Falling back to the \u0027python\u0027 engine because the \u0027c\u0027 engine does not support regex separators (separators \u003e 1 char and different from \u0027\\s+\u0027 are interpreted as regex); you can avoid this warning by specifying engine=\u0027python\u0027.\n  # validate(samples, schema=\"../schemas/samples.schema.yaml\")\nLints for snakefile /tmp/tmp4blo6wgc/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp4blo6wgc/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_transposable_elements (line 267, /tmp/tmp4blo6wgc/workflow/rules/03_prepare_marker_files.smk):\n    * Param vcf_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607072584.0
  },
  {
    "data_format": 1,
    "description": "This workflow performs CLIP peak calling using omniCLIP.",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "simojoe/omniCLIP-workflow",
    "linting": "Lints for snakefile /tmp/tmp4sryeztz/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp4sryeztz/workflow/rules/common.smk:\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_chrs (line 2, /tmp/tmp4sryeztz/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_annotation (line 24, /tmp/tmp4sryeztz/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_genome (line 52, /tmp/tmp4sryeztz/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_SRR_PE_datasets (line 76, /tmp/tmp4sryeztz/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_SRR_SE_datasets (line 101, /tmp/tmp4sryeztz/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule STAR_genomeGenerate (line 1, /tmp/tmp4sryeztz/workflow/rules/rnaseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule STAR_CLIP_alignment (line 54, /tmp/tmp4sryeztz/workflow/rules/rnaseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule sort_index_BAM (line 118, /tmp/tmp4sryeztz/workflow/rules/rnaseq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule git_omniCLIP (line 2, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param omni_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule compile_omniCLIP_requirements (line 34, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prepare_omniCLIP_annotation (line 62, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param omni_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule prepare_bg_dat (line 102, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prepare_clip_dat (line 145, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_omniCLIP (line 190, /tmp/tmp4sryeztz/workflow/rules/omniCLIP.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param omni_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595617585.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "thomasbtf/snakemake-workflow-book-popularity",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614969349.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for GWAS.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "maehler/gwas-workflow",
    "linting": "Lints for snakefile /tmp/tmpvn078350/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule cluster_config (line 57, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gemma (line 78, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule link_relatedness (line 128, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gemma_relatedness (line 148, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule setup_plink_traits (line 196, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule link_plink (line 220, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule plink (line 242, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param plink_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule link_variants (line 290, /tmp/tmpvn078350/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1575357364.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpq7r0soer/Snakefile\":  \u0027InvalidParameterSyntax: 284scale_constant=lambda wildcards:10000 if config[\u0027reference_spikein\u0027] else10000000\n\u0027\n[INFO] In file \"/tmp/tmpq7r0soer/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "robinmeyers/cutnrun-snakemake",
    "linting": "WorkflowError in line 19 of /tmp/tmpq7r0soer/Snakefile:\nError validating config file.\nValidationError: None is not of type \u0027string\u0027\n\nFailed validating \u0027type\u0027 in schema[\u0027properties\u0027][\u0027reference\u0027]:\n    OrderedDict([(\u0027type\u0027, \u0027string\u0027)])\n\nOn instance[\u0027reference\u0027]:\n    None\n  File \"/tmp/tmpq7r0soer/Snakefile\", line 19, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618946901.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpzwl419s1/workflow/Snakefile\":  \u0027NoParametersError: L34: In input definition.\u0027\n[INFO] In file \"/tmp/tmpzwl419s1/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "royhaast/b1corr-smk",
    "linting": "FileNotFoundError in line 5 of /tmp/tmpzwl419s1/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpzwl419s1/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1596235610.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for running cellphonedb.",
    "formatting": "[WARNING] In file \"/tmp/tmpnr_pqyub/rules/cellphonedb.smk\":  Keyword \"output\" at line 16 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "racng/single-cell-cellphonedb",
    "linting": "Lints for snakefile /tmp/tmpnr_pqyub/rules/cellphonedb.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samples (line 1, /tmp/tmpnr_pqyub/rules/cellphonedb.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prep_counts (line 24, /tmp/tmpnr_pqyub/rules/cellphonedb.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_cellphoendb (line 50, /tmp/tmpnr_pqyub/rules/cellphonedb.smk):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1606168873.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "hans-vg/snakemake_basespace_merge_qc",
    "linting": "Lints for snakefile /tmp/tmphvi0smpk/Snakefile:\n    * Absolute path \"/{proj}-{randint}_L{lanenum1}_ds.{bsstring}/{sample}_L{lanenum}_R1_001.fastq.gz\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{proj}-{randint}_L{lanenum1}_ds.{bsstring}/{sample}_L{lanenum}_R2_001.fastq.gz\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*/{sample}_L*_R1_001.fastq.gz\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*/{sample}_L*_R2_001.fastq.gz\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule mergeFastqR1 (line 29, /tmp/tmphvi0smpk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mergeFastqR2 (line 45, /tmp/tmphvi0smpk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612985230.0
  },
  {
    "data_format": 1,
    "description": "Test usage of renv in a Snakemake pipeline",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kevinrue/snakemake_renv",
    "linting": "Lints for snakefile /tmp/tmp82g4ctpg/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp82g4ctpg/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule installed_packages (line 3, /tmp/tmp82g4ctpg/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule session_info (line 23, /tmp/tmp82g4ctpg/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule renv (line 43, /tmp/tmp82g4ctpg/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593955977.0
  },
  {
    "data_format": 1,
    "description": "workflow-rymo1354 created by GitHub Classroom",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cu-swe4s-fall-2019/workflow-rymo1354",
    "linting": "Lints for rule tissue_samples (line 15, /tmp/tmpa0f15bn6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TISSUE_GROUP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule gene_sample_counts (line 34, /tmp/tmpa0f15bn6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GENES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule box_plot (line 57, /tmp/tmpa0f15bn6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GENES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TISSUES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TISSUE_GROUP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1574380615.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 12 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "LabLuco/snakemake_align_MAJIQ",
    "linting": "FileNotFoundError in line 6 of /tmp/tmpr3jqx50u/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027../config/config.yaml\u0027\n  File \"/tmp/tmpr3jqx50u/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpr3jqx50u/workflow/rules/common.smk\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618475550.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kevinrue/snakemake_minimal_example",
    "linting": "Lints for snakefile /tmp/tmpq6za4t6m/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpq6za4t6m/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cat (line 3, /tmp/tmpq6za4t6m/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592238824.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for 10X using salmon alevin",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_alevin_10x",
    "linting": "Lints for snakefile /tmp/tmppvxzu7fl/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome (line 5, /tmp/tmppvxzu7fl/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule transcriptome (line 21, /tmp/tmppvxzu7fl/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genesets (line 37, /tmp/tmppvxzu7fl/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule decoys (line 53, /tmp/tmppvxzu7fl/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concatenate_genome_transcriptome (line 81, /tmp/tmppvxzu7fl/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1594369040.0
  },
  {
    "data_format": 1,
    "description": "align and deduplicate paired reads from MPE-seq protocol",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "bfairkun/201907_mpe-seq_processing",
    "linting": "Lints for snakefile /tmp/tmpbvjmzrb6/Snakefile:\n    * Absolute path \"/bin/bash\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk:\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 39:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 64:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 100:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 112:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 114:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 122:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 125:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 127:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule STAR_make_index (line 2, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param genomeDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule umi_tools_extract (line 34, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule STAR_alignment (line 68, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable genomeDir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable output_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable output_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule index_RNA_seq_bams (line 106, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_unextended_primer (line 199, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable use from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable print from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule CountJunctions_Orig (line 226, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule CountJunctions_Dedup (line 254, /tmp/tmpbvjmzrb6/rules/STAR_Alignment.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1566398972.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for 10X using cellranger count",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_cellranger_count",
    "linting": "Lints for snakefile /tmp/tmpfypu71me/workflow/rules/cellranger.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule cellranger_count (line 58, /tmp/tmpfypu71me/workflow/rules/cellranger.smk):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1594824773.0
  },
  {
    "data_format": 1,
    "description": "Snakemake example",
    "formatting": "[ERROR] In file \"/tmp/tmp1c2ox1ta/workflow/Snakefile\":  \u0027NoParametersError: L17: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "karatugo/snakemake-example",
    "linting": "Lints for snakefile /tmp/tmp1c2ox1ta/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1c2ox1ta/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599650945.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for processing functional data of the 7T MELAS study",
    "formatting": "[WARNING] In file \"/tmp/tmphcn364sn/workflow/Snakefile\":  Keyword \"input\" at line 23 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "royhaast/smk-melas",
    "linting": "Lints for snakefile /tmp/tmphcn364sn/workflow/rules/preprocessing.smk:\n    * Deprecated singularity directive used for container definition in line 9.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 30.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 48.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 67.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 84.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 103.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 123.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 136.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmphcn364sn/workflow/rules/coregistration.smk:\n    * Deprecated singularity directive used for container definition in line 16.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 34.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 69.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 93.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 108.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 131.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 149.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 164.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmphcn364sn/workflow/rules/cifti.smk:\n    * Deprecated singularity directive used for container definition in line 124.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 140.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 153.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 164.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule slicetimer (line 1, /tmp/tmphcn364sn/workflow/rules/preprocessing.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule apply_gradient_unwarp (line 88, /tmp/tmphcn364sn/workflow/rules/preprocessing.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prep_topup (line 134, /tmp/tmphcn364sn/workflow/rules/preprocessing.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mask_epi (line 299, /tmp/tmphcn364sn/workflow/rules/preprocessing.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule inverse_bbr (line 49, /tmp/tmphcn364sn/workflow/rules/coregistration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule apply_bbr (line 86, /tmp/tmphcn364sn/workflow/rules/coregistration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule inverse_fnirt (line 214, /tmp/tmphcn364sn/workflow/rules/coregistration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ica_aroma (line 295, /tmp/tmphcn364sn/workflow/rules/coregistration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule transform_ica_aroma (line 331, /tmp/tmphcn364sn/workflow/rules/coregistration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_surfaces (line 3, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_midthickness (line 30, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_tkr2scanner (line 57, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_scanner2epi (line 84, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule apply_surf_tkr2scanner (line 113, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule apply_surf_scanner2epi (line 141, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule apply_surf_scanner2mni (line 169, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule resample_surface (line 200, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule inflate_surface (line 226, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule volume_to_surface (line 259, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule volume_to_surface_aroma (line 292, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule resample_surface_data (line 327, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule resample_surface_data_aroma (line 358, /tmp/tmphcn364sn/workflow/rules/surface.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule transform_rois (line 3, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_rois (line 49, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_subcortical_labels (line 95, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_dense_labels (line 120, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_dtseries (line 146, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_dtseries_aroma (line 180, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_confounds (line 214, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule parcellate_dtseries (line 330, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule correlate_ptseries (line 362, /tmp/tmphcn364sn/workflow/rules/cifti.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1614212243.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "thomasbtf/document-anonymization",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1619381234.0
  },
  {
    "data_format": 1,
    "description": "Template repository for snakemake neuroimaging workflows",
    "formatting": "[ERROR] In file \"/tmp/tmp5jmmj0n9/workflow/Snakefile\":  \u0027NoParametersError: L26: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "akhanf/snakemake_neuroworkflow",
    "linting": "Lints for snakefile /tmp/tmp5jmmj0n9/workflow/rules/common.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1596039808.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline running `cellranger mkref`",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_cellranger_mkref",
    "linting": "Lints for snakefile /tmp/tmp1rcti4f1/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1rcti4f1/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1rcti4f1/workflow/rules/cellranger.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule cellranger_mkref (line 72, /tmp/tmp1rcti4f1/workflow/rules/cellranger.smk):\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1594824950.0
  },
  {
    "data_format": 1,
    "description": "Snakemake example workflow with singularity",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zavolanlab/snakemake_example_workflow",
    "linting": "Lints for snakefile /tmp/tmpy7va10h3/Snakefile:\n    * Deprecated singularity directive used for container definition in line 12.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 25.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 47.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 67.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1578410137.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for quantitation of single-cell data using salmon alevin",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_alevin_quant",
    "linting": "Lints for snakefile /tmp/tmpoytyuef8/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpoytyuef8/workflow/rules/common.smk:\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule alevin (line 4, /tmp/tmpoytyuef8/workflow/rules/alevin.smk):\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595585578.0
  },
  {
    "data_format": 1,
    "description": "Template snakemake pipeline for minimal tests",
    "formatting": "[ERROR] In file \"/tmp/tmp11xwkisd/workflow/Snakefile\":  \u0027NoParametersError: L17: In input definition.\u0027\n[INFO] In file \"/tmp/tmp11xwkisd/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmp11xwkisd/workflow/Snakefile\":  1 file(s) would be changed \ud83d\ude2c\n[INFO] In file \"/tmp/tmp11xwkisd/workflow/Snakefile\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_test",
    "linting": "Lints for snakefile /tmp/tmp11xwkisd/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp11xwkisd/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1598438409.0
  },
  {
    "data_format": 1,
    "description": "Tract-weighted functional connectivity snakemake workflow - WIP",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "akhanf/tract-fconn-smk",
    "linting": "Lints for snakefile /tmp/tmppg9t7vc6/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule visualize_connectome (line 375, /tmp/tmppg9t7vc6/workflow/rules/mrtrix.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1590467060.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to evaluate peak calling",
    "formatting": "[WARNING] In file \"/tmp/tmpssu7i1p0/workflow/Snakefile\":  Keyword \"input\" at line 12 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "knutdrand/peak-eval",
    "linting": "Lints for snakefile /tmp/tmpssu7i1p0/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpssu7i1p0/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpssu7i1p0/workflow/rules/encode.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule macs_wo_control (line 6, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable genomesize from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule hmmacs_wo_control (line 25, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter_hmmacs (line 45, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule callpeak_w_control (line 64, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable genomesize from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule move_macs_peaks (line 85, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summit_regions (line 104, /tmp/tmpssu7i1p0/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_meme (line 4, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_peaks (line 16, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_peak_sequences (line 35, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule motif_enrichment (line 58, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule motif_plot (line 78, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule join_plot (line 99, /tmp/tmpssu7i1p0/workflow/rules/motifs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule encode_download (line 1, /tmp/tmpssu7i1p0/workflow/rules/encode.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule region_bamtobed (line 29, /tmp/tmpssu7i1p0/workflow/rules/encode.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sortbed (line 49, /tmp/tmpssu7i1p0/workflow/rules/encode.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603349411.0
  },
  {
    "data_format": 1,
    "description": "Detect circRNAs in fastq files using CIRCExplorer and CIRI2",
    "formatting": "[ERROR] In file \"/tmp/tmp9sxj0kkh/workflow/Snakefile\":  \u0027NoParametersError: L17: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "mywanuo/circRNADetection",
    "linting": "Lints for snakefile /tmp/tmp9sxj0kkh/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp9sxj0kkh/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1616086967.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for slamDunk toolkit",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jafors/rna-seq-slamseq",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_annotation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmpgnfk2otz/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpgnfk2otz/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpgnfk2otz/workflow/rules/slamdunk.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule summary (line 219, /tmp/tmpgnfk2otz/workflow/rules/alleyoop.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule multiqc (line 362, /tmp/tmpgnfk2otz/workflow/rules/alleyoop.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615395248.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline to generate a reference index for Salmon Alevin",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_alevin_index",
    "linting": "Lints for snakefile /tmp/tmpn_mgg2pm/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 12.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpn_mgg2pm/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome (line 5, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule transcriptome (line 21, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genesets (line 37, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule decoys (line 53, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gentrome (line 81, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_index (line 106, /tmp/tmpn_mgg2pm/workflow/rules/salmon.smk):\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DATETIME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1595515551.0
  },
  {
    "data_format": 1,
    "description": "Benchmark of methods for 10X single-cell RNA-sequencing",
    "formatting": "[ERROR] In file \"/tmp/tmpbu7dnj6f/workflow/Snakefile\":  \u0027NoParametersError: L17: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/10x-benchmark-rna",
    "linting": "Lints for snakefile /tmp/tmpbu7dnj6f/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpbu7dnj6f/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1594393499.0
  },
  {
    "data_format": 1,
    "description": "Demonstration Snakemake workflow for Ronin/AWS Slurm-based auto-scaling cluster",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nathanhaigh/ronin_snakemake_demo",
    "linting": "Lints for snakefile /tmp/tmpr171r3dr/Snakefile:\n    * Absolute path \"/\u003e\u003cAttribute name = \" in line 41:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Deprecated singularity directive used for container definition in line 3.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule annotation (line 28, /tmp/tmpr171r3dr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genome (line 48, /tmp/tmpr171r3dr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule transcriptome (line 67, /tmp/tmpr171r3dr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule reads (line 83, /tmp/tmpr171r3dr/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "ronin",
      "aws",
      "aws-parallelcluster"
    ],
    "updated_at": 1588569193.0
  },
  {
    "data_format": 1,
    "description": "Genome assembly evaluation workflow",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "percyfal/assemblyeval-smk",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1605568468.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmprblomc0r/Snakefile\":  \u0027IndexError: pop from empty list\u0027\n[INFO] In file \"/tmp/tmprblomc0r/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "skurscheid/snakemake",
    "linting": "SyntaxError in line 16 of /tmp/tmprblomc0r/Snakefile:\nEOF in multi-line statement (Snakefile, line 16)\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1570492914.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpfmcb2s4d/rules/filtering.smk\":  Keyword \"input\" at line 55 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpfmcb2s4d/rules/vqsr_indel.smk\":  Keyword \"input\" at line 10 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 11 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rimjhimroy/SnakeGATK",
    "linting": "Lints for snakefile /tmp/tmpfmcb2s4d/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 104, /tmp/tmpfmcb2s4d/rules/processbam.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concordance (line 65, /tmp/tmpfmcb2s4d/rules/calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 2, /tmp/tmpfmcb2s4d/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 34, /tmp/tmpfmcb2s4d/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concordance_all (line 65, /tmp/tmpfmcb2s4d/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1578575623.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpgcrxsq3c/workflow/Snakefile\":  Keyword \"input\" at line 22 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "data-exp-lab/topostellar-oceans",
    "linting": "KeyError in line 23 of /tmp/tmpgcrxsq3c/workflow/Snakefile:\n\u0027workdir\u0027\n  File \"/tmp/tmpgcrxsq3c/workflow/Snakefile\", line 23, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1615486544.0
  },
  {
    "data_format": 1,
    "description": "adat",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Mar111tiN/DNAseq-GATK-Haplo",
    "linting": "FileNotFoundError in line 18 of /tmp/tmp51tt0b06/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmp51tt0b06/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp51tt0b06/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1551012620.0
  },
  {
    "data_format": 1,
    "description": "Clinical Analyis Pipeline - Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "clinical-genomics-uppsala/CAPS",
    "linting": "WorkflowError in line 7 of /tmp/tmpj_65dytg/Snakefile:\nConfig file must be given as JSON or YAML with keys at top level.\n  File \"/tmp/tmpj_65dytg/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1543834180.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to setup data sources",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "percyfal/datasources-smk",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615498512.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmph3xltbxl/Snakefile\":  \u0027NoParametersError: L14: In input definition.\u0027\n[INFO] In file \"/tmp/tmph3xltbxl/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmph3xltbxl/Snakefile\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "Phoenix-Effect/gatk",
    "linting": "WorkflowError in line 5 of /tmp/tmph3xltbxl/Snakefile:\nConfig file must be given as JSON or YAML with keys at top level.\n  File \"/tmp/tmph3xltbxl/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1529387615.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jensenrichardson/dna-variant-calling",
    "linting": "FileNotFoundError in line 11 of /tmp/tmp_a48c8yo/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027/scratch/07467/jwr2735/dna-preprocess/samples.tsv\u0027\n  File \"/tmp/tmp_a48c8yo/workflow/Snakefile\", line 11, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618587603.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpxbay4xce/rules/deepTools_data_prep.smk\":  \u0027NoParametersError: L128: In params definition.\u0027\n[ERROR] In file \"/tmp/tmpxbay4xce/rules/run_idr.smk\":  \u0027NoParametersError: L25: In params definition.\u0027\n[ERROR] In file \"/tmp/tmpxbay4xce/rules/bam_post_processing.smk\":  \u0027NoParametersError: L36: In params definition.\u0027\n[ERROR] In file \"/tmp/tmpxbay4xce/rules/run_fastp.smk\":  \u0027IndexError: pop from empty list\u0027\n[INFO] 4 file(s) raised parsing errors \ud83e\udd15\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "skurscheid/cellcycle_workflow",
    "linting": "KeyError in line 8 of /tmp/tmpxbay4xce/Snakefile:\n\u0027references\u0027\n  File \"/tmp/tmpxbay4xce/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1588924284.0
  },
  {
    "data_format": 1,
    "description": "A workflow for RNA-seq analyses in Snakemake",
    "formatting": "[INFO] 28 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jma1991/rnaseq",
    "linting": "KeyError in line 12 of /tmp/tmpr32w96mr/workflow/rules/project.smk:\n\u0027samples\u0027\n  File \"/tmp/tmpr32w96mr/workflow/Snakefile\", line 9, in \u003cmodule\u003e\n  File \"/tmp/tmpr32w96mr/workflow/rules/project.smk\", line 12, in __init__\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "rnaseq",
      "snakemake"
    ],
    "updated_at": 1612775589.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpwkk5l3dp/rules/mapping.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 4:7: output:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fischuu/Pipeline-WGS-GATK",
    "linting": "KeyError in line 9 of /tmp/tmpwkk5l3dp/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpwkk5l3dp/Snakefile\", line 9, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593208342.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for structural connectome using mrtrix3 (loading BIDS data from prepdwi, fmriprep)",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "akhanf/structural_connectome_smk",
    "linting": "ModuleNotFoundError in line 3 of /tmp/tmp12p_7jim/Snakefile:\nNo module named \u0027bids\u0027\n  File \"/tmp/tmp12p_7jim/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1587729220.0
  },
  {
    "data_format": 1,
    "description": "tweaking chipseq workflow for ATACseq",
    "formatting": "[WARNING] In file \"/tmp/tmp3k7u7w3_/workflow/rules/filtering.smk\":  Keyword \"params\" at line 6 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp3k7u7w3_/workflow/rules/filtering.smk\":  Inline-formatted keyword \"threads\" at line 56 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AnnaLorenc/ATACseq",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603204022.0
  },
  {
    "data_format": 1,
    "description": "Experimental workflow for ATAC-seq data analysis using snakemake",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mhebrard/smk-ATAC-seq",
    "linting": "KeyError in line 7 of /tmp/tmprho0xztw/Snakefile:\n\u0027genome\u0027\n  File \"/tmp/tmprho0xztw/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1546502933.0
  },
  {
    "data_format": 1,
    "description": "A workflow for ChIP-seq analyses in Snakemake",
    "formatting": "Usage: snakefmt [OPTIONS] [SRC]...\n\nError: Invalid value for \u0027[SRC]...\u0027: Path \u0027/tmp/tmpnnw226yy/workflow/rules/.#markdup.smk\u0027 does not exist.\n",
    "full_name": "jma1991/chipseq",
    "linting": "WorkflowError in line 2 of /tmp/tmpnnw226yy/Snakefile:\nConfig file config.yaml not found.\n  File \"/tmp/tmpnnw226yy/Snakefile\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612775523.0
  },
  {
    "data_format": 1,
    "description": "CODA microbiome-analysis using aldex2, pylofactorisation",
    "formatting": "[WARNING] In file \"/tmp/tmpgay5bxe9/Snakefile\":  Keyword \"input\" at line 19 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpgay5bxe9/Snakefile\":  Keyword \"input\" at line 108 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SilasK/microbiome-analysis",
    "linting": "KeyError in line 10 of /tmp/tmpgay5bxe9/Snakefile:\n\u0027Comparisons\u0027\n  File \"/tmp/tmpgay5bxe9/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618240784.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for Kaiju analysis",
    "formatting": "[ERROR] In file \"/tmp/tmpib3o_fzv/Snakefile\":  \u0027NoParametersError: L14: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "ropolomx/kaiju-snake",
    "linting": "WorkflowError in line 5 of /tmp/tmpib3o_fzv/Snakefile:\nConfig file must be given as JSON or YAML with keys at top level.\n  File \"/tmp/tmpib3o_fzv/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1550001482.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for RNA-seq mapping",
    "formatting": "[WARNING] In file \"/tmp/tmpbrt_ct13/workflow/Snakefile\":  Keyword \"input\" at line 17 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "percyfal/rna-seq-map-smk",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1602190673.0
  },
  {
    "data_format": 1,
    "description": "test snakemake workflow",
    "formatting": "[WARNING] In file \"/tmp/tmp92cv06s0/workflow/Snakefile\":  Keyword \"input\" at line 38 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "visze/new_project",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610716160.0
  },
  {
    "data_format": 1,
    "description": "Repo for analyses of Amplicon Sequencing data at LSTM within the Vector informatics and Genomics group",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sanjaynagi/AmpSeqVIGG",
    "linting": "FileNotFoundError in line 10 of /tmp/tmp3zqc8a6p/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027resources/present.samples.txt\u0027\n  File \"/tmp/tmp3zqc8a6p/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618918642.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpvy8vo8mn/workflow/Snakefile\":  Keyword \"input\" at line 23 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "jensenrichardson/NeoAntigen-Prediction",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpvy8vo8mn/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027/scratch/07467/jwr2735/neoant-pred/config/rna_samples.tsv\u0027\n  File \"/tmp/tmpvy8vo8mn/workflow/Snakefile\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619391572.0
  },
  {
    "data_format": 1,
    "description": "computes all variants including indels from Illumina sequences compared to a reference sequence",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "medvir/variant-calling",
    "linting": "Lints for rule sample_reads (line 16, /tmp/tmpiif534w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_to_ref (line 39, /tmp/tmpiif534w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule indelqual (line 69, /tmp/tmpiif534w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule call_variants (line 93, /tmp/tmpiif534w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1568204022.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "skurscheid/aks_snakemake",
    "linting": "Lints for snakefile /tmp/tmp0ilggxra/Snakefile:\n    * Absolute path \"/data/cellcycledata/aks_test\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Environment variable HOME used but not asserted with envvars directive in line 6.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp0ilggxra/rules/run_fastp.smk:\n    * Environment variable HOME used but not asserted with envvars directive in line 20.:\n      Asserting existence of environment variables with the envvars directive\n      ensures proper error messages if the user fails to invoke a workflow with\n      all required environment variables defined. Further, it allows snakemake\n      to pass them on in case of distributed execution.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#environment-variables\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 23.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 24.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_fastp (line 26, /tmp/tmp0ilggxra/rules/run_fastp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * The version directive is deprecated:\n      It was meant for documenting tool version, but this has been replaced by\n      using the conda or container directive.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1551148919.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "andrewsanchez/GenBankQC-Workflow",
    "linting": "Lints for snakefile /tmp/tmpsrx99ezf/rules/mash.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpsrx99ezf/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule download (line 4, /tmp/tmpsrx99ezf/rules/genome-download.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable outdir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable section from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable format from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable assembly_level from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable taxid from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable group from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sketch (line 11, /tmp/tmpsrx99ezf/rules/mash.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule paste (line 27, /tmp/tmpsrx99ezf/rules/mash.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable section_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule dist (line 50, /tmp/tmpsrx99ezf/rules/mash.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mean_dist (line 69, /tmp/tmpsrx99ezf/rules/mash.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_stats (line 12, /tmp/tmpsrx99ezf/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc (line 29, /tmp/tmpsrx99ezf/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule metadata (line 1, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biosample (line 17, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule xtract_biosample (line 33, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule parse_biosample (line 49, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sra (line 67, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule summary (line 83, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule names (line 99, /tmp/tmpsrx99ezf/rules/metadata.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1581689344.0
  },
  {
    "data_format": 1,
    "description": "A workflow for scRNA-seq processing in Snakemake",
    "formatting": "[INFO] 14 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jma1991/scrnaseq",
    "linting": "Lints for snakefile /tmp/tmpea14p7jm/workflow/rules/star.smk:\n    * Absolute path \"/tmp/star\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule bustools_count_spliced (line 137, /tmp/tmpea14p7jm/workflow/rules/bustools.smk):\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bustools_count_unspliced (line 175, /tmp/tmpea14p7jm/workflow/rules/bustools.smk):\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule eisar (line 6, /tmp/tmpea14p7jm/workflow/rules/eisar.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genomepy_gentrome (line 70, /tmp/tmpea14p7jm/workflow/rules/genomepy.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genomepy_decoys (line 90, /tmp/tmpea14p7jm/workflow/rules/genomepy.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule curl_barcodes (line 6, /tmp/tmpea14p7jm/workflow/rules/resources.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule singlecellexperiment_kallisto (line 6, /tmp/tmpea14p7jm/workflow/rules/singlecellexperiment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule singlecellexperiment_salmon (line 38, /tmp/tmpea14p7jm/workflow/rules/singlecellexperiment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule singlecellexperiment_star (line 70, /tmp/tmpea14p7jm/workflow/rules/singlecellexperiment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612775318.0
  },
  {
    "data_format": 1,
    "description": "A small example to test DENTIST\u0027s workflow",
    "formatting": "[ERROR] In file \"/tmp/tmps__pdpzj/Snakefile\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7f10145320d0\u003eNewline expected after keyword \u0027run\u0027\u0027\n[INFO] In file \"/tmp/tmps__pdpzj/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "a-ludi/dentist-example",
    "linting": "KeyError in line 712 of /tmp/tmps__pdpzj/Snakefile:\n\u0027inputs\u0027\n  File \"/tmp/tmps__pdpzj/Snakefile\", line 712, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619011244.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for building an ANTS template (from antsMultivariateTemplateConstruction2.sh)",
    "formatting": "[ERROR] In file \"/tmp/tmpgkdexgwq/workflow/Snakefile\":  \u0027InvalidParameterSyntax: 57cmd=lambda wildcards, input, output:\n\u0027ResampleImageBySpacing {dim} {input} {output} {vox_dims}\u0027. format(\n dim= config[\u0027ants\u0027][\u0027dim\u0027], input= input, output= output,\n vox_dims=\u0027 \u0027. join([ str( d) for d in config[\u0027resample_vox_dims\u0027]]))\n if config[\u0027resample_init_template\u0027] elsef\"cp -v {input} {output}\"\n\u0027\n[INFO] In file \"/tmp/tmpgkdexgwq/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "akhanf/ants_build_template_smk",
    "linting": "Lints for rule cp_nii_to_templateflow_naming (line 433, /tmp/tmpgkdexgwq/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_template_json (line 452, /tmp/tmpgkdexgwq/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1601493502.0
  },
  {
    "data_format": 1,
    "description": "Repository for m5c data analysis",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 2 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "pd321/m5c",
    "linting": "FileNotFoundError in line 9 of /tmp/tmpf2p_te4j/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027config/samples.tsv\u0027\n  File \"/tmp/tmpf2p_te4j/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpf2p_te4j/workflow/rules/common.smk\", line 9, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614161374.0
  },
  {
    "data_format": 1,
    "description": "Fast profiler using mmseqs",
    "formatting": "[ERROR] In file \"/tmp/tmpto0zod6e/Snakefile\":  \u0027NoParametersError: L14: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "SilasK/profiler",
    "linting": "WorkflowError in line 5 of /tmp/tmpto0zod6e/Snakefile:\nConfig file must be given as JSON or YAML with keys at top level.\n  File \"/tmp/tmpto0zod6e/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1544104547.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "marcjwilliams1/lpWGS",
    "linting": "PermissionError in line 8 of /tmp/tmp_5owge5m/Snakefile:\n[Errno 13] Permission denied: \u0027/data/BCI-EvoCa2\u0027\n  File \"/tmp/tmp_5owge5m/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1567023448.0
  },
  {
    "data_format": 1,
    "description": "The evaluation of MERFISHtools and the underyling model as published in the corresponding article.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "merfishtools/merfishtools-evaluation",
    "linting": "Lints for snakefile /tmp/tmpv22r2qw7/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule format (line 89, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_mhd2_codebook (line 113, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_mhd4_codebook (line 151, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cell_props (line 187, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule raw_counts (line 211, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule count_matrix (line 235, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule estimate_real_error_rates (line 278, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule estimate_simulated_error_rates (line 309, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule expression_matrix (line 390, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule scale_factors (line 417, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule normalize_expression_matrix (line 445, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule normalize_cdf (line 470, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule diffexp (line 504, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multidiffexp (line 548, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule enrichment (line 584, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule simulate_counts (line 615, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule simulate (line 644, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_error_rates (line 694, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_error_rate_uncertainty (line 722, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_cv_raw_vs_posterior (line 756, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_naive_vs_map (line 781, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_enrichment (line 807, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_multidiffexp (line 833, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_expression_pmf (line 864, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_foldchange_cdf (line 890, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_qq (line 915, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_expression_dist (line 943, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_overdispersion (line 967, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_correlation (line 991, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_tsne (line 1015, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_simulation (line 1048, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_m_vs_errors (line 1083, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_simulation_pmf (line 1111, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dataset_correlation (line 1140, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dataset_correlation_error (line 1165, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_neighborhood (line 1192, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_em (line 1223, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_thbs1_bias (line 1250, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_error_rates (line 1276, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_error_rate_uncertainty (line 1301, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_naive_vs_map (line 1326, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_example (line 1352, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_simulation (line 1378, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_simulation_supp (line 1407, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_simulation_ci_error (line 1436, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_clustering (line 1463, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_multidiffexp (line 1494, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_cv_raw_vs_posterior (line 1523, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_exact_vs_corrected (line 1548, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_model (line 1571, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule figure_dataset_correlation (line 1598, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_svg (line 1625, /tmp/tmpv22r2qw7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "workflow",
      "snakemake-workflows"
    ],
    "updated_at": 1529095773.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake pipeline to call variants (SNPs, InDels) from DNA-seq Illumina fastq files. ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "BleekerLab/freebayes_snp_calling",
    "linting": "AttributeError in line 176 of /tmp/tmpp0ci7847/Snakefile:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmpp0ci7847/Snakefile\", line 176, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1598344305.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp07x7m0c9/workflow/rules/unused.smk\":  Keyword \"shell\" at line 47 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp07x7m0c9/workflow/rules/segmentation.smk\":  Keyword \"shell\" at line 22 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp07x7m0c9/workflow/rules/registration.smk\":  Keyword \"shell\" at line 49 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp07x7m0c9/workflow/rules/registration.smk\":  Keyword \"shell\" at line 64 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp07x7m0c9/workflow/rules/registration.smk\":  Keyword \"shell\" at line 141 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmp07x7m0c9/workflow/rules/registration.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:0:     \u0027{params.deform_model} {params.deform_metric} {params.deform_multires} \u0027# deformable registration\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "akhanf/clinical-atlasreg",
    "linting": "Lints for snakefile /tmp/tmp07x7m0c9/workflow/rules/common.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp07x7m0c9/workflow/rules/registration.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp07x7m0c9/workflow/rules/segmentation.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp07x7m0c9/workflow/rules/electrodes.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule tar_results (line 42, /tmp/tmp07x7m0c9/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tar_tsv_only (line 67, /tmp/tmp07x7m0c9/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule import_subj_t1 (line 11, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule affine_aladin (line 31, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_xfm_ras2itk (line 58, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule warp_brainmask_from_template_affine (line 83, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule warp_tissue_probseg_from_template_affine (line 111, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule n4biasfield (line 150, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mask_template_t1w (line 182, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mask_subject_t1w (line 209, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ants_syn_affine_init (line 236, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule warp_dseg_from_template (line 306, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule warp_tissue_probseg_from_template (line 345, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule warp_brainmask_from_template (line 383, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dilate_brainmask (line 421, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dilate_atlas_labels (line 452, /tmp/tmp07x7m0c9/workflow/rules/registration.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tissue_seg_kmeans_init (line 8, /tmp/tmp07x7m0c9/workflow/rules/segmentation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_channels_to_tissue (line 47, /tmp/tmp07x7m0c9/workflow/rules/segmentation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tissue_seg_to_4d (line 72, /tmp/tmp07x7m0c9/workflow/rules/segmentation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule brainmask_from_tissue (line 98, /tmp/tmp07x7m0c9/workflow/rules/segmentation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vis_electrodes (line 8, /tmp/tmp07x7m0c9/workflow/rules/electrodes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule label_electrodes_atlas (line 38, /tmp/tmp07x7m0c9/workflow/rules/electrodes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_reg (line 2, /tmp/tmp07x7m0c9/workflow/rules/visqc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_probseg (line 33, /tmp/tmp07x7m0c9/workflow/rules/visqc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_dseg (line 58, /tmp/tmp07x7m0c9/workflow/rules/visqc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1598102324.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp7p38eirt/Snakefile\":  Keyword \"input\" at line 35 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SilasK/abinner",
    "linting": "Lints for snakefile /tmp/tmp7p38eirt/Snakefile:\n    * Absolute path \"/home/kiesers/atlas/atlas/Snakefile\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule get_contig_covarage_from_bb (line 5, /tmp/tmp7p38eirt/rules/binning.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule concoct (line 93, /tmp/tmp7p38eirt/rules/binning.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1532076008.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for the analysis of bacterial WGS data",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ropolomx/boa-wgs",
    "linting": "FileNotFoundError in line 8 of /tmp/tmpwt7drko_/Snakefile:\n[Errno 2] No such file or directory: \u0027samples.tsv\u0027\n  File \"/tmp/tmpwt7drko_/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1591815603.0
  },
  {
    "data_format": 1,
    "description": "Hello, World! example for Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nathanhaigh/snakemake-hello-world",
    "linting": "Lints for rule hello_world (line 13, /tmp/tmp9mtdcpna/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1565589479.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for non-model organism analyses",
    "formatting": "[INFO] 13 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NBISweden/manticore-smk",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1608284514.0
  },
  {
    "data_format": 1,
    "description": "Creating a synthetic tumor/normal sample pair from CHM1/CHM13 haploid cell lines.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "koesterlab/chm-synthetic-tumor-normal",
    "linting": "Lints for snakefile /tmp/tmpkdir11x5/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_reads (line 27, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_eval_kit (line 48, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_confident_vcf (line 66, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_confident_bed (line 89, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compose_truth (line 109, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mix_reads (line 136, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule copy_normal (line 173, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_chr (line 196, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1580296029.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to assemble and query short haplotype genomes.",
    "formatting": "[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 87 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 115 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 142 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 166 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 187 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 211 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 236 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 282 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "JoeBlackSci/SAHDA",
    "linting": "Lints for snakefile /tmp/tmptf38mad7/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule file_append (line 125, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule helper_condense (line 446, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fasta_condense (line 475, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618580267.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "alexpenson/rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmp1p10qkuj/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmp1p10qkuj/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603375395.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "christacaggiano/GATK_exome_seq",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmps8i2a4lu/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmps8i2a4lu/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmps8i2a4lu/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmps8i2a4lu/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmps8i2a4lu/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmps8i2a4lu/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592336464.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sheucke/dna-seq-gatk-variant-calling",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmphhqaufon/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmphhqaufon/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmphhqaufon/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmphhqaufon/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmphhqaufon/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmphhqaufon/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603098218.0
  },
  {
    "data_format": 1,
    "description": "standard RNA-seq analysis pipeline (cloned from snakemake example workflow)",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yztxwd/rna-seq-standard-pipeline",
    "linting": "Lints for snakefile /tmp/tmpm7lmdsei/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/trim.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 17, /tmp/tmpm7lmdsei/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 41, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 80, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 182, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 250, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 285, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1604400794.0
  },
  {
    "data_format": 1,
    "description": "RNAseq workflow using snakemake workflow with Kallisto.",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pburnham50/KallistoPipeline",
    "linting": "Lints for snakefile /tmp/tmp4bsa7obp/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmp4bsa7obp/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599757051.0
  },
  {
    "data_format": 1,
    "description": "KAS-Seq Analysis Workflow",
    "formatting": null,
    "full_name": "lparsons/kas-seq-workflow",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614794946.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for RNAseq data processing on computing clusters with SLURM.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pawel125/RNAseq_snakemake",
    "linting": "Lints for snakefile /tmp/tmphw91_js2/workflow/Snakefile:\n    * Absolute path \"/home/pkus/resources/rseqc/mm10_RefSeq.chr.bed\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/pkus/resources/rnaseq_files/mm_gencode/gencode.vM25.primary_assembly.annotation.gtf\" in line 172:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule make_links (line 36, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_1 (line 61, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_1 (line 81, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_1 (line 114, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trimming (line 147, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_2 (line 169, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_2 (line 189, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_2 (line 222, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star (line 259, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 315, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_3 (line 335, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule read_GC_py (line 364, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tin_py (line 384, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule inner_distance_py (line 420, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genebody_coverage_py (line 442, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_stats (line 464, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_3 (line 481, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_counts_all (line 524, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "rnaseq",
      "snakemake-workflow"
    ],
    "updated_at": 1616146140.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Ducklegs17/intro_to_snakemake",
    "linting": "Lints for rule get_data (line 20, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule subset_data (line 42, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trim (line 69, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assemble (line 99, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613090143.0
  },
  {
    "data_format": 1,
    "description": "Using https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling as the template",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "weizhu365/gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmp7ogyfgvd/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmp7ogyfgvd/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp7ogyfgvd/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp7ogyfgvd/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp7ogyfgvd/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1588106733.0
  },
  {
    "data_format": 1,
    "description": "template SNP calling",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gitbackspacer/jitu-seq-gatk-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpr578vu71/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpr578vu71/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpr578vu71/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1564416750.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for GATK following best practice guidelines",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "srikanthkris/GATK",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpgdmc9nnh/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpgdmc9nnh/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpgdmc9nnh/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1566779188.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "MYB201560476/rna-seq-kallisto-sleuth",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613657705.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "silastittes/dna-seq-gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmpms22e1nb/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpms22e1nb/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpms22e1nb/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpms22e1nb/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpms22e1nb/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1582860563.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dsobral/dna-seq-gatk-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpmw3dag7v/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpmw3dag7v/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpmw3dag7v/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1569592010.0
  },
  {
    "data_format": 1,
    "description": "WGS pipeline using Snakemake",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "wenliangz/wgs-variant-calling",
    "linting": "Lints for snakefile /tmp/tmp8jtlu89x/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmp8jtlu89x/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp8jtlu89x/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp8jtlu89x/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp8jtlu89x/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1578374061.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "whtns/dna-seq-gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmpw3qpetm1/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpw3qpetm1/rules/scna.smk:\n    * Absolute path \"/CNAprofiles/segment.Rdata\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpw3qpetm1/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpw3qpetm1/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpw3qpetm1/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpw3qpetm1/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CopywriteR (line 14, /tmp/tmpw3qpetm1/rules/scna.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param copywriter_output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param input_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1584738254.0
  },
  {
    "data_format": 1,
    "description": "snakemake repository for alevin quantification of single cell RNAseq data",
    "formatting": "[ERROR] In file \"/tmp/tmpl5zx5at5/workflow/rules/alevin.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 4:10: resources:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "deevdevil88/snakemake_alevin_10x",
    "linting": "Lints for snakefile /tmp/tmpl5zx5at5/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592388734.0
  },
  {
    "data_format": 1,
    "description": "\u962a\u5927\u7528",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nkimoto/GATK_pipeline",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpvv46p3ct/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpvv46p3ct/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpvv46p3ct/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1571966680.0
  },
  {
    "data_format": 1,
    "description": "Using snakemake-workflows/dna-seq-gatk-variant-calling as a template",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dchebotarov/varcall-snakemake-test",
    "linting": "Lints for snakefile /tmp/tmpzefi8wlm/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpzefi8wlm/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpzefi8wlm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpzefi8wlm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpzefi8wlm/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1582785108.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "bioinfo-dirty-jobs/rna-seq-star-deseq2",
    "linting": "WorkflowError in line 10 of /tmp/tmpq25gzdtd/Snakefile:\nSchema file /tmp/tmpq25gzdtd/schemas/config.schema.yaml not found.\n  File \"/tmp/tmpq25gzdtd/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1558032161.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nsteinau/kallisto-snakemake",
    "linting": "FileNotFoundError in line 12 of /tmp/tmpna5qcclm/Snakefile:\n[Errno 2] No such file or directory: \u0027SRR_Acc_List.txt\u0027\n  File \"/tmp/tmpna5qcclm/Snakefile\", line 12, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1569993994.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dence/rnastar_deseq2_isoseq_trial",
    "linting": "FileNotFoundError in line 15 of /tmp/tmpzjl93g08/rules/common.smk:\n[Errno 2] No such file or directory: \u0027Jan_29_2021_units.tsv\u0027\n  File \"/tmp/tmpzjl93g08/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmpzjl93g08/rules/common.smk\", line 15, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612445934.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpzzlbywn3/rules/diffexp.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     \"../scripts/normalized_counts.R\"\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpzzlbywn3/rules/diffexp.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpzzlbywn3/rules/diffexp.smk\":  9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dence/rnastar_dseq2_old_RNAseq",
    "linting": "FileNotFoundError in line 12 of /tmp/tmpzzlbywn3/rules/common.smk:\n[Errno 2] No such file or directory: \u0027MJC_cambium_vs_MJ_cambium.samples.tsv\u0027\n  File \"/tmp/tmpzzlbywn3/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmpzzlbywn3/rules/common.smk\", line 12, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613397973.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 5 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "williamjeong2/snakemake_DNA-seq",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nNameError in line 13 of /tmp/tmpr7rhswao/rules/mapping.smk:\nname \u0027THREADS\u0027 is not defined\n  File \"/tmp/tmpr7rhswao/Snakefile\", line 19, in \u003cmodule\u003e\n  File \"/tmp/tmpr7rhswao/rules/mapping.smk\", line 13, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1616743570.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dence/rnaseq_star_iseq_trial",
    "linting": "WorkflowError in line 10 of /tmp/tmpjn7m_vil/rules/common.smk:\nError validating config file.\nValidationError: {\u0027Miami_of_Ohio\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/t3352.cdna.v1.contig\u0027, \u0027PacBio\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/PacBio\u0027, \u0027Clustered\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/clustered_longer_than_500.est_0.99\u0027, \u0027v2.01_annnotations\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/v2.01_genome_annots_index\u0027} is not of type \u0027string\u0027\n\nFailed validating \u0027type\u0027 in schema[\u0027properties\u0027][\u0027ref\u0027][\u0027properties\u0027][\u0027index\u0027]:\n    OrderedDict([(\u0027type\u0027, \u0027string\u0027)])\n\nOn instance[\u0027ref\u0027][\u0027index\u0027]:\n    {\u0027Clustered\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/clustered_longer_than_500.est_0.99\u0027,\n     \u0027Miami_of_Ohio\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/t3352.cdna.v1.contig\u0027,\n     \u0027PacBio\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/PacBio\u0027,\n     \u0027v2.01_annnotations\u0027: \u0027/home/d.ence/d.ence_peter_share/pinus_taeda_L/reanalysis_old_RNAseq/rna-seq-star-deseq2/star_indices/v2.01_genome_annots_index\u0027}\n  File \"/tmp/tmpjn7m_vil/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmpjn7m_vil/rules/common.smk\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608505416.0
  },
  {
    "data_format": 1,
    "description": "Analysis of 4sU RNA-Seq data and calculation of rates using the INSPEcT package",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "marykthompson/foursu_steady_state",
    "linting": "Lints for snakefile /tmp/tmpbtbxkb8u/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpbtbxkb8u/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpbtbxkb8u/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpbtbxkb8u/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmpbtbxkb8u/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmpbtbxkb8u/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmpbtbxkb8u/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmpbtbxkb8u/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmpbtbxkb8u/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmpbtbxkb8u/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1576768126.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ZunsongHu/pipeline",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmpr1wbyi6d/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmpr1wbyi6d/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmpr1wbyi6d/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpr1wbyi6d/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpr1wbyi6d/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpr1wbyi6d/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1600902432.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "jereiard/msto",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617953046.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "sam0per/gatk-variant-calling",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612527479.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "MLKaufman/rna-seq-star-deseq2",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611085208.0
  },
  {
    "data_format": 1,
    "description": "GATK best-practiced Snakemake variant calling pipeline",
    "formatting": null,
    "full_name": "ttumkaya/snakemake_variant_calling",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617236099.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpwogi3kpl/Snakefile\":  Keyword \"input\" at line 27 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dence/snakemake_pipeline_kallisto_deseq2",
    "linting": "FileNotFoundError in line 12 of /tmp/tmpwogi3kpl/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/d.ence/applications/snakemake_pipeline_kallisto_deseq2//samples.tsv\u0027\n  File \"/tmp/tmpwogi3kpl/Snakefile\", line 12, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1563858713.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "chaodi51/snakemake-workflows-rna-seq-star-deseq2",
    "linting": "Lints for snakefile /tmp/tmp3njv298t/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp3njv298t/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp3njv298t/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp3njv298t/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmp3njv298t/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmp3njv298t/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmp3njv298t/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmp3njv298t/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmp3njv298t/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmp3njv298t/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603853328.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sheucke/rna-seq-star-deseq2",
    "linting": "Lints for snakefile /tmp/tmp1je1cimx/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1je1cimx/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1je1cimx/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1je1cimx/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmp1je1cimx/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmp1je1cimx/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmp1je1cimx/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmp1je1cimx/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmp1je1cimx/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmp1je1cimx/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603098288.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "eriqande/dna-seq-gatk-variant-calling",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for rule compose_regions (line 3, /tmp/tmpvkmopjxq/rules/calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613064823.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vdjonsson/rna-seq-pizzly",
    "linting": "FileNotFoundError in line 3 of /tmp/tmpd4vn1s6n/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpd4vn1s6n/workflow/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1588021783.0
  },
  {
    "data_format": 1,
    "description": "An RNASeq snakemake workflow using the Tuxedo suite.",
    "formatting": "[ERROR] In file \"/tmp/tmp0dkuf9nk/rules/align.smk\":  \u0027InvalidParameterSyntax: 2gtf=\u0027\n[ERROR] In file \"/tmp/tmp0dkuf9nk/Snakefile\":  \u0027NoParametersError: L15: In input definition.\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "vallenderlab/rnaseq-tuxedo-suite",
    "linting": "SyntaxError in line 2 of /tmp/tmp0dkuf9nk/rules/align.smk:\nexpression cannot contain assignment, perhaps you meant \"==\"?\n  File \"/tmp/tmp0dkuf9nk/Snakefile\", line 15, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "rnaseq-tuxedo",
      "rnaseq-pipeline",
      "rna-seq-pipeline",
      "rna-seq-snakemake",
      "snakemake",
      "snakemake-workflow"
    ],
    "updated_at": 1563988276.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Iweirdo/NGS",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmppl8ip2up/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmppl8ip2up/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmppl8ip2up/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmppl8ip2up/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmppl8ip2up/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmppl8ip2up/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599010927.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "vojtab84/ACGT_snakemake",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613383850.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp2d06bb92/Snakefile\":  Keyword \"input\" at line 8 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 5 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "xiangyugits/wgs",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619084039.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ravimore8386/gatk",
    "linting": "WorkflowError in line 13 of /tmp/tmp1pe_nfsn/rules/common.smk:\nSchema file /tmp/tmp1pe_nfsn/rules/../schemas/config.schema.yaml not found.\n  File \"/tmp/tmp1pe_nfsn/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp1pe_nfsn/rules/common.smk\", line 13, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1606573687.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "babramso/vcf_snake",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmp5cthgvmm/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmp5cthgvmm/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmp5cthgvmm/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp5cthgvmm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp5cthgvmm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp5cthgvmm/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603407965.0
  },
  {
    "data_format": 1,
    "description": "gatk-variant-calling",
    "formatting": null,
    "full_name": "manuelconst/gatk-variant-calling",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611155078.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpjauz77xt/Snakefile\":  Keyword \"params\" at line 104 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "khavarilab/featurebarcode-qc-snakemake",
    "linting": "/tmp/tmpjauz77xt/Snakefile:118: SyntaxWarning: \"is not\" with a literal. Did you mean \"!=\"?\n  shell:\n/tmp/tmpjauz77xt/Snakefile:119: SyntaxWarning: \"is not\" with a literal. Did you mean \"!=\"?\n  \"umi_tools extract --extract-method string \"\n/tmp/tmpjauz77xt/Snakefile:218: SyntaxWarning: \"is not\" with a literal. Did you mean \"!=\"?\n  \nModuleNotFoundError in line 10 of /tmp/tmpjauz77xt/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpjauz77xt/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1603673074.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "TasnubaS/Snakemake-WorkFlow",
    "linting": "Lints for snakefile /tmp/tmpchm2w250/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpchm2w250/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpchm2w250/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpchm2w250/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmpchm2w250/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmpchm2w250/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmpchm2w250/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmpchm2w250/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmpchm2w250/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmpchm2w250/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603859651.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "arabidopsisca/snakemake-rna-seq-star-deseq2",
    "linting": "Lints for snakefile /tmp/tmpjphyka0q/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpjphyka0q/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpjphyka0q/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpjphyka0q/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 8, /tmp/tmpjphyka0q/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 32, /tmp/tmpjphyka0q/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 71, /tmp/tmpjphyka0q/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 173, /tmp/tmpjphyka0q/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 241, /tmp/tmpjphyka0q/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 276, /tmp/tmpjphyka0q/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607216871.0
  },
  {
    "data_format": 1,
    "description": "RNA-seq analysis snakemake pipeline with Kallisto and Sleuth",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bigmonty12/BRASS",
    "linting": "Lints for snakefile /tmp/tmphdlf4e0j/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmphdlf4e0j/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmphdlf4e0j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule sleuth_init (line 30, /tmp/tmphdlf4e0j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sleuth_diffexp (line 62, /tmp/tmphdlf4e0j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_bootstrap (line 91, /tmp/tmphdlf4e0j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_pca (line 115, /tmp/tmphdlf4e0j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1570567959.0
  },
  {
    "data_format": 1,
    "description": "Adaption from https://github.com/snakemake-workflows/rna-seq-star-deseq2 for sRNA",
    "formatting": "[WARNING] In file \"/tmp/tmp7jmfq39i/rules/align.smk\":  Keyword \"output\" at line 16 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp7jmfq39i/rules/align.smk\":  Keyword \"params\" at line 26 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp7jmfq39i/rules/align.smk\":  Keyword \"wrapper\" at line 33 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp7jmfq39i/rules/count.smk\":  Keyword \"wrapper\" at line 40 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "moritzschaefer/srna-seq-star-deseq2",
    "linting": "KeyError in line 13 of /tmp/tmp7jmfq39i/rules/trim.smk:\n\u0027adapter\u0027\n  File \"/tmp/tmp7jmfq39i/Snakefile\", line 19, in \u003cmodule\u003e\n  File \"/tmp/tmp7jmfq39i/rules/trim.smk\", line 13, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1556265811.0
  },
  {
    "data_format": 1,
    "description": "pipeline to find unique primer for genome bins in metagenomes",
    "formatting": "[ERROR] In file \"/tmp/tmpy3sb0xjr/Snakefile\":  \u0027NoParametersError: L13: In localrules definition.\u0027\n[INFO] In file \"/tmp/tmpy3sb0xjr/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpy3sb0xjr/Snakefile\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "jiarong/uniqprimer",
    "linting": "SyntaxError in line 26 of /tmp/tmpy3sb0xjr/Snakefile:\ninvalid syntax\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1541590754.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp4k5sz2kw/rules/aligned_report.smk\":  Keyword \"input\" at line 8 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp4k5sz2kw/rules/aligned_report.smk\":  Keyword \"input\" at line 40 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp4k5sz2kw/rules/erccdashboard.smk\":  Keyword \"input\" at line 2 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmp4k5sz2kw/rules/erccdashboard.smk\":  \u0027InvalidParameterSyntax: 5\" results/ spike_ins/ transgenic. TRT. CTL. pdf\n\u0027\n[WARNING] In file \"/tmp/tmp4k5sz2kw/rules/qc.smk\":  Keyword \"output\" at line 9 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 11 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dence/rnastar_deseq2_MeJA_timecourse",
    "linting": "Multiple includes of /tmp/tmp4k5sz2kw/rules/common.smk ignored\nModuleNotFoundError in line 4 of /tmp/tmp4k5sz2kw/rules/aligned_report.smk:\nNo module named \u0027scipy\u0027\n  File \"/tmp/tmp4k5sz2kw/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/tmp/tmp4k5sz2kw/rules/aligned_report.smk\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619107585.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fuzhiliang/dna-seq-gatk-variant-calling_test",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmpht22afd5/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmpht22afd5/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmpht22afd5/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpht22afd5/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpht22afd5/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpht22afd5/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599717836.0
  },
  {
    "data_format": 1,
    "description": "A quality control pipeline for Oxford Nanopore sequencing",
    "formatting": "[WARNING] In file \"/tmp/tmpr0n_nhk6/rules/eval.smk\":  Keyword \"shell\" at line 59 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpr0n_nhk6/rules/preprocess.smk\":  Keyword \"shell\" at line 23 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "koesterlab/nanopore-qc",
    "linting": "Detecting available FASTQ files...\nLints for snakefile /tmp/tmpr0n_nhk6/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule calc_tree_colormap (line 1, /tmp/tmpr0n_nhk6/rules/utils.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_fastq (line 1, /tmp/tmpr0n_nhk6/rules/preprocess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule crimson (line 29, /tmp/tmpr0n_nhk6/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_classification_tree (line 76, /tmp/tmpr0n_nhk6/rules/kraken.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_raw (line 3, /tmp/tmpr0n_nhk6/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_read_lengths (line 31, /tmp/tmpr0n_nhk6/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_quals (line 55, /tmp/tmpr0n_nhk6/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_kmer_mapping (line 79, /tmp/tmpr0n_nhk6/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_classification_tree (line 104, /tmp/tmpr0n_nhk6/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule per_sample_report (line 42, /tmp/tmpr0n_nhk6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1543929190.0
  },
  {
    "data_format": 1,
    "description": "Joint variant calling with GATK4 HaplotypeCaller, Google DeepVariant 1.0.0 and Strelka2, coordinated via Snakemake.",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "NCI-CGR/GEMSCAN",
    "linting": "FileNotFoundError in line 29 of /tmp/tmpvc0gv16g/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027../config/config.yaml\u0027\n  File \"/tmp/tmpvc0gv16g/workflow/Snakefile\", line 29, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [
      "snakemake",
      "workflow",
      "variant-calling"
    ],
    "updated_at": 1619026362.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"input\" at line 45 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"params\" at line 149 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"shell\" at line 155 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"params\" at line 211 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"output\" at line 299 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpc7srq7wv/Snakefile\":  Keyword \"input\" at line 311 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpc7srq7wv/Snakefile\":  \u0027NoParametersError: L354: In params definition.\u0027\n[INFO] In file \"/tmp/tmpc7srq7wv/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "kingyang728/DNA_seq_pipeline_conda",
    "linting": "Lints for snakefile /tmp/tmpc7srq7wv/Snakefile:\n    * Absolute path \"/home/kingyang/harddisk/SSH_server_Data/{dir}/{file}\" in line 2:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/SSH_server_Data/Fastq_test/{file}_001_test.fastq.gz\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/SSH_server_Data/Fastq_test\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample_name}_{sample_num}_{lane}_{read}_{tail}.fastq.gz\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample_name}_{sample_num}.bam\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample_name}_{sample_num}.bam.bai\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample_name}_{sample_num}.bam\" in line 37:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample_name}_{sample_num}_{lane}_R1_{tail}_val_1.fq.gz\" in line 39:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_filterd_calls/{sample_name}_{sample_num}.vcf.gz\" in line 41:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/filterd_calls/{sample_name}_{sample_num}.vcf.gz\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample_name}_{sample_num}.vcf\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/CNV/{sample_name}_{sample_num}.csv\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample_name}_{sample_num}.bam\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample_name}_{sample_num}.bam\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample_name}_{sample_num}_{lane}_R1_{tail}_val_1.fq.gz\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample_name}_{sample_num}_{lane}_R1_{tail}.fastq.gz_trimming_report.txt\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample_name}_{sample_num}_{lane}_R2_{tail}_val_2.fq.gz\" in line 61:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample_name}_{sample_num}_{lane}_R2_{tail}.fastq.gz_trimming_report.txt\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample_name}_{sample_num}_{lane}.bam\" in line 66:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample_name}_{sample_num}.vcf.gz.stats\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Genome_NAME.fa\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 99:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}.fai\" in line 101:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.fai\" in line 102:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 110:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 112:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.amb\" in line 113:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.ann\" in line 114:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.bwt\" in line 115:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.pac\" in line 116:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/kingyang/harddisk/VariantWorkFlow/Fasta_Index/{genome}.sa\" in line 117:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}.fa\" in line 126:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}.dict\" in line 128:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}_{lane}_R1_\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}_{lane}_R2_\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R1_\" in line 138:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R1_\" in line 139:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R2_\" in line 140:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R2_\" in line 141:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/trim_galore/{sample}_{lane}.log\" in line 154:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\"${RunNumber}\" in line 157:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tmp/ --fastqc\" in line 157:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 162:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R1_\" in line 167:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_{lane}_R2_\" in line 168:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample}_{lane}.bam\" in line 170:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/bwa_mem/{sample}_{lane}.log\" in line 177:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample}_L001.bam\" in line 189:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample}_L002.bam\" in line 190:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample}_L003.bam\" in line 191:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mapped_reads/{sample}_L004.bam\" in line 192:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/merged/{sample}.bam\" in line 194:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/merged/{sample}.bam\" in line 206:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample}.bam\" in line 208:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/samtools/sort_sam/{sample}.log\" in line 210:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample}.bam\" in line 222:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample}.bam.bai\" in line 224:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample}.bam\" in line 232:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sorted/{sample}.bam.bai\" in line 233:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample}.bam\" in line 235:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample}.metrics.txt\" in line 236:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/picard/dedup/{sample}.log\" in line 238:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample}.bam\" in line 251:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 252:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 253:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.recal_data.table\" in line 256:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/gatk/baserecalibrator/{sample}.log\" in line 258:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dedup/{sample}.bam\" in line 268:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 269:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 270:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.recal_data.table\" in line 271:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 273:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/gatk/gatk_applybqsr/{sample}.log\" in line 275:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 285:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam.bai\" in line 287:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 297:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PON_VCF/{PON_samples}.vcf.gz\" in line 301:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 310:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 326:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 343:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 344:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.vcf.gz\" in line 350:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.f1r2.tar.gz\" in line 351:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.f1r2.tar.gz\" in line 363:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.read-orientation-model.tar.gz\" in line 365:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 374:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.pileupsummaries.table\" in line 378:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.pileupsummaries.table\" in line 387:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.segments.table\" in line 389:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.contamination.table\" in line 390:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 400:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.vcf.gz\" in line 401:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.segments.table\" in line 402:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.contamination.table\" in line 403:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.read-orientation-model.tar.gz\" in line 404:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/filterd_calls/{sample}.vcf.gz\" in line 406:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 422:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 423:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam.bai\" in line 424:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_calls/{sample}.vcf.gz\" in line 428:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_calls/{sample}.vcf.gz\" in line 440:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_calls/{sample}.vcf.gz.tbi\" in line 442:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{genome}\" in line 449:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.vcf.gz\" in line 450:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/calls/{sample}.vcf.gz.stats\" in line 452:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_calls/{sample}.vcf.gz.tbi\" in line 461:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_calls/{sample}.vcf.gz\" in line 462:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samtools_filterd_calls/{sample}.vcf.gz\" in line 464:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 477:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.discordants.unsorted.bam\" in line 479:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 487:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.splitters.unsorted.bam\" in line 489:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.discordants.unsorted.bam\" in line 500:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.splitters.unsorted.bam\" in line 501:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.discordants.bam\" in line 503:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.splitters.bam\" in line 504:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 517:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.discordants.bam\" in line 518:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.splitters.bam\" in line 519:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.vcf\" in line 521:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/recal/{sample}.bam\" in line 533:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lumpySV/{sample}.splitters.bam\" in line 536:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/CNV/{sample}.csv\" in line 538:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 39:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 41:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 138:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 139:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 140:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 141:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 154:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 162:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 167:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 167:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 177:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 189:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 190:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 191:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 192:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 194:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 206:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 208:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 210:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 222:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 224:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 232:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 233:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 235:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 236:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 238:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 251:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 252:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 253:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 258:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 268:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 269:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 270:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 273:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 275:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 285:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 287:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 297:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 301:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 310:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 326:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 343:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 350:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 351:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 363:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 365:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 374:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 378:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 387:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 389:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 390:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 400:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 402:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 403:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 404:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 406:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 422:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 424:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 428:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 440:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 442:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 449:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 452:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 461:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 462:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 464:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 477:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 479:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 487:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 489:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 500:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 501:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 503:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 504:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 517:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 518:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 519:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 521:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 533:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 536:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 538:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 138:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 139:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 140:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 141:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 167:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_refGDC (line 85, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore_pe (line 154, /tmp/tmpc7srq7wv/Snakefile):\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_merge (line 244, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 313, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index_bqsr (line 440, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Mutect2_PON (line 465, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule GenomicsDBImport (line 491, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CreateSomaticPanelOfNormals (line 522, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Mutect2 (line 552, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule LearnReadOrientationModel (line 589, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule GetPileupSummaries (line 615, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CalculateContamination (line 643, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FilterMutectCalls (line 671, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Ext_discordant_align (line 766, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Ext_splitread_align (line 789, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_align (line 817, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule lumpy_SV (line 852, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule panelcn_mop (line 884, /tmp/tmpc7srq7wv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612280801.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "whtns/rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmpj5a5rslf/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpj5a5rslf/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpj5a5rslf/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpj5a5rslf/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmpj5a5rslf/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1585499023.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to analyze bulk RNA-seq ",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "khayer/rna_seq_standard_pipeline",
    "linting": "KeyError in line 4 of /tmp/tmp39ldaj9a/workflow/rules/common.smk:\n\u0027samples\u0027\n  File \"/tmp/tmp39ldaj9a/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmp39ldaj9a/workflow/rules/common.smk\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618923292.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for non-model organism analyses",
    "formatting": "[INFO] 13 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NBISweden/manticore-smk",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1608284514.0
  },
  {
    "data_format": 1,
    "description": "Creating a synthetic tumor/normal sample pair from CHM1/CHM13 haploid cell lines.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "koesterlab/chm-synthetic-tumor-normal",
    "linting": "Lints for snakefile /tmp/tmpkdir11x5/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_reads (line 27, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_eval_kit (line 48, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_confident_vcf (line 66, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_confident_bed (line 89, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compose_truth (line 109, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mix_reads (line 136, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule copy_normal (line 173, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_chr (line 196, /tmp/tmpkdir11x5/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1580296029.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to assemble and query short haplotype genomes.",
    "formatting": "[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 87 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 115 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 142 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 166 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 187 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 211 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 236 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmptf38mad7/workflow/Snakefile\":  Keyword \"shell\" at line 282 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "JoeBlackSci/SAHDA",
    "linting": "Lints for snakefile /tmp/tmptf38mad7/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule file_append (line 125, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule helper_condense (line 446, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fasta_condense (line 475, /tmp/tmptf38mad7/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618580267.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "alexpenson/rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmp1p10qkuj/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp1p10qkuj/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmp1p10qkuj/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603375395.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "christacaggiano/GATK_exome_seq",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmps8i2a4lu/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmps8i2a4lu/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmps8i2a4lu/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmps8i2a4lu/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmps8i2a4lu/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmps8i2a4lu/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592336464.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sheucke/dna-seq-gatk-variant-calling",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmphhqaufon/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmphhqaufon/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmphhqaufon/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmphhqaufon/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmphhqaufon/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmphhqaufon/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603098218.0
  },
  {
    "data_format": 1,
    "description": "standard RNA-seq analysis pipeline (cloned from snakemake example workflow)",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yztxwd/rna-seq-standard-pipeline",
    "linting": "Lints for snakefile /tmp/tmpm7lmdsei/Snakefile:\n    * Deprecated singularity directive used for container definition in line 35.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/trim.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpm7lmdsei/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule count_matrix (line 17, /tmp/tmpm7lmdsei/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rseqc_junction_annotation (line 41, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 80, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 182, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 250, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 285, /tmp/tmpm7lmdsei/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1604400794.0
  },
  {
    "data_format": 1,
    "description": "RNAseq workflow using snakemake workflow with Kallisto.",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pburnham50/KallistoPipeline",
    "linting": "Lints for snakefile /tmp/tmp4bsa7obp/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp4bsa7obp/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmp4bsa7obp/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599757051.0
  },
  {
    "data_format": 1,
    "description": "KAS-Seq Analysis Workflow",
    "formatting": null,
    "full_name": "lparsons/kas-seq-workflow",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614794946.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for RNAseq data processing on computing clusters with SLURM.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pawel125/RNAseq_snakemake",
    "linting": "Lints for snakefile /tmp/tmphw91_js2/workflow/Snakefile:\n    * Absolute path \"/home/pkus/resources/rseqc/mm10_RefSeq.chr.bed\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/pkus/resources/rnaseq_files/mm_gencode/gencode.vM25.primary_assembly.annotation.gtf\" in line 172:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule make_links (line 36, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_1 (line 61, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_1 (line 81, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_1 (line 114, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trimming (line 147, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_2 (line 169, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_2 (line 189, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_2 (line 222, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star (line 259, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 315, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_3 (line 335, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule read_GC_py (line 364, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tin_py (line 384, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule inner_distance_py (line 420, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genebody_coverage_py (line 442, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_stats (line 464, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc_3 (line 481, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_counts_all (line 524, /tmp/tmphw91_js2/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "rnaseq",
      "snakemake-workflow"
    ],
    "updated_at": 1616146140.0
  },
  {
    "data_format": 1,
    "description": "bwa map fastq files ",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "weizhu365/bwa-align",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmp_fjq8hfv/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmp_fjq8hfv/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmp_fjq8hfv/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp_fjq8hfv/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp_fjq8hfv/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp_fjq8hfv/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1589988148.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Ducklegs17/intro_to_snakemake",
    "linting": "Lints for rule get_data (line 20, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule subset_data (line 42, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trim (line 69, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assemble (line 99, /tmp/tmpo85_q7ph/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613090143.0
  },
  {
    "data_format": 1,
    "description": "Using https://github.com/snakemake-workflows/dna-seq-gatk-variant-calling as the template",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "weizhu365/gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmp7ogyfgvd/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmp7ogyfgvd/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp7ogyfgvd/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp7ogyfgvd/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp7ogyfgvd/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1588106733.0
  },
  {
    "data_format": 1,
    "description": "template SNP calling",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gitbackspacer/jitu-seq-gatk-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpr578vu71/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpr578vu71/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpr578vu71/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1564416750.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for GATK following best practice guidelines",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "srikanthkris/GATK",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpgdmc9nnh/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpgdmc9nnh/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpgdmc9nnh/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1566779188.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "MYB201560476/rna-seq-kallisto-sleuth",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613657705.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "silastittes/dna-seq-gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmpms22e1nb/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpms22e1nb/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpms22e1nb/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpms22e1nb/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpms22e1nb/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1582860563.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dsobral/dna-seq-gatk-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpmw3dag7v/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpmw3dag7v/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpmw3dag7v/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1569592010.0
  },
  {
    "data_format": 1,
    "description": "WGS pipeline using Snakemake",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "wenliangz/wgs-variant-calling",
    "linting": "Lints for snakefile /tmp/tmp8jtlu89x/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmp8jtlu89x/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp8jtlu89x/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp8jtlu89x/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp8jtlu89x/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1578374061.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "whtns/dna-seq-gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmpw3qpetm1/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpw3qpetm1/rules/scna.smk:\n    * Absolute path \"/CNAprofiles/segment.Rdata\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpw3qpetm1/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpw3qpetm1/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpw3qpetm1/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpw3qpetm1/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CopywriteR (line 14, /tmp/tmpw3qpetm1/rules/scna.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param copywriter_output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param input_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1584738254.0
  },
  {
    "data_format": 1,
    "description": "snakemake repository for alevin quantification of single cell RNAseq data",
    "formatting": "[ERROR] In file \"/tmp/tmpl5zx5at5/workflow/rules/alevin.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 4:10: resources:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "deevdevil88/snakemake_alevin_10x",
    "linting": "Lints for snakefile /tmp/tmpl5zx5at5/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592388734.0
  },
  {
    "data_format": 1,
    "description": "\u962a\u5927\u7528",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nkimoto/GATK_pipeline",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpvv46p3ct/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmpvv46p3ct/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpvv46p3ct/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1571966680.0
  },
  {
    "data_format": 1,
    "description": "Using snakemake-workflows/dna-seq-gatk-variant-calling as a template",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dchebotarov/varcall-snakemake-test",
    "linting": "Lints for snakefile /tmp/tmpzefi8wlm/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpzefi8wlm/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpzefi8wlm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpzefi8wlm/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpzefi8wlm/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1582785108.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp_z2zhwr3/Snakefile\":  Keyword \"shell\" at line 95 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yuxuth/snake_mRNA_v2",
    "linting": "Lints for rule trim_fastqs (line 60, /tmp/tmp_z2zhwr3/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc (line 98, /tmp/tmp_z2zhwr3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hisat_mapping (line 133, /tmp/tmp_z2zhwr3/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable hisat from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable STARINDEX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule featureCount_fq (line 188, /tmp/tmp_z2zhwr3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable gtf from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sortBam (line 221, /tmp/tmp_z2zhwr3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 258, /tmp/tmp_z2zhwr3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_bigwigs (line 294, /tmp/tmp_z2zhwr3/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595002256.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmp410jtj4n/workflow/rules/ilastik.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 111:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[ERROR] In file \"/tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 124:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk\":  2 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk\":  1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ndamond/ImcSegmentationSnakemake",
    "linting": "Lints for snakefile /tmp/tmp410jtj4n/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 472:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp410jtj4n/workflow/rules/ilastik.smk:\n    * Absolute path \"/ilastik-release/run_ilastik.sh\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule generate_sample_list (line 396, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mcdfolder2imcfolder (line 431, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ome2analysis (line 477, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule analysis2channelmeta (line 514, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule exportacmeta (line 537, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param fol_path_out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule compensation_copy_sm (line 641, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule modify_measurement_basic_pipeline (line 698, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule move_panel_to_cpout (line 725, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_untrained_ilastik (line 738, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_channelmeta_probab (line 764, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule zip_output_data (line 776, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 20 (line 123, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 21 (line 123, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 22 (line 123, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 23 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 24 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 25 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 26 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 27 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 28 (line 148, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 29 (line 123, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 30 (line 123, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_get_plugins (line 174, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_file_list (line 196, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cp_create_batch_data (line 221, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outfolder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cp_run_batch (line 285, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_get_groups (line 348, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_combine_batch_output (line 372, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule open_project (line 392, /tmp/tmp410jtj4n/workflow/rules/cellprofiler.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 38 (line 110, /tmp/tmp410jtj4n/workflow/rules/ilastik.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ilastik_run_batch (line 164, /tmp/tmp410jtj4n/workflow/rules/ilastik.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ilastik_combine_batch_output (line 226, /tmp/tmp410jtj4n/workflow/rules/ilastik.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_example_data (line 828, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule download_spillover_data (line 848, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean (line 861, /tmp/tmp410jtj4n/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable fol_path_base from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608206436.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "geparada/rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmpizj4dl38/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpizj4dl38/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpizj4dl38/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpizj4dl38/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmpizj4dl38/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610599646.0
  },
  {
    "data_format": 1,
    "description": "Long read alignment pipelines",
    "formatting": "[WARNING] In file \"/tmp/tmpc88jmos5/rules/nanosim.smk\":  Keyword \"params\" at line 52 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpc88jmos5/rules/mapping.smk\":  \u0027InvalidParameterSyntax: 35\" minimap2- t{ threads}- ax map- ont{ params. genome}{ input}|\n\"samtools view -bS | \"\n\"samtools sort -@{threads} -o {output.bam} 2\u003e {log}\"\n\n\n\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zihhuafang/seqoccinlr",
    "linting": "PermissionError in line 6 of /tmp/tmpc88jmos5/rules/common.smk:\n[Errno 13] Permission denied: \u0027/work\u0027\n  File \"/tmp/tmpc88jmos5/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmpc88jmos5/rules/common.smk\", line 6, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1615297138.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "HenningTimm/pcr_deduplication_analysis_workflow",
    "linting": "Lints for rule compile_custom_rbt (line 23, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule simulate_pcr_duplicates (line 55, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param output_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule dedup (line 98, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule evaluate (line 137, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_locus_results (line 163, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_scatter (line 191, /tmp/tmp4e4miynu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615409216.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "YTLogos/snakemake-gatk",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmp8i3m3v5m/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmp8i3m3v5m/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmp8i3m3v5m/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp8i3m3v5m/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp8i3m3v5m/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp8i3m3v5m/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1589374377.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for evaluation of assemblies",
    "formatting": "[INFO] 13 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "NBISweden/assemblyeval-smk",
    "linting": "generating id column from [\u0027species\u0027, \u0027version\u0027]\nWorkflowError in line 50 of /tmp/tmpk7dd1peq/workflow/rules/common.smk:\nError validating row 0 of data frame.\nValidationError: \u0027read1\u0027 is a required property\n\nFailed validating \u0027required\u0027 in schema:\n    OrderedDict([(\u0027$schema\u0027, \u0027http://json-schema.org/draft-07/schema#\u0027),\n                 (\u0027description\u0027,\n                  \u0027Read schema definition. List read files.\u0027),\n                 (\u0027type\u0027, \u0027object\u0027),\n                 (\u0027properties\u0027,\n                  OrderedDict([(\u0027id\u0027,\n                                OrderedDict([(\u0027type\u0027, \u0027string\u0027),\n                                             (\u0027description\u0027,\n                                              \u0027read set identifier\u0027)])),\n                               (\u0027read1\u0027,\n                                OrderedDict([(\u0027description\u0027,\n                                              \u0027fastq/sam/bam/cram file\u0027),\n                                             (\u0027type\u0027, \u0027string\u0027),\n                                             (\u0027format\u0027, \u0027uri\u0027),\n                                             (\u0027pattern\u0027,\n                                              \u0027.+\\\\.(fastq|fastq\\\\.gz|sam|bam|cram)$\u0027)])),\n                               (\u0027read2\u0027,\n                                OrderedDict([(\u0027description\u0027, \u0027fastq file\u0027),\n                                             (\u0027type\u0027, \u0027string\u0027),\n                                             (\u0027format\u0027, \u0027uri\u0027),\n                                             (\u0027pattern\u0027,\n                                              \u0027.+\\\\.(fastq|fastq\\\\.gz)$\u0027)])),\n                               (\u0027description\u0027,\n                                OrderedDict([(\u0027type\u0027, [\u0027string\u0027, \u0027null\u0027]),\n                                             (\u0027default\u0027, None)]))])),\n                 (\u0027required\u0027, [\u0027id\u0027, \u0027read1\u0027])])\n\nOn instance:\n    {\u0027description\u0027: None,\n     \u0027id\u0027: \u0027A.reads\u0027,\n     \u0027reads\u0027: \u0027data/raw/reads/A.reads_1.fastq.gz\u0027}\n  File \"/tmp/tmpk7dd1peq/workflow/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmpk7dd1peq/workflow/rules/common.smk\", line 60, in \u003cmodule\u003e\n  File \"/tmp/tmpk7dd1peq/workflow/rules/common.smk\", line 50, in _read\n",
    "stargazers_count": 0,
    "subscribers_count": 9,
    "topics": [],
    "updated_at": 1616010110.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline to automate running multiple samples through the ENCODE ATAC-seq workflow",
    "formatting": "[ERROR] In file \"/tmp/tmpzc9lvuln/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 60:4:     groupings_dict[group] = {\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpzc9lvuln/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "robinmeyers/atac-encode-snakemake",
    "linting": "Lints for snakefile /tmp/tmpzc9lvuln/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 158:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 220:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 26.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule all (line 124, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gather_qc (line 134, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_tag_align_files (line 156, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_cromwell_workflow (line 203, /tmp/tmpzc9lvuln/Snakefile):\n    * Shell command directly uses variable caper_params from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_json_input (line 244, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_grouped_json_input (line 275, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_grouped_tagalign (line 287, /tmp/tmpzc9lvuln/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1586798423.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for generating native space hcp-mmp segmentations (requires Freesurfer run)",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "khanlab-snakemake/hcp_mmp_to_native",
    "linting": "Lints for snakefile /tmp/tmpvzdfzqno/workflow/rules/common.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611335938.0
  },
  {
    "data_format": 1,
    "description": "parse vcf",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lqsae/vcf-tools",
    "linting": "Lints for snakefile /tmp/tmpqcxqkm3_/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpqcxqkm3_/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpqcxqkm3_/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpqcxqkm3_/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpqcxqkm3_/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610980693.0
  },
  {
    "data_format": 1,
    "description": "A pipeline for processing 16S / ITS data",
    "formatting": "[WARNING] In file \"/tmp/tmpeytah4rp/rules/phyloseq.smk\":  Keyword \"input\" at line 18 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpeytah4rp/Snakefile\":  Keyword \"input\" at line 56 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "welch16/microbiome-pipeline",
    "linting": "FileNotFoundError in line 8 of /tmp/tmpeytah4rp/Snakefile:\n[Errno 2] No such file or directory: \u0027samples.tsv\u0027\n  File \"/tmp/tmpeytah4rp/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615261484.0
  },
  {
    "data_format": 1,
    "description": "Conifer short-read polishing workflow",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tallgran/conifer-short-read-polishing",
    "linting": "FileNotFoundError in line 13 of /tmp/tmp7ukr4d36/Snakefile:\n[Errno 2] No such file or directory: \u0027read_metadata.tsv\u0027\n  File \"/tmp/tmp7ukr4d36/Snakefile\", line 13, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "workflow",
      "snakemake",
      "bioinformatics"
    ],
    "updated_at": 1592376010.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpul9chxa4/workflow/rules/ilastik.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 111:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[ERROR] In file \"/tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 130:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Zhanmengtao/ImcSegmentationSnakemake",
    "linting": "Lints for snakefile /tmp/tmpul9chxa4/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 473:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpul9chxa4/workflow/rules/ilastik.smk:\n    * Absolute path \"/ilastik-release/run_ilastik.sh\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule generate_sample_list (line 396, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mcdfolder2imcfolder (line 431, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ome2analysis (line 478, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule analysis2channelmeta (line 515, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule exportacmeta (line 538, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param fol_path_out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule compensation_copy_sm (line 642, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule modify_measurement_pipeline (line 703, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule move_panel_to_cpout (line 730, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_untrained_ilastik (line 743, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_channelmeta_probab (line 769, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule zip_output_data (line 781, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 20 (line 129, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 21 (line 129, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 22 (line 129, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 23 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 24 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 25 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 26 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 27 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 28 (line 154, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 29 (line 129, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule 30 (line 129, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_get_plugins (line 180, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_file_list (line 202, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cp_create_batch_data (line 227, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outfolder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cp_run_batch (line 291, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_get_groups (line 354, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cp_combine_batch_output (line 378, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule open_project (line 398, /tmp/tmpul9chxa4/workflow/rules/cellprofiler.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 38 (line 110, /tmp/tmpul9chxa4/workflow/rules/ilastik.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ilastik_run_batch (line 164, /tmp/tmpul9chxa4/workflow/rules/ilastik.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ilastik_combine_batch_output (line 226, /tmp/tmpul9chxa4/workflow/rules/ilastik.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_example_data (line 833, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule download_spillover_data (line 853, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean (line 866, /tmp/tmpul9chxa4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable fol_path_base from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611711242.0
  },
  {
    "data_format": 1,
    "description": "\ud83d\udeb0 Snakemake files for dahak workflows. Snakemake files for dahak workflows. (This repo will be absorbed into dahak-taco.)",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "charlesreid1/dahak-flot",
    "linting": "KeyError in line 8 of /tmp/tmpefkxv41g/rules/dahak/biocontainers.rule:\n\u0027data_dir\u0027\n  File \"/tmp/tmpefkxv41g/Snakefile\", line 14, in \u003cmodule\u003e\n  File \"/tmp/tmpefkxv41g/rules/dahak/biocontainers.rule\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1522181911.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fuzhiliang/dna-seq-gatk-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmptd05xsmw/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmptd05xsmw/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmptd05xsmw/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1571622378.0
  },
  {
    "data_format": 1,
    "description": "snakemake gatk best practices 20200522",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "juntkym/dna-seq-gatk-variant-calling",
    "linting": "Workflow defines that rule get_genome is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_faidx is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule genome_dict is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule get_known_variation is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule remove_iupac_codes is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule tabix_known_variants is eligible for caching between workflows (use the --cache argument to enable this).\nWorkflow defines that rule bwa_index is eligible for caching between workflows (use the --cache argument to enable this).\nLints for snakefile /tmp/tmp9miagi1r/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_vep_plugins (line 243, /tmp/tmp9miagi1r/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 160, /tmp/tmp9miagi1r/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp9miagi1r/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp9miagi1r/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp9miagi1r/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1590146243.0
  },
  {
    "data_format": 1,
    "description": "Conifer long-read polishing workflow",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tallgran/conifer-long-read-polishing",
    "linting": "FileNotFoundError in line 13 of /tmp/tmpcq6iroqi/Snakefile:\n[Errno 2] No such file or directory: \u0027read_metadata.tsv\u0027\n  File \"/tmp/tmpcq6iroqi/Snakefile\", line 13, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 684, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 457, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 814, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1045, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1862, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1357, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 639, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "workflow",
      "snakemake",
      "bioinformatics"
    ],
    "updated_at": 1592375960.0
  },
  {
    "data_format": 1,
    "description": "Clone of the Snakemake workflow for variant calling with the GATK pipeline",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mptrsen/gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmp7_rnglkg/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmp7_rnglkg/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmp7_rnglkg/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmp7_rnglkg/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmp7_rnglkg/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1579776898.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpqtbxtvh7/workflow/rules/semapping.smk\":  Inline-formatted keyword \"threads\" at line 66 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk\":  Keyword \"input\" at line 56 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk\":  Inline-formatted keyword \"threads\" at line 74 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "knutdrand/mu-chip",
    "linting": "Lints for snakefile /tmp/tmpqtbxtvh7/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpqtbxtvh7/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort_pe (line 120, /tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter_reads (line 171, /tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule remove_small_fragments_bam (line 194, /tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bamtobedpe (line 213, /tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule remove_small_fragments (line 232, /tmp/tmpqtbxtvh7/workflow/rules/pemapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_sort (line 112, /tmp/tmpqtbxtvh7/workflow/rules/semapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter (line 135, /tmp/tmpqtbxtvh7/workflow/rules/semapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bamtobed (line 186, /tmp/tmpqtbxtvh7/workflow/rules/semapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_reads (line 209, /tmp/tmpqtbxtvh7/workflow/rules/semapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_domains (line 71, /tmp/tmpqtbxtvh7/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clip_bed (line 94, /tmp/tmpqtbxtvh7/workflow/rules/peakcalling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule heatplot (line 7, /tmp/tmpqtbxtvh7/workflow/rules/plots.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tssplot (line 32, /tmp/tmpqtbxtvh7/workflow/rules/plots.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule averageplot (line 57, /tmp/tmpqtbxtvh7/workflow/rules/plots.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule joinplots (line 82, /tmp/tmpqtbxtvh7/workflow/rules/plots.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_genomes_file (line 3, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_trackhub (line 21, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule create_trackdb (line 44, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ucsc_sort (line 68, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_bw_track (line 95, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trunctate_score (line 115, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_peak_track (line 134, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_domain_track (line 158, /tmp/tmpqtbxtvh7/workflow/rules/trackhub.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prefetch (line 7, /tmp/tmpqtbxtvh7/workflow/rules/sra.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastq_dump_pe (line 26, /tmp/tmpqtbxtvh7/workflow/rules/sra.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastq_dump_se (line 50, /tmp/tmpqtbxtvh7/workflow/rules/sra.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rename_dumped_sra_pe (line 73, /tmp/tmpqtbxtvh7/workflow/rules/sra.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rename_dumped_sra_se (line 92, /tmp/tmpqtbxtvh7/workflow/rules/sra.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_chrom_sizes (line 6, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean_chrom_sizes (line 21, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_genes (line 42, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_genes_bed (line 57, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_unique_tss (line 79, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule unzip_bowtie_index (line 117, /tmp/tmpqtbxtvh7/workflow/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607344531.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "arabidopsisca/snakemake-rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmp2li2o44l/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp2li2o44l/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp2li2o44l/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp2li2o44l/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmp2li2o44l/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607217225.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpf3f9efsx/workflow/Snakefile\":  \u0027NoParametersError: L17: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "eho-tacc/aisd_shapefile_integration",
    "linting": "Lints for snakefile /tmp/tmpf3f9efsx/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpf3f9efsx/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 6.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617990927.0
  },
  {
    "data_format": 1,
    "description": "AW",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rm93/Awesome-Workflow",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpmdahb3xc/Snakefile:\nNo module named \u0027tqdm\u0027\n  File \"/tmp/tmpmdahb3xc/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1528488237.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp75ef5vzi/Snakefile\":  Keyword \"shell\" at line 87 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yuxuth/snake_hisat_cutadapt",
    "linting": "Lints for rule trim_fastqs (line 54, /tmp/tmp75ef5vzi/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc (line 92, /tmp/tmp75ef5vzi/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hisat_mapping (line 126, /tmp/tmp75ef5vzi/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable hisat from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable STARINDEX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sortBam (line 178, /tmp/tmp75ef5vzi/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 215, /tmp/tmp75ef5vzi/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule stringtie_FPKM_caculation (line 250, /tmp/tmp75ef5vzi/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable stringtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gtf from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_bigwigs (line 290, /tmp/tmp75ef5vzi/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiQC (line 333, /tmp/tmp75ef5vzi/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ballgown (line 363, /tmp/tmp75ef5vzi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1580763096.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lxhuang7/snakemake-workflows-rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmpmxy_ocwv/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpmxy_ocwv/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpmxy_ocwv/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpmxy_ocwv/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmpmxy_ocwv/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1573602799.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lparsons/lamaya_tn5_demultiplex",
    "linting": "Lints for snakefile /tmp/tmp1t2kqu4t/Snakefile:\n    * Absolute path \"/{sample}.fastq.gz\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/{sample}-read-{read}.fastq\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/unmatched-read-{read}.fastq\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/multimatched-read-{read}.fastq\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/barcode_splitter.log\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/{file}.fastq\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{file}.fq\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/unmatched-read-1.fastq\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/unmatched-read-2.fastq\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_tag/unmatched-read-3.fastq\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_neb/{sample}-read-{read}.fastq\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_neb/unmatched-read-{read}.fastq\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_neb/multimatched-read-{read}.fastq\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_neb/barcode_splitter.log\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastq_neb/{file}.fastq\" in line 68:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{file}.fq\" in line 69:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}-read-1.fq\" in line 75:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}-read-3.fq\" in line 76:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}.fastq.gz\" in line 78:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/logs/{sample}_umi_tools_extract.log\" in line 80:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule demultiplex_tag (line 29, /tmp/tmp1t2kqu4t/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule move_tag_fastq (line 68, /tmp/tmp1t2kqu4t/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule demultiplex_neb (line 84, /tmp/tmp1t2kqu4t/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule move_neb_fastq (line 125, /tmp/tmp1t2kqu4t/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_umi (line 142, /tmp/tmp1t2kqu4t/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1533243373.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gitbackspacer/rna-seq-kallisto-sleuth-test",
    "linting": "Lints for snakefile /tmp/tmpxnpbjrco/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpxnpbjrco/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpxnpbjrco/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpxnpbjrco/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmpxnpbjrco/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1601639009.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Adrianzo/dna-seq-variant-calling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmp48gb65ti/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmp48gb65ti/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp48gb65ti/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1564840753.0
  },
  {
    "data_format": 1,
    "description": "test",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zhujack/dna-seq-gatk-variant-calling",
    "linting": "Lints for snakefile /tmp/tmpnamt3yf2/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule samtools_index (line 157, /tmp/tmpnamt3yf2/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vcf_to_tsv (line 1, /tmp/tmpnamt3yf2/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_stats (line 29, /tmp/tmpnamt3yf2/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 1, /tmp/tmpnamt3yf2/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1585348152.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "myRNASeq/scRNASeqPipeline",
    "linting": "Lints for snakefile /tmp/tmpbdur8hsn/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpbdur8hsn/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpbdur8hsn/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpbdur8hsn/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmpbdur8hsn/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610483547.0
  },
  {
    "data_format": 1,
    "description": "Analysis of publicly available MCF10A ChIP-Seq data",
    "formatting": "[ERROR] In file \"/tmp/tmp1v8bzgw5/rules/deeptools.smk\":  \u0027NoParametersError: L109: In params definition.\u0027\n[ERROR] In file \"/tmp/tmp1v8bzgw5/rules/sra_download.smk\":  \u0027NoParametersError: L14: In input definition.\u0027\n[ERROR] In file \"/tmp/tmp1v8bzgw5/Snakefile\":  \u0027NoParametersError: L29: In input definition.\u0027\n[ERROR] In file \"/tmp/tmp1v8bzgw5/rules/align.smk\":  \u0027InvalidParameterSyntax: L60:Invalid key [\u0027.end1.fastq.gz\u0027,\u0027.end2.fastq.gz\u0027]\n index\u0027\n[INFO] 4 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "skurscheid/mcf10_promoter_profiling",
    "linting": "KeyError in line 18 of /tmp/tmp1v8bzgw5/Snakefile:\n\u0027project\u0027\n  File \"/tmp/tmp1v8bzgw5/Snakefile\", line 18, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611913414.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/pipeline_cellranger_citeseq",
    "linting": "Lints for snakefile /tmp/tmpamjbyabm/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 13.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cellranger_count (line 31, /tmp/tmpamjbyabm/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1591797476.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Mira0507/snakemake_sra",
    "linting": "Lints for snakefile /tmp/tmpbzf8r0a1/Snakefile:\n    * Absolute path \"/bin/bash\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule get_fastq (line 31, /tmp/tmpbzf8r0a1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618854024.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpq9i97iiy/Snakefile\":  Keyword \"input\" at line 48 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpq9i97iiy/Snakefile\":  Keyword \"shell\" at line 60 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpq9i97iiy/Snakefile\":  Keyword \"params\" at line 102 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yuxuth/snakemake_RNAseq",
    "linting": "Lints for rule merge_fastqs (line 55, /tmp/tmpq9i97iiy/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_adapter (line 101, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable bbbuk from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable adaptors from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fastqc (line 138, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hisat_mapping (line 177, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable hisat from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable STARINDEX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable splicesite_index from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sortBam (line 230, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 269, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule stringtie_FPKM_caculation (line 302, /tmp/tmpq9i97iiy/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable stringtie from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable gtf from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_bigwigs (line 342, /tmp/tmpq9i97iiy/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1561151133.0
  },
  {
    "data_format": 1,
    "description": "Assembling short genomic reads from bacteria",
    "formatting": "[WARNING] In file \"/tmp/tmp2dzul041/rules/preprocess.smk\":  Keyword \"shell\" at line 27 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "matrs/assembly-illumina-bact",
    "linting": "RuleException in line 60 of /tmp/tmp2dzul041/rules/preprocess.smk:\nConda environments are only allowed with shell, script, notebook, or wrapper directives (not with run).\n  File \"/tmp/tmp2dzul041/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmp2dzul041/rules/preprocess.smk\", line 60, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1569499618.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake Workflow for Gene Set Enrichment Analysis",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "LittleFool/snakemake-workflow-gsea",
    "linting": "Lints for snakefile /tmp/tmpde5_3ssb/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpde5_3ssb/rules/star.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpde5_3ssb/rules/gsea.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule cutadapt_pe (line 7, /tmp/tmpde5_3ssb/rules/trim.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule cutadapt_se (line 47, /tmp/tmpde5_3ssb/rules/trim.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align (line 17, /tmp/tmpde5_3ssb/rules/star.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule count_matrix (line 36, /tmp/tmpde5_3ssb/rules/count.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gsea (line 1, /tmp/tmpde5_3ssb/rules/gsea.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule gsea_cleanup (line 40, /tmp/tmpde5_3ssb/rules/gsea.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1587047806.0
  },
  {
    "data_format": 1,
    "description": "Small snakemake pipeline to explore RNA-Seq data with deepTools.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AngryMaciek/snakemake_deeptools",
    "linting": "KeyError in line 46 of /tmp/tmpt3ixh75z/Snakefile:\n\u0027output_dir\u0027\n  File \"/tmp/tmpt3ixh75z/Snakefile\", line 46, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake-pipeline",
      "rna-seq"
    ],
    "updated_at": 1593795952.0
  },
  {
    "data_format": 1,
    "description": "Using the snakemake-workflow as a template to create my own WES pipeline",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "misrak/snakemake_WES",
    "linting": "FileNotFoundError in line 18 of /tmp/tmp6_nlnlf5/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmp6_nlnlf5/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp6_nlnlf5/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1562254819.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "blairperry/snakemake_gatk_variantCalling",
    "linting": "FileNotFoundError in line 18 of /tmp/tmp8xi2wk7b/rules/common.smk:\n[Errno 2] No such file or directory: \u0027data/ref/genome.chr21.fa.fai\u0027\n  File \"/tmp/tmp8xi2wk7b/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp8xi2wk7b/rules/common.smk\", line 18, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1568490127.0
  },
  {
    "data_format": 1,
    "description": "snakemake pipeline for calling uk biobank exomes",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "alavertu/snakemake-ukb-exome",
    "linting": "FileNotFoundError in line 13 of /tmp/tmpurtmm3i8/rules/common.smk:\n[Errno 2] No such file or directory: \u0027/scratch/groups/rbaltman/ukbiobank/ab_samples.txt\u0027\n  File \"/tmp/tmpurtmm3i8/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpurtmm3i8/rules/common.smk\", line 13, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1591861173.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for single cell RNAseq",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rpauly/snakemake_workflow_sc_rnaseq",
    "linting": "Lints for snakefile /tmp/tmp9mixu6ku/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp9mixu6ku/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp9mixu6ku/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp9mixu6ku/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmp9mixu6ku/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1600719050.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "kogolts/scRNAseq_SNAKELIKE",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611270875.0
  },
  {
    "data_format": 1,
    "description": "template of single cell sequencing from scilife",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sunjiangming/scRNASeq",
    "linting": "Lints for snakefile /tmp/tmph_s1jd3j/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmph_s1jd3j/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmph_s1jd3j/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmph_s1jd3j/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmph_s1jd3j/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593436345.0
  },
  {
    "data_format": 1,
    "description": "Rebasing changes made by @lucren on newest stag-mwc branch",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kcivkulis/stag-mwc",
    "linting": "FileNotFoundError in line 31 of /tmp/tmp_fh6ji54/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp_fh6ji54/Snakefile\", line 31, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618315861.0
  },
  {
    "data_format": 1,
    "description": "This Snakemake pipeline implements the GATK best-practices workflow for calling short germline variants.",
    "formatting": "[WARNING] In file \"/tmp/tmpooynuzdi/workflow/rules/annotation.smk\":  Keyword \"output\" at line 5 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ehyunim/gatk-germline-short-variant-pipeline",
    "linting": "FileNotFoundError in line 11 of /tmp/tmpooynuzdi/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpooynuzdi/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpooynuzdi/workflow/rules/common.smk\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1605860893.0
  },
  {
    "data_format": 1,
    "description": "learning scRNA workflow",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dming1024/scRNA-seq_snakemake",
    "linting": "Lints for snakefile /tmp/tmpe885fiaj/Snakefile:\n    * Deprecated singularity directive used for container definition in line 70.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpe885fiaj/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpe885fiaj/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1570629741.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nchris5/diffparc-smk_piriform",
    "linting": "FileNotFoundError in line 8 of /tmp/tmprwkxu3jh/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config/participants_testing.tsv\u0027\n  File \"/tmp/tmprwkxu3jh/workflow/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618864079.0
  },
  {
    "data_format": 1,
    "description": "minimal snakemake repo for mixing R and python scripts",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "erblast/snakemake_minimal",
    "linting": "Lints for rule load_dataset (line 22, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_rmd_direct (line 45, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_rmd_via_script (line 69, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_execute_nb_plot (line 94, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_nb_2_html (line 122, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule test_utilR (line 152, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule check_utilR (line 172, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule readme (line 196, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_md (line 215, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_rst (line 230, /tmp/tmps1cvgacu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1576666535.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "skelviper/scRNA-seq",
    "linting": "Lints for snakefile /tmp/tmpgrkyypzi/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpgrkyypzi/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpgrkyypzi/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpgrkyypzi/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmpgrkyypzi/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597370397.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpycaton3e/workflow/Snakefile\":  Keyword \"input\" at line 42 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpycaton3e/workflow/Snakefile\":  \u0027SyntaxError: L49: Unrecognised keyword \u0027expand\u0027 in rule definition\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fuzhiliang/rna-seq-kallisto-sleuth",
    "linting": "Lints for snakefile /tmp/tmpycaton3e/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 7.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpycaton3e/workflow/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpycaton3e/workflow/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compose_sample_sheet (line 6, /tmp/tmpycaton3e/workflow/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1571735273.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline to automate a standard PSASS workflow",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SexGenomicsToolkit/PSASS-workflow",
    "linting": "Lints for snakefile /tmp/tmpitzpsrxi/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpitzpsrxi/rules/resources.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpitzpsrxi/rules/processing.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpitzpsrxi/rules/analyses.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616763983.0
  },
  {
    "data_format": 1,
    "description": "Example Snakemake workflow describing the use of input functions.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jakevc/snakemake_inputfuncs",
    "linting": "Lints for rule first (line 17, /tmp/tmpqjzn7wrh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1562179304.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for CircRNA Detection and Quantification",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "andremrsantos/circrna_smk",
    "linting": "Lints for snakefile /tmp/tmpm8f541f1/Snakefile:\n    * Absolute path \"/{sample}_R1.fastq.gz\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample,[\\w\\d-]+}.fastq.gz\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Deprecated singularity directive used for container definition in line 36.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpm8f541f1/rules/trim.smk:\n    * Absolute path \"/{sample}.fastq.gz\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}_R1.fastq.gz\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}_R2.fastq.gz\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpm8f541f1/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpm8f541f1/rules/quant.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Deprecated singularity directive used for container definition in line 29.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 103.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_fastp_se (line 6, /tmp/tmpm8f541f1/rules/trim.smk):\n    * Shell command directly uses variable trim from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_fastp_pe (line 41, /tmp/tmpm8f541f1/rules/trim.smk):\n    * Shell command directly uses variable trim from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_star (line 42, /tmp/tmpm8f541f1/rules/align.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule align_star_mate (line 137, /tmp/tmpm8f541f1/rules/align.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule dcc_se_samplesheet (line 128, /tmp/tmpm8f541f1/rules/quant.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dcc_pe_samplesheet (line 142, /tmp/tmpm8f541f1/rules/quant.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule dcc (line 171, /tmp/tmpm8f541f1/rules/quant.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param wkdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1558358461.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow for mappind single/paired datas",
    "formatting": "[ERROR] In file \"/tmp/tmpryo01ean/Snakefile\":  \u0027InvalidParameterSyntax: 273R1=rules. run_atropos. output. R1 if cleaning elsef\"{fastq_dir}{{samples}}_R1{rattlersnp.fastq_files_ext}\"\n\u0027\n[INFO] In file \"/tmp/tmpryo01ean/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "sravel/RattlerSNP",
    "linting": "ModuleNotFoundError in line 14 of /tmp/tmpryo01ean/Snakefile:\nNo module named \u0027pysam\u0027\n  File \"/tmp/tmpryo01ean/Snakefile\", line 14, in \u003cmodule\u003e\n  File \"/tmp/tmpryo01ean/script/module.py\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1616596779.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AlexanderLabWHOI/tara-download-snakemake",
    "linting": "Lints for snakefile /tmp/tmpu0cgs55n/Snakefile:\n    * Absolute path \"/\u0027+STUDY+\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 62:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027+STUDY+\" in line 64:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 62:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 64:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 62:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_fastq (line 50, /tmp/tmpu0cgs55n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fastq2 (line 70, /tmp/tmpu0cgs55n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule move_fastq1 (line 91, /tmp/tmpu0cgs55n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule move_fastq2 (line 107, /tmp/tmpu0cgs55n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule md5sum (line 123, /tmp/tmpu0cgs55n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1550193430.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bricoletc/single-cell-rna-seq-4predocs",
    "linting": "Lints for snakefile /tmp/tmp9_se01vw/Snakefile:\n    * Deprecated singularity directive used for container definition in line 70.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp9_se01vw/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp9_se01vw/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1570173635.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lasejour/snakemake-scRNA-seq",
    "linting": "Lints for snakefile /tmp/tmpgdfpbgev/Snakefile:\n    * Deprecated singularity directive used for container definition in line 70.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpgdfpbgev/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpgdfpbgev/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1568736565.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for characterizing and organizing microbial isolate genome sequences",
    "formatting": "[WARNING] In file \"/tmp/tmp8lo8s6ny/Snakefile\":  Keyword \"input\" at line 45 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "tanaes/culture-collector",
    "linting": "Lints for snakefile /tmp/tmp8lo8s6ny/Snakefile:\n    * Deprecated singularity directive used for container definition in line 30.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp8lo8s6ny/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule file_rename (line 5, /tmp/tmp8lo8s6ny/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assemble_spades (line 1, /tmp/tmp8lo8s6ny/rules/assemble.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule assemble_quast (line 58, /tmp/tmp8lo8s6ny/rules/assemble.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule checkm_setup (line 3, /tmp/tmp8lo8s6ny/rules/checkm.smk):\n    * Shell command directly uses variable checkm_data from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable checkm_data from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bwa_index (line 99, /tmp/tmp8lo8s6ny/rules/checkm.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bwa_mem (line 132, /tmp/tmp8lo8s6ny/rules/checkm.smk):\n    * Param index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_index (line 169, /tmp/tmp8lo8s6ny/rules/checkm.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gtdb_setup (line 3, /tmp/tmp8lo8s6ny/rules/taxonomy.smk):\n    * Shell command directly uses variable gtdb_data from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule gtdb_prep (line 38, /tmp/tmp8lo8s6ny/rules/taxonomy.smk):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule taxonomy_gtdb (line 63, /tmp/tmp8lo8s6ny/rules/taxonomy.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Shell command directly uses variable gtdb_data from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597261596.0
  },
  {
    "data_format": 1,
    "description": "WRF post processing",
    "formatting": "[ERROR] In file \"/tmp/tmputx19e99/Snakefile\":  \u0027NoParametersError: L18: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "eelsirhc/wrf_pp",
    "linting": "Lints for snakefile /tmp/tmputx19e99/Snakefile:\n    * Deprecated singularity directive used for container definition in line 9.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmputx19e99/rules/other.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule combined_wrfout (line 18, /tmp/tmputx19e99/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule diag_simple (line 76, /tmp/tmputx19e99/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_index (line 96, /tmp/tmputx19e99/rules/other.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1556896075.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "myRNASeq/FF2",
    "linting": "Lints for snakefile /tmp/tmpnjinwivn/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpnjinwivn/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpnjinwivn/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpnjinwivn/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmpnjinwivn/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610482432.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "myRNASeq/FF",
    "linting": "Lints for snakefile /tmp/tmp24cc9ifc/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp24cc9ifc/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp24cc9ifc/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp24cc9ifc/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmp24cc9ifc/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610482406.0
  },
  {
    "data_format": 1,
    "description": "RNA-Seq data analysis: Quantification, Differential Gene Expression, Differential Transcripts Usage",
    "formatting": "[WARNING] In file \"/tmp/tmpp9wl3_0p/Snakefile\":  Keyword \"input\" at line 233 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AngryMaciek/snakemake_qdgedtu",
    "linting": "KeyError in line 48 of /tmp/tmpp9wl3_0p/Snakefile:\n\u0027output_dir\u0027\n  File \"/tmp/tmpp9wl3_0p/Snakefile\", line 48, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake-pipeline",
      "rna-seq",
      "differential-gene-expression",
      "differential-transcripts"
    ],
    "updated_at": 1593795884.0
  },
  {
    "data_format": 1,
    "description": "Workflow to call structural and copy number variants in somatic whole genome data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 4 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "marrip/wgs_somatic_cnv_sv_viper",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619086211.0
  },
  {
    "data_format": 1,
    "description": "concensus caller for circular amplified long read nanopore data",
    "formatting": "[ERROR] In file \"/tmp/tmp32dqsvo2/Snakefile\":  \u0027NoParametersError: L18: In input definition.\u0027\n[INFO] In file \"/tmp/tmp32dqsvo2/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmp32dqsvo2/Snakefile\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "zztin/concaller",
    "linting": "WorkflowError in line 5 of /tmp/tmp32dqsvo2/Snakefile:\nConfig file must be given as JSON or YAML with keys at top level.\n  File \"/tmp/tmp32dqsvo2/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1584550308.0
  },
  {
    "data_format": 1,
    "description": "Search for variant in ENA/SRA to determine allele frequency",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SichongP/VariantFrequency",
    "linting": "Lints for snakefile /tmp/tmp7w3vw56i/Snakefile:\n    * Absolute path \"/data/accessions.tsv\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/accessions.tsv\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/VCF/output.vcf\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp7w3vw56i/rules/prepareFiles.smk:\n    * Absolute path \"/data/genome.fa\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/chrom.sizes\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/unzip.yaml\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/genome.fa\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/variants.bed\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/chrom.sizes\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.bed\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/variantMapping.txt\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.vcf\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/python.yaml\" in line 37:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/scripts/generateRef.py\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.vcf\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.vcf.idx\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa.fai\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.dict\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/samtools.yaml\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/accessions.tsv\" in line 65:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/fastqLinks.csv\" in line 66:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/python.yaml\" in line 67:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/scripts/getFASTQLinks.py\" in line 68:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/runs/{{run}}/{{run}}_{read}.fastq.gz\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/runs/{run}/{run}_1.fastq.gz\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/runs/{run}/{run}_2.fastq.gz\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/{sample}_R1.fq\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/{sample}_R2.fq\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 19:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 65:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 67:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 74:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp7w3vw56i/rules/Alignment.smk:\n    * Absolute path \"/data/regionsRef.fa\" in line 2:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa.bwt\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/bwa.yaml\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/{sample}_R1.fq\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/temp_fastq/{sample}_R2.fq\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa.bwt\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.sam\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/bwa.yaml\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.sam\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.filtered.bam\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/samtools.yaml\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.filtered.bam\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.bam\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/sambamba.yaml\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.bam\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.sorted.bam\" in line 73:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/samtools.yaml\" in line 79:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.sorted.bam\" in line 86:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.sorted.bam.bai\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/envs/samtools.yaml\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 19:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 79:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 86:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp7w3vw56i/rules/Genotype.smk:\n    * Absolute path \"/Results/bams/{sample}.markDup.sorted.bam\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/bams/{sample}.markDup.sorted.bam.bai\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa.fai\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.dict\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/GVCFs/{sample}.g.vcf.gz\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/GVCFs/{sample}.g.vcf.gz\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/GVCFs/combined.g.vcf.gz\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/GVCFs/combined.g.vcf.gz\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/regionsRef.fa\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.vcf\" in line 46:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tools/gatk-4.2.0.0/gatk\" in line 47:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/data/remappedVariants.vcf.idx\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/Results/VCF/output.vcf\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 46:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 47:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule getGenomeFASTA (line 4, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule getGATK (line 35, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generateRef (line 60, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule indexVCF (line 88, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule indexRef (line 111, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generateFASTQLink (line 141, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getFASTQ (line 162, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outDir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule mergeFASTQ (line 194, /tmp/tmp7w3vw56i/rules/prepareFiles.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule indexFASTA (line 1, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mapping (line 33, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filterBAM (line 72, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule markDuplicate (line 105, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sortBAM (line 156, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule indexBAM (line 189, /tmp/tmp7w3vw56i/rules/Alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule HaplotypeCaller (line 3, /tmp/tmp7w3vw56i/rules/Genotype.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combineGVCF (line 39, /tmp/tmp7w3vw56i/rules/Genotype.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genotypeGVCF (line 76, /tmp/tmp7w3vw56i/rules/Genotype.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618254506.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "MehdiFard/Skill_test_McGill_Snakemake",
    "linting": "Lints for snakefile /tmp/tmpkk65gdsy/Snakefile:\n    * Absolute path \"/ID=chr{wildcards.file}/p\" in line 154:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/contig/d\" in line 154:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/##bcftools/d\" in line 154:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule parsing_metadata (line 33, /tmp/tmpkk65gdsy/Snakefile):\n    * Shell command directly uses variable VCF_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611403896.0
  },
  {
    "data_format": 1,
    "description": "Code to setup and maintain reference assemblies and accompanying data",
    "formatting": "[WARNING] In file \"/tmp/tmpg2tc7k43/workflow/Snakefile\":  Keyword \"input\" at line 150 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpg2tc7k43/workflow/Snakefile\":  Keyword \"wildcard_constraints\" at line 153 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpg2tc7k43/workflow/Snakefile\":  Keyword \"input\" at line 213 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "DKFZ-ODCF/reference-data",
    "linting": "KeyError in line 15 of /tmp/tmpg2tc7k43/workflow/Snakefile:\n\u0027genome_config\u0027\n  File \"/tmp/tmpg2tc7k43/workflow/Snakefile\", line 15, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1614000715.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "LCrossman/ribosomal_snakemake",
    "linting": "FileNotFoundError in line 1 of /tmp/tmpqnkbr0_j/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpqnkbr0_j/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619171024.0
  },
  {
    "data_format": 1,
    "description": "Adaptation of Derek Croote\u0027s pipeline to process assembled BCR sequences",
    "formatting": "[WARNING] In file \"/tmp/tmpuwbbimtz/Snakefile\":  Keyword \"input\" at line 19 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "iosonofabio/singlecell-bcr",
    "linting": "Lints for rule get_immcantation_image (line 39, /tmp/tmpuwbbimtz/rules/get_container.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule transfer_contigs (line 7, /tmp/tmpuwbbimtz/rules/assemble.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule blast_constant_region (line 49, /tmp/tmpuwbbimtz/rules/assemble.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule igblast_changeo (line 99, /tmp/tmpuwbbimtz/rules/assemble.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule merge_changeo_constant (line 158, /tmp/tmpuwbbimtz/rules/assemble.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combine_assemblies (line 200, /tmp/tmpuwbbimtz/rules/assemble.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1560287830.0
  },
  {
    "data_format": 1,
    "description": "Simplified SAMap workflow",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ebi-gene-expression-group/samap-workflow",
    "linting": "FileNotFoundError in line 4 of /tmp/tmp93g3ln7x/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp93g3ln7x/workflow/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1614701111.0
  },
  {
    "data_format": 1,
    "description": "A small snakemake pipeline to perform quality analysis of RNA-Seq data with FastQC.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AngryMaciek/snakemake_fastqc",
    "linting": "KeyError in line 49 of /tmp/tmppjnepwcp/Snakefile:\n\u0027output_dir\u0027\n  File \"/tmp/tmppjnepwcp/Snakefile\", line 49, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake-pipeline",
      "rna-seq",
      "quality-control"
    ],
    "updated_at": 1593795936.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for generation of GISTIC focal alteration frequencies from sWGS absolute copy number profiles",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Phil9S/swgs-gistic",
    "linting": "FileNotFoundError in line 12 of /tmp/tmpvvz3q3b_/rules/common.smk:\n[Errno 2] No such file or directory: \u0027sample_sheet.tsv\u0027\n  File \"/tmp/tmpvvz3q3b_/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpvvz3q3b_/rules/common.smk\", line 12, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1602260789.0
  },
  {
    "data_format": 1,
    "description": "Analysis of xenograft data",
    "formatting": "[ERROR] In file \"/tmp/tmpfh9o0k_y/Snakefile\":  \u0027NoParametersError: L25: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "skurscheid/mcf10a_xenografts",
    "linting": "KeyError in line 12 of /tmp/tmpfh9o0k_y/Snakefile:\n\u0027units\u0027\n  File \"/tmp/tmpfh9o0k_y/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1590034234.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "myRNASeq/zoe",
    "linting": "Lints for snakefile /tmp/tmpq29kmizw/Snakefile:\n    * Deprecated singularity directive used for container definition in line 82.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpq29kmizw/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpq29kmizw/rules/cell-type.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpq29kmizw/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule plot_expression (line 57, /tmp/tmpq29kmizw/rules/diffexp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610482751.0
  },
  {
    "data_format": 1,
    "description": "iSeq processing for Apprill Lab",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "microlei/apprill-iseq",
    "linting": "Lints for rule clean (line 39, /tmp/tmp3ka2qdbf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plotQP (line 61, /tmp/tmp3ka2qdbf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter (line 78, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule learnError (line 107, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule uniques (line 132, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule removeChimeras (line 158, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trackReads (line 183, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule taxonomy (line 214, /tmp/tmp3ka2qdbf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615571177.0
  },
  {
    "data_format": 1,
    "description": "Evaluation workflow for Bitpacked Hopscotch Hashtable.",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "HenningTimm/bpht_evaluation_workflow",
    "linting": "Lints for snakefile /tmp/tmp3oh6y_qx/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp3oh6y_qx/rules/plotting.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule compile_bpht_eval (line 3, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule eval_fill_rate (line 34, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule eval_fill_time_genome (line 57, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_bpht_access_time (line 80, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule evaluate_bpht_access_time (line 104, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compare_access_times_numa0 (line 128, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compare_access_times_numa1 (line 151, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule new_compare_access_times_numa0 (line 174, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule new_compare_access_times_numa1 (line 197, /tmp/tmp3oh6y_qx/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_random_fill_rate (line 3, /tmp/tmp3oh6y_qx/rules/plotting.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_all_genome_fill_times (line 35, /tmp/tmp3oh6y_qx/rules/plotting.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_access_time_comparison (line 72, /tmp/tmp3oh6y_qx/rules/plotting.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_new_access_time_comparison (line 115, /tmp/tmp3oh6y_qx/rules/plotting.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_mxanthus_genome (line 9, /tmp/tmp3oh6y_qx/rules/files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_pfalciparum_genome (line 31, /tmp/tmp3oh6y_qx/rules/files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_hops_genome (line 52, /tmp/tmp3oh6y_qx/rules/files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_human_genome (line 74, /tmp/tmp3oh6y_qx/rules/files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1609867598.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for quality control using Fastqc and Fastq Screen",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "matrs/Quality_control",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1569509367.0
  },
  {
    "data_format": 1,
    "description": "A benchmark of benchmark workflow",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kokitsuyuzaki/workflow_benchmark",
    "linting": "Lints for snakefile /tmp/tmp2nbnhd46/Snakefile:\n    * Absolute path \"/home/koki/Software/R-3.6.0/bin/Rscript\" in line 1:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/pca.csv\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/nmf.csv\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mds.csv\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lda.csv\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/som.csv\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/isomap.csv\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lle.csv\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dmap.csv\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tsne.csv\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/umap.csv\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tumap.csv\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fatlas2.csv\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/pca.csv\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/nmf.csv\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mds.csv\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lda.csv\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/som.csv\" in line 66:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/isomap.csv\" in line 76:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lle.csv\" in line 86:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dmap.csv\" in line 96:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tsne.csv\" in line 106:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/umap.csv\" in line 116:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/tumap.csv\" in line 126:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fatlas2.csv\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 86:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 106:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule pca (line 27, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule nmf (line 50, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule mds (line 73, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule lda (line 96, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule som (line 120, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule isomap (line 143, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule lle (line 166, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule dmap (line 189, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tsne (line 212, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule umap (line 235, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tumap (line 258, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fatlas2 (line 281, /tmp/tmp2nbnhd46/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable Rscript from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1574841216.0
  },
  {
    "data_format": 1,
    "description": "pipeline i used for calling sQTL in chimps",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "bfairkun/sQTL",
    "linting": "FileNotFoundError in line 13 of /tmp/tmpboqvni46/rules/common.smk:\n[Errno 2] No such file or directory: \u0027RNASeqFileList.tsv\u0027\n  File \"/tmp/tmpboqvni46/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpboqvni46/rules/common.smk\", line 13, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1571675952.0
  },
  {
    "data_format": 1,
    "description": "Snakemake, OpenMS, and more",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mafreitas/regius",
    "linting": "Lints for snakefile /tmp/tmp481yxgqm/Snakefile:\n    * Absolute path \"/data\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 62:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 188:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 248:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 79:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 189:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule DecoyDatabaseRule (line 61, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule VBConvertmzML (line 86, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule VBConvertmzXML (line 111, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MyriMatchAdapterRule (line 140, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule IDPosteriorErrorProbabilityMyr (line 162, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MzTabExporterMyr (line 185, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MSGFPlusAdapter (line 212, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule IDPosteriorErrorProbabilityMsgf (line 237, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MzTabExporterMsgf (line 261, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule IDMergeConsensusPep (line 289, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule ConsensusID (line 315, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule PeptideIndexer (line 340, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MzTabExporter (line 364, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule IDMergeFido (line 393, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule FidoAdapter (line 419, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule TextExporterProt (line 442, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule TextExporter (line 466, /tmp/tmp481yxgqm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1513267939.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "TatyanaLev/EE283HW4",
    "linting": "Lints for rule r (line 11, /tmp/tmp1y7sf3u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule py (line 30, /tmp/tmp1y7sf3u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 51, /tmp/tmp1y7sf3u1/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612316564.0
  },
  {
    "data_format": 1,
    "description": "A workflow for getting gene information based on RNA-Seq data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "DaanJG98/RNA-Seq-Workflow",
    "linting": "Lints for snakefile /tmp/tmpt9gg1zth/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 154:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule filter_ids (line 13, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_orthologs (line 34, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_ncbi_ids (line 53, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_gene_info (line 72, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_sequence (line 107, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule conv_kegg_ids (line 140, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_kegg_ids (line 159, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_genes_per_pubmed (line 194, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule create_gc_graphs (line 230, /tmp/tmpt9gg1zth/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report (line 260, /tmp/tmpt9gg1zth/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1528392888.0
  },
  {
    "data_format": 1,
    "description": "Tutorial to getting started with the Snakemake workflow manager",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "susheelbhanu/snakemake-tutorial",
    "linting": "Lints for rule stats (line 212, /tmp/tmpbto4xhcn/Snakefile):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613661716.0
  },
  {
    "data_format": 1,
    "description": "Collection of scripts for conveniently downloading data from the archs4 resource",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jhawe/archs4_loader",
    "linting": "Lints for snakefile /tmp/tmpob6v2c0n/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 60:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 62:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 86:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 101:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 117:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule init_data (line 14, /tmp/tmpob6v2c0n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule explore_data (line 130, /tmp/tmpob6v2c0n/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_gtex_tissue_summaries (line 168, /tmp/tmpob6v2c0n/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule explore_gtex (line 187, /tmp/tmpob6v2c0n/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1549681231.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pp4rs/2019-snakemake-zurich",
    "linting": "Lints for rule tab01 (line 42, /tmp/tmp9i0ype6c/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param filepath is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_figs (line 80, /tmp/tmp9i0ype6c/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule solow_model (line 111, /tmp/tmp9i0ype6c/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rename_vars (line 147, /tmp/tmp9i0ype6c/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_variables (line 175, /tmp/tmp9i0ype6c/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean (line 206, /tmp/tmp9i0ype6c/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_windows (line 217, /tmp/tmp9i0ype6c/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1575955912.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpmpx96o1o/rules/transfer.smk\":  \u0027InvalidParameterSyntax: L8:Invalid key config[\"tranfer\"][\"port\"]\n username\u0027\n[ERROR] In file \"/tmp/tmpmpx96o1o/rules/common.smk\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 42)\u0027\n[ERROR] In file \"/tmp/tmpmpx96o1o/rules/pcrmonitor.smk\":  \u0027EmptyContextError: L1: rule has no keywords attached to it.\u0027\n[INFO] 3 file(s) raised parsing errors \ud83e\udd15\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mwhitesi/apl_assignment",
    "linting": "IndentationError in line 42 of \u003ctokenize\u003e:\nunindent does not match any outer indentation level (\u003ctokenize\u003e, line 42)\n  File \"/tmp/tmpmpx96o1o/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/tokenize.py\", line 512, in _tokenize\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599630850.0
  },
  {
    "data_format": 1,
    "description": "Workflow for generating reference phylogenies of AMR sequences in CARD",
    "formatting": "[WARNING] In file \"/tmp/tmpmgm6rfwx/workflow/Snakefile\":  Keyword \"input\" at line 331 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fmaguire/card-phylo",
    "linting": "Lints for snakefile /tmp/tmpmgm6rfwx/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule extract_prevalence (line 105, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tree (line 230, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cluster_by_mmseq_0 (line 268, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cluster_by_mmseq_70 (line 308, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cluster_by_mmseq_80 (line 347, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cluster_by_mmseq_90 (line 387, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cluster_by_mmseq_95 (line 427, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule mmseqs (line 626, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule organise_by_family (line 652, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * Param prev_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule family (line 700, /tmp/tmpmgm6rfwx/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1596120817.0
  },
  {
    "data_format": 1,
    "description": "project for pp4rs 2020",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bellni/2020-pp4rs-bellert-project",
    "linting": "Lints for rule paper (line 17, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_table (line 41, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule ols (line 71, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule figs (line 109, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_data (line 135, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 162, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filegraph (line 177, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rulegraph (line 196, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule find_packages (line 218, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule install_packages (line 234, /tmp/tmpyv4ngovu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1584046003.0
  },
  {
    "data_format": 1,
    "description": "Evaluation Workflow for Protein Complex Similarity based on Weisfeiler-Lehman labeling",
    "formatting": "[WARNING] In file \"/tmp/tmp8_57dfxy/Snakefile\":  Keyword \"output\" at line 113 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp8_57dfxy/Snakefile\":  Keyword \"params\" at line 203 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp8_57dfxy/Snakefile\":  Keyword \"params\" at line 219 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "BiancaStoecker/complex-similarity-evaluation",
    "linting": "Lints for snakefile /tmp/tmp8_57dfxy/Snakefile:\n    * Absolute path \"/usr/bin/time -v numactl --physcpubind={params.cpu_list} --membind=1 -- java -jar tools/rangequery/rangequery-29dfc07.jar --queries {input} --dataset simulated_complexes/true_constraints/ --resultfolder $out_dir --thresh {wildcards.thresh} --falsenegativerate {wildcards.false_negative_rate} {params.minhash} \u0026\u003e $out_dir/log.txt ;\" in line 209:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/usr/bin/time -v numactl --physcpubind={params.cpu_list} --membind=1 -- tools/ged_rangequery/ged_rangequery ../../simulated_complexes/true_constraints.gml ../../{input} {wildcards.thresh} ../../{output} \u0026\u003e $out_dir/log.txt\" in line 226:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule compute_threshold_graph (line 34, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule write_unique_complexes (line 71, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_candidate_pairs (line 102, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calculate_edit_similarities (line 126, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calculate_wl_values (line 155, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule compare_distances (line 184, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule subsample_complexes (line 218, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule benchmark_wl_2iter (line 247, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule benchmark_wl_3iter (line 276, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_benchmarks (line 345, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule database_search_wl (line 371, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule database_search_ges (line 407, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_times (line 438, /tmp/tmp8_57dfxy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1611763432.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cshlwyang/RNAseq",
    "linting": "Lints for snakefile /tmp/tmpt2bgidf4/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpt2bgidf4/rules/align.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpt2bgidf4/rules/diffexp.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule rseqc_junction_annotation (line 31, /tmp/tmpt2bgidf4/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_junction_saturation (line 79, /tmp/tmpt2bgidf4/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_innerdis (line 188, /tmp/tmpt2bgidf4/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readdup (line 264, /tmp/tmpt2bgidf4/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rseqc_readgc (line 298, /tmp/tmpt2bgidf4/rules/qc.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616724776.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to process circRNA AmpliconSeq datasets generated to study circRNAs in KSHV and human hosts using divergent primers.",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "CCBR/CCBR_circRNA_AmpliconSeq",
    "linting": "IndexError in line 8 of /tmp/tmp3ojgfhu0/workflow/rules/init.smk:\nlist index out of range\n  File \"/tmp/tmp3ojgfhu0/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmp3ojgfhu0/workflow/rules/init.smk\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 5,
    "topics": [
      "snakemake-workflow",
      "amplicon-sequencing",
      "amplicon-pipeline",
      "circrna",
      "circrna-pipeline"
    ],
    "updated_at": 1616619140.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpy6lnf51w/rules/allelfeqplots.rules.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:7: script:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpy6lnf51w/rules/allelfeqplots.rules.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpy6lnf51w/rules/allelfeqplots.rules.smk\":  4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AnzaGhaffar/QTL-Snakemake-Workflow",
    "linting": "Lints for rule VCF_Homozygous_Filtering (line 14, /tmp/tmpy6lnf51w/rules/filtering.rules.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule QTL_VCF_to_Table_Parser (line 14, /tmp/tmpy6lnf51w/rules/vcfparser.rules.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule QTL_Plotting (line 21, /tmp/tmpy6lnf51w/rules/qtlplots.rules.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule Allel_Frequency_Tsv_Generator (line 21, /tmp/tmpy6lnf51w/rules/allelfeqplots.rules.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule Allel_Plots (line 50, /tmp/tmpy6lnf51w/rules/allelfeqplots.rules.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593177001.0
  },
  {
    "data_format": 1,
    "description": "Snakemake Pipeline for RNASeq Workflow ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "alipe2018/Snakemake-RNAseq-Workflow",
    "linting": "/tmp/tmpyp3biglf/Snakefile:9: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  SAMPLES = sorted(FILES.keys())\nLints for snakefile /tmp/tmpyp3biglf/Snakefile:\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 77:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 112:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 114:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 128:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 139:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 151:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 152:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 154:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 156:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 158:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 169:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 171:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 184:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 185:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule ReName (line 46, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule hisat2Align (line 104, /tmp/tmpyp3biglf/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable HISAT2_INDEX_PREFIX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sortBam (line 141, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule stringtieMergeList (line 202, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule stringtieMerge (line 220, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule produce_sample_express_list (line 278, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule prepDE (line 298, /tmp/tmpyp3biglf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1544405955.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "max-hoffman/snakemake-ex",
    "linting": "Lints for rule hello (line 15, /tmp/tmp57yvm1c4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hello2 (line 30, /tmp/tmp57yvm1c4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1560542869.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "amberr098/snakemakeOWE11",
    "linting": "Lints for rule entrezID (line 13, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule lpID (line 34, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule function (line 54, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule sequentie (line 75, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule names (line 94, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pmids (line 114, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule sort (line 140, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule comb (line 160, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule kegginf (line 180, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pathway (line 199, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ortho (line 218, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule workflow (line 238, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report (line 254, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean (line 276, /tmp/tmpubtwilpq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1528467751.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to run some basic quality control programs ",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pawel125/quality_control_CellRanger",
    "linting": "Lints for rule multiqc (line 14, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mklink (line 35, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc (line 64, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule read_GC (line 94, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bam_stat (line 120, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule tin (line 141, /tmp/tmphv1rub3e/workflow/rules/bam_quality_control.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule emptydrops (line 29, /tmp/tmphv1rub3e/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param input is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608713691.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for detecting viral variants.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "RachelRodgers/VirusVariantViewR",
    "linting": "Lints for snakefile /tmp/tmp8_egso3y/workflow/Snakefile:\n    * Absolute path \"/renamed/\", PATTERN_R1 + \" in line 82:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/renamed/\", PATTERN_R2 + \" in line 83:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule trim_adapters (line 87, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable XMX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule quality_trimming (line 125, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable XMX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule deduplicate (line 158, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable XMX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule deinterleave (line 190, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable XMX from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_to_ref_genome (line 216, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_view (line 250, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort (line 278, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_index (line 306, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genomecov (line 331, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_indel_quality (line 356, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_variants (line 385, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_coverage_depth (line 414, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_alignment_data (line 439, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule concat_alignment_data (line 466, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule annotate_mutations (line 489, /tmp/tmp8_egso3y/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable R from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1601653425.0
  },
  {
    "data_format": 1,
    "description": "Process raw Quant-Seq data and run DE-Seq",
    "formatting": "[ERROR] In file \"/tmp/tmp9vziguhd/rules/build_indices.smk\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[ERROR] In file \"/tmp/tmp9vziguhd/rules/prepare_seqs.smk\":  \u0027NoParametersError: L8: In input definition.\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "marykthompson/ribopop_rnaseq",
    "linting": "FileNotFoundError in line 12 of /tmp/tmp9vziguhd/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp9vziguhd/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597225570.0
  },
  {
    "data_format": 1,
    "description": "The ENCODE-DCC demo workflow in Snakemake (under development)",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "vsoch/encode-demo-workflow",
    "linting": "Lints for snakefile /tmp/tmp_qlshebr/Snakefile:\n    * Deprecated singularity directive used for container definition in line 11.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot (line 3, /tmp/tmp_qlshebr/rules/plot.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "encode-dcc",
      "snakemake",
      "snakemake-workflows"
    ],
    "updated_at": 1567344079.0
  },
  {
    "data_format": 1,
    "description": "Simple Snakemake example",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ctmrbio/sm3",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1613383888.0
  },
  {
    "data_format": 1,
    "description": "Workflow to generate gene expression box plot for tissues",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cu-swe4s-fall-2019/workflow-qyang13",
    "linting": "Lints for rule get_tissue_samples (line 14, /tmp/tmp0vawe3i0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TISSUES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule get_gene_counts (line 37, /tmp/tmp0vawe3i0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GENES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule box (line 60, /tmp/tmp0vawe3i0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GENES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TISSUES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1574270982.0
  },
  {
    "data_format": 1,
    "description": "SV calling pipeline",
    "formatting": "[ERROR] In file \"/tmp/tmp_9rbr3dn/Snakefile\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 214)\u0027\n[INFO] In file \"/tmp/tmp_9rbr3dn/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "Triciasnow/HiFi-sv-snakemake",
    "linting": "IndentationError in line 214 of \u003ctokenize\u003e:\nunindent does not match any outer indentation level (\u003ctokenize\u003e, line 214)\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/tokenize.py\", line 512, in _tokenize\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619315805.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dirkrepsilber/metagenomics-workflow",
    "linting": "WorkflowError in line 58 of /tmp/tmps35mw57w/Snakefile:\nFailed to open source file /glob/brynjar/github/snakemake-workflows/bio/ngs/rules/quality_control/fastqc.rules\nFileNotFoundError: [Errno 2] No such file or directory: \u0027/glob/brynjar/github/snakemake-workflows/bio/ngs/rules/quality_control/fastqc.rules\u0027\n  File \"/tmp/tmps35mw57w/Snakefile\", line 58, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1481113840.0
  },
  {
    "data_format": 1,
    "description": "Simple text analysis using Snakemake",
    "formatting": "[ERROR] In file \"/tmp/tmpkfqp5592/Snakefile\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7f9294b120d0\u003eNewline expected after keyword \u0027run\u0027\u0027\n[INFO] In file \"/tmp/tmpkfqp5592/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "sipv/snakemake-text-analysis",
    "linting": "ModuleNotFoundError in line 1 of /tmp/tmpkfqp5592/Snakefile:\nNo module named \u0027matplotlib\u0027\n  File \"/tmp/tmpkfqp5592/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpkfqp5592/util/__init__.py\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpkfqp5592/util/plot.py\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1572337568.0
  },
  {
    "data_format": 1,
    "description": "Basic Snakemake wf for wordcloud generation.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NajlaBH/snaketmpl",
    "linting": "Lints for rule get_text (line 5, /tmp/tmpbb_o01bj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_plot (line 32, /tmp/tmpbb_o01bj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_shape (line 59, /tmp/tmpbb_o01bj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613065388.0
  },
  {
    "data_format": 1,
    "description": "A workflow for metagenomics analysis",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ORU-BIOINF/metagenomics-workflow",
    "linting": "WorkflowError in line 49 of /tmp/tmpxl_ohn3l/Snakefile:\nFailed to open source file https://raw.githubusercontent.com/binnisb/snakemake-workflows/master/bio/ngs/rules/quality_control/fastqc.rules\nHTTPError: 404 Client Error: Not Found for url: https://raw.githubusercontent.com/binnisb/snakemake-workflows/master/bio/ngs/rules/quality_control/fastqc.rules\n  File \"/tmp/tmpxl_ohn3l/Snakefile\", line 49, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1427278649.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow: Toy example, compliant with HPC Sumner at JAX",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "TheJacksonLaboratory/toymake",
    "linting": "FileNotFoundError in line 25 of /tmp/tmpv1oe9nrm/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/runner/work/snakemake-workflow-catalog/snakemake-workflow-catalog/config.yaml\u0027\n  File \"/tmp/tmpv1oe9nrm/Snakefile\", line 25, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "workflow",
      "python3",
      "slurm",
      "slurm-job-scheduler",
      "hpc",
      "cluster-computing",
      "centos7"
    ],
    "updated_at": 1589224874.0
  },
  {
    "data_format": 1,
    "description": "This is a pipeline for the beginning analysis of paired end ATAC seq data",
    "formatting": "[WARNING] In file \"/tmp/tmpmqt6xltr/rules/peak_catalog.smk\":  Keyword \"input\" at line 50 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpmqt6xltr/Snakefile\":  Keyword \"input\" at line 101 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpmqt6xltr/rules/quality_and_align.smk\":  \u0027EmptyContextError: L47: rule has no keywords attached to it.\u0027\n[WARNING] In file \"/tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk\":  Keyword \"shell\" at line 138 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "maxsonBraunLab/atac_seq---old",
    "linting": "Lints for snakefile /tmp/tmpmqt6xltr/Snakefile:\n    * Absolute path \"/home/groups/MaxsonLab/indices/GRch38/hg38.blacklist.v2.bed\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/groups/MaxsonLab/callahro/atac_seq/fastqscreen.conf\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/groups/MaxsonLab/callahro/atac_seq/MOLM_ATAC_metadata.txt\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/list/to/file/\" + name + \" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 79:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 80:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout/{merged_sample}_macs_peaks.broadPeak\" in line 81:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_homerout/homerMotifs.all.motifs\" in line 82:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/raw_bigwigs/{merged_sample}_raw.bw\" in line 83:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/non_downsampled_bigwigs/{merged_sample}_non_downsampled.bw\" in line 84:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqc/{sample}_fastqc_done.txt\" in line 85:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}_fastqscreen_done.txt\" in line 86:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_information_done.txt\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_flagstat.txt\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_smooth.bw\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_rough.bw\" in line 90:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_count_peaks_done.txt\" in line 92:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals_nodownsample.bed_{sample_condition}.bed\" in line 93:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{sample_condition}_merged.bam\" in line 94:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_counts.bed\" in line 95:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals_nodownsample.bed\" in line 96:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/intersected_bedfile_nodownsample.bed\" in line 97:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_nodownsample_counts.bed\" in line 98:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/intersected_bedfile_nodownsample_peakinfo.txt\" in line 99:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/intersected_catalog.bed\" in line 100:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_nodownsample_counts_ondownsamplepeaks.bed\" in line 101:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/done.txt\" in line 103:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 79:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 85:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 86:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 92:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 93:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 94:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 97:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 98:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 100:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 101:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk:\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_smooth.bw\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_rough.bw\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/downsample_list.txt\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam.bai\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_sorted.bam.bai\" in line 37:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_sorted.bam\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/downsample_list.txt\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/samples_downsample_information.txt\" in line 60:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality.bam\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality.bam.bai\" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 91:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality.bam\" in line 107:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_merged.bam\" in line 108:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality.bam.bai\" in line 110:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_merged.bam.bai\" in line 111:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup.bam\" in line 127:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality.bam\" in line 130:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 132:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM.bam\" in line 150:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup.bam\" in line 153:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 155:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_sorted.bam\" in line 172:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM.bam\" in line 175:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 176:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_readswmito.txt\" in line 182:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}\" in line 198:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}\" in line 199:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_merged.bam\" in line 201:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 37:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 60:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 89:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 91:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 107:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 111:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 127:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 132:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 153:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 155:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 172:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 175:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 176:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 198:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 199:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 201:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 198:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/peak_catalog_no_downsample.smk:\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_nodownsample_counts.bed\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_nodownsample_counts.bed\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals_nodownsample.bed\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_nodownsample_counts.bed\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_sorted.bam\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample/{merged_sample}_macs_peaks.broadPeak\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals_nodownsample.bed\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals_nodownsample.bed_{sample_condition}.bed\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/all_broad_peaks_nodownsample.bed\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample/{merged_sample}_macs_peaks.broadPeak\" in line 61:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 76:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample/{merged_sample}_macs_peaks.broadPeak\" in line 78:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample\" in line 85:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 85:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/peak_catalog.smk:\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_counts.bed\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_counts.bed\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_counts.bed\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/reads_catalog_intervals.bed\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_peaks_information.txt\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout/{merged_sample}_macs_peaks.broadPeak\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed_{sample_condition}.bed\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/all_broad_peaks.bed\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout/{merged_sample}_macs_peaks.broadPeak\" in line 66:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads_downSample.bam\" in line 81:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout/{merged_sample}_macs_peaks.broadPeak\" in line 83:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout\" in line 90:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 19:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 22:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 38:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 81:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/non_downsampled_bigwig.smk:\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_non_downsampled.bedGraph\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_non_downsampled.bedGraph\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_non_downsampled_sorted.bedGraph\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_non_downsampled_sorted.bedGraph\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/non_downsampled_bigwigs/{merged_sample}_non_downsampled.bw\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/groups/MaxsonLab/indices/mm10/mm10.chrom.sizes\" in line 39:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk:\n    * Absolute path \"/results/sample_quality_information.tsv\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/correllation_plot.png\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/intersected_catalog.bed\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_read_catalog_nodownsample_counts_ondownsamplepeaks.bed\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/sample_quality_information.tsv\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/correlation_plot.png\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/intersected_catalog.bed\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/reads_catalog_intervals.bed\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/all_read_catalog_nodownsample_counts_ondownsamplepeaks.bed\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_count_peaks_done.txt\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/*.csv\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/all_sample_information_nospaces.csv\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/results/sample_quality_information.tsv\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/^\\s*$/d\" in line 39:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_rough.bw\" in line 46:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_tracks_5window_rough.bw\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/sample_scores.npz\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/correllation_plot.png\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed_{sample_condition}_formatted.bed\" in line 69:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/intersected_catalog.bed\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed_{sample_condition}_formatted.bed\" in line 73:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed_{sample_condition}.bed\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed\" in line 88:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/reads_catalog_intervals.bed_{sample_condition}_formatted.bed\" in line 90:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_information_done.txt\" in line 102:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample/{merged_sample}_macs_peaks.broadPeak\" in line 103:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_count_peaks_done.txt\" in line 105:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 107:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_macsout_nodownsample/{merged_sample}_macs_peaks.broadPeak\" in line 119:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}.bam\" in line 120:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_nodownsample_counts.bed\" in line 121:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_catalog_nodownsample_counts.bed\" in line 122:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{merged_sample}_rmChrM_dedup_quality_shiftedReads.bam\" in line 123:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_information_done.txt\" in line 125:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_flagstat.txt\" in line 126:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_read_depths.csv\" in line 128:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{merged_sample}_flagstat.txt\" in line 129:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 36:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 46:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 52:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 69:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 88:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 90:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 102:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 107:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 121:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 122:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 125:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 128:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 129:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/merged_bigwig.smk:\n    * Absolute path \"/bamfiles/\" + name + \" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/\" + name + \" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{sample_condition}_merged.bam\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fully_filtered/{sample_condition}_merged.bw\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpmqt6xltr/rules/quality_and_align.smk:\n    * Absolute path \"/bamfiles/{sample}.bam\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{sample}_sorted.bam\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samfiles/{sample}.sam\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/bamfiles/{sample}.bam\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_1_fastqc_tpaired.fastq\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_2_fastqc_tpaired.fastq\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/samfiles/{sample}.sam\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_1_fastqc_tpaired.fastq\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_2_fastqc_tpaired.fastq\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_1_fastqc_tunpaired.fastq\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/trimmed/{sample}_2_fastqc_tunpaired.fastq\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqc/{sample}_fastqc.done\" in line 70:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/multiqc/done.txt\" in line 73:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqc/{sample}_fastqc_done.txt\" in line 82:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqc/{sample}_2_fastqc.zip\" in line 83:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqc/{sample}\" in line 85:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}/\" + prefix + \" in line 108:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}/\" + prefix + \" in line 109:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}\" in line 110:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fastqscreen/{sample}\" in line 114:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 79:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 85:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 114:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 51:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 80:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 106:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule makeBigwig (line 4, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule downSample (line 50, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule createDownsample (line 98, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule shiftReads (line 139, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index (line 178, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtoolsQuality (line 219, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dedup (line 264, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mergeBam (line 358, /tmp/tmpmqt6xltr/rules/filter_shift_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_catalog_counts_no_downsmpl (line 2, /tmp/tmpmqt6xltr/rules/peak_catalog_no_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule peak_catalog_counts_no_downsmpl (line 28, /tmp/tmpmqt6xltr/rules/peak_catalog_no_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule peak_catalog_no_downsmpl (line 79, /tmp/tmpmqt6xltr/rules/peak_catalog_no_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MACS_no_downsmpl (line 125, /tmp/tmpmqt6xltr/rules/peak_catalog_no_downsample.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule merge_catalog_counts (line 1, /tmp/tmpmqt6xltr/rules/peak_catalog.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule peak_catalog_counts (line 24, /tmp/tmpmqt6xltr/rules/peak_catalog.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule annotate_peak_names (line 60, /tmp/tmpmqt6xltr/rules/peak_catalog.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule peak_catalog (line 92, /tmp/tmpmqt6xltr/rules/peak_catalog.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param reads_catalog_bed is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule MACS (line 141, /tmp/tmpmqt6xltr/rules/peak_catalog.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule non_downsampled_bedGraph (line 1, /tmp/tmpmqt6xltr/rules/non_downsampled_bigwig.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule non_downsampled_sort (line 34, /tmp/tmpmqt6xltr/rules/non_downsampled_bigwig.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule non_downsampled_bedGraphToBigWig (line 67, /tmp/tmpmqt6xltr/rules/non_downsampled_bigwig.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plotCorrelation (line 44, /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multi_intersect (line 90, /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_union_interval_from_sample_interval (line 125, /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_macs_peaks (line 155, /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_info (line 188, /tmp/tmpmqt6xltr/rules/final_qc_metrics.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param flagstat is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule makeMergedBigwig (line 13, /tmp/tmpmqt6xltr/rules/merged_bigwig.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sortbam (line 1, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdirectory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule sam2bam (line 35, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule align (line 65, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trimming (line 102, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 151, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fastqscreen (line 199, /tmp/tmpmqt6xltr/rules/quality_and_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1613080707.0
  },
  {
    "data_format": 1,
    "description": "no more boiler plate in your execution scripts.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "RossLab/snakemake-cluster-template",
    "linting": "Lints for rule help (line 26, /tmp/tmphafra4lv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_data (line 37, /tmp/tmphafra4lv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_results (line 53, /tmp/tmphafra4lv/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1596106536.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "skyriakidis/Snakemake_haps",
    "linting": "FileNotFoundError in line 6 of /tmp/tmpnxw2l6qq/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpnxw2l6qq/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1544099658.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpk48nheob/workflow/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:0: elif config[\u0027input_type\u0027] == \u0027assemblies\u0027:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "maxlcummins/pipelord2_0",
    "linting": "Cloning into \u0027resources/tools/pointfinder\u0027...\nCloning into \u0027resources/tools/pointfinder/pointfinder_db\u0027...\nCloning into \u0027resources/tools/pmlst\u0027...\nCloning into \u0027resources/tools/pmlst/pmlst_db\u0027...\nLints for snakefile /tmp/tmpk48nheob/workflow/Snakefile:\n    * Absolute path \"/{sample}.R1.fastq.gz\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}.R1.fastq.gz\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}.fasta\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assembly_stats/{sample}_assembly_stats.txt\" in line 49:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/abricate/{gene_db}/{sample}.tab\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/{sample}.out/results_named.txt\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/{sample}/{sample}_blastn_results_named.tsv\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/{sample}_clean.report\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.R1.fastq.gz\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/mlst/{sample}_mlst.txt\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/fastp_summary.json\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/assembly_stats.txt\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/{prefix}_simple_summary_N\" in line 60:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/mlst.txt\" in line 61:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/Pointfinder.txt\" in line 62:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/abricate/{gene_db}/{sample}.tab\" in line 63:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/abricate_hits.txt\" in line 64:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/pMLST.txt\" in line 65:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/kraken2_full_summary.txt\" in line 66:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/read_cleaning.smk:\n    * Absolute path \"/{sample}.R1.fastq.gz\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}.R2.fastq.gz\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.R1.fastq.gz\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.R2.fastq.gz\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.fastp.json\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.fastp.html\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/run/{sample}.log\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/genome_assembly.smk:\n    * Absolute path \"/\"+config[\u0027prefix\u0027]+\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}.fasta\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assembly_stats/{sample}_assembly_stats.txt\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.R1.fastq.gz\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/fastp/{sample}.R2.fastq.gz\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/shovill_out/{sample}.out\" in line 37:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/run/{sample}_out.log\" in line 40:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/run/{sample}_err.log\" in line 41:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 53:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/shovill/assembly_stats/{sample}_assembly_stats.txt\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/summarise.smk:\n    * Absolute path \"/{prefix}/summaries/mlst.txt\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/Pointfinder.txt\" in line 4:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/abricate_hits.txt\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/pMLST.txt\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/kraken2_full_summary.txt\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/assembly_stats.txt\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summaries/{prefix}_simple_summary_N\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summarise/summarise_out.log\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/summarise/summarise_err.log\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/species_id.smk:\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/{sample}.out\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/{sample}.report\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/run/{sample}_err.log\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/{sample}.report\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/kraken2/{sample}_clean.report\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/genotype_abricate.smk:\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/abricate/{gene_db}/{sample}.tab\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/abricate/run/{gene_db}/{sample}.log\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/point_mutations.smk:\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/{sample}/{sample}_blastn_results.tsv\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/pointfinder_run/{sample}.log\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/{sample}\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/{sample}/{sample}_blastn_results.tsv\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pointfinder/{sample}/{sample}_blastn_results_named.tsv\" in line 37:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/strain_mlst.smk:\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/mlst/{sample}_mlst.txt\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpk48nheob/workflow/rules/plasmid_mlst.smk:\n    * Absolute path \"/{prefix}/shovill/assemblies/{sample}.fasta\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/{sample}.out/results.txt\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/{sample}.out\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/temp_{sample}\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/{sample}.out/results.txt\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{prefix}/pMLST/{scheme}/{sample}.out/results_named.txt\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule run_assembly_stats (line 95, /tmp/tmpk48nheob/workflow/rules/genome_assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule name_kraken2_inline (line 42, /tmp/tmpk48nheob/workflow/rules/species_id.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pointfinder_run (line 16, /tmp/tmpk48nheob/workflow/rules/point_mutations.smk):\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule name_append (line 52, /tmp/tmpk48nheob/workflow/rules/point_mutations.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mlst_run (line 13, /tmp/tmpk48nheob/workflow/rules/strain_mlst.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pMLST_run (line 18, /tmp/tmpk48nheob/workflow/rules/plasmid_mlst.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule name_files_pMLST (line 58, /tmp/tmpk48nheob/workflow/rules/plasmid_mlst.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614230000.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rr-mrc-bsu/containerization-example",
    "linting": "Lints for snakefile /tmp/tmp0e06tcrf/Snakefile:\n    * Deprecated singularity directive used for container definition in line 1.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_report (line 5, /tmp/tmp0e06tcrf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1551699396.0
  },
  {
    "data_format": 1,
    "description": "Snakemake Workflow: Impute HLA and/or KIR alleles from SNPs",
    "formatting": "[ERROR] In file \"/tmp/tmpkhs_qy37/Snakefile\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7f281fc300d0\u003eNewline expected after keyword \u0027run\u0027\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "bjohnnyd/hla-kir-imputation",
    "linting": "Lints for snakefile /tmp/tmpkhs_qy37/Snakefile:\n    * Deprecated singularity directive used for container definition in line 15.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpkhs_qy37/rules/freq_encode_snps.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule print_defaults (line 65, /tmp/tmpkhs_qy37/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_dag (line 71, /tmp/tmpkhs_qy37/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_kirimp_panel (line 6, /tmp/tmpkhs_qy37/rules/kirimp_panel.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_overchain (line 14, /tmp/tmpkhs_qy37/rules/liftover.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_ucsc_bed (line 30, /tmp/tmpkhs_qy37/rules/liftover.smk):\n    * Param baseout is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule create_plink_lifted (line 91, /tmp/tmpkhs_qy37/rules/liftover.smk):\n    * Param baseout is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule create_vcf (line 132, /tmp/tmpkhs_qy37/rules/liftover.smk):\n    * Param baseout is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule shapeit4 (line 27, /tmp/tmpkhs_qy37/rules/shapeit.smk):\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1589639041.0
  },
  {
    "data_format": 1,
    "description": "Code to run the analysis of the Ultra Deep Sequencing",
    "formatting": "[WARNING] In file \"/tmp/tmp4p4mjaep/Snakefile\":  Keyword \"input\" at line 34 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp4p4mjaep/Snakefile\":  Keyword \"input\" at line 59 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "knights-lab/uds-analysis",
    "linting": "ValueError in line 28 of /tmp/tmp4p4mjaep/Snakefile:\nnot enough values to unpack (expected 2, got 0)\n  File \"/tmp/tmp4p4mjaep/Snakefile\", line 28, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1508439216.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpnjqlfmt_/Snakefile\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 4)\u0027\n[INFO] In file \"/tmp/tmpnjqlfmt_/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "PrincetonUniversity/gtex-gatk4",
    "linting": "FileNotFoundError in line 19 of /tmp/tmpnjqlfmt_/Snakefile:\n[Errno 2] No such file or directory: \u0027/tigress/AKEY/akey_vol1/home/selinav/GTEx/SampInfo/sample_details.txt\u0027\n  File \"/tmp/tmpnjqlfmt_/Snakefile\", line 19, in \u003cmodule\u003e\n  File \"/tmp/tmpnjqlfmt_/scripts/get_samples.py\", line 18, in get_samples\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1558379531.0
  },
  {
    "data_format": 1,
    "description": "Estimate pi using a snakemake workflow",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PabRod/snakePi",
    "linting": "Lints for rule gather (line 8, /tmp/tmpai3xsco5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule set_path (line 25, /tmp/tmpai3xsco5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_pi (line 47, /tmp/tmpai3xsco5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 68, /tmp/tmpai3xsco5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608222544.0
  },
  {
    "data_format": 1,
    "description": "Integrative Pipeline for Splicing Analysis",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Leoberium/pyIPSA",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmpkkl4ykxb/workflow/Snakefile:\nNo module named \u0027pysam\u0027\n  File \"/tmp/tmpkkl4ykxb/workflow/Snakefile\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614342277.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "csoneson/WagnerEMT2020",
    "linting": "Lints for snakefile /tmp/tmp2ft57b_c/Snakefile:\n    * Absolute path \"/versionInfo.json\" in line 173:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/SA\" in line 199:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/chrNameLength.txt\" in line 200:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/versionInfo.json\" in line 351:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/SA\" in line 378:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/chrNameLength.txt\" in line 421:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/versionInfo.json\" in line 448:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 77:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 94:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 96:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 103:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 113:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 124:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 156:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 158:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 175:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 177:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 202:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 204:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 225:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 229:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 233:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 245:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 247:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 249:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 253:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 266:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 267:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 269:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 270:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 272:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 277:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 293:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 297:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 307:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 311:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 313:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 315:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 317:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 334:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 338:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 352:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 352:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 356:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 358:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 363:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 379:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 379:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 385:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 387:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 390:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 404:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 406:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 408:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 410:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 420:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 423:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 425:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 429:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 445:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 446:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 451:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 453:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 455:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 457:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 482:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 484:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 486:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 513:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 514:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 518:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 519:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 521:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 529:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 225:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 330:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 352:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 379:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule pkginstall (line 93, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule listpackages (line 180, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule softwareversions (line 206, /tmp/tmp2ft57b_c/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmonindex (line 271, /tmp/tmp2ft57b_c/Snakefile):\n    * Param salmonoutdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule starindex (line 323, /tmp/tmp2ft57b_c/Snakefile):\n    * Param STARindex is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fastqc (line 376, /tmp/tmp2ft57b_c/Snakefile):\n    * Param FastQC is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fastqctrimmed (line 418, /tmp/tmp2ft57b_c/Snakefile):\n    * Param FastQC is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule multiqc (line 483, /tmp/tmp2ft57b_c/Snakefile):\n    * Param MultiQCdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule qcsummary (line 522, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param multiqcdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule trimgaloreSE (line 564, /tmp/tmp2ft57b_c/Snakefile):\n    * Param FASTQtrimmeddir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule salmonSE (line 605, /tmp/tmp2ft57b_c/Snakefile):\n    * Param salmonindex is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param salmondir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule starSE (line 655, /tmp/tmp2ft57b_c/Snakefile):\n    * Param STARindex is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param STARdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bigwig (line 740, /tmp/tmp2ft57b_c/Snakefile):\n    * Param STARbigwigdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule tximeta (line 788, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param salmondir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule checkinputs (line 841, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule edgeR (line 896, /tmp/tmp2ft57b_c/Snakefile):\n    * Shell command directly uses variable Rbin from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1616493309.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 16 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "shukwong/dna-seq-varlociraptor-test",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608310889.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ohsu-cedar-comp-hub/DIDA",
    "linting": "FileNotFoundError in line 24 of /tmp/tmp_zvjn1pu/Snakefile:\n[Errno 2] No such file or directory: \u0027data/snakemake_pipeline_NovaSeq_8bp_barcodes.txt\u0027\n  File \"/tmp/tmp_zvjn1pu/Snakefile\", line 24, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1589323783.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ORU-BIOI/metagenomics-testproject",
    "linting": "WorkflowError in line 58 of /tmp/tmpfda1a8c9/Snakefile:\nFailed to open source file /glob/brynjar/github/snakemake-workflows/bio/ngs/rules/quality_control/fastqc.rules\nFileNotFoundError: [Errno 2] No such file or directory: \u0027/glob/brynjar/github/snakemake-workflows/bio/ngs/rules/quality_control/fastqc.rules\u0027\n  File \"/tmp/tmpfda1a8c9/Snakefile\", line 58, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1432992248.0
  },
  {
    "data_format": 1,
    "description": "Build indices for STAR, Kallisto etc. Add data for specified spike-in RNAs.",
    "formatting": "[ERROR] In file \"/tmp/tmp_94005yh/rules/build_indices.smk\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[ERROR] In file \"/tmp/tmp_94005yh/rules/prepare_seqs.smk\":  \u0027NoParametersError: L8: In input definition.\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "marykthompson/build_genome_indices",
    "linting": "FileNotFoundError in line 11 of /tmp/tmp_94005yh/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp_94005yh/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1605549114.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "populationgenomics/utr-variants",
    "linting": "Workflow defines that rule genomepy is eligible for caching between workflows (use the --cache argument to enable this).\nDefaultCredentialsError in line 7 of /tmp/tmp6xkl03ai/workflow/rules/evaluation.smk:\nCould not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started\n  File \"/tmp/tmp6xkl03ai/workflow/Snakefile\", line 21, in \u003cmodule\u003e\n  File \"/tmp/tmp6xkl03ai/workflow/rules/evaluation.smk\", line 7, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/snakemake/remote/GS.py\", line 124, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/storage/client.py\", line 121, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 249, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 201, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 216, in _determine_default\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/_helpers.py\", line 186, in _determine_default_project\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/auth/_default.py\", line 483, in default\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1617932354.0
  },
  {
    "data_format": 1,
    "description": "Alignment based workflow for Illumina Short reads",
    "formatting": "[ERROR] In file \"/tmp/tmprasacgu_/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     \"FC/without_dups_gene_level/counts.txt.summary\"\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmprasacgu_/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "tgstoecker/RAW-ABS",
    "linting": "Lints for rule STAR_index (line 22, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_rRNA_gtf (line 72, /tmp/tmprasacgu_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_rRNA_fasta (line 92, /tmp/tmprasacgu_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bbduk (line 316, /tmp/tmprasacgu_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc (line 347, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trimmed_fastqc (line 414, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule STAR (line 442, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule index_sorted_bams_with_dups (line 527, /tmp/tmprasacgu_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_duplicates_samtools (line 579, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_sorted_bams_without_dups (line 619, /tmp/tmprasacgu_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule featureCounts (line 656, /tmp/tmprasacgu_/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1596061036.0
  },
  {
    "data_format": 1,
    "description": "Instructions and examples for using PanVC with founders",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "algbio/panvc-founders",
    "linting": "WorkflowError in line 4 of /tmp/tmpbhp9v3k6/Snakefile:\nFailed to open source file /tmp/tmpbhp9v3k6/panvc-sample-workflow/Snakefile.install\nFileNotFoundError: [Errno 2] No such file or directory: \u0027/tmp/tmpbhp9v3k6/panvc-sample-workflow/Snakefile.install\u0027\n  File \"/tmp/tmpbhp9v3k6/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1614592014.0
  },
  {
    "data_format": 1,
    "description": "Repository to process ITS amplicon sequencing data and produce figures for Schneider et al (2021)",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "andnischneider/its_workflow",
    "linting": "SystemExit in line 19 of /tmp/tmp5ymnidfl/workflow/rules/common.smk:\n\n\n###### WORKFLOW ERROR ######\nSample its.mock.none.t5.0-0 has no SRA id and missing at least one of\nR1: data/ITS_needle/its.mock.none.t5.0_R1.fastq.gz\nR2: data/ITS_needle/its.mock.none.t5.0_R2.fastq.gz\n###### WORKFLOW ERROR ######\n\n            \n  File \"/tmp/tmp5ymnidfl/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp5ymnidfl/workflow/rules/common.smk\", line 19, in \u003cmodule\u003e\n  File \"/tmp/tmp5ymnidfl/workflow/scripts/common.py\", line 63, in get_samples\n  File \"/tmp/tmp5ymnidfl/workflow/scripts/common.py\", line 5, in exit\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [
      "amplicon-data-preprocessing",
      "snakemake-workflow"
    ],
    "updated_at": 1614616263.0
  },
  {
    "data_format": 1,
    "description": "old repo. see dib-lab/elvers for this snakemake workflow",
    "formatting": "[ERROR] In file \"/tmp/tmpk3s9_i2g/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     fastqc_targs = get_targets(units , base , QC_DIR)\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpk3s9_i2g/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "bluegenes/2018-snakemake-eel-pond",
    "linting": "KeyError in line 12 of /tmp/tmpk3s9_i2g/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpk3s9_i2g/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1554845901.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for analyzing AMBER molecular dynamics simulations.",
    "formatting": "[ERROR] In file \"/tmp/tmpln7a0yq3/rules/eda.smk\":  \u0027EmptyContextError: L4: rule has no keywords attached to it.\u0027\n[ERROR] In file \"/tmp/tmpln7a0yq3/rules/figs.smk\":  \u0027EmptyContextError: L4: rule has no keywords attached to it.\u0027\n[ERROR] In file \"/tmp/tmpln7a0yq3/Snakefile\":  \u0027EmptyContextError: L16: rule has no keywords attached to it.\u0027\n[ERROR] In file \"/tmp/tmpln7a0yq3/rules/cpptraj.smk\":  \u0027EmptyContextError: L4: rule has no keywords attached to it.\u0027\n[INFO] In file \"/tmp/tmpln7a0yq3/rules/cpptraj.smk\":  4 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpln7a0yq3/rules/cpptraj.smk\":  2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "emleddin/md-analysis-workflow",
    "linting": "Lints for snakefile /tmp/tmpln7a0yq3/rules/common.smk:\n    * Path composition with \u0027+\u0027 in line 203:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule help (line 118, /tmp/tmpln7a0yq3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 154, /tmp/tmpln7a0yq3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule eda_diff (line 4, /tmp/tmpln7a0yq3/rules/eda.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule eda_avg (line 36, /tmp/tmpln7a0yq3/rules/eda.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule eda_run (line 76, /tmp/tmpln7a0yq3/rules/eda.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule eda_copy (line 122, /tmp/tmpln7a0yq3/rules/eda.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule eda_write (line 155, /tmp/tmpln7a0yq3/rules/eda.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cpptraj_analysis (line 4, /tmp/tmpln7a0yq3/rules/cpptraj.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cpptraj_strip (line 65, /tmp/tmpln7a0yq3/rules/cpptraj.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cpptraj_write (line 102, /tmp/tmpln7a0yq3/rules/cpptraj.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mc (line 4, /tmp/tmpln7a0yq3/rules/figs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule nma (line 35, /tmp/tmpln7a0yq3/rules/figs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rms_gnuplot (line 66, /tmp/tmpln7a0yq3/rules/figs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule rms_conv (line 108, /tmp/tmpln7a0yq3/rules/figs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule second_struct (line 139, /tmp/tmpln7a0yq3/rules/figs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflow",
      "molecular-dynamics",
      "reproducible-research",
      "amber"
    ],
    "updated_at": 1616312236.0
  },
  {
    "data_format": 1,
    "description": "My personal snakemake pipeline for VAST-TOOLS execution",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AngryMaciek/snakemake_vast_tools",
    "linting": "KeyError in line 65 of /tmp/tmpnwu0e0lc/Snakefile:\n\u0027output_dir\u0027\n  File \"/tmp/tmpnwu0e0lc/Snakefile\", line 65, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "snakemake-pipeline",
      "rna-seq",
      "alternative-splicing"
    ],
    "updated_at": 1599235514.0
  },
  {
    "data_format": 1,
    "description": "Multiplex Accurate Sensitive Quantitation (MASQ) analysis and primer design",
    "formatting": "[ERROR] In file \"/tmp/tmp4g8awdke/Snakefile\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] In file \"/tmp/tmp4g8awdke/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "amoffitt/MASQ",
    "linting": "Lints for snakefile /tmp/tmp4g8awdke/Snakefile:\n    * Absolute path \"/r1.fastq.gz\" in line 143:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/r2.fastq.gz\" in line 144:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/r1.fastq.gz\" in line 176:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/r2.fastq.gz\" in line 177:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/extended_var_table.txt\" in line 178:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/extended_var_table.txt\" in line 223:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 65:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 178:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 223:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 110:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 425:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 426:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule convert_SNV_table (line 120, /tmp/tmp4g8awdke/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule barcode_split (line 145, /tmp/tmp4g8awdke/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fastqc (line 176, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule check_loci_extend_and_plot (line 207, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_data_by_tag_and_locus (line 239, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule all_base_report (line 290, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_within_tag_errors (line 337, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tag_count_graphs_per_region (line 363, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tag_count_graphs_combined (line 395, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rollup_tags (line 426, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_reports_alignment (line 462, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_reports_basecount (line 488, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_reports_per_base (line 513, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_reports_rollup (line 538, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_reports_readspertag (line 563, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_variant_bases_from_report (line 589, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_rollup_results (line 614, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule final_report (line 643, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bad_loci (line 671, /tmp/tmp4g8awdke/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule qc_plots (line 684, /tmp/tmp4g8awdke/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_base_report (line 708, /tmp/tmp4g8awdke/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1563039774.0
  },
  {
    "data_format": 1,
    "description": "The assignment for the course PP4RS 2019 ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lucaMoretti90/PP4RS_assignment_KG",
    "linting": "Lints for rule plot (line 28, /tmp/tmp557nlou2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule analysis (line 56, /tmp/tmp557nlou2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge (line 84, /tmp/tmp557nlou2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LOG_ALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean (line 154, /tmp/tmp557nlou2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1552322006.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpcz43mphn/Snakefile\":  \u0027TokenError: (\u0027EOF in multi-line statement\u0027, (109, 0))\u0027\n[INFO] In file \"/tmp/tmpcz43mphn/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "bjmorgan/data_argyrodite_disorder",
    "linting": "SyntaxError in line 83 of /tmp/tmpcz43mphn/Snakefile:\nEOF in multi-line statement (Snakefile, line 83)\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608221695.0
  },
  {
    "data_format": 1,
    "description": "build xaringan slides with snakemake",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lachlandeer/xaringan-snakemake",
    "linting": "Lints for rule build_html (line 16, /tmp/tmp95b4qui3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable run_r from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable log_all from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_pdf (line 43, /tmp/tmp95b4qui3/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable run_r from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable log_all from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean (line 82, /tmp/tmp95b4qui3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_install (line 10, /tmp/tmp95b4qui3/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule renv_consent (line 22, /tmp/tmp95b4qui3/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_init (line 34, /tmp/tmp95b4qui3/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_snap (line 46, /tmp/tmp95b4qui3/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_restore (line 58, /tmp/tmp95b4qui3/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "xaringan",
      "xaringan-slides",
      "snakemake",
      "snakemake-workflow"
    ],
    "updated_at": 1615736927.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 14 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vsoch/dna-seq-varlociraptor",
    "linting": "Lints for snakefile /tmp/tmpiqdadxh9/workflow/Snakefile:\n    * Deprecated singularity directive used for container definition in line 9.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpiqdadxh9/workflow/rules/annotation/pass2.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpiqdadxh9/workflow/rules/filtering.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_srr (line 1, /tmp/tmpiqdadxh9/workflow/rules/download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_genome (line 1, /tmp/tmpiqdadxh9/workflow/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genome_faidx (line 24, /tmp/tmpiqdadxh9/workflow/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_known_variants (line 68, /tmp/tmpiqdadxh9/workflow/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule remove_iupac_codes (line 95, /tmp/tmpiqdadxh9/workflow/rules/ref.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_index (line 147, /tmp/tmpiqdadxh9/workflow/rules/ref.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule merge_fastqs (line 67, /tmp/tmpiqdadxh9/workflow/rules/trimming.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_reads (line 1, /tmp/tmpiqdadxh9/workflow/rules/mapping.smk):\n    * Param index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule render_scenario (line 1, /tmp/tmpiqdadxh9/workflow/rules/calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcftools_concat (line 97, /tmp/tmpiqdadxh9/workflow/rules/calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule annotate_vcfs (line 20, /tmp/tmpiqdadxh9/workflow/rules/annotation/pass2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule annotate_dgidb (line 51, /tmp/tmpiqdadxh9/workflow/rules/annotation/pass2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter_by_annotation (line 1, /tmp/tmpiqdadxh9/workflow/rules/filtering.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule control_fdr (line 29, /tmp/tmpiqdadxh9/workflow/rules/filtering.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_calls (line 69, /tmp/tmpiqdadxh9/workflow/rules/filtering.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule igv_report (line 1, /tmp/tmpiqdadxh9/workflow/rules/report.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule vg2svg (line 1, /tmp/tmpiqdadxh9/workflow/rules/vega.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bcf_index (line 1, /tmp/tmpiqdadxh9/workflow/rules/utils.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bam_index (line 25, /tmp/tmpiqdadxh9/workflow/rules/utils.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule build_oncoprint_table (line 1, /tmp/tmpiqdadxh9/workflow/rules/oncoprint.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_oncoprint (line 24, /tmp/tmpiqdadxh9/workflow/rules/oncoprint.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1581227362.0
  },
  {
    "data_format": 1,
    "description": "Houses a snakemake workflow used for multiple sequence alignment and maximum likelihood phylogeny construction of genes or proteins. Originally used to study sex determination genes in mosquitoes.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "JamesTanShengYi/Snakemake-Alignment_Phylogeny",
    "linting": "Lints for rule GeneAlignmentClustal (line 1, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GeneAlignmentFasta (line 34, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GeneAlignmentTrimmed (line 67, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GeneTree (line 105, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ProteinAlignmentClustal (line 148, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ProteinAlignmentFasta (line 181, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ProteinAlignmentTrimmed (line 211, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ProteinTree (line 249, /tmp/tmptbkuhh5d/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "phylogenetic-trees",
      "alignment",
      "snakemake-workflows"
    ],
    "updated_at": 1607966955.0
  },
  {
    "data_format": 1,
    "description": "RADSex paper analyses workflow",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "RomainFeron/paper-sexdetermination-radsex",
    "linting": "Lints for snakefile /tmp/tmpd9r7jcti/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpd9r7jcti/rules/data.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpd9r7jcti/rules/default_workflow.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpd9r7jcti/rules/supplementary.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule medaka_update_popmap (line 99, /tmp/tmpd9r7jcti/rules/medaka.smk):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule medaka_chromosomes_file (line 346, /tmp/tmpd9r7jcti/rules/medaka.smk):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "workflow",
      "snakemake",
      "analyses"
    ],
    "updated_at": 1613771177.0
  },
  {
    "data_format": 1,
    "description": "An introduction to Snakemake with simple examples of central snakemake features",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "oskarvid/snakemake-intro",
    "linting": "Lints for rule BasicRule (line 18, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ParallelProcessingRule (line 37, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MultipleInputsToOneToolRule (line 56, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MoreThanOneFileAsInputRule (line 75, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule CondaInstallBCFToolsRule (line 95, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CondaShowBCFToolsVersionWithParamsRule (line 118, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CondaBCFToolsMultipleParamsRule (line 148, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MultipleThreadsRule (line 179, /tmp/tmpq1itut_e/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1604660348.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpgh5ehjmg/Snakefile\":  \u0027NoParametersError: L78: In input definition.\u0027\n[INFO] In file \"/tmp/tmpgh5ehjmg/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "deanpettinga/ecc_workflow",
    "linting": "Wildcard constraints in inputs are ignored. (rule: merge_tech_reps)\nWildcard constraints in inputs are ignored. (rule: merge_tech_reps)\nWildcard constraints in inputs are ignored. (rule: merge_tech_reps)\nLints for rule download_ref (line 83, /tmp/tmpgh5ehjmg/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore (line 168, /tmp/tmpgh5ehjmg/Snakefile):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule gunzip_reads (line 240, /tmp/tmpgh5ehjmg/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ecc_caller_align (line 314, /tmp/tmpgh5ehjmg/Snakefile):\n    * Param outname is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule ecc_caller_callEccRegions (line 380, /tmp/tmpgh5ehjmg/Snakefile):\n    * Param sample is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule assign_confidence (line 438, /tmp/tmpgh5ehjmg/Snakefile):\n    * Param sample is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule filter_by_conf (line 500, /tmp/tmpgh5ehjmg/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_tech_reps (line 548, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_bio_reps (line 588, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_flagstat (line 647, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prepare_chip_datasets (line 726, /tmp/tmpgh5ehjmg/Snakefile):\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule get_centromere_bw (line 983, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_TE_bw (line 1071, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_GC_bw (line 1149, /tmp/tmpgh5ehjmg/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_random_eccs (line 1379, /tmp/tmpgh5ehjmg/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614012450.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ImperialCollegeLondon/rcs-snakemake-tutorial",
    "linting": "Lints for rule bwa_map (line 15, /tmp/tmpdvcydprp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort (line 36, /tmp/tmpdvcydprp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_index (line 58, /tmp/tmpdvcydprp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bcftools_call (line 78, /tmp/tmpdvcydprp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_quals (line 102, /tmp/tmpdvcydprp/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1596555560.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 15 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "whtns/dna-seq-varlociraptor",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1596732037.0
  },
  {
    "data_format": 1,
    "description": "Workflow to simulate an RNA-seq reads and perform DE-analysis",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "rstats-tartu/simulate-rnaseq",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmp3volloql/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmp3volloql/Snakefile\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "rna-seq-pipeline",
      "simulation",
      "snakemake-workflow",
      "deseq2",
      "polyester"
    ],
    "updated_at": 1611645822.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "qbicsoftware-archive/rnaseq",
    "linting": "FileNotFoundError in line 9 of /tmp/tmpg8gc2uf5/Snakefile:\n[Errno 2] No such file or directory: \u0027config.json\u0027\n  File \"/tmp/tmpg8gc2uf5/Snakefile\", line 9, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 8,
    "topics": [
      "workflow",
      "ngs",
      "rna"
    ],
    "updated_at": 1600945244.0
  },
  {
    "data_format": 1,
    "description": "Detect replication slippage variants at microsatellite regions",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tdayris-perso/bam-msisensor",
    "linting": "FileNotFoundError in line 21 of /tmp/tmpo0jtku78/rules/common.smk:\n[Errno 2] No such file or directory: \u0027design.tsv\u0027\n  File \"/tmp/tmpo0jtku78/Snakefile\", line 12, in \u003cmodule\u003e\n  File \"/tmp/tmpo0jtku78/rules/common.smk\", line 21, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [
      "bioinformatics-pipeline",
      "snakemake-workflows",
      "reproducible-workflows",
      "microsatellite-instability"
    ],
    "updated_at": 1586510173.0
  },
  {
    "data_format": 1,
    "description": "Scripts associated with our study, \"Genomic and transcriptomic characterization of relapsed small cell lung cancer through rapid research autopsy\"",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "OSU-SRLab/SCLC-autopsy-genomics",
    "linting": "PermissionError in line 16 of /tmp/tmpyhc_ei4j/Snakefile:\n[Errno 13] Permission denied: \u0027/parh\u0027\n  File \"/tmp/tmpyhc_ei4j/Snakefile\", line 16, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 0,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1614368088.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "LucilleWerner/owe11_workflows",
    "linting": "Lints for rule get_data_from_ncbi (line 11, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_data_from_uniprot (line 35, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_data_from_eggnog (line 59, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_data_from_oma (line 83, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mine_text (line 107, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_gc_plot (line 131, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_report (line 155, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cleanup (line 180, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule surprise (line 195, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dag (line 210, /tmp/tmpb8jusuqi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1528846840.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpf3im9zwb/Snakefile\":  \u0027SyntaxError: L69: Unrecognised keyword \u0027Ensembl\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmpf3im9zwb/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "RnRoger/API_course_11",
    "linting": "ModuleNotFoundError in line 1 of /tmp/tmpf3im9zwb/Snakefile:\nNo module named \u0027pytools\u0027\n  File \"/tmp/tmpf3im9zwb/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1597860000.0
  },
  {
    "data_format": 1,
    "description": "Snakemake tutorial ready to be used",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mstamenk/snakemake-tutorial",
    "linting": "TypeError in line 11 of /tmp/tmpxgbvd7iy/Snakefile:\nunsupported operand type(s) for +: \u0027NoneType\u0027 and \u0027str\u0027\n  File \"/tmp/tmpxgbvd7iy/Snakefile\", line 11, in \u003cmodule\u003e\n  File \"/tmp/tmpxgbvd7iy/routines/Analysis.py\", line 18, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1500753613.0
  },
  {
    "data_format": 1,
    "description": "Demonstrating an RNA-seq workflow in Snakemake.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cjsifuen/Basic-Bulk-RNA-seq-Workflow",
    "linting": "Lints for snakefile /tmp/tmpcp1gjon4/workflow/Snakefile:\n    * Absolute path \"/^#/d\" in line 189:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 101:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 127:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 134:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule fastqc (line 82, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param qc_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule trim_galore (line 131, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule star (line 180, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule featureCounts (line 249, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean_counts (line 312, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule deseq2 (line 342, /tmp/tmpcp1gjon4/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618581958.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpp02qg31h/rules/2.1.BQSR.smk\":  \u0027InvalidParameterSyntax: L41:Invalid key REFERENCE\n recal\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rimjhimroy/snakeGATK4",
    "linting": "FileNotFoundError in line 15 of /tmp/tmpp02qg31h/Snakefile:\n[Errno 2] No such file or directory: \u0027data/genome/biscutV2.1_new.fa\u0027\n  File \"/tmp/tmpp02qg31h/Snakefile\", line 15, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607589582.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 14 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zhuakexi/scDrop-Seq",
    "linting": "FileNotFoundError in line 20 of /tmp/tmp3wjweskn/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp3wjweskn/Snakefile\", line 20, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1601219664.0
  },
  {
    "data_format": 1,
    "description": "run dropseq - slight modifications from original dropSeqPipe",
    "formatting": "[ERROR] In file \"/tmp/tmpm3wfkq7m/rules/map.smk\":  \u0027SyntaxError: L227: Unrecognised keyword \u0027STAR_output\u0027 in rule definition\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 13 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dylkot/dropSeqPipe-dak",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpm3wfkq7m/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/runDropseqPipeGCP/dropSeqPipe-dak/templates/gtf_biotypes.yaml\u0027\n  File \"/tmp/tmpm3wfkq7m/Snakefile\", line 18, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1606611634.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow that reanalyzes proteomics data from a PRIDE Archive project to create a spectral library",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "RalfG/workflow_pxd_to_speclib",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpdjw3jr6k/get_data.smk:\nNo module named \u0027wget\u0027\n  File \"/tmp/tmpdjw3jr6k/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpdjw3jr6k/get_data.smk\", line 4, in \u003cmodule\u003e\n  File \"/tmp/tmpdjw3jr6k/scripts/download_pride_project.py\", line 16, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [
      "proteomics",
      "msgfplus",
      "percolator",
      "thermorawfileparser",
      "snakemake",
      "pride-database",
      "mass-spectrometry"
    ],
    "updated_at": 1583943853.0
  },
  {
    "data_format": 1,
    "description": "Workflow for Burrows wheeler alignment on CRAM input files using snakemake  and Healthrecord system analytics",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Umar-Ayub/Human-Genome-Analysis-Pipeline",
    "linting": "Lints for rule bwa_mem (line 11, /tmp/tmpqlvg1t2r/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1544046285.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "hobrien/GENEX-FB1",
    "linting": "Lints for snakefile /tmp/tmporwc3v1w/Snakefile:\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 105:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule format_bed (line 18, /tmp/tmporwc3v1w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule transcript_seqs (line 37, /tmp/tmporwc3v1w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_index (line 59, /tmp/tmporwc3v1w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_kalliso (line 82, /tmp/tmporwc3v1w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule estimate_counts (line 112, /tmp/tmporwc3v1w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule join_factors (line 181, /tmp/tmporwc3v1w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule format_cov (line 201, /tmp/tmporwc3v1w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule gene_level (line 221, /tmp/tmporwc3v1w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule transcript_level (line 255, /tmp/tmporwc3v1w/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1531409849.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for Salmon quantification and FastQC quality controls",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tdayris-perso/rna-count-salmon",
    "linting": "FileNotFoundError in line 33 of /tmp/tmpsp14r_nf/rules/common.smk:\n[Errno 2] No such file or directory: \u0027tests/design.tsv\u0027\n  File \"/tmp/tmpsp14r_nf/Snakefile\", line 11, in \u003cmodule\u003e\n  File \"/tmp/tmpsp14r_nf/rules/common.smk\", line 33, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [
      "snakemake-workflows",
      "bioinformatics-pipeline",
      "reproducible-workflows",
      "rnaseq-pipeline",
      "fastqc",
      "salmon",
      "multiqc"
    ],
    "updated_at": 1613463894.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Adrian-Cantu/frap_maker",
    "linting": null,
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619211705.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "z-sector/stepik_snakemake",
    "linting": "Lints for rule copy (line 15, /tmp/tmp8mufigvh/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1560418918.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SmitsG/Workflows-bi11",
    "linting": "ModuleNotFoundError in line 5 of /tmp/tmpyys0yke8/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpyys0yke8/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1531178141.0
  },
  {
    "data_format": 1,
    "description": " ",
    "formatting": "[ERROR] In file \"/tmp/tmp8_o7kah1/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:0:     \"cutadapt/tophat/htseq/map_count.txt\"\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmp8_o7kah1/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "ORU-BIOI/rna-seq-workflow",
    "linting": "WorkflowError in line 93 of /tmp/tmp8_o7kah1/Snakefile:\nDuplicate output file pattern in rule qc. First two duplicate for entries qcout and zipout\n  File \"/tmp/tmp8_o7kah1/Snakefile\", line 93, in \u003cmodule\u003e\n",
    "stargazers_count": 0,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1433000612.0
  },
  {
    "data_format": 1,
    "description": "The LCPsCluster module for the on-rep-seq",
    "formatting": "[ERROR] In file \"/tmp/tmpzcvz9cj3/Snakefile\":  \u0027SyntaxError: L32: Unrecognised keyword \u0027work_directory\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmpzcvz9cj3/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "yanhui09/LCPsCluster",
    "linting": "Lints for snakefile /tmp/tmpzcvz9cj3/Snakefile:\n    * Absolute path \"/{barcode}.txt\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.html\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.ipynb\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{barcode}.txt\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.ipynb\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.html\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmap_large.png\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmap.png\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_flowgrams_clustering_order.pdf\" in line 27:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCPsClusteringData\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/r_saved_images\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*.txt \" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*.txt \" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 24:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 12.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule LCPsCluster (line 26, /tmp/tmpzcvz9cj3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param work_directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 0,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615980524.0
  },
  {
    "data_format": 1,
    "description": "Integrated workflow for SV calling from single-cell Strand-seq data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "friendsofstrandseq/pipeline",
    "linting": "IndexError in line 362 of /tmp/tmpzjwbiwrv/Snakefile:\nlist index out of range\n  File \"/tmp/tmpzjwbiwrv/Snakefile\", line 362, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 8,
    "topics": [],
    "updated_at": 1607095752.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "fbartusch/snakemake_tutorial",
    "linting": "PermissionError in line 2 of /tmp/tmp5iygb5ke/Snakefile:\n[Errno 13] Permission denied: \u0027/scif\u0027\n  File \"/tmp/tmp5iygb5ke/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1555340554.0
  },
  {
    "data_format": 1,
    "description": "Creating a variant calling pipeline ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "LandiMi2/Variant_Calling_Project-",
    "linting": "Lints for rule sort_bam (line 157, /tmp/tmpi0h0s3n_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bam_index (line 176, /tmp/tmpi0h0s3n_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule removal_deduplication (line 197, /tmp/tmpi0h0s3n_/Snakefile):\n    * Param sam_index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule ref_samtools_index (line 261, /tmp/tmpi0h0s3n_/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule Create_Ref_Dictonary (line 283, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule Index_Feature_File (line 312, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule AddOrReplaceReadGroups (line 336, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule BaseRecalibrator (line 362, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ApplyBQSR (line 391, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule HaplotypeCaller (line 415, /tmp/tmpi0h0s3n_/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1599728428.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow for hisat2 and htseq-count counts generation for use with deseq2 etc",
    "formatting": "[ERROR] In file \"/tmp/tmpp2pdpx4i/Snakefile\":  \u0027NoParametersError: L119: In shell definition.\u0027\n[INFO] In file \"/tmp/tmpp2pdpx4i/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "dwheelerau/snakemake-rnaseq-counts",
    "linting": "SyntaxError in line 109 of /tmp/tmpp2pdpx4i/Snakefile:\nCommand must be given as string after the shell keyword. (Snakefile, line 109)\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1566427487.0
  },
  {
    "data_format": 1,
    "description": "A workflow for some simple Mass spectrometry QC.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "KeepCalmAndRushB/QCSlowPipeline",
    "linting": "Lints for rule copy_raw_file (line 26, /tmp/tmp7cd7y9dw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prepare_max_quant_analysis (line 42, /tmp/tmp7cd7y9dw/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param adir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule run_max_quant_analysis (line 67, /tmp/tmp7cd7y9dw/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable maxquant_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_qc_metrics (line 88, /tmp/tmp7cd7y9dw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule add_metrics_to_database (line 112, /tmp/tmp7cd7y9dw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1555575202.0
  },
  {
    "data_format": 1,
    "description": "AMBIC Epigenomics project analysis",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "timplab/ambic-epigenome",
    "linting": "PermissionError in line 7 of /tmp/tmpce203yfu/Snakefile:\n[Errno 13] Permission denied: \u0027/home/[user]\u0027\n  File \"/tmp/tmpce203yfu/Snakefile\", line 7, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1603387944.0
  },
  {
    "data_format": 1,
    "description": "snakemake files of the tuxedo v2 pipeline from Pertea et al 2016",
    "formatting": "[WARNING] In file \"/tmp/tmpm20ppa4r/Snakefile\":  Keyword \"input\" at line 70 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/smsk_tuxedo2",
    "linting": "/tmp/tmpm20ppa4r/Snakefile:12: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  params = yaml.load(open(\"params.yml\", \"r\"))\n/tmp/tmpm20ppa4r/Snakefile:13: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.\n  features = yaml.load(open(\"features.yml\", \"r\"))\nAttributeError in line 21 of /tmp/tmpm20ppa4r/src/snakefiles/raw.py:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmpm20ppa4r/Snakefile\", line 26, in \u003cmodule\u003e\n  File \"/tmp/tmpm20ppa4r/src/snakefiles/raw.py\", line 21, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "tuxedo2",
      "samtools",
      "hisat2",
      "stringtie",
      "ballgown",
      "rskittlebrewer",
      "r",
      "smsk",
      "differential",
      "expression",
      "rnaseq",
      "rna-seq",
      "conda",
      "conda-environment"
    ],
    "updated_at": 1557312196.0
  },
  {
    "data_format": 1,
    "description": "Generalized Snakemake workflow for R.4Cker",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "MediciPrime/4C-Workflow",
    "linting": "Lints for snakefile /tmp/tmphk6b27a0/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule reduced_genome (line 40, /tmp/tmphk6b27a0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bowtie2_build (line 72, /tmp/tmphk6b27a0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bowtie2 (line 109, /tmp/tmphk6b27a0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bedGraph_Counts (line 154, /tmp/tmphk6b27a0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule purify_helper (line 188, /tmp/tmphk6b27a0/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule purify_aligned_bedGraph (line 212, /tmp/tmphk6b27a0/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule R_script (line 243, /tmp/tmphk6b27a0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1477424546.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake pipeline for RRBS data analysis",
    "formatting": "[ERROR] In file \"/tmp/tmphp9eamtt/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:10: benchmark:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmphp9eamtt/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "ftabaro/MethylSnake",
    "linting": "KeyError in line 3 of /tmp/tmphp9eamtt/Snakefile:\n\u0027singularity_container\u0027\n  File \"/tmp/tmphp9eamtt/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "bioinformatics-pipeline",
      "methylation",
      "rrbs-pipeline",
      "snakemake",
      "rrbs-data-analysis"
    ],
    "updated_at": 1613445122.0
  },
  {
    "data_format": 1,
    "description": "Example workflow using snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "panoptes-organization/snakemake_example_workflow",
    "linting": "Lints for rule merge_results (line 129, /tmp/tmpizxxvkmi/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1603918413.0
  },
  {
    "data_format": 1,
    "description": "A basic boilerplate structure for snakemake based pipelines.",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "codingene/snakemake-base",
    "linting": "Lints for rule fastqc (line 1, /tmp/tmph4y_r4oe/rules/pre-align-qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [
      "snakemake",
      "conda",
      "codingene"
    ],
    "updated_at": 1592542294.0
  },
  {
    "data_format": 1,
    "description": "snakemake pipeline to generate art_illumina profiles from public datasets",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "hzi-bifo/art_illumina_profiles",
    "linting": "Lints for snakefile /tmp/tmpmmm1h1vn/Snakefile:\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule samtools_bam2fq_separate (line 72, /tmp/tmpmmm1h1vn/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 1,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1589987504.0
  },
  {
    "data_format": 1,
    "description": "Finding virus integration sites from whole genome nanopore sequence",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kpalin/VirusIntegrationFinder",
    "linting": "FileNotFoundError in line 12 of /tmp/tmp7jbbfdrx/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027/mnt/cgnano/projects/promethion/190612/ont_pipe_Yin/lenti_cas9_find/possible insert.txt\u0027\n  File \"/tmp/tmp7jbbfdrx/workflow/Snakefile\", line 10, in \u003cmodule\u003e\n  File \"/tmp/tmp7jbbfdrx/workflow/rules/common.smk\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617584807.0
  },
  {
    "data_format": 1,
    "description": "Kallisto Bus for 10X",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_kallisto_10x",
    "linting": "Lints for snakefile /tmp/tmpfx1qvgks/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpfx1qvgks/workflow/rules/kallisto.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592919803.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "limbo1996/GATK-snakemake-pipeline",
    "linting": "SyntaxError in line 16 of /tmp/tmpquf88ol6/Snakefile:\ninvalid syntax\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1598956758.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for the OrthoEvol python package.",
    "formatting": "[ERROR] In file \"/tmp/tmpx112p856/Snakefile\":  \u0027NoParametersError: L8: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "datasnakes/orthoevol-snakemake",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmpx112p856/rules/setup.smk:\nNo module named \u0027OrthoEvol\u0027\n  File \"/tmp/tmpx112p856/Snakefile\", line 8, in \u003cmodule\u003e\n  File \"/tmp/tmpx112p856/rules/setup.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "snakemake-workflow",
      "snakemake-pipelines",
      "orthology-inference",
      "orthoevol"
    ],
    "updated_at": 1562624607.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for the chain of models approach to hydrologic modeling",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jhamman/chains",
    "linting": "ModuleNotFoundError in line 11 of /tmp/tmp84n34lwj/Snakefile:\nNo module named \u0027xarray\u0027\n  File \"/tmp/tmp84n34lwj/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1568320138.0
  },
  {
    "data_format": 1,
    "description": "Snakemake Workflow For WGBS Data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vari-bbc/WGBS_Biscuit_Snakemake",
    "linting": "Lints for snakefile /tmp/tmpbb033gh3/Snakefile:\n    * Absolute path \"/secondary/projects/bbc/research/shared_snakemake/rules/fastq_qc/fastq_lengths\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule rename_fastq (line 73, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore (line 105, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule biscuit_align (line 177, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param unmapped is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule biscuit_pileup (line 283, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biscuit_mergecg (line 358, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biscuit_qc (line 417, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiQC (line 505, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule control_vectors (line 576, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1614964766.0
  },
  {
    "data_format": 1,
    "description": "A simple Snakemake workflow to perform network analyses using the Python Geneset Network Analysis (PyGNA) package.",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "stracquadaniolab/workflow-pygna",
    "linting": "Lints for snakefile /tmp/tmp9qhs5mae/Snakefile:\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp9qhs5mae/rules/pygna.smk:\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 102:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 134:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 181:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule generate_matrix_sp (line 6, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_matrix_rwr (line 28, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_module (line 50, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_internal_degree (line 79, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_total_degree (line 107, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_sp (line 135, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_rwr (line 164, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule association_RW (line 193, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule association_SP (line 224, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule comparison_RW (line 255, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule comparison_SP (line 286, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule within_comparison_RW (line 318, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule within_comparison_SP (line 348, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_topology_module (line 7, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_internal_degree (line 26, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_total_degree (line 45, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_sp (line 64, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_rwr (line 83, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_within_comparison_rwr (line 102, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_within_comparison_sp (line 121, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_association_rwr (line 140, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_association_sp (line 159, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593442526.0
  },
  {
    "data_format": 1,
    "description": "snakemake snp calling pipeline based on freebayes",
    "formatting": "[WARNING] In file \"/tmp/tmp5o5_uc9h/Snakefile\":  Keyword \"input\" at line 30 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dwheelerau/snakemake-snp",
    "linting": "SyntaxError:\nNot all output, log and benchmark files of rule sample_qc contain the same wildcards. This is crucial though, in order to avoid that two or more jobs write to the same file.\n  File \"/tmp/tmp5o5_uc9h/Snakefile\", line 97, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1563507584.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmp4o39eycy/rules/download.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:16: download_genome:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "SilasK/FastDrep",
    "linting": "KeyError in line 9 of /tmp/tmp4o39eycy/compare.smk:\n\u0027genome_folder\u0027\n  File \"/tmp/tmp4o39eycy/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmp4o39eycy/compare.smk\", line 9, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616758915.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpmiwsrwpn/Snakefile\":  Keyword \"input\" at line 9 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ctmrbio/mgi-ncov19-snakemake",
    "linting": "Lints for rule coverage_bwa_filter (line 415, /tmp/tmpmiwsrwpn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule coverage_samtools (line 461, /tmp/tmpmiwsrwpn/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1594914099.0
  },
  {
    "data_format": 1,
    "description": "Snakemake Workflow For WGBS Data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vari-bbc/WGBS_Biscuit_Snakemake",
    "linting": "Lints for snakefile /tmp/tmpbb033gh3/Snakefile:\n    * Absolute path \"/secondary/projects/bbc/research/shared_snakemake/rules/fastq_qc/fastq_lengths\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule rename_fastq (line 73, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore (line 105, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule biscuit_align (line 177, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param unmapped is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule biscuit_pileup (line 283, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biscuit_mergecg (line 358, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule biscuit_qc (line 417, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiQC (line 505, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule control_vectors (line 576, /tmp/tmpbb033gh3/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1614964766.0
  },
  {
    "data_format": 1,
    "description": "A cookiecutter template for a snakemake-workflow readme file",
    "formatting": "[ERROR] In file \"/tmp/tmpmgk1o7b0/workflow/Snakefile\":  \u0027NoParametersError: L12: In input definition.\u0027\n[INFO] In file \"/tmp/tmpmgk1o7b0/workflow/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpmgk1o7b0/workflow/Snakefile\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "snakemake-workflows/snakemake-workflow-template",
    "linting": null,
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1618822754.0
  },
  {
    "data_format": 1,
    "description": "A simple Snakemake workflow to perform network analyses using the Python Geneset Network Analysis (PyGNA) package.",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "stracquadaniolab/workflow-pygna",
    "linting": "Lints for snakefile /tmp/tmp9qhs5mae/Snakefile:\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp9qhs5mae/rules/pygna.smk:\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 87:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 102:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 134:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 181:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule generate_matrix_sp (line 6, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_matrix_rwr (line 28, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_module (line 50, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_internal_degree (line 79, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_total_degree (line 107, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_sp (line 135, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule topology_rwr (line 164, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule association_RW (line 193, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule association_SP (line 224, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule comparison_RW (line 255, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule comparison_SP (line 286, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule within_comparison_RW (line 318, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule within_comparison_SP (line 348, /tmp/tmp9qhs5mae/rules/pygna.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_topology_module (line 7, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_internal_degree (line 26, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_total_degree (line 45, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_sp (line 64, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_topology_rwr (line 83, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_within_comparison_rwr (line 102, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_within_comparison_sp (line 121, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_association_rwr (line 140, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_association_sp (line 159, /tmp/tmp9qhs5mae/rules/pygna_visualise.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1593442526.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for paired-end sanger sequences of 16s rRNA",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "matrs/16s-rRNA-Sanger",
    "linting": "Wildcard constraints in inputs are ignored.\nLints for rule abi_to_fastq (line 2, /tmp/tmpl5vau7rp/rules/abi_to_fastq.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_pre (line 1, /tmp/tmpl5vau7rp/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule qual_trim (line 21, /tmp/tmpl5vau7rp/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc_post (line 41, /tmp/tmpl5vau7rp/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merger_quality (line 89, /tmp/tmpl5vau7rp/rules/qc.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule reverse_complement (line 1, /tmp/tmpl5vau7rp/rules/merge.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_fastq (line 20, /tmp/tmpl5vau7rp/rules/merge.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule blastn (line 1, /tmp/tmpl5vau7rp/rules/blast.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule blast_top (line 35, /tmp/tmpl5vau7rp/rules/blast.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule seqmatch (line 1, /tmp/tmpl5vau7rp/rules/rdp.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule classifier (line 28, /tmp/tmpl5vau7rp/rules/rdp.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genus_list (line 53, /tmp/tmpl5vau7rp/rules/rdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ncbi_tree (line 72, /tmp/tmpl5vau7rp/rules/rdp.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1600235906.0
  },
  {
    "data_format": 1,
    "description": "Arquivos e c\u00f3digos em bash, R e Python para confec\u00e7\u00e3o da minha tese de doutorado.",
    "formatting": "[ERROR] In file \"/tmp/tmp_q_yi89v/Snakefile\":  \u0027SyntaxError: L24: Unrecognised keyword \u0027expand\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmp_q_yi89v/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "cmcouto-silva/tese",
    "linting": "Lints for rule setup_env (line 64, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rehh (line 80, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pbs (line 106, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule results_bin (line 132, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule enrichr (line 160, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_eQTL (line 185, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule webgestalt (line 211, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule goatools (line 238, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule metanalysis (line 264, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule pheno_annot (line 291, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summarise_all (line 326, /tmp/tmp_q_yi89v/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615839331.0
  },
  {
    "data_format": 1,
    "description": "A computational method to generate causal explanations for proteomic profiles using prior mechanistic knowledge in the literature, as recorded in cellular pathway maps. ",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ohsu-cedar-comp-hub/causal-path-pipeline",
    "linting": "FileNotFoundError in line 16 of /tmp/tmpbr8mjanw/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/groups/CEDAR/estabroj/experiments/causal-path-pipeline/data/JE_modified_meta.txt\u0027\n  File \"/tmp/tmpbr8mjanw/Snakefile\", line 16, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1613583007.0
  },
  {
    "data_format": 1,
    "description": "Pipeline running \u0027salmon alevin\u0027 for CITE-seq data",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "kevinrue/snakemake_alevin_citeseq_tutorial",
    "linting": "Lints for snakefile /tmp/tmp0ueyridj/workflow/Snakefile:\n    * Absolute path \"/alevin/quants_mat.gz\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/alevin/quants_mat.gz\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/alevin/quants_mat.gz\" in line 87:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule alevin_rna (line 25, /tmp/tmp0ueyridj/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule alevin_adt (line 84, /tmp/tmp0ueyridj/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule alevin_hto (line 142, /tmp/tmp0ueyridj/workflow/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592323315.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sraorao/snakemake_code_clinic",
    "linting": "Lints for snakefile /tmp/tmp5f5mdezq/Snakefile:\n    * Absolute path \"/home/srao/PycharmProjects/snakemake_cc\" in line 2:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule count_words (line 6, /tmp/tmp5f5mdezq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_reduplicated_words (line 27, /tmp/tmp5f5mdezq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_lines (line 62, /tmp/tmp5f5mdezq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_counts (line 98, /tmp/tmp5f5mdezq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_data_files (line 119, /tmp/tmp5f5mdezq/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1606139625.0
  },
  {
    "data_format": 1,
    "description": "Template for running mikRopML with Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SchlossLab/mikropml-snakemake-workflow",
    "linting": "Lints for rule preprocess_data (line 23, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_ml (line 59, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_results (line 100, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_hp_performance (line 128, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine_benchmarks (line 156, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_performance (line 180, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_hp_performance (line 204, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_benchmarks (line 228, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_report (line 252, /tmp/tmpmum_6zlj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 286, /tmp/tmpmum_6zlj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 3,
    "topics": [
      "snakemake",
      "machine-learning",
      "rstats"
    ],
    "updated_at": 1617911521.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to process and analyse Bisulfite-Sequencing data",
    "formatting": "[ERROR] In file \"/tmp/tmpyj2pgmc2/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     \"2_bismark_mapped/{sample}-{unit}.1_bismark_bt2_PE_report.txt\"\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpyj2pgmc2/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "davidries84/bismark_snakemake",
    "linting": "Lints for snakefile /tmp/tmpyj2pgmc2/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 146:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 147:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 161:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 177:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 178:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 377:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 423:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 428:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 191:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 428:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule fastqc_trimmed (line 150, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_genome_preparation (line 180, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BISMARK_GENOME_PREPARATION from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule samtools_index (line 214, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_alignment (line 243, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BISMARK from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule deduplicate_bismark (line 293, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DEDUPLICATE_BISMARK from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bam2nuc (line 329, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BAM2NUC from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bismark_methylation_extractor (line 357, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BISMARK_METHYLATION_EXTRACTOR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule genomewidecytosinemethylationreport (line 400, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable COVERAGE2CYTOSINE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule summarizedinucleotides (line 427, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable COVERAGE2CYTOSINE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule cContextCovFileWithStrandInfoFromGWCMR (line 476, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cytosinecoveragetiling (line 527, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BISMARKCOV2TILES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule bgzipgwcmr (line 554, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule csigwcmr (line 581, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ViewBS_samplefile (line 608, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CREATEVIEWBSFILELIST from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ViewBS_MethCoverage (line 634, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VIEWBS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ViewBS_GlobalMethLev (line 665, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VIEWBS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ViewBS_MethLevDist (line 694, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VIEWBS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ViewBS_MethGeno (line 727, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VIEWBS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ViewBS_BisNonConvRate (line 761, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VIEWBS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule multiqcSummaryReport (line 821, /tmp/tmpyj2pgmc2/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619158166.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp5gkuirnr/rules/batchRm.smk\":  Keyword \"output\" at line 14 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/batchRm.smk\":  Keyword \"input\" at line 38 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/batchRm.smk\":  Keyword \"output\" at line 44 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/batchRm.smk\":  Keyword \"input\" at line 151 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/normalization.smk\":  Keyword \"input\" at line 50 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/dimensionReduct.smk\":  Keyword \"params\" at line 76 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/qc.smk\":  Keyword \"params\" at line 39 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/qc.smk\":  Keyword \"input\" at line 101 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/clustering.smk\":  Keyword \"input\" at line 130 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/evalClustStab.smk\":  Keyword \"input\" at line 112 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/evalClustStab.smk\":  Keyword \"output\" at line 120 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5gkuirnr/rules/evalClustStab.smk\":  Keyword \"input\" at line 129 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 16 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lybird300/single_cell_rna_seq_snakemake",
    "linting": "FileNotFoundError in line 9 of /tmp/tmp5gkuirnr/Snakefile:\n[Errno 2] No such file or directory: \u0027config/samples_GEJ_IM.txt\u0027\n  File \"/tmp/tmp5gkuirnr/Snakefile\", line 9, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618674070.0
  },
  {
    "data_format": 1,
    "description": "Extended genecatalog workflow for metagenome-atlas",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "metagenome-atlas/genecatalog_atlas",
    "linting": "KeyError in line 6 of /tmp/tmp7v7fc2bv/rules/clustering.smk:\n\u0027input_faa\u0027\n  File \"/tmp/tmp7v7fc2bv/Snakefile\", line 35, in \u003cmodule\u003e\n  File \"/tmp/tmp7v7fc2bv/rules/clustering.smk\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1592410796.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflows for processing TCGA RNA-Seq data for the CAMDA2019 challenge",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "skurscheid/camda2019-workflows",
    "linting": "FileNotFoundError in line 28 of /tmp/tmptxvf0gtr/Snakefile:\n[Errno 2] No such file or directory: \u0027kirc_ids.tsv\u0027\n  File \"/tmp/tmptxvf0gtr/Snakefile\", line 28, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611071862.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "metagenome-atlas/plasmid_atlas",
    "linting": "FileNotFoundError in line 21 of /tmp/tmpugfv7kab/rules/sample_table.smk:\n[Errno 2] No such file or directory: \u0027samples.tsv\u0027\n  File \"/tmp/tmpugfv7kab/Snakefile\", line 14, in \u003cmodule\u003e\n  File \"/tmp/tmpugfv7kab/rules/spades.smk\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpugfv7kab/rules/sample_table.smk\", line 26, in \u003cmodule\u003e\n  File \"/tmp/tmpugfv7kab/rules/sample_table.smk\", line 21, in load_sample_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1588839730.0
  },
  {
    "data_format": 1,
    "description": "atlas extension for phage discovery",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "metagenome-atlas/virome_atlas",
    "linting": "KeyError in line 8 of /tmp/tmp76u9oyju/rules/vibrant.smk:\n\u0027database_dir\u0027\n  File \"/tmp/tmp76u9oyju/Snakefile\", line 3, in \u003cmodule\u003e\n  File \"/tmp/tmp76u9oyju/rules/vibrant.smk\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1588054541.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for Illumina RNA-sequencing experiments, including variant calling and more - relating to pop gen and insecticide resistance",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "sanjaynagi/rna-seq-ir",
    "linting": "FileNotFoundError in line 11 of /tmp/tmpdkrdyd8m/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config/config.yaml\u0027\n  File \"/tmp/tmpdkrdyd8m/workflow/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflow",
      "rna-seq",
      "variant-calling",
      "differential-expression",
      "population-genomics",
      "snakemake",
      "transcriptomics"
    ],
    "updated_at": 1617983776.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "steveped/snakemake_rnaseq_se",
    "linting": "Lints for snakefile /tmp/tmpl6p2bvh0/Snakefile:\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpl6p2bvh0/rules/trimming.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpl6p2bvh0/rules/qc.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpl6p2bvh0/rules/staralign.smk:\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule make_rulegraph (line 1, /tmp/tmpl6p2bvh0/rules/rulegraph.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index_bam (line 37, /tmp/tmpl6p2bvh0/rules/staralign.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule count (line 1, /tmp/tmpl6p2bvh0/rules/count.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_rproj (line 26, /tmp/tmpl6p2bvh0/rules/workflowr.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1610515548.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow to perform Alternative Splicing analysis from RNA-Seq with SUPPA2.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AngryMaciek/snakemake_suppa2",
    "linting": "KeyError in line 31 of /tmp/tmppm3hdrjx/Snakefile:\n\u0027result_dir\u0027\n  File \"/tmp/tmppm3hdrjx/Snakefile\", line 31, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "snakemake-pipeline",
      "rna-seq",
      "alternative-splicing"
    ],
    "updated_at": 1593795916.0
  },
  {
    "data_format": 1,
    "description": "BIDS app to facilitate determining which set of experimental confounds to include in a standard task-based fMRI GLM analysis.",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "switt4/confounder-snakemake",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmp6gtjnxul/Snakefile:\nNo module named \u0027bids\u0027\n  File \"/tmp/tmp6gtjnxul/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597222665.0
  },
  {
    "data_format": 1,
    "description": "High throughput Next Generation Sequencing (NGS) data analysis using Python 3 Snakemake",
    "formatting": "[ERROR] In file \"/tmp/tmpzwrbjbf9/Snakefile\":  \u0027InvalidParameterSyntax: 87\"~/ miniconda2/ bin/ featureCounts- a genome/ gencode. v32. annotation. gtf- o{ output. counts}{ input. bam}- T{ threads}\n\n\u0027\n[INFO] In file \"/tmp/tmpzwrbjbf9/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "ACSoupir/SnakeMake",
    "linting": "SyntaxError in line 88 of /tmp/tmpzwrbjbf9/Snakefile:\nEOL while scanning string literal\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608092170.0
  },
  {
    "data_format": 1,
    "description": "Having fun plotting my activity data from Strava!",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kelly-sovacool/strava",
    "linting": "Lints for rule download (line 28, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule process (line 47, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_activities (line 68, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_summary (line 97, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_report (line 118, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cat_figures_readme (line 140, /tmp/tmp51a2t2co/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619372461.0
  },
  {
    "data_format": 1,
    "description": "WORK IN PROGRESS!",
    "formatting": "[ERROR] In file \"/tmp/tmpjyrv19rt/rules/analysis.smk\":  \u0027NoParametersError: L25: In log definition.\u0027\n[INFO] In file \"/tmp/tmpjyrv19rt/rules/analysis.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpjyrv19rt/rules/analysis.smk\":  8 file(s) would be changed \ud83d\ude2c\n[INFO] In file \"/tmp/tmpjyrv19rt/rules/analysis.smk\":  1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "lachlandeer/snakemake-econ-python",
    "linting": "Lints for snakefile /tmp/tmpjyrv19rt/Snakefile:\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpjyrv19rt/rules/paper.smk:\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpjyrv19rt/rules/slides.smk:\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule clean_all (line 45, /tmp/tmpjyrv19rt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_main (line 60, /tmp/tmpjyrv19rt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gen_regression_vars (line 7, /tmp/tmpjyrv19rt/rules/data_mgt.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rename_vars (line 35, /tmp/tmpjyrv19rt/rules/data_mgt.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ols_model (line 21, /tmp/tmpjyrv19rt/rules/analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule figures (line 19, /tmp/tmpjyrv19rt/rules/figures.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule paper2root (line 21, /tmp/tmpjyrv19rt/rules/paper.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_pdf (line 41, /tmp/tmpjyrv19rt/rules/paper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule slides2root (line 8, /tmp/tmpjyrv19rt/rules/slides.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_slides (line 27, /tmp/tmpjyrv19rt/rules/slides.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pdf_clean (line 68, /tmp/tmpjyrv19rt/rules/slides.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_output (line 7, /tmp/tmpjyrv19rt/rules/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_log (line 19, /tmp/tmpjyrv19rt/rules/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dag (line 8, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filegraph (line 29, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rulegraph (line 49, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rulegraph_to_png (line 69, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule graphviz (line 92, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule graphviz_mac (line 104, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule install_poppler (line 116, /tmp/tmpjyrv19rt/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_analysis (line 6, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_data_mgt (line 21, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_figures (line 36, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_paper (line 51, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_slides (line 66, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_tables (line 81, /tmp/tmpjyrv19rt/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1609866020.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kipkurui/snakemake-demo",
    "linting": "Lints for snakefile /tmp/tmp95vt4rqj/Snakefile:\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule quantify_genes (line 13, /tmp/tmp95vt4rqj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collate_outputs (line 34, /tmp/tmp95vt4rqj/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1563092837.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "j2moreno/biobakery",
    "linting": "Lints for snakefile /tmp/tmphye_p3iu/Snakefile:\n    * Absolute path \"/shares/hii/images/morenoj/biobakery/test.simg\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/sw/singularity/latest/bin/singularity exec \" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/sw/bowtie2/2.2.9/bin/bowtie2\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/sw/bowtie2/2.2.9/bin/bowtie2-build\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/sw/bowtie2/2.2.9/bin\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/data/bt2/GRCh38\" in line 22:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/bioinfo/ref/biobakery-workflows/humann2/chocophlan\" in line 23:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/bioinfo/ref/biobakery-workflows/humann2/uniref\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/shares/hii/bioinfo/ref/biobakery-workflows/humann2/utility_mapping/map_level4ec_uniref90.txt.gz\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule kneaddata (line 42, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable BOWTIE_GRCH38_BUILD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable BOWTIE_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TRIMMOMATIC_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kneaddata_post_processing (line 88, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable KNEADDATA_LOGS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule metaphlan (line 122, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable METAPHLAN_PY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable BOWTIE2_EXEC from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable BOWTIE2_BUILD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule metaphlan_post_processing (line 155, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable METAPHLAN_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule humann (line 181, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable BOWTIE_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable METAPHLAN_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable NUCLEO_DB from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PROTEIN_DB from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule huamnn_post_processing (line 224, /tmp/tmphye_p3iu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable UTILITY_MAPPING from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SINGULARITY from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1538959919.0
  },
  {
    "data_format": 1,
    "description": "deepvariant snakemake wrapper",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tetsuro90/snakemake-deepvariant",
    "linting": "FileNotFoundError in line 11 of /tmp/tmprx3_ywny/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmprx3_ywny/workflow/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmprx3_ywny/workflow/rules/common.smk\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1611601961.0
  },
  {
    "data_format": 1,
    "description": "Whole exome sequencing data analysis pipeline developed on Snakemake workflow management system.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "huzuner/WES-data-analysis-pipeline",
    "linting": "Lints for snakefile /tmp/tmpxz1lmq7z/Snakefile:\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule bwa_map (line 41, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort (line 69, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule align_stat (line 92, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_rg (line 117, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mark_dup (line 145, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_recal (line 175, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule appy_bqsr (line 203, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule variant_call (line 229, /tmp/tmpxz1lmq7z/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1596581785.0
  },
  {
    "data_format": 1,
    "description": "Notes on Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dantaki/snakemake_tutorial",
    "linting": "Lints for rule bwa_map (line 14, /tmp/tmpp4s54chk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort (line 34, /tmp/tmpp4s54chk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_index (line 55, /tmp/tmpp4s54chk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bcftools_call (line 74, /tmp/tmpp4s54chk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_quals (line 97, /tmp/tmpp4s54chk/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1551028733.0
  },
  {
    "data_format": 1,
    "description": "Perform Cytoscans/Oncoscans CNV analysis with EaCoN",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tdayris/cel-cnv-eacon",
    "linting": "WorkflowError in line 18 of /tmp/tmpvn4ll9_t/rules/common.smk:\nError validating config file.\nValidationError: \u0027threads\u0027 is a required property\n\nFailed validating \u0027required\u0027 in schema:\n    OrderedDict([(\u0027$schema\u0027, \u0027http://json-schema.org/draft-04/schema#\u0027),\n                 (\u0027description\u0027, \u0027Snakemake configuration file\u0027),\n                 (\u0027type\u0027, \u0027object\u0027),\n                 (\u0027proprerties\u0027,\n                  OrderedDict([(\u0027workdir\u0027,\n                                OrderedDict([(\u0027type\u0027, \u0027string\u0027)])),\n                               (\u0027threads\u0027,\n                                OrderedDict([(\u0027type\u0027, \u0027integer\u0027)])),\n                               (\u0027singularity_docker_image\u0027,\n                                OrderedDict([(\u0027type\u0027, \u0027string\u0027)])),\n                               (\u0027cold_storage\u0027,\n                                OrderedDict([(\u0027type\u0027, \u0027array\u0027),\n                                             (\u0027items\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027uniqueItems\u0027, True),\n                                             (\u0027minItems\u0027, 1)]))])),\n                 (\u0027params\u0027,\n                  OrderedDict([(\u0027type\u0027, \u0027object\u0027),\n                               (\u0027proprerties\u0027,\n                                OrderedDict([(\u0027arraytype\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027baf_filter\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027float\u0027)])),\n                                             (\u0027genome\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027ldb\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027nar\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027nrf\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027scripts\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027segmenter\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027string\u0027)])),\n                                             (\u0027ser_pen\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027integer\u0027)])),\n                                             (\u0027smooth_k\u0027,\n                                              OrderedDict([(\u0027type\u0027,\n                                                            \u0027integer\u0027)])),\n                                             (\u0027required\u0027,\n                                              [\u0027arraytype\u0027,\n                                               \u0027baf_filter\u0027,\n                                               \u0027genome\u0027,\n                                               \u0027nar\u0027,\n                                               \u0027nrf\u0027,\n                                               \u0027scripts\u0027,\n                                               \u0027segmenter\u0027,\n                                               \u0027ser_pen\u0027,\n                                               \u0027smooth_k\u0027])]))])),\n                 (\u0027required\u0027,\n                  [\u0027threads\u0027,\n                   \u0027params\u0027,\n                   \u0027singularity_docker_image\u0027,\n                   \u0027workdir\u0027])])\n\nOn instance:\n    {}\n  File \"/tmp/tmpvn4ll9_t/Snakefile\", line 11, in \u003cmodule\u003e\n  File \"/tmp/tmpvn4ll9_t/rules/common.smk\", line 18, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617359746.0
  },
  {
    "data_format": 1,
    "description": "Workflow for MAG construction",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "WatsonLab/GoogleMAGs",
    "linting": "Lints for snakefile /tmp/tmpo6h2i4ez/Snakefile:\n    * Absolute path \"/bin/bash\" in line 1:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule download (line 16, /tmp/tmpo6h2i4ez/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cutadapt (line 39, /tmp/tmpo6h2i4ez/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule megahit (line 63, /tmp/tmpo6h2i4ez/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_index (line 90, /tmp/tmpo6h2i4ez/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param idx is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bwa_mem (line 118, /tmp/tmpo6h2i4ez/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 1,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1580245202.0
  },
  {
    "data_format": 1,
    "description": "What can we infer about an orbit from the RV or astrometric jitter?",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "one-datum/pipeline",
    "linting": null,
    "stargazers_count": 1,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1618576970.0
  },
  {
    "data_format": 1,
    "description": "Dosage Compensation Analyses Pipeline using Snakemake ",
    "formatting": "[WARNING] In file \"/tmp/tmpppf0m25q/Snakefile\":  Keyword \"input\" at line 32 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "rameshbalan/ARChES",
    "linting": "Lints for snakefile /tmp/tmpppf0m25q/Snakefile:\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule summary_table (line 739, /tmp/tmpppf0m25q/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 1,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1600383202.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to upload Sars-Cov-2 reads to ENA",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "avilab/ena-upload",
    "linting": "WorkflowError in line 8 of /tmp/tmp15puv5em/workflow/Snakefile:\nConfig file is not valid JSON or YAML. In case of YAML, make sure to not mix whitespace and tab indentation.\n  File \"/tmp/tmp15puv5em/workflow/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflow",
      "sars-cov-2",
      "korogeno-est",
      "covid-19-estonia",
      "snakemake",
      "snakefile",
      "upload",
      "ena"
    ],
    "updated_at": 1615296939.0
  },
  {
    "data_format": 1,
    "description": "Cancer Selection",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pmonnahan/CanSel",
    "linting": "FileNotFoundError in line 30 of /tmp/tmpauvetyn7/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027/home/spectorl/pmonnaha/misc/h1kgp_samps.txt\u0027\n  File \"/tmp/tmpauvetyn7/workflow/Snakefile\", line 30, in \u003cmodule\u003e\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616402483.0
  },
  {
    "data_format": 1,
    "description": "Snakemake and cluster-based exome workflows for the XiaoLab",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "stevenpastor/snakemake-exome-workflow",
    "linting": "Lints for rule bwa_map (line 15, /tmp/tmpkhzei4_l/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_sort (line 53, /tmp/tmpkhzei4_l/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_index (line 78, /tmp/tmpkhzei4_l/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bcftools_call (line 98, /tmp/tmpkhzei4_l/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report (line 123, /tmp/tmpkhzei4_l/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 1,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1573225052.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for benchmarking taxonomy assignments",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gwarmstrong/taxa-assign-benchmarking",
    "linting": "ModuleNotFoundError in line 1 of /tmp/tmp49p1eg7n/Snakefile:\nNo module named \u0027scipy\u0027\n  File \"/tmp/tmp49p1eg7n/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp49p1eg7n/benchutils/metrics.py\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1580305945.0
  },
  {
    "data_format": 1,
    "description": "Code for replicating \"Right and left, partisanship predicts vulnerability to misinformation\" by Dimitar Nikolov, Alessandro Flammini and Filippo Menczer",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "osome-iu/misinfo-partisanship-hksmisinforeview-2021",
    "linting": "FileNotFoundError in line 1 of /tmp/tmpxe_9ozjd/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config.json\u0027\n  File \"/tmp/tmpxe_9ozjd/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1613920056.0
  },
  {
    "data_format": 1,
    "description": "GWAS Summary Statistics Data Harmonisation",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "EBISPOT/gwas-sumstats-harmoniser",
    "linting": "KeyError in line 7 of /tmp/tmpuxhwlf68/Snakefile:\n\u0027local_resources\u0027\n  File \"/tmp/tmpuxhwlf68/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1616008004.0
  },
  {
    "data_format": 1,
    "description": "A scalable machine learning pipeline for probe specificity prediction.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "beliveau-lab/PaintSHOP_pipeline",
    "linting": "KeyError in line 11 of /tmp/tmp5rzllik_/workflow/Snakefile:\n\u0027genome_fasta\u0027\n  File \"/tmp/tmp5rzllik_/workflow/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618617145.0
  },
  {
    "data_format": 1,
    "description": "A pipeline for extracting SSU rRNA gene from wgs data and applying them for further diversity analysis",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jiarong/SSUsearch",
    "linting": "KeyError in line 14 of /tmp/tmpt2zwt_ce/Snakefile:\n\u0027Workdir\u0027\n  File \"/tmp/tmpt2zwt_ce/Snakefile\", line 14, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1574156379.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpdgwntzku/Snakefile\":  \u0027NoParametersError: L199: In input definition.\u0027\n[INFO] In file \"/tmp/tmpdgwntzku/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "PEDIA-Charite/classifier",
    "linting": "WorkflowError in line 1 of /tmp/tmpdgwntzku/Snakefile:\nConfig file ../config.yml not found.\n  File \"/tmp/tmpdgwntzku/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1578247121.0
  },
  {
    "data_format": 1,
    "description": "A simple Snakemake workflow to annotate variant call format (VCF) data using GATK4, SnpSift, VEP and genmod. Designed to be used after human_genomics_pipeline.",
    "formatting": "[INFO] 21 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ESR-NZ/vcf_annotation_pipeline",
    "linting": "KeyError in line 12 of /tmp/tmpo3qfp4y9/workflow/Snakefile:\n\u0027DATA\u0027\n  File \"/tmp/tmpo3qfp4y9/workflow/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1604002264.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpndad_1lm/rules/align_reads.smk\":  \u0027NoParametersError: L4: In input definition.\u0027\n[ERROR] In file \"/tmp/tmpndad_1lm/rules/merge_expr.smk\":  \u0027NoParametersError: L4: In input definition.\u0027\n[ERROR] In file \"/tmp/tmpndad_1lm/rules/merge_alignment_statistics.smk\":  \u0027NoParametersError: L4: In input definition.\u0027\n[INFO] 3 file(s) raised parsing errors \ud83e\udd15\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "acshetty/sncRNA-seq-analysis",
    "linting": "FileNotFoundError in line 14 of /tmp/tmpndad_1lm/Snakefile:\n[Errno 2] No such file or directory: \u0027.test/info/samples.txt\u0027\n  File \"/tmp/tmpndad_1lm/Snakefile\", line 14, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1604381140.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for evaluating varlociraptor",
    "formatting": "[INFO] 16 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "varlociraptor/varlociraptor-evaluation",
    "linting": "Lints for snakefile /tmp/tmp62t3b4u7/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp62t3b4u7/rules/mapping.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp62t3b4u7/rules/lancet.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp62t3b4u7/rules/neusomatic.smk:\n    * Deprecated singularity directive used for container definition in line 26.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmp62t3b4u7/rules/varlociraptor.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp62t3b4u7/rules/eval.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule prepare_bam (line 7, /tmp/tmp62t3b4u7/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bam2fq (line 34, /tmp/tmp62t3b4u7/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_index (line 60, /tmp/tmp62t3b4u7/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_sort (line 142, /tmp/tmp62t3b4u7/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 198, /tmp/tmp62t3b4u7/rules/mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule delly_concat (line 39, /tmp/tmp62t3b4u7/rules/delly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule delly_samples (line 63, /tmp/tmp62t3b4u7/rules/delly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule delly_adhoc (line 81, /tmp/tmp62t3b4u7/rules/delly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule build_lancet (line 1, /tmp/tmp62t3b4u7/rules/lancet.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fix_lancet (line 102, /tmp/tmp62t3b4u7/rules/lancet.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_lancet (line 127, /tmp/tmp62t3b4u7/rules/lancet.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule lancet_adhoc (line 155, /tmp/tmp62t3b4u7/rules/lancet.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule manta_raw (line 51, /tmp/tmp62t3b4u7/rules/manta.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule manta_default (line 75, /tmp/tmp62t3b4u7/rules/manta.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule manta_adhoc (line 103, /tmp/tmp62t3b4u7/rules/manta.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule strelka_default (line 51, /tmp/tmp62t3b4u7/rules/strelka.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule strelka_adhoc (line 80, /tmp/tmp62t3b4u7/rules/strelka.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_region_bed (line 1, /tmp/tmp62t3b4u7/rules/neusomatic.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fix_neusomatic (line 101, /tmp/tmp62t3b4u7/rules/neusomatic.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule neusomatic_adhoc (line 131, /tmp/tmp62t3b4u7/rules/neusomatic.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule novobreak_adhoc (line 38, /tmp/tmp62t3b4u7/rules/novobreak.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule novobreak_all (line 62, /tmp/tmp62t3b4u7/rules/novobreak.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bpi_convert (line 41, /tmp/tmp62t3b4u7/rules/bpi.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bpi_adhoc (line 65, /tmp/tmp62t3b4u7/rules/bpi.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule varlociraptor_call (line 20, /tmp/tmp62t3b4u7/rules/varlociraptor.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule varlociraptor_merge (line 72, /tmp/tmp62t3b4u7/rules/varlociraptor.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule varlociraptor_filter_by_odds (line 96, /tmp/tmp62t3b4u7/rules/varlociraptor.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule varlociraptor_control_fdr (line 120, /tmp/tmp62t3b4u7/rules/varlociraptor.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule adhoc_varlociraptor (line 148, /tmp/tmp62t3b4u7/rules/varlociraptor.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule adhoc_filter (line 1, /tmp/tmp62t3b4u7/rules/adhoc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule annotate_truth (line 7, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule match_varlociraptor_calls (line 44, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule match_other_calls (line 75, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule truth_to_tsv (line 106, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule varlociraptor_calls_to_tsv (line 154, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule varlociraptor_all_calls_to_tsv (line 183, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule other_calls_to_tsv (line 212, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule obtain_tp_fp (line 248, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_precision_recall (line 307, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_allelefreq_recall (line 346, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_fdr (line 379, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_allelefreq (line 410, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_allelefreq_scatter (line 440, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_score_dist (line 470, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_softclips (line 499, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concordance_match (line 523, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_concordance (line 558, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concordance_to_tsv (line 589, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_concordance_upset (line 619, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_concordance (line 650, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule lancet_frequencies (line 680, /tmp/tmp62t3b4u7/rules/eval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule depth (line 31, /tmp/tmp62t3b4u7/rules/stats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule svg2pdf (line 1, /tmp/tmp62t3b4u7/rules/tools.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index_bcf (line 128, /tmp/tmp62t3b4u7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rank_fps (line 152, /tmp/tmp62t3b4u7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule testcase (line 182, /tmp/tmp62t3b4u7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "workflow",
      "reproducible-research"
    ],
    "updated_at": 1615147146.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmp2yt_40wr/Snakefile\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7fc6e9ac50d0\u003eNewline expected after keyword \u0027onsuccess\u0027\u0027\n[INFO] In file \"/tmp/tmp2yt_40wr/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "sraorao/snakemake_code_clinic_2",
    "linting": "Lints for rule align (line 40, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_bams (line 71, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_dups (line 96, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dupmetrics (line 123, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dupmetrics_python (line 145, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608227648.0
  },
  {
    "data_format": 1,
    "description": "Snakemake implementation of GATK4 pipeline for GTEx Project",
    "formatting": "[ERROR] In file \"/tmp/tmpgkz3_gpq/Snakefile\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 4)\u0027\n[INFO] In file \"/tmp/tmpgkz3_gpq/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "troycomi/gtex-gatk4",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpgkz3_gpq/Snakefile:\nNo module named \u0027slurm_scrub\u0027\n  File \"/tmp/tmpgkz3_gpq/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1575296570.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for CAGE data analysis using icetea",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vivekbhr/cage_pipeline",
    "linting": "KeyError in line 38 of /tmp/tmph9zuzcwx/internals.Snakefile:\n\u0027knownAnnotations\u0027\n  File \"/tmp/tmph9zuzcwx/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmph9zuzcwx/internals.Snakefile\", line 38, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [
      "cage-seq"
    ],
    "updated_at": 1585033965.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for TAP-seq data processing",
    "formatting": "[WARNING] In file \"/tmp/tmpr_i3id0k/rules/align_reads.smk\":  Keyword \"shell\" at line 172 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "argschwind/TAPseq_workflow",
    "linting": "Lints for snakefile /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk:\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpr_i3id0k/rules/align_reads.smk:\n    * Absolute path \"/*\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/genomeDir\" in line 164:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.fasta\" in line 206:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.dict\" in line 207:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.refFlat\" in line 226:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpr_i3id0k/rules/extract_dge.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule create_vector_ref (line 74, /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param output_bn is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule create_cropseq_ref (line 107, /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule star_align (line 320, /tmp/tmpr_i3id0k/rules/align_reads.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outprefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule align_report (line 529, /tmp/tmpr_i3id0k/rules/align_reads.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dge_report (line 144, /tmp/tmpr_i3id0k/rules/extract_dge.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595252030.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for the vg toolkit.",
    "formatting": "[ERROR] In file \"/tmp/tmp6wkz3i01/Snakefile\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] In file \"/tmp/tmp6wkz3i01/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "vgteam/vg_snakemake",
    "linting": "KeyError in line 11 of /tmp/tmp6wkz3i01/Snakefile:\n\u0027chrs\u0027\n  File \"/tmp/tmp6wkz3i01/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1600434993.0
  },
  {
    "data_format": 1,
    "description": "Amplicon sequencing workflow with snakemake and dada2",
    "formatting": "[ERROR] In file \"/tmp/tmpjia0qmmz/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 18:4:     return expand(\"output/fastqc/{sample}\" + config[\"R1\"] + \"_fastqc.html\" , sample = SAMPLES)\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpjia0qmmz/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "ramay/dada2_snakemake_workflow",
    "linting": "Lints for snakefile /tmp/tmpjia0qmmz/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpjia0qmmz/utils/rules/qc_cutadapt.smk:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule combine_read_counts (line 64, /tmp/tmpjia0qmmz/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cutadapt (line 60, /tmp/tmpjia0qmmz/utils/rules/qc_cutadapt.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plotQualityProfile (line 4, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dada2_filter (line 36, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * Param nread is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule IDtaxa (line 221, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613140246.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmp2yt_40wr/Snakefile\":  \u0027SyntaxError: \u003cfunction line_nb at 0x7fc6e9ac50d0\u003eNewline expected after keyword \u0027onsuccess\u0027\u0027\n[INFO] In file \"/tmp/tmp2yt_40wr/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "sraorao/snakemake_code_clinic_2",
    "linting": "Lints for rule align (line 40, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sort_bams (line 71, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_dups (line 96, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dupmetrics (line 123, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dupmetrics_python (line 145, /tmp/tmp2yt_40wr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608227648.0
  },
  {
    "data_format": 1,
    "description": "Snakemake implementation of GATK4 pipeline for GTEx Project",
    "formatting": "[ERROR] In file \"/tmp/tmpgkz3_gpq/Snakefile\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 4)\u0027\n[INFO] In file \"/tmp/tmpgkz3_gpq/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "troycomi/gtex-gatk4",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpgkz3_gpq/Snakefile:\nNo module named \u0027slurm_scrub\u0027\n  File \"/tmp/tmpgkz3_gpq/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1575296570.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for CAGE data analysis using icetea",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vivekbhr/cage_pipeline",
    "linting": "KeyError in line 38 of /tmp/tmph9zuzcwx/internals.Snakefile:\n\u0027knownAnnotations\u0027\n  File \"/tmp/tmph9zuzcwx/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmph9zuzcwx/internals.Snakefile\", line 38, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [
      "cage-seq"
    ],
    "updated_at": 1585033965.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for TAP-seq data processing",
    "formatting": "[WARNING] In file \"/tmp/tmpr_i3id0k/rules/align_reads.smk\":  Keyword \"shell\" at line 172 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "argschwind/TAPseq_workflow",
    "linting": "Lints for snakefile /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk:\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpr_i3id0k/rules/align_reads.smk:\n    * Absolute path \"/*\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/genomeDir\" in line 164:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.fasta\" in line 206:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.dict\" in line 207:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cropseq_ref.refFlat\" in line 226:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 12:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpr_i3id0k/rules/extract_dge.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule create_vector_ref (line 74, /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param output_bn is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule create_cropseq_ref (line 107, /tmp/tmpr_i3id0k/rules/create_alignment_refs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule star_align (line 320, /tmp/tmpr_i3id0k/rules/align_reads.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outprefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule align_report (line 529, /tmp/tmpr_i3id0k/rules/align_reads.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dge_report (line 144, /tmp/tmpr_i3id0k/rules/extract_dge.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595252030.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for the vg toolkit.",
    "formatting": "[ERROR] In file \"/tmp/tmp6wkz3i01/Snakefile\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] In file \"/tmp/tmp6wkz3i01/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "vgteam/vg_snakemake",
    "linting": "KeyError in line 11 of /tmp/tmp6wkz3i01/Snakefile:\n\u0027chrs\u0027\n  File \"/tmp/tmp6wkz3i01/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1600434993.0
  },
  {
    "data_format": 1,
    "description": "Amplicon sequencing workflow with snakemake and dada2",
    "formatting": "[ERROR] In file \"/tmp/tmpjia0qmmz/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 18:4:     return expand(\"output/fastqc/{sample}\" + config[\"R1\"] + \"_fastqc.html\" , sample = SAMPLES)\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpjia0qmmz/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "ramay/dada2_snakemake_workflow",
    "linting": "Lints for snakefile /tmp/tmpjia0qmmz/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpjia0qmmz/utils/rules/qc_cutadapt.smk:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 17:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule combine_read_counts (line 64, /tmp/tmpjia0qmmz/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cutadapt (line 60, /tmp/tmpjia0qmmz/utils/rules/qc_cutadapt.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plotQualityProfile (line 4, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dada2_filter (line 36, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * Param nread is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule IDtaxa (line 221, /tmp/tmpjia0qmmz/utils/rules/dada2.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613140246.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vaquerizaslab/Ing-Simmons_et_al_dorsoventral_3D_genome",
    "linting": "Lints for snakefile /tmp/tmpnp5d8akq/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/make_hi-c.smk:\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 39:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/insulation_domains.smk:\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/fan-c/scripts/domain_boundaries.Rmd\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/fan-c/scripts/domain_boundaries.html\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/chess.smk:\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/fan-c/scripts/analyse_chess_comparisons_merged.Rmd\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/fan-c/scripts/analyse_chess_comparisons_merged.html\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/figures.smk:\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/plot_enhancers_figures.Rmd\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/plot_enhancers_figures.html\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk:\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/differential_H3K27ac.Rmd\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/differential_H3K27ac.html\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/make_candidate_enhancers.Rmd\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/make_candidate_enhancers.html\" in line 25:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/csaw_enhancer_domains.Rmd\" in line 51:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/research/vaquerizas/liz/dorsal_ventral/for_paper/scripts/csaw_enhancer_domains.html\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpnp5d8akq/workflow/micro-c.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule touch_data (line 180, /tmp/tmpnp5d8akq/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastqc (line 195, /tmp/tmpnp5d8akq/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multiqc (line 215, /tmp/tmpnp5d8akq/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 5 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 6 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 7 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 8 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 9 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 10 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 11 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 12 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 13 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 14 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 15 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 16 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 17 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 18 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 19 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 20 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 21 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 22 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 23 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 24 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 25 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 26 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 27 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 28 (line 16, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_fragments (line 47, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 30 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 31 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 32 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 33 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 34 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 35 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 36 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 37 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 38 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 39 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 40 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 41 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 42 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 43 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 44 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 45 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 46 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 47 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 48 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 49 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 50 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 51 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 52 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 53 (line 69, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 54 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 55 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 56 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 57 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 58 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 59 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 60 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 61 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 62 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 63 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 64 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 65 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 66 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 67 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 68 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 69 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 70 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 71 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 72 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 73 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 74 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 75 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 76 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 77 (line 108, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule process_hic (line 127, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule export_for_pca (line 158, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fanc_pca (line 182, /tmp/tmpnp5d8akq/workflow/make_hi-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cis_trans_ratio (line 1, /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cis_local_ratio (line 19, /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_hic_stats (line 41, /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_alignment_stats_bwa (line 85, /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_stats (line 107, /tmp/tmpnp5d8akq/workflow/hi-c_qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 86 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 87 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 88 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 89 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 90 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 91 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 92 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 93 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 94 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 95 (line 14, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fanc_process_hic_merged (line 36, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule balance_whole_genome (line 66, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_cooler (line 94, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_diff_hic (line 116, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calc_distance_decay (line 141, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule analyse_distance_decay (line 163, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule export_marginals (line 186, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_resolution (line 205, /tmp/tmpnp5d8akq/workflow/process_merged_replicates.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule process_published_loop_sets (line 2, /tmp/tmpnp5d8akq/workflow/loops.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_published_loop_aggregates (line 18, /tmp/tmpnp5d8akq/workflow/loops.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule write_loop_strengths (line 46, /tmp/tmpnp5d8akq/workflow/loops.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule insulation_score (line 5, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_boundaries (line 25, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_boundaries (line 48, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_boundaries_and_domains (line 71, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_boundaries_hitile (line 97, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_domains_hitile (line 122, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_domain_aggregates (line 147, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_boundary_aggregates (line 175, /tmp/tmpnp5d8akq/workflow/insulation_domains.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule custom_eigenvectors (line 5, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ref_eigenvector (line 25, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule compartment_aggregates (line 43, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mask_heterochromatin (line 76, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule custom_eigenvectors_masked (line 96, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ref_eigenvector_masked (line 116, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule compartment_aggregates_masked (line 134, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compartment_aggregates_masked_collapsed (line 168, /tmp/tmpnp5d8akq/workflow/compartments.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule export_hic_as_text (line 5, /tmp/tmpnp5d8akq/workflow/chess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule chess_make_pairs (line 25, /tmp/tmpnp5d8akq/workflow/chess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule chess_sim (line 45, /tmp/tmpnp5d8akq/workflow/chess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule analyse_chess (line 78, /tmp/tmpnp5d8akq/workflow/chess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_chess_regions (line 142, /tmp/tmpnp5d8akq/workflow/chess.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_de_gene_regions (line 5, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_de_gene_regions_nc14 (line 22, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_de_gene_regions_stg10 (line 41, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_fig1_regions (line 62, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule enhancers_figures (line 85, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_enhancer_heatmap_matrix_5kb (line 117, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_enhancer_heatmap_5kb (line 167, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_fig4_regions (line 212, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_fig4_regions_large (line 232, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_overview_figure (line 256, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_whole_genome (line 277, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_microc_regions (line 299, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_ep_loop_aggregates (line 320, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule write_ep_loop_strengths (line 348, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_ep_loop_aggregates_microc (line 369, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule write_ep_loop_strengths_microc (line 401, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_schematic (line 425, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_microc_overview_figure (line 459, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_stg10_regions (line 482, /tmp/tmpnp5d8akq/workflow/figures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule differential_h3k27ac_calculations (line 1, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_enhancers (line 27, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_differential_chipseq_hitile (line 55, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_tf_peak_pairs (line 96, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_tf_aggregates (line 113, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_tf_aggregates (line 138, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_tf_aggregates_microc (line 168, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_tf_aggregates_microc (line 193, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_enhancer_loop_aggregates (line 224, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_enhancer_loop_aggregates_microc (line 253, /tmp/tmpnp5d8akq/workflow/chip-seq_analysis.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 157 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 158 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 159 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 160 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 161 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule 162 (line 30, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule get_microc_alignment_stats (line 61, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bin_genome (line 83, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_sam (line 103, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 166 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 167 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 168 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 169 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 170 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 171 (line 126, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_microc_pairs (line 152, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_microc (line 188, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 174 (line 217, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule 175 (line 217, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fanc_process_microc_merged (line 239, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_cooler_microc (line 271, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule export_marginals_microc (line 293, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule insulation_score_microc (line 313, /tmp/tmpnp5d8akq/workflow/micro-c.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618324900.0
  },
  {
    "data_format": 1,
    "description": "Automated de novo transcriptome assembly",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SchulzLab/SOS",
    "linting": "WorkflowError in line 3 of /tmp/tmpuut5ihey/Snakefile:\nConfig file is not valid JSON or YAML. In case of YAML, make sure to not mix whitespace and tab indentation.\n  File \"/tmp/tmpuut5ihey/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1598040049.0
  },
  {
    "data_format": 1,
    "description": "Python tools for metagenomic data",
    "formatting": "[ERROR] In file \"/tmp/tmpdkm08tu5/rules/annotation.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     \"-db {params} -query {input} -outfmt \u00276 std qlen slen\u0027 -out {output.ssu}\\n\"\n```\nwhile formatting code with black.\u0027\n[WARNING] In file \"/tmp/tmpdkm08tu5/rules/athena.smk\":  Keyword \"shell\" at line 65 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpdkm08tu5/rules/beadsCluster.smk\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     o6 = \"{sample}/mash/lv\" + str(config[\"p_dist_lv\"]) + \"/tree.cluster.count\"\n```\nwhile formatting code with black.\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ZeweiSong/metaSeq",
    "linting": "KeyError in line 17 of /tmp/tmpdkm08tu5/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpdkm08tu5/Snakefile\", line 17, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1609227820.0
  },
  {
    "data_format": 1,
    "description": "Quality control for mass spec proteomics",
    "formatting": "[ERROR] In file \"/tmp/tmp1hcjgbbw/Snakefile\":  \u0027InvalidParameterSyntax: L191: Over-indented recognised keyword found: \u0027params\u0027\u0027\n[INFO] In file \"/tmp/tmp1hcjgbbw/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "qbicsoftware-archive/qcprot",
    "linting": "WorkflowError in line 16 of /tmp/tmp1hcjgbbw/Snakefile:\nConfig file config.json not found.\n  File \"/tmp/tmp1hcjgbbw/Snakefile\", line 16, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 14,
    "topics": [
      "workflow",
      "ms",
      "px"
    ],
    "updated_at": 1600936668.0
  },
  {
    "data_format": 1,
    "description": "Trade-offs between geographic scale, cost, and infrastructure requirements for fully renewable electricity in Europe",
    "formatting": "[ERROR] In file \"/tmp/tmpetzeck2i/Snakefile\":  \u0027InvalidParameterSyntax: 35\"build/output/{resolution}/uncertainty/weather-diff-diff.txt\". format( resolution= config[\"weather-uncertainty\"][\"resolution\"][\"space\"])\n\"build/output/{resolution}/figures/figure1.tiff\". format( resolution= config[\"resolution\"][\"space\"])\n\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "timtroendle/geographic-scale",
    "linting": "SyntaxError in line 36 of /tmp/tmpetzeck2i/Snakefile:\ninvalid syntax\n",
    "stargazers_count": 2,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1601294507.0
  },
  {
    "data_format": 1,
    "description": "Fastq to bam mapping pipeline with marked duplicates, indexing, and insert size metric collection",
    "formatting": "[ERROR] In file \"/tmp/tmpixz7mjo8/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:0: else:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpixz7mjo8/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "bnelsj/bwa_mem_mapping",
    "linting": "Lints for snakefile /tmp/tmpixz7mjo8/Snakefile:\n    * Absolute path \"/\" + manifest_sn.sn + \" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + manifest_sn.lane + \" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule get_flagstat (line 81, /tmp/tmpixz7mjo8/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_idxstats (line 105, /tmp/tmpixz7mjo8/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect_isize_metrics (line 128, /tmp/tmpixz7mjo8/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_merged_bams (line 151, /tmp/tmpixz7mjo8/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bams (line 173, /tmp/tmpixz7mjo8/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_mem_map_and_mark_dups (line 237, /tmp/tmpixz7mjo8/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608787032.0
  },
  {
    "data_format": 1,
    "description": "Using the `bookdown` and `rticles` packages together to write papers",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lachlandeer/bookdown_rticles",
    "linting": "Lints for snakefile /tmp/tmpzmvofki0/Snakefile:\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule all (line 27, /tmp/tmpzmvofki0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pdf (line 47, /tmp/tmpzmvofki0/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule html (line 75, /tmp/tmpzmvofki0/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean (line 102, /tmp/tmpzmvofki0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [
      "bookdown",
      "rticles",
      "journal-article"
    ],
    "updated_at": 1547572234.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for neoantigen prediction",
    "formatting": "[WARNING] In file \"/tmp/tmpsn4or5ay/workflow/rules/calling.smk\":  Keyword \"input\" at line 114 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 21 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "snakemake-workflows/dna-seq-neoantigen-prediction",
    "linting": null,
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1607525163.0
  },
  {
    "data_format": 1,
    "description": "Benchmarking variant calling in polyploids",
    "formatting": "[WARNING] In file \"/tmp/tmpu09ywk85/workflow/rules/octopus.smk\":  Keyword \"input\" at line 20 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "luntergroup/polyploid",
    "linting": "ModuleNotFoundError in line 3 of /tmp/tmpu09ywk85/workflow/rules/data.smk:\nNo module named \u0027wget\u0027\n  File \"/tmp/tmpu09ywk85/workflow/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpu09ywk85/workflow/rules/data.smk\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1618278084.0
  },
  {
    "data_format": 1,
    "description": "A target captured medical exome ONT sequencing with amplicon variant calling workflow",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "HKU-BAL/ECNano",
    "linting": "Lints for rule analysis (line 62, /tmp/tmpod9_jl42/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule porechop (line 101, /tmp/tmpod9_jl42/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule align_index (line 121, /tmp/tmpod9_jl42/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clair_ensemble (line 145, /tmp/tmpod9_jl42/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1617772219.0
  },
  {
    "data_format": 1,
    "description": "Neural Networks based unified physics parameterization for atmospheric models",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nbren12/uwnet",
    "linting": "ModuleNotFoundError in line 6 of /tmp/tmp8wm5p7ww/Snakefile:\nNo module named \u0027f90nml\u0027\n  File \"/tmp/tmp8wm5p7ww/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmp8wm5p7ww/src/data/__init__.py\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmp8wm5p7ww/src/data/sam.py\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614740714.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for de novo transcriptome assembly with 454 reads",
    "formatting": "[WARNING] In file \"/tmp/tmpb26y05d_/Snakefile\":  Keyword \"input\" at line 47 has comments under a value.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/smsk_454",
    "linting": "Lints for snakefile /tmp/tmpb26y05d_/bin/snakefiles/folders:\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpb26y05d_/bin/snakefiles/raw:\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpb26y05d_/bin/snakefiles/qc_sff:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 55:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 112:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpb26y05d_/bin/snakefiles/qc_fastq:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpb26y05d_/bin/snakefiles/assembly:\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule clean (line 1, /tmp/tmpb26y05d_/bin/snakefiles/clean):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_raw (line 18, /tmp/tmpb26y05d_/bin/snakefiles/clean):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable raw_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_qc (line 34, /tmp/tmpb26y05d_/bin/snakefiles/clean):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable qc_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_assembly (line 50, /tmp/tmpb26y05d_/bin/snakefiles/clean):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable assembly_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_report (line 66, /tmp/tmpb26y05d_/bin/snakefiles/clean):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable report_dir from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule raw_sample_sff_make_link_sff (line 1, /tmp/tmpb26y05d_/bin/snakefiles/raw):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule raw_sample_fastq_make_link (line 40, /tmp/tmpb26y05d_/bin/snakefiles/raw):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule raw_sample_sff_to_fastq (line 79, /tmp/tmpb26y05d_/bin/snakefiles/raw):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule raw_adaptors_make_link (line 128, /tmp/tmpb26y05d_/bin/snakefiles/raw):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_sff_pyrobayes (line 1, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param stub is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule qc_sample_from_sff_join_fasta_qual (line 49, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_sff_decompress_pyrobayes (line 95, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_sff_snowhite (line 142, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_sff_trimmomatic (line 325, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_sff_decompress_trimmomatic (line 380, /tmp/tmpb26y05d_/bin/snakefiles/qc_sff):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_fastq_decompress (line 1, /tmp/tmpb26y05d_/bin/snakefiles/qc_fastq):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_snowhite_from_fastq_sample (line 48, /tmp/tmpb26y05d_/bin/snakefiles/qc_fastq):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_fastq_trimmomatic (line 225, /tmp/tmpb26y05d_/bin/snakefiles/qc_fastq):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qc_sample_from_fastq_decompress_trimmomatic (line 278, /tmp/tmpb26y05d_/bin/snakefiles/qc_fastq):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule assembly_newbler (line 1, /tmp/tmpb26y05d_/bin/snakefiles/assembly):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_sff_raw_fastqc (line 1, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_sff_pyrobayes_fastqc (line 51, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_sff_snowhite_fastqc (line 101, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_fastqc_trimmomatic_from_sff_sample (line 151, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_sff_multiqc (line 201, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_sample_from_fastq_raw_fastqc (line 275, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_fastq_snowhite_fastqc (line 325, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_fastq_trimmomatic_fastqc (line 375, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule report_sample_from_fastq_multiqc (line 425, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_all_multiqc (line 500, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_raw_multiqc (line 592, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_pyrobayes_multiqc (line 664, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_snowhite_multiqc (line 732, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report_trimmomatic_multiqc (line 804, /tmp/tmpb26y05d_/bin/snakefiles/report):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1515950115.0
  },
  {
    "data_format": 1,
    "description": "Consolidation of tools in the MutPred Suite to work with VCF files",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NCBI-Hackathons/MutPredMerge",
    "linting": "Lints for snakefile /tmp/tmptyyiox9u/Snakefile:\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 69:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 93:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 104:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 106:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 128:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 131:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 132:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 28:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 93:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 104:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 106:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 108:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 118:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 128:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 130:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 131:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule annovar_convert (line 46, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule annovar_annotate (line 70, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule splitter (line 96, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule coding_change (line 131, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MutPred2 (line 162, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MutPred_LOF (line 186, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outfile_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule MutPred_indel (line 213, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outfile_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule Merge (line 240, /tmp/tmptyyiox9u/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1564703075.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow for large-scale SARS-CoV-2 analyses.",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cbg-ethz/SARS-CoV-2_Analysis",
    "linting": "Lints for snakefile /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule download_fastq (line 29, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule bwa_index (line 280, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bwa_mem (line 312, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_index (line 409, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule compute_coverage (line 436, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule aggregate_results (line 476, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule plot_coverage_per_locus (line 523, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule plot_coverage_per_sample (line 677, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule retrieve_sra_metadata (line 760, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule compute_additional_properties (line 824, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule assemble_final_dataframe (line 892, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule select_samples (line 946, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule distill_data_frame (line 998, /tmp/tmp6sdmmp7o/workflow/rules/data_retrieval.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule lofreq (line 1, /tmp/tmp6sdmmp7o/workflow/rules/variant_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule gather_vcf_files (line 116, /tmp/tmp6sdmmp7o/workflow/rules/variant_calling.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_heatmaps (line 1, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_histograms (line 23, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_mutation_histograms (line 45, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compute_regression (line 66, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snv_coverage_plot (line 87, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compute_top_positions (line 120, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_extended_top_positions_table (line 144, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_extended_top_samples_table (line 165, /tmp/tmp6sdmmp7o/workflow/rules/visualize_results.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 3,
    "topics": [
      "snakemake",
      "sars-cov-2"
    ],
    "updated_at": 1618915205.0
  },
  {
    "data_format": 1,
    "description": "Tools and workflows for evaluating bacterial and archaeal genome contamination with sourmash and GTDB.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dib-lab/sourmash-oddify",
    "linting": "Lints for snakefile /tmp/tmp0s7f284q/Snakefile:\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule sigs (line 49, /tmp/tmp0s7f284q/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gtdbtk_gather_matches (line 72, /tmp/tmp0s7f284q/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_lineages_csv (line 102, /tmp/tmp0s7f284q/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outputs_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_lca_db (line 125, /tmp/tmp0s7f284q/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_oddities_txt (line 150, /tmp/tmp0s7f284q/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outputs_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_oddities_examine_txt (line 174, /tmp/tmp0s7f284q/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outputs_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 2,
    "subscribers_count": 5,
    "topics": [
      "sourmash"
    ],
    "updated_at": 1614979996.0
  },
  {
    "data_format": 1,
    "description": "Benchmarking adapter and quality trimming tools",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "boulund/adapter_benchmark",
    "linting": "Lints for rule download_fastq (line 29, /tmp/tmpkw55vqy0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule grade (line 354, /tmp/tmpkw55vqy0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot (line 386, /tmp/tmpkw55vqy0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597668499.0
  },
  {
    "data_format": 1,
    "description": "Explore car crash prediction using NZTA Crash Analysis System data",
    "formatting": "[ERROR] In file \"/tmp/tmp3l1tyiwf/Snakefile\":  \u0027InvalidParameterSyntax: 311 if USE_SLURM else8\n\u0027\n[INFO] In file \"/tmp/tmp3l1tyiwf/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "neon-ninja/crash_prediction",
    "linting": "Lints for rule download_data (line 18, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_data (line 33, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fit (line 52, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable N_ITER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fit_knn (line 94, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable N_ITER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fit_mlp (line 133, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable N_ITER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fit_gbdt (line 176, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable N_ITER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule predict (line 218, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule evaluate (line 238, /tmp/tmp3l1tyiwf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [
      "crash-prediction",
      "nzta",
      "cas",
      "road",
      "traffic",
      "safety"
    ],
    "updated_at": 1617804426.0
  },
  {
    "data_format": 1,
    "description": "A pipeline to annotate miRNAs, phased siRNAs and other types using a reference genome and experimental sRNA-Seq data",
    "formatting": "[WARNING] In file \"/tmp/tmp2l3ol_6d/Snakefile\":  Keyword \"shell\" at line 208 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp2l3ol_6d/Snakefile\":  Keyword \"shell\" at line 242 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp2l3ol_6d/Snakefile\":  Keyword \"shell\" at line 332 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "BleekerLab/small-rna-seq-pipeline",
    "linting": "ModuleNotFoundError in line 7 of /tmp/tmp2l3ol_6d/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmp2l3ol_6d/Snakefile\", line 7, in \u003cmodule\u003e\n  File \"/tmp/tmp2l3ol_6d/helpers.py\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 2,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619278887.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmpkmyfckk4/Snakefile\":  \u0027DuplicateKeyWordError: L92: \u0027rule all\u0027 specified twice.\u0027\n[INFO] In file \"/tmp/tmpkmyfckk4/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "weng-lab/psychip_snakemake",
    "linting": "WorkflowError in line 3 of /tmp/tmpkmyfckk4/Snakefile:\nConfig file is not valid JSON or YAML. In case of YAML, make sure to not mix whitespace and tab indentation.\n  File \"/tmp/tmpkmyfckk4/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 21,
    "topics": [],
    "updated_at": 1566354708.0
  },
  {
    "data_format": 1,
    "description": "Consistency pipeline for hierarchies of orthologous groups (OGs) based on subsampling and tree reconciliation",
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "meringlab/og_consistency_pipeline",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmprqv0qksu/rules/preprocessing.smk:\nNo module named \u0027ete3\u0027\n  File \"/tmp/tmprqv0qksu/Snakefile\", line 69, in \u003cmodule\u003e\n  File \"/tmp/tmprqv0qksu/rules/preprocessing.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1608601207.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmp4v9osoff/Snakefile\":  Keyword \"input\" at line 132 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AlexanderLabWHOI/tara-euk-metaG",
    "linting": "PermissionError in line 78 of /tmp/tmp4v9osoff/Snakefile:\n[Errno 13] Permission denied: \u0027/vortexfs1\u0027\n  File \"/tmp/tmp4v9osoff/Snakefile\", line 78, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1312, in mkdir\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1597853507.0
  },
  {
    "data_format": 1,
    "description": "Command line bioinformatics workflows, created with Snakemake workflow management tool.",
    "formatting": "[WARNING] In file \"/tmp/tmpbuyvlgak/Snakefile\":  Keyword \"input\" at line 56 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpbuyvlgak/Snakefile\":  Keyword \"shell\" at line 175 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AAFC-BICoE/snakemake-amplicon-metagenomics",
    "linting": "KeyError in line 18 of /tmp/tmpbuyvlgak/Snakefile:\n\u0027workdir\u0027\n  File \"/tmp/tmpbuyvlgak/Snakefile\", line 18, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 11,
    "topics": [],
    "updated_at": 1616791690.0
  },
  {
    "data_format": 1,
    "description": "Snakemake ProtoWorkflow for DNA Analysis. ",
    "formatting": "[WARNING] In file \"/tmp/tmp5be2q0er/rules/stats.rules.smk\":  Keyword \"input\" at line 12 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/stats.rules.smk\":  Keyword \"input\" at line 46 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/denovo.rules.smk\":  Keyword \"shell\" at line 84 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/varcall.rules.smk\":  Keyword \"shell\" at line 62 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/varcall.rules.smk\":  Keyword \"shell\" at line 103 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/align.rules.smk\":  Keyword \"shell\" at line 29 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/align.rules.smk\":  Keyword \"shell\" at line 71 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/align.rules.smk\":  Keyword \"shell\" at line 121 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/align.rules.smk\":  Keyword \"shell\" at line 208 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp5be2q0er/rules/align.rules.smk\":  Keyword \"shell\" at line 297 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mv-lab/dna-proto-workflow",
    "linting": "ModuleNotFoundError in line 5 of /tmp/tmp5be2q0er/rules/init.rules.smk:\nNo module named \u0027yamlordereddictloader\u0027\n  File \"/tmp/tmp5be2q0er/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmp5be2q0er/rules/init.rules.smk\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmp5be2q0er/utils/snkmk.py\", line 7, in \u003cmodule\u003e\n  File \"/tmp/tmp5be2q0er/utils/check_config.py\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "dna",
      "bioconda"
    ],
    "updated_at": 1606597344.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline to produce figures for \"Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios\"",
    "formatting": "UnicodeDecodeError: \u0027utf-8\u0027 codec can\u0027t decode byte 0xb0 in position 37: invalid start byte\n",
    "full_name": "marcjwilliams1/dnds-clonesize",
    "linting": "The flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nThe flag \u0027directory\u0027 used in rule all is only valid for outputs, not inputs.\nLints for snakefile /tmp/tmpa34sqetd/Snakefile:\n    * Absolute path \"/bin/bash\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Deprecated singularity directive used for container definition in line 10.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpa34sqetd/rules/CalculatedNdS-normal.smk:\n    * Deprecated singularity directive used for container definition in line 20.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 42.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 56.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 71.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpa34sqetd/rules/fitdNdSnormal.smk:\n    * Deprecated singularity directive used for container definition in line 21.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 44.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 61.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 95.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpa34sqetd/rules/GenerateFigures.smk:\n    * Deprecated singularity directive used for container definition in line 4.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 22.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 39.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 54.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 67.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 75.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 87.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 97.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 106.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 115.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 124.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 135.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpa34sqetd/rules/clonesizefit.smk:\n    * Deprecated singularity directive used for container definition in line 8.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 19.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 29.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 46.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 63.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 73.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ModellingNormalTissue (line 1, /tmp/tmpa34sqetd/rules/ModellingNormalTissue.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ModellingNormalTissueDistribution (line 37, /tmp/tmpa34sqetd/rules/ModellingNormalTissue.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ModellingNormalTissuePower (line 67, /tmp/tmpa34sqetd/rules/ModellingNormalTissue.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ModellingNormalTissueCloneSize (line 100, /tmp/tmpa34sqetd/rules/ModellingNormalTissue.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ModellingNormalTissueHitchikers (line 126, /tmp/tmpa34sqetd/rules/ModellingNormalTissue.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule CalculatedNdSNormal (line 1, /tmp/tmpa34sqetd/rules/CalculatedNdS-normal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CalculatedNdSNormalSNV (line 38, /tmp/tmpa34sqetd/rules/CalculatedNdS-normal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule MakeFilesSSB (line 75, /tmp/tmpa34sqetd/rules/CalculatedNdS-normal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule CalculateSitedNdSNormal (line 104, /tmp/tmpa34sqetd/rules/CalculatedNdS-normal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fitdNdSnormal (line 1, /tmp/tmpa34sqetd/rules/fitdNdSnormal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fitdNdSnormalSNV (line 37, /tmp/tmpa34sqetd/rules/fitdNdSnormal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule formatresultsSSB (line 73, /tmp/tmpa34sqetd/rules/fitdNdSnormal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fitdNdSnormalSSB (line 150, /tmp/tmpa34sqetd/rules/fitdNdSnormal.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figure1 (line 1, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figure2 (line 18, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figure3 (line 49, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figure4 (line 81, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figurecomparednds (line 111, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figurebinsize (line 137, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule Figure5 (line 158, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FigureCloneSizeData2 (line 183, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FigureCloneSizeSims (line 206, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FiguredNdSsimsdist (line 228, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FigureCloneSizeSimsDist (line 250, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule FigureHitchikers (line 272, /tmp/tmpa34sqetd/rules/GenerateFigures.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule brms (line 1, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule brmssites (line 26, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clonesizesims (line 52, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clonesizesimsdist (line 90, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clonesizedata (line 128, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clonesizedatacompare (line 153, /tmp/tmpa34sqetd/rules/clonesizefit.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1598708428.0
  },
  {
    "data_format": 1,
    "description": "Germline Variant Calling Pipeline built in Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "oskarvid/snakemake_germline_legacy",
    "linting": "Lints for snakefile /tmp/tmp7zivndiu/Snakefile:\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 17:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 101:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/dbsnp_146.hg38.vcf.gz\" in line 102:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/1000G_phase1.snps.high_confidence.hg38.vcf.gz\" in line 103:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz\" in line 104:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 137:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 157:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 188:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 221:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 250:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/dbsnp_146.hg38.vcf.gz\" in line 251:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/1000G_phase1.snps.high_confidence.hg38.vcf.gz\" in line 252:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/1000G_omni2.5.hg38.vcf.gz\" in line 253:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/hapmap_3.3.hg38.vcf.gz\" in line 254:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 280:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/dbsnp_146.hg38.vcf.gz\" in line 281:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz\" in line 282:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 306:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/references/Homo_sapiens_assembly38.fasta\" in line 326:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule BwaMem (line 21, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule FastqtoSam (line 48, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MergeBamAlignment (line 91, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MarkDup (line 158, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule BaseRecalibrator (line 182, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GatherBQSR (line 225, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule ApplyBQSR (line 244, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ApplyBQSRunmapped (line 283, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GatherBamFiles (line 323, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule HaplotypeCaller (line 343, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GatherVCFs (line 385, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule GenotypeGVCFs (line 405, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GatherVCFs2 (line 438, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule VariantRecalibratorSNP (line 458, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule VariantRecalibratorINDEL (line 515, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ApplyVqsrSnp (line 566, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ApplyVqsrIndel (line 606, /tmp/tmp7zivndiu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [
      "bwa",
      "snakemake",
      "pipeline"
    ],
    "updated_at": 1573430022.0
  },
  {
    "data_format": 1,
    "description": "A comprehensive quality-control and quantification RNA-seq pipeline",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "skchronicles/RNA-seek",
    "linting": "FileNotFoundError in line 9 of /tmp/tmp2ake4cnf/workflow/Snakefile:\n[Errno 2] No such file or directory: \u0027config.json\u0027\n  File \"/tmp/tmp2ake4cnf/workflow/Snakefile\", line 9, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [
      "rna-seq",
      "singularity",
      "quality-control",
      "gene-expression",
      "gene-fusions",
      "snakemake-pipeline"
    ],
    "updated_at": 1619215261.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": null,
    "full_name": "koesterlab/snakemake-workflow-sars-cov2",
    "linting": null,
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618939130.0
  },
  {
    "data_format": 1,
    "description": "RNA Biology Pipeline to Characterize protein-RNA Interactions",
    "formatting": "[WARNING] In file \"/tmp/tmp6tjk5ady/workflow/Snakefile\":  Keyword \"input\" at line 303 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp6tjk5ady/workflow/Snakefile\":  Keyword \"output\" at line 1290 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmp6tjk5ady/workflow/Snakefile\":  \u0027NoParametersError: L1378: In input definition.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "RBL-NCI/iCLIP",
    "linting": "KeyError in line 35 of /tmp/tmp6tjk5ady/workflow/Snakefile:\n\u0027source_dir\u0027\n  File \"/tmp/tmp6tjk5ady/workflow/Snakefile\", line 35, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1612792793.0
  },
  {
    "data_format": 1,
    "description": "minimal example to demonstrate flexible targeting in snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bluegenes/2020-flexible-targets-snakemake",
    "linting": "Lints for rule A (line 38, /tmp/tmpx5cq25tb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule B (line 59, /tmp/tmpx5cq25tb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1587592116.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "metagenome-atlas/genome-annotator",
    "linting": "OSError in line 10 of /tmp/tmpawck2xdf/Snakefile:\nneed a \u0027input\u0027 argument in the config file\n  File \"/tmp/tmpawck2xdf/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616758216.0
  },
  {
    "data_format": 1,
    "description": "quantify-virome: identify and quantify viruses from metagenomic shotgun sequences",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "avilab/quantify-virome",
    "linting": "FileNotFoundError in line 21 of /tmp/tmp1j_5oh6r/Snakefile:\n[Errno 2] No such file or directory: \u0027samples.tsv\u0027\n  File \"/tmp/tmp1j_5oh6r/Snakefile\", line 21, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflows",
      "virome",
      "metagenomics",
      "metagenomic-pipeline",
      "snakemake",
      "virome-workflow",
      "taxonomy"
    ],
    "updated_at": 1610656983.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for processing CNV-seq data.",
    "formatting": "[INFO] 8 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jrderuiter/snakemake-cnvseq",
    "linting": "ValueError in line 4 of /tmp/tmp34fqjytj/Snakefile:\nA config file must be provided using --configfile\n  File \"/tmp/tmp34fqjytj/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1611995550.0
  },
  {
    "data_format": 1,
    "description": "This is an human RNAseq variant calling workflow, following the GATK pipeline. Also includes ADAR-site elimination.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "durrantmm/rnaseq_variant_calling_workflow",
    "linting": "Lints for snakefile /tmp/tmp328lrm3k/Snakefile:\n    * Path composition with \u0027+\u0027 in line 254:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule all (line 39, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_genome_dictionary (line 52, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index_genome (line 72, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bgzip_dbsnp_vcf (line 92, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tabix_dbsnp_vcf (line 112, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_exons_bed (line 132, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_genome (line 153, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_align (line 185, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule add_read_groups (line 233, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_duplicates (line 253, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule split_reads (line 274, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule recalibrate_bases (line 297, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule print_reads (line 322, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule haplotype_caller (line 344, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule variant_quality_filter (line 365, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_adar_bed (line 386, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule filter_adar_sites (line 410, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_heterozygous_snps_only (line 431, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1580595897.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AlexanderLabWHOI/EUKHeist",
    "linting": "PermissionError in line 77 of /tmp/tmpi9yln2od/Snakefile:\n[Errno 13] Permission denied: \u0027/vortexfs1\u0027\n  File \"/tmp/tmpi9yln2od/Snakefile\", line 77, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1312, in mkdir\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618266767.0
  },
  {
    "data_format": 1,
    "description": "This is an human RNAseq variant calling workflow, following the GATK pipeline. Also includes ADAR-site elimination.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "durrantmm/rnaseq_variant_calling_workflow",
    "linting": "Lints for snakefile /tmp/tmp328lrm3k/Snakefile:\n    * Path composition with \u0027+\u0027 in line 254:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule all (line 39, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_genome_dictionary (line 52, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index_genome (line 72, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bgzip_dbsnp_vcf (line 92, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tabix_dbsnp_vcf (line 112, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_exons_bed (line 132, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_genome (line 153, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule star_align (line 185, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param out_prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule add_read_groups (line 233, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mark_duplicates (line 253, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule split_reads (line 274, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule recalibrate_bases (line 297, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule print_reads (line 322, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule haplotype_caller (line 344, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule variant_quality_filter (line 365, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule make_adar_bed (line 386, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule filter_adar_sites (line 410, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_heterozygous_snps_only (line 431, /tmp/tmp328lrm3k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1580595897.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AlexanderLabWHOI/EUKHeist",
    "linting": "PermissionError in line 77 of /tmp/tmpi9yln2od/Snakefile:\n[Errno 13] Permission denied: \u0027/vortexfs1\u0027\n  File \"/tmp/tmpi9yln2od/Snakefile\", line 77, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1316, in mkdir\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/pathlib.py\", line 1312, in mkdir\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618266767.0
  },
  {
    "data_format": 1,
    "description": "Unified sub-grid-scale parametrization using neural networks",
    "formatting": "[INFO] 2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nbren12/nn_atmos_param",
    "linting": "ModuleNotFoundError in line 3 of /tmp/tmpl4wtipsj/Snakefile:\nNo module named \u0027xarray\u0027\n  File \"/tmp/tmpl4wtipsj/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1607254069.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake pipeline to analyse RNA-Seq reads",
    "formatting": "[WARNING] In file \"/tmp/tmpknuwc4w7/Snakefile\":  Keyword \"input\" at line 74 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpknuwc4w7/Snakefile\":  Keyword \"shell\" at line 281 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "KoesGroup/Snakemake_hisat-DESeq",
    "linting": "Lints for snakefile /tmp/tmpknuwc4w7/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 102:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 120:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 121:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 125:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 148:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 154:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 162:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 164:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 165:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 191:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 192:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 194:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 207:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 225:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 226:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 247:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 268:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 271:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 150:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule get_genome_fasta (line 99, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable genome_url from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule get_transcriptome_gtf (line 120, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable transcriptome_gtf_url from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule index (line 192, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hisat_mapping (line 222, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param indexName is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule create_counts_table (line 265, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule DESeq2_analysis (line 294, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule results_function (line 329, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_for_plots (line 369, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_plots (line 410, /tmp/tmpknuwc4w7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 3,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1600240776.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake and Docker based pipeline to create a Neo4j graph. ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "elswob/neo4j-build-pipeline",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpf7zrey9b/workflow/Snakefile:\nNo module named \u0027environs\u0027\n  File \"/tmp/tmpf7zrey9b/workflow/Snakefile\", line 4, in \u003cmodule\u003e\n  File \"/tmp/tmpf7zrey9b/workflow/scripts/utils/settings.py\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [
      "neo4j",
      "docker",
      "snakemake",
      "python"
    ],
    "updated_at": 1616598321.0
  },
  {
    "data_format": 1,
    "description": "Barebones framework for creating new snakemake workflows.",
    "formatting": null,
    "full_name": "CCBR/CCBR_SnakemakePipelineCookiecutter",
    "linting": null,
    "stargazers_count": 3,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1615943784.0
  },
  {
    "data_format": 1,
    "description": "RNA-seq pipeline for AML diagnostics developed in collaboration with the LUMC Hematology department",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "LUMC/HAMLET",
    "linting": "KeyError in line 28 of /tmp/tmpcraq6pls/includes/qc-seq/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmpcraq6pls/Snakefile\", line 7, in \u003cmodule\u003e\n  File \"/tmp/tmpcraq6pls/includes/qc-seq/Snakefile\", line 59, in \u003cmodule\u003e\n  File \"/tmp/tmpcraq6pls/includes/qc-seq/Snakefile\", line 60, in \u003cgenexpr\u003e\n  File \"/tmp/tmpcraq6pls/includes/qc-seq/Snakefile\", line 28, in get_readgroup_per_sample\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1606137588.0
  },
  {
    "data_format": 1,
    "description": "single-cell sequencing of flu infected cells enriched for IFN+ ones",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jbloomlab/IFNsorted_flu_single_cell",
    "linting": "Lints for snakefile /tmp/tmpz906bayl/Snakefile:\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 15:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule make_paper (line 57, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PAPER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PAPER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PAPER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PAPER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PAPER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule analyze_IFN_vs_viral_mutations (line 96, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PAPER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable R_FIGS_SUBDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule call_PacBio_mutations (line 142, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PAPER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PYTHON_FIGS_SUBDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule get_PacBio_CCSs (line 189, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_PacBio_subreads (line 236, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_cellgene_matrix (line 254, /tmp/tmpz906bayl/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1606116910.0
  },
  {
    "data_format": 1,
    "description": "scRNA-seq analysis beyond gene annotations using transcriptionally active regions (TARs) generated from sequence alignment data",
    "formatting": "[ERROR] In file \"/tmp/tmp8yor4by0/Snakefile\":  \u0027EmptyContextError: L46: rule has no keywords attached to it.\u0027\n[INFO] In file \"/tmp/tmp8yor4by0/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "fw262/TAR-scRNA-seq",
    "linting": "Lints for snakefile /tmp/tmp8yor4by0/Snakefile:\n    * Absolute path \"/{sample}_R1.fastq.gz\" in line 104:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}_R2.fastq.gz\" in line 105:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/^#/ d\" in line 361:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule createGenomeDict (line 68, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generateStar (line 95, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable STAREXEC from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CORES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule convertToRefFlat1 (line 137, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable gtfToGenePred from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fastq_to_sam (line 166, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule stage1 (line 199, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sam_to_fastq (line 318, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule STAR_align (line 344, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable STAREXEC from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CORES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sort (line 385, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sortCoord (line 414, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule indexBamFiles (line 443, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mergeBamFiles (line 464, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calcHMMbed (line 489, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CORES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable MERGEBP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable THRESH from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule calcHMMrefFlat (line 513, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule MergeBamAlignment (line 535, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TMPDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PICARD from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable GENOMEREF from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule stage3 (line 571, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule stage3_withDir (line 605, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule stage3_noDir (line 639, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_expression (line 678, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule getCellsList (line 710, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_HMM_expression_withDir (line 739, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract_HMM_expression_noDir (line 771, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DROPSEQ from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SAMPLEWDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule getDiffMarkers (line 805, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule getDiffSeqsToBlast (line 829, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule getDiffSeqsToBlastFa (line 855, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ruleBlast (line 881, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CORES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule labelDiffuTARs (line 909, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule getMatsSteps (line 932, /tmp/tmp8yor4by0/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1614281905.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for reproduction of NealeLab 2018 UKB GWAS",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "related-sciences/ukb-gwas-pipeline-nealelab",
    "linting": "DefaultCredentialsError in line 6 of /tmp/tmpgt1n6p7f/Snakefile:\nCould not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started\n  File \"/tmp/tmpgt1n6p7f/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/snakemake/remote/GS.py\", line 124, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/storage/client.py\", line 121, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 249, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 201, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 216, in _determine_default\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/_helpers.py\", line 186, in _determine_default_project\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/auth/_default.py\", line 483, in default\n",
    "stargazers_count": 3,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1611231921.0
  },
  {
    "data_format": 1,
    "description": "Bulk Typing of Bacterial Species down to Strain Level",
    "formatting": "[ERROR] In file \"/tmp/tmp3_85g2fs/rules/LCPs.smk\":  \u0027SyntaxError: L80: Unrecognised keyword \u0027png1\u0027 in rule definition\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lauramilena3/On-rep-seq",
    "linting": "Lints for snakefile /tmp/tmp3_85g2fs/Snakefile:\n    * Absolute path \"/bin/bash\" in line 1:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/ON-rep-seq_DATA\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{sample}\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_plots.pdf\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.html\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.ipynb\" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/check.txt\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp3_85g2fs/rules/demultiplex.smk:\n    * Absolute path \"/01_porechopped_data/{sample}\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data/temporal_{barcode}.fastq\" in line 28:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data\" in line 30:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data/temporal_{barcode}.fastq\" in line 42:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data/{barcode}_demultiplexed.fastq\" in line 44:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data/rejected_{barcode}.fastq\" in line 46:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 26:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 46:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp3_85g2fs/rules/LCPs.smk:\n    * Absolute path \"/01_porechopped_data/{barcode}_demultiplexed.fastq\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/{barcode}.txt\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs\" in line 7:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/{barcode}.txt\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_plots.pdf\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\")[-1].split(\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_plots.pdf\" in line 50:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/{barcode}.txt\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/peaks_{barcode}.txt\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/peaks_{barcode}.pdf\" in line 57:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/{barcode}.txt\" in line 68:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.ipynb\" in line 70:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmaps.html\" in line 71:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmap_large.png\" in line 72:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_heatmap.png\" in line 73:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCP_clustering_flowgrams_clustering_order.pdf\" in line 74:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/LCPsClusteringData\" in line 75:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs/r_saved_images\" in line 76:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/02_LCPs\" in line 78:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*.txt \" in line 89:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 56:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 71:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 73:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 74:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 75:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 76:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp3_85g2fs/rules/peakCorrection.smk:\n    * Absolute path \"/03_LCPs_peaks/peaks_{barcode}.txt\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/input-peaks-{barcode}.txt\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/01_porechopped_data\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/input-peaks-{barcode}.txt\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fixed_{barcode}.txt\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/fixed_{barcode}.txt\" in line 54:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/vsearch_fixed_{barcode}.txt\" in line 56:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/00_peak_consensus\" in line 59:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 54:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp3_85g2fs/rules/taxonomyAssignment.smk:\n    * Absolute path \"/vsearch_fixed_{barcode}.txt\" in line 3:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/merged_fixed_{barcode}.fasta\" in line 5:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/01_taxonomic_assignments/taxonomy_{barcode}.txt\" in line 6:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/00_peak_consensus\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/01_taxonomic_assignments\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/01_taxonomic_assignments/taxonomy_assignments.txt\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks/01_taxonomic_assignments/taxonomy_{barcode}.txt\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/check.txt\" in line 31:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/03_LCPs_peaks\" in line 33:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 10:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 31:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule demultiplexing_1 (line 1, /tmp/tmp3_85g2fs/rules/demultiplex.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable BARCODES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule merge_first_demultiplexing (line 51, /tmp/tmp3_85g2fs/rules/demultiplex.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule demultiplexing_2 (line 87, /tmp/tmp3_85g2fs/rules/demultiplex.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getLCPs (line 1, /tmp/tmp3_85g2fs/rules/LCPs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plotLCPs (line 27, /tmp/tmp3_85g2fs/rules/LCPs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule peakPicker (line 74, /tmp/tmp3_85g2fs/rules/LCPs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule LCPsCluster (line 105, /tmp/tmp3_85g2fs/rules/LCPs.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param work_directory is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cutAdapt (line 1, /tmp/tmp3_85g2fs/rules/peakCorrection.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param peaks is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule correctReads (line 56, /tmp/tmp3_85g2fs/rules/peakCorrection.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule vSearch (line 110, /tmp/tmp3_85g2fs/rules/peakCorrection.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule taxonomyAssignment (line 1, /tmp/tmp3_85g2fs/rules/taxonomyAssignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param taxonomy is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule checkOutputs (line 54, /tmp/tmp3_85g2fs/rules/taxonomyAssignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param peaks is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 3,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1594993695.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 15 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "vindarbot/RNA_Seq_Pipeline",
    "linting": "Lints for snakefile /tmp/tmpnqz6dueh/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 45:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 48:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 63:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 66:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/get_ref_files.smk:\n    * Absolute path \"/,\"mitochondria\"); sub(/\u0027ChrC\" in line 108:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/,\"\");print}}\" in line 108:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 61:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/trimming.smk:\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 14:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/classic_mapping.smk:\n    * Path composition with \u0027+\u0027 in line 5:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/1st_pass.smk:\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 59:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/2sd_pass.smk:\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 7:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 57:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 58:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 116:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpnqz6dueh/rules/init_CSE.smk:\n    * Absolute path \"/ChromatineStateEnrichment/html/index.html\" in line 29:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule get_featureCounts (line 10, /tmp/tmpnqz6dueh/rules/get_packages.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule experimental_design (line 49, /tmp/tmpnqz6dueh/rules/get_ref_files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_reference_files (line 87, /tmp/tmpnqz6dueh/rules/get_ref_files.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trimming_PE (line 10, /tmp/tmpnqz6dueh/rules/trimming.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule indexation_genome (line 11, /tmp/tmpnqz6dueh/rules/classic_mapping.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mapping_PE (line 62, /tmp/tmpnqz6dueh/rules/classic_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_bam (line 151, /tmp/tmpnqz6dueh/rules/classic_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_bam (line 177, /tmp/tmpnqz6dueh/rules/classic_mapping.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule featureCounts_PE (line 13, /tmp/tmpnqz6dueh/rules/counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule FPKM (line 74, /tmp/tmpnqz6dueh/rules/counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule DESeq2 (line 11, /tmp/tmpnqz6dueh/rules/run_DESeq2.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outprefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule trimming_AS_PE (line 22, /tmp/tmpnqz6dueh/rules/trimming_AS.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_for_pass1 (line 28, /tmp/tmpnqz6dueh/rules/1st_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pass1_PE (line 74, /tmp/tmpnqz6dueh/rules/1st_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param direct is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule filt_SJ_out (line 153, /tmp/tmpnqz6dueh/rules/1st_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule SJ_db (line 181, /tmp/tmpnqz6dueh/rules/1st_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_for_pass2 (line 31, /tmp/tmpnqz6dueh/rules/2sd_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param genomedir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule pass2_PE (line 72, /tmp/tmpnqz6dueh/rules/2sd_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param direct is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_b_files (line 168, /tmp/tmpnqz6dueh/rules/2sd_pass.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule rMATS_PE (line 31, /tmp/tmpnqz6dueh/rules/run_rMATS.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rMATS_sig (line 21, /tmp/tmpnqz6dueh/rules/sign_DAS.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule salmon_index (line 17, /tmp/tmpnqz6dueh/rules/salmon_pseudomapping.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule salmon_mapping_PE (line 59, /tmp/tmpnqz6dueh/rules/salmon_pseudomapping.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule run_RATS (line 16, /tmp/tmpnqz6dueh/rules/run_RATs.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outprefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule get_CSE (line 13, /tmp/tmpnqz6dueh/rules/init_CSE.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_CSE (line 28, /tmp/tmpnqz6dueh/rules/init_CSE.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule format_DEG_CSE_results (line 63, /tmp/tmpnqz6dueh/rules/init_CSE.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1617701837.0
  },
  {
    "data_format": 1,
    "description": "Scripts and results for estimating the dates of integrated HIV viruses using RNA samples from the same host",
    "formatting": "[WARNING] In file \"/tmp/tmpn9b1zrb7/Snakefile\":  Keyword \"shell\" at line 107 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "veg/ogv-dating",
    "linting": "Lints for snakefile /tmp/tmpn9b1zrb7/Snakefile:\n    * Absolute path \"/usr/local/bin/hyphy\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/usr/local/bin/muscle\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/usr/local/bin/raxml-ng\" in line 14:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/usr/local/bin/FastTree\" in line 15:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule read_qc (line 50, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_protein_msa_rna (line 79, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_protein_msa_combined (line 103, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_codon_msa_rna (line 127, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_codon_msa_combined (line 152, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule infer_ml_rna_tree (line 177, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule infer_ml_tree (line 206, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule perform_placement (line 235, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule perform_classification (line 269, /tmp/tmpn9b1zrb7/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1613170116.0
  },
  {
    "data_format": 1,
    "description": "Mutational antigenic profiling of Crowe-lab antibodies to SARS-CoV-2 RBD",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_MAP_Crowe_antibodies",
    "linting": "Lints for snakefile /tmp/tmpv1keff58/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule make_summary (line 47, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule make_dag (line 124, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule escape_selections (line 144, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_supp_data (line 171, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule crowe_evol (line 197, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule output_pdbs (line 236, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule circulating_variants (line 270, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mds_escape_profiles (line 305, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule analyze_escape_profiles (line 332, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule scores_to_frac_escape (line 363, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule counts_to_scores (line 392, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_variants (line 424, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_codon_variant_table (line 452, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_mut_bind_expr (line 466, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_variant_expr (line 479, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_variant_bind (line 492, /tmp/tmpv1keff58/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1616871931.0
  },
  {
    "data_format": 1,
    "description": "Code for market value paper with Lina Reichenberg",
    "formatting": "[ERROR] In file \"/tmp/tmpa_eskgeb/Snakefile\":  \u0027NoParametersError: L42: In input definition.\u0027\n[INFO] In file \"/tmp/tmpa_eskgeb/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "nworbmot/market-value",
    "linting": "Lints for snakefile /tmp/tmpa_eskgeb/Snakefile:\n    * Absolute path \"/\" + config[\u0027run\u0027] + \" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + config[\u0027run\u0027] + \" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027 + config[\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/configs/config.yaml\" in line 18:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + config[\u0027run\u0027] + \" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + config[\u0027run\u0027] + \" in line 32:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\" + config[\u0027run\u0027] + \" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/\u0027 + config[\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/configs/config.yaml\" in line 43:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 9:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule plot_summary (line 9, /tmp/tmpa_eskgeb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 33, /tmp/tmpa_eskgeb/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_summary (line 64, /tmp/tmpa_eskgeb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule copy_config (line 89, /tmp/tmpa_eskgeb/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1584096057.0
  },
  {
    "data_format": 1,
    "description": "CallVars is an automated NGS workflow that takes paired-end sample FastQ\u0027s directly to a handful clinically relevant variants (SNPs+Indels) of interest.  ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "IAnalyzeGenomes/CallVars",
    "linting": "Lints for snakefile /tmp/tmpen1sw4y2/rules/Funcotator.py:\n    * Absolute path \"/ --ref-version \" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule BWA_Mapping (line 3, /tmp/tmpen1sw4y2/rules/Mapping.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule SAMTOOLS_Sort (line 3, /tmp/tmpen1sw4y2/rules/BamPrep.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GATK_MarkDuplicates (line 34, /tmp/tmpen1sw4y2/rules/BamPrep.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule SAMTOOLS_Index (line 60, /tmp/tmpen1sw4y2/rules/BamPrep.py):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule SAMTOOLS_BCF (line 3, /tmp/tmpen1sw4y2/rules/SamtoolsVC.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule SAMTOOLS_VCF (line 35, /tmp/tmpen1sw4y2/rules/SamtoolsVC.py):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule Bedtools_PerBaseCov (line 2, /tmp/tmpen1sw4y2/rules/PerBaseCov.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GATK_HaplotypeCaller (line 3, /tmp/tmpen1sw4y2/rules/HaplotypeCaller.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule GATK_Funcotator_Germline (line 3, /tmp/tmpen1sw4y2/rules/Funcotator.py):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1617629619.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow for random forest based feature selection on chromHMM data  ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-chromForest",
    "linting": "Lints for rule tile_seg (line 36, /tmp/tmpqc0ymvrc/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule state_recode_seg (line 66, /tmp/tmpqc0ymvrc/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule combine_sample_segs (line 104, /tmp/tmpqc0ymvrc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_epilogos_input_by_chr (line 132, /tmp/tmpqc0ymvrc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_epilogos_by_chr (line 163, /tmp/tmpqc0ymvrc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge_epilogos (line 194, /tmp/tmpqc0ymvrc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 3,
    "subscribers_count": 1,
    "topics": [
      "snakemake-workflows",
      "chromhmm",
      "epigenomics",
      "machine-learning"
    ],
    "updated_at": 1539946273.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow for bulk RNA-seq workflow using STAR-edgeR ",
    "formatting": "[WARNING] In file \"/tmp/tmpxk0ls1sw/Snakefile\":  Keyword \"input\" at line 84 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpxk0ls1sw/Snakefile\":  \u0027NoParametersError: L91: In input definition.\u0027\n[INFO] In file \"/tmp/tmpxk0ls1sw/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "vari-bbc/rnaseq_workflow",
    "linting": "Lints for snakefile /tmp/tmpxk0ls1sw/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 405:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 532:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule mergeLanesAndRename_SE (line 96, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mergeLanesAndRename_PE (line 127, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_PE (line 168, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_screen_SE (line 216, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore_PE (line 268, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule trim_galore_SE (line 337, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule STAR (line 409, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param outprefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule multiqc (line 504, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule edgeR (line 569, /tmp/tmpxk0ls1sw/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule addRG (line 610, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule markdups (line 677, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule splitncigar (line 735, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_recalibrate (line 792, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule applyBQSR (line 855, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule haplotypecaller (line 917, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combinevar (line 989, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule jointgeno (line 1049, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_and_filter_vcf (line 1106, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule variant_annot (line 1235, /tmp/tmpxk0ls1sw/Snakefile):\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snprelate (line 1333, /tmp/tmpxk0ls1sw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 4,
    "subscribers_count": 3,
    "topics": [
      "snakemake",
      "rnaseq",
      "pipeline"
    ],
    "updated_at": 1615408546.0
  },
  {
    "data_format": 1,
    "description": "A complete Snakemake pipeline for detecting allele specific expression",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "aryarm/as_analysis",
    "linting": "FileNotFoundError in line 27 of /tmp/tmpabmu387g/Snakefile:\n[Errno 2] No such file or directory: \u0027/iblm/netapp/home/amassarat/allele_specific_analysis/snakemake/samples.tsv\u0027\n  File \"/tmp/tmpabmu387g/Snakefile\", line 44, in \u003cmodule\u003e\n  File \"/tmp/tmpabmu387g/Snakefile\", line 27, in read_samples\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "allele-specific-expression",
      "rna-seq",
      "atac-seq",
      "whole-genome-sequencing"
    ],
    "updated_at": 1617738010.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ctb/2018-snakemake-cli",
    "linting": "KeyError in line 1 of /tmp/tmpk_ppwo25/Snakefile:\n\u0027name\u0027\n  File \"/tmp/tmpk_ppwo25/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1595453385.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpkicsr1qu/Snakefile\":  Inline-formatted keyword \"singularity\" at line 33 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpkicsr1qu/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 3:7: params:\n```\nwhile formatting code with black.\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nathanhaigh/snakemake-tutorial",
    "linting": "Lints for snakefile /tmp/tmpkicsr1qu/Snakefile:\n    * Path composition with \u0027+\u0027 in line 135:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 210:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 112:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 33.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_chromosome (line 5, /tmp/tmpkicsr1qu/rules/setup_data.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_chromosome_subregion (line 44, /tmp/tmpkicsr1qu/rules/setup_data.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule extract_reads (line 83, /tmp/tmpkicsr1qu/rules/setup_data.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule index_fasta (line 129, /tmp/tmpkicsr1qu/rules/setup_data.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc_raw (line 94, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multiqc_raw (line 130, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_trimmomatic_pe_adapters (line 161, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trimmomatic_pe (line 188, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc_trimmed (line 231, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule multiqc_trimmed (line 263, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_index (line 294, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bwa_mem (line 340, /tmp/tmpkicsr1qu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param index is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1619259171.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for processing exome data.",
    "formatting": "[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jrderuiter/snakemake-exome",
    "linting": "Lints for snakefile /tmp/tmp442uomcs/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 3:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmp442uomcs/rules/input.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp442uomcs/rules/alignment.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmp442uomcs/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule copy_input (line 56, /tmp/tmp442uomcs/rules/input.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_merge (line 234, /tmp/tmp442uomcs/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 302, /tmp/tmp442uomcs/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 52, /tmp/tmp442uomcs/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_stats (line 77, /tmp/tmp442uomcs/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1573224353.0
  },
  {
    "data_format": 1,
    "description": "Transcriptome  assembly via digital normalization with khmer and Trinity",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/smsk_khmer_trinity",
    "linting": "AttributeError in line 26 of /tmp/tmpefwc2ueu/src/snakefiles/raw.py:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmpefwc2ueu/Snakefile\", line 37, in \u003cmodule\u003e\n  File \"/tmp/tmpefwc2ueu/src/snakefiles/raw.py\", line 26, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 3,
    "topics": [
      "transcriptome-assembly",
      "ngs",
      "trinity",
      "khmer",
      "digital-normalization",
      "assembler",
      "transrate",
      "snakemake"
    ],
    "updated_at": 1595859289.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "mohimanilab/AssociationNetworks",
    "linting": "Lints for rule decoy_database (line 18, /tmp/tmp7curevk5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule associations_serial (line 42, /tmp/tmp7curevk5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule associations_decoy_serial (line 68, /tmp/tmp7curevk5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fdr (line 243, /tmp/tmp7curevk5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tree_biclustering (line 266, /tmp/tmp7curevk5/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1575273758.0
  },
  {
    "data_format": 1,
    "description": "PyPSA Model of the South African Energy System",
    "formatting": "[WARNING] In file \"/tmp/tmpr6uilntt/Snakefile\":  Keyword \"input\" at line 99 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PyPSA/pypsa-za",
    "linting": "Lints for snakefile /tmp/tmpr6uilntt/Snakefile:\n    * Absolute path \"/plots/scenario_{param}.html\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/networks/{cost}_{resarea}_{sectors}_{opts}.nc\" in line 109:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/networks/{cost}_{resarea}_{sectors}_{opts}.nc\" in line 121:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/plots/network_{cost}_{resarea}_{sectors}_{opts}_{attr}\" in line 125:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/plots/network_{cost}_{resarea}_{sectors}_{opts}_{attr}_ext\" in line 126:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/plots/scenario_{param}.html\" in line 137:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/plots\" in line 140:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/networks/{cost}_{resarea}_{sectors}_{opts}.nc\" in line 145:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/summaries/{cost}_{resarea}_{sectors}_{opts}\" in line 151:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 13:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 121:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 125:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 126:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 141:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 145:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule build_landuse_remove_protected_and_conservation_areas (line 25, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_landuse_map_to_tech_and_supply_region (line 53, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_population (line 83, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_topology (line 133, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_network (line 157, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_electricity (line 185, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_sectors (line 220, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 244, /tmp/tmpr6uilntt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_network (line 277, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule scenario_comparison (line 301, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param plot_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_summary (line 333, /tmp/tmpr6uilntt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 6,
    "topics": [],
    "updated_at": 1611705213.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow to download and/or align reads to targets and produce useful outputs.",
    "formatting": "[ERROR] In file \"/tmp/tmpg_icq0lo/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 28:0: \u003cline number missing in source\u003e\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpg_icq0lo/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "louiejtaylor/hisss",
    "linting": "KeyError in line 11 of /tmp/tmpg_icq0lo/Snakefile:\n\u0027io\u0027\n  File \"/tmp/tmpg_icq0lo/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1601902657.0
  },
  {
    "data_format": 1,
    "description": "discover-virome: identify and quantify viruses from metagenomic assemblies",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 5 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "avilab/discover-virome",
    "linting": "FileNotFoundError in line 24 of /tmp/tmplflhrbw9/Snakefile:\n[Errno 2] No such file or directory: \u0027samples.tsv\u0027\n  File \"/tmp/tmplflhrbw9/Snakefile\", line 24, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [
      "virome",
      "virome-workflow",
      "snakemake",
      "snakemake-workflow",
      "metagenomic-analysis",
      "metagenomic-classification"
    ],
    "updated_at": 1611671697.0
  },
  {
    "data_format": 1,
    "description": "snakemake pipeline for Hi-C post-processing",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n[INFO] 2 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "open2c/quaich",
    "linting": "ModuleNotFoundError in line 5 of /tmp/tmpkd4ovbmv/workflow/Snakefile:\nNo module named \u0027cooler\u0027\n  File \"/tmp/tmpkd4ovbmv/workflow/Snakefile\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [
      "workflow",
      "snakemake",
      "analysis",
      "hi-c"
    ],
    "updated_at": 1616348021.0
  },
  {
    "data_format": 1,
    "description": ":snake::hamburger: A beginner\u0027s guide to modular snakemake workflows.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dohlee/snakemake-hamburger",
    "linting": "Lints for rule make_hamburger (line 11, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_tomato (line 36, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule wash_tomato (line 51, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_cucumber (line 70, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_vinegar (line 85, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_vinegar (line 100, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_onion (line 120, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule wash_onion (line 135, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule roast_onion (line 154, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_cheese (line 173, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_bun (line 188, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule warm_bun (line 203, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_lettuce (line 222, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule wash_lettuce (line 237, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule buy_beef (line 256, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_patty (line 271, /tmp/tmpu56_t5f3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 0,
    "topics": [
      "snakemake",
      "snakemake-workflow",
      "reproducible-research",
      "bioinformatics-pipeline"
    ],
    "updated_at": 1609773164.0
  },
  {
    "data_format": 1,
    "description": "Brief tutorial for using Snakemake to create and submit jobs on a HPC cluster.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SchlossLab/snakemake_cluster_tutorial",
    "linting": "Lints for rule clean (line 42, /tmp/tmpg4_t25j3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_words (line 64, /tmp/tmpg4_t25j3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_plot (line 86, /tmp/tmpg4_t25j3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule zipf_test (line 108, /tmp/tmpg4_t25j3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_archive (line 134, /tmp/tmpg4_t25j3/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614188537.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "darneson/DropSeq",
    "linting": "Lints for snakefile /tmp/tmpb_vowibm/Snakefile:\n    * Absolute path \"/PATH/TO/STAR/INDEX/FOR/GENOME\" in line 19:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PATH/TO/REFERENCE/GENOME/FASTA/FILE\" in line 20:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PATH/TO/REFERENCE/GENOME/GTF/FILE\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PATH/TO/DROPSEQTOOLS/FOLDER\" in line 24:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/3rdParty/picard/picard.jar\" in line 34:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PATH/TO/STAR/EXECUTABLE\" in line 35:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PATH/TO/JAVA/EXECUTABLE\" in line 36:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 151:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 196:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 211:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 224:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 238:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 255:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 68:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 82:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 95:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 123:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 137:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 151:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 182:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 196:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 211:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 224:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 238:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule fastq_to_bam (line 50, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JAVA_distro from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable picard_jar from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tag_cells (line 91, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TagBamWithReadSequenceExtended from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tag_molecules (line 123, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TagBamWithReadSequenceExtended from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule filter_bam (line 155, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable FilterBAM from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_starting_sequence (line 186, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TrimStartingSequence from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_poly_a (line 218, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PolyATrimmer from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sam_to_fastq (line 251, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable picard_jar from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule star_alignment (line 282, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable STAR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sort_aligned (line 315, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable picard_jar from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule merge_bam (line 348, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable picard_jar from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tag_with_gene_exon (line 381, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable TagReadWithGeneExon from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule synth_err (line 413, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DetectBeadSynthesisErrors from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Number_Barcodes from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule duplicate_umi (line 446, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable GatherMolecularBarcodeDistributionByGene from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tag_hist (line 477, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable BAMTagHistogram from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule dig_expr (line 510, /tmp/tmpb_vowibm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DigitalExpression from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable Number_Core_Barcodes from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 4,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1579740374.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "taylorreiter/olive_genome",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmp3sid5c8y/blast.snakefile:\nNo module named \u0027pysam\u0027\n  File \"/tmp/tmp3sid5c8y/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmp3sid5c8y/blast.snakefile\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1569622662.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake pipeline for cellranger to process 10x single-cell RNAseq data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-cellranger",
    "linting": "Lints for snakefile /tmp/tmpdo5iykqj/Snakefile:\n    * Absolute path \"/RunInfo.xml\" in line 21:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/RunInfo.xml\" in line 86:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/outs/fastq_path/\" + FILES[wildcards.sample][batch][1] + \" in line 125:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 99:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 119:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule download_raw (line 64, /tmp/tmpdo5iykqj/Snakefile):\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n    * Param jobname is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule make_fastq (line 118, /tmp/tmpdo5iykqj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count (line 177, /tmp/tmpdo5iykqj/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1583491203.0
  },
  {
    "data_format": 1,
    "description": "WES HLA Typing based on multiple alternative tools",
    "formatting": "[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lkuchenb/MultiHLA",
    "linting": "Lints for snakefile /tmp/tmpvai4bz15/workflow/rules/multihla.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/trim.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/map/bwa.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 49:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/map/star.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/hla_vbseq.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/xhla.smk:\n    * Absolute path \"/usr/bin/diamond blastx -p {threads} \"$@\"\" in line 58:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/hla_la.smk:\n    * Absolute path \"/{input:q} \u0026\u003e\u003e\" in line 52:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/opt/hla-la/bin/HLA-LA --action prepareGraph --PRG_graph_dir \" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/PRG_MHC_GRCh38_withIMGT \u0026\u003e\u003e\" in line 55:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/opt/hla-la/src/* . \u0026\u003e\u003e\" in line 93:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/opt/hla-la/src/HLA-LA.pl . \u0026\u003e\u003e\" in line 95:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/opt/hla-la/bin/HLA-LA bin/ \u0026\u003e\u003e\" in line 99:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for snakefile /tmp/tmpvai4bz15/workflow/rules/arcasHLA.smk:\n    * Absolute path \"/{input:q} \u0026\u003e\u003e\" in line 45:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule xhla_conversion (line 75, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule hla_xhla_dataset (line 120, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hla_la_conversion (line 153, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule hla_la_dataset (line 198, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule hla_vbseq_conversion (line 231, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule hla_vbseq_dataset (line 268, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule optitype_conversion (line 302, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule optitype_dataset (line 348, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collect (line 382, /tmp/tmpvai4bz15/workflow/rules/multihla.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ucsc_hg_download (line 24, /tmp/tmpvai4bz15/workflow/rules/ucsc_hg_download.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule compare (line 21, /tmp/tmpvai4bz15/workflow/rules/compare.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trim (line 37, /tmp/tmpvai4bz15/workflow/rules/trim.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule bwa_index (line 25, /tmp/tmpvai4bz15/workflow/rules/map/bwa.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_index (line 70, /tmp/tmpvai4bz15/workflow/rules/map/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bwa_mem_dataset_collector (line 150, /tmp/tmpvai4bz15/workflow/rules/map/bwa.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 4,
    "topics": [
      "workflow",
      "immunology",
      "immunoinformatics",
      "sequencing",
      "exome",
      "genome"
    ],
    "updated_at": 1616231374.0
  },
  {
    "data_format": 1,
    "description": "Viral VDAP: a viral alignment, variant discovery, and annotation pipeline launched at the NCBI-Hackathon 2019",
    "formatting": "[INFO] 10 file(s) would be changed \ud83d\ude2c\n[INFO] 1 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "NCBI-Hackathons/Viral-VDAP",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmpop84jtxn/rules/common.smk:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpop84jtxn/Snakefile\", line 1, in \u003cmodule\u003e\n  File \"/tmp/tmpop84jtxn/rules/common.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 7,
    "topics": [
      "womenled-nih-2019"
    ],
    "updated_at": 1613063565.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for modelling-to-generate-alternatives with PyPSA-Eur",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PyPSA/pypsa-eur-mga",
    "linting": "Lints for snakefile /tmp/tmp9xta_mcm/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule solve_base (line 34, /tmp/tmp9xta_mcm/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_list_of_alternatives (line 83, /tmp/tmp9xta_mcm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule generate_alternative (line 99, /tmp/tmp9xta_mcm/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule extract_results (line 201, /tmp/tmp9xta_mcm/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1613402443.0
  },
  {
    "data_format": 1,
    "description": "GIAB pipeline for benchmarking genome assemblies",
    "formatting": "[WARNING] In file \"/tmp/tmpz93_1284/workflow/Snakefile\":  Keyword \"input\" at line 249 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpz93_1284/workflow/Snakefile\":  Keyword \"input\" at line 307 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpz93_1284/workflow/Snakefile\":  Keyword \"input\" at line 349 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "usnistgov/giab-asm-benchmarking",
    "linting": "AttributeError in line 304 of /tmp/tmpz93_1284/workflow/Snakefile:\n\u0027Rules\u0027 object has no attribute \u0027get_benchmark_bed_index\u0027\n  File \"/tmp/tmpz93_1284/workflow/Snakefile\", line 304, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1616102318.0
  },
  {
    "data_format": 1,
    "description": "Characterizing the neurogenomics of parental care in the rock dove",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "macmanes-lab/DoveParentsRNAseq",
    "linting": "Lints for rule wrangle (line 1, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fig1 (line 22, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule deseq2 (line 42, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule vsd (line 66, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule degs (line 88, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule figs23 (line 110, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule figsup1 (line 131, /tmp/tmpvzmmd4q4/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 4,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1617146094.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NCBI-Codeathons/MASCARA",
    "linting": "Lints for snakefile /tmp/tmp69x4tvak/Snakefile:\n    * Deprecated singularity directive used for container definition in line 12.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 28.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 41.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 54.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 69.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule actionet_rna (line 13, /tmp/tmp69x4tvak/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cicero (line 41, /tmp/tmp69x4tvak/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule chromVAR (line 74, /tmp/tmp69x4tvak/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule actionet_inferred_rna (line 102, /tmp/tmp69x4tvak/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule run_match_cellstates (line 130, /tmp/tmp69x4tvak/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 4,
    "subscribers_count": 8,
    "topics": [
      "sc-nygc-jan2020",
      "cshl-codeathon-2020"
    ],
    "updated_at": 1616183984.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for Somatic Variant Calling with WES and WGS data",
    "formatting": "[WARNING] In file \"/tmp/tmpawult3vz/rules/preprocessing.smk\":  Keyword \"input\" at line 81 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/preprocessing.smk\":  Keyword \"input\" at line 148 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 40 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 86 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 101 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 161 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"shell\" at line 177 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tjbencomo/ngs-pipeline",
    "linting": "FileNotFoundError in line 17 of /tmp/tmpawult3vz/rules/common.smk:\n[Errno 2] No such file or directory: \u0027patients_test.csv\u0027\n  File \"/tmp/tmpawult3vz/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmpawult3vz/rules/common.smk\", line 17, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "bioinformatics",
      "somatic-variants",
      "exome-sequencing"
    ],
    "updated_at": 1611946910.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for Somatic Variant Calling with WES and WGS data",
    "formatting": "[WARNING] In file \"/tmp/tmpawult3vz/rules/preprocessing.smk\":  Keyword \"input\" at line 81 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/preprocessing.smk\":  Keyword \"input\" at line 148 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 40 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 86 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 101 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"input\" at line 161 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpawult3vz/rules/calling.smk\":  Keyword \"shell\" at line 177 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tjbencomo/ngs-pipeline",
    "linting": "FileNotFoundError in line 17 of /tmp/tmpawult3vz/rules/common.smk:\n[Errno 2] No such file or directory: \u0027patients_test.csv\u0027\n  File \"/tmp/tmpawult3vz/Snakefile\", line 2, in \u003cmodule\u003e\n  File \"/tmp/tmpawult3vz/rules/common.smk\", line 17, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 610, in read_csv\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 4,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "bioinformatics",
      "somatic-variants",
      "exome-sequencing"
    ],
    "updated_at": 1611946910.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow comparing Genrich and MACS2",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/Genrich_compare",
    "linting": "FileNotFoundError in line 10 of /tmp/tmpcs34tk21/Snakefile:\n[Errno 2] No such file or directory: \u0027./samples.json\u0027\n  File \"/tmp/tmpcs34tk21/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 4,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1604895310.0
  },
  {
    "data_format": 1,
    "description": "PEDIA-workflow prioritizes the disease-causing gene by integrating facial analysis and exome sequencing data.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PEDIA-Charite/PEDIA-workflow",
    "linting": "Lints for snakefile /tmp/tmp09p2_jnt/Snakefile:\n    * Path composition with \u0027+\u0027 in line 1:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule decompress (line 34, /tmp/tmp09p2_jnt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort (line 57, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule index_sorted (line 83, /tmp/tmp09p2_jnt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter (line 102, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable data_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule index_filter (line 128, /tmp/tmp09p2_jnt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule annotate (line 151, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable data_path from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule index_annotated (line 181, /tmp/tmp09p2_jnt/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule json (line 200, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable sample_index from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule test (line 232, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable classify_file from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable param_c from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable train_pickle from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable exclude_pheno from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule map_pedia (line 264, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable mapping_file from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule map_vcf (line 295, /tmp/tmp09p2_jnt/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable mapping_vcf_file from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\n",
    "stargazers_count": 5,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1600865596.0
  },
  {
    "data_format": 1,
    "description": "cluster stability measurement by subsampling and reclustering with Seurat V3 and V4",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow_seurat_parameter",
    "linting": "Lints for snakefile /tmp/tmpfoyu0o61/Snakefile:\n    * Deprecated singularity directive used for container definition in line 36.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 47.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 57.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 70.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 5,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1619407013.0
  },
  {
    "data_format": 1,
    "description": "A snakemake workflow for metagenomic projects",
    "formatting": "[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NBISweden/nbis-meta",
    "linting": "Lints for snakefile /tmp/tmppsondrq_/workflow/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmppsondrq_/workflow/rules/common.smk:\n    * Deprecated singularity directive used for container definition in line 10.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmppsondrq_/workflow/rules/classification.smk:\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 100:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 148:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 183:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 264:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 292:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 25:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 72:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 100:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 148:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 183:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 264:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 380.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 402.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 427.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmppsondrq_/workflow/rules/preprocessing.smk:\n    * Path composition with \u0027+\u0027 in line 637:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 41:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 298:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 369:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 396:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 467:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 506:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 547:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 578:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmppsondrq_/workflow/rules/quantification.smk:\n    * Path composition with \u0027+\u0027 in line 83:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 16:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule infernal (line 205, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Param db is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule pfam_scan (line 369, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_pfam (line 418, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule emapper_homology_search (line 508, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param resource_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule emapper_annotate_hits (line 634, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_ko_annotations (line 686, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Param outbase is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param resource_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rgi (line 769, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * Param out is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_rgi (line 829, /tmp/tmppsondrq_/workflow/rules/annotation.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule megahit (line 218, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param output_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule fasta2bed (line 337, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bowtie_build (line 362, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bowtie_map_pe (line 407, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bowtie_map_se (line 467, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule samtools_flagstat (line 529, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assembly_stats (line 581, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_assembly_stats (line 605, /tmp/tmppsondrq_/workflow/rules/assembly.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule concoct_coverage_table (line 238, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_cutup (line 385, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule extract_fasta (line 419, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule contig_map (line 473, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule binning_stats (line 500, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_binning_stats (line 525, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule remove_checkm_zerocols (line 849, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_checkm_profiles (line 927, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_checkm_stats (line 947, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_gtdbtk (line 1064, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule count_rRNA (line 1180, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule count_tRNA (line 1286, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_bin_annot (line 1309, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_ref_genome (line 1339, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_fastANI_lists (line 1358, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastANI (line 1393, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule cluster_genomes (line 1444, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule binning_report (line 1475, /tmp/tmppsondrq_/workflow/rules/binning.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule kraken_pe (line 117, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param db is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kraken_se (line 170, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param db is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule centrifuge_pe (line 261, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule centrifuge_se (line 324, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule centrifuge_kreport (line 385, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule build_metaphlan (line 427, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule metaphlan_pe (line 474, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule metaphlan_se (line 527, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule merge_metaphlan (line 578, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule metaphlan2krona_table (line 605, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_metaphlan (line 665, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule classifier2krona (line 733, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param tax is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule all2krona (line 778, /tmp/tmppsondrq_/workflow/rules/classification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * Param tax is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule link_files (line 31, /tmp/tmppsondrq_/workflow/rules/preprocessing.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sortmerna_link_pe (line 538, /tmp/tmppsondrq_/workflow/rules/preprocessing.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sortmerna_link_se (line 560, /tmp/tmppsondrq_/workflow/rules/preprocessing.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filter_phix_pe (line 922, /tmp/tmppsondrq_/workflow/rules/preprocessing.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule filter_phix_se (line 997, /tmp/tmppsondrq_/workflow/rules/preprocessing.smk):\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule write_featurefile (line 28, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule remove_mark_duplicates (line 51, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule featurecount (line 137, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n\nLints for rule clean_featurecount (line 191, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_featurecount (line 212, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule count_features (line 270, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sum_to_taxa (line 360, /tmp/tmppsondrq_/workflow/rules/quantification.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule contigtax_assign (line 350, /tmp/tmppsondrq_/workflow/rules/taxonomy.smk):\n    * Param taxdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule contigtax_assign_orfs (line 547, /tmp/tmppsondrq_/workflow/rules/taxonomy.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 5,
    "subscribers_count": 8,
    "topics": [
      "metagenomics",
      "snakemake",
      "workflow",
      "assembly"
    ],
    "updated_at": 1612908430.0
  },
  {
    "data_format": 1,
    "description": "Data analysis related to the bioconda paper",
    "formatting": "[WARNING] In file \"/tmp/tmpuwc7ohin/Snakefile\":  Keyword \"shell\" at line 86 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bioconda/bioconda-paper",
    "linting": "Lints for snakefile /tmp/tmpuwc7ohin/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_package_data (line 49, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_package_data (line 77, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clone (line 101, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule git_log (line 127, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_dag (line 167, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule parse_git_log (line 191, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_pr_data (line 215, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule collect_summaries (line 238, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_adddel (line 261, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_package_degree (line 281, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_dag (line 305, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_colored_dag (line 335, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_downloads (line 375, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_ecosystems (line 400, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_comparison (line 426, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_contributions (line 452, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_age_vs_downloads (line 476, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_turnaround (line 501, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule stats (line 527, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule author_list (line 552, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule author_list_tex (line 576, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fig1 (line 603, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fig2 (line 630, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_svg (line 659, /tmp/tmpuwc7ohin/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 5,
    "subscribers_count": 16,
    "topics": [
      "snakemake",
      "reproducibility",
      "bioconda",
      "analysis-workflows"
    ],
    "updated_at": 1619024958.0
  },
  {
    "data_format": 1,
    "description": "A showcase and characterisation of pipeline frameworks (aka workflow managers)",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "komparo/rosetta-pipeline",
    "linting": "Lints for snakefile /tmp/tmpypxvt931/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 29:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule framework_docker (line 57, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule task_docker (line 82, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_example (line 107, /tmp/tmpypxvt931/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregate_tasks (line 173, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_readme (line 192, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_task_readmes (line 210, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule render_comparison (line 229, /tmp/tmpypxvt931/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 5,
    "subscribers_count": 1,
    "topics": [
      "workflow-management",
      "pipeline-framework",
      "workflow-engine",
      "rosettacode"
    ],
    "updated_at": 1609432839.0
  },
  {
    "data_format": 1,
    "description": "Benchmarking of CNV calling tools",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cphgeno/CNVbench",
    "linting": "SyntaxError:\nInput and output files have to be specified as strings or lists of strings.\n  File \"/tmp/tmpwv7kw90f/Snakefile\", line 44, in \u003cmodule\u003e\n  File \"/tmp/tmpwv7kw90f/tools/align.smk\", line 176, in \u003cmodule\u003e\n",
    "stargazers_count": 5,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1607055579.0
  },
  {
    "data_format": 1,
    "description": "Indels are not ideal - quick test for interrupted ORFs in bacterial/microbial genomes",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "phiweger/ideel",
    "linting": "KeyError in line 6 of /tmp/tmpl0elcx3n/Snakefile:\n\u0027threads\u0027\n  File \"/tmp/tmpl0elcx3n/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 5,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1605624407.0
  },
  {
    "data_format": 1,
    "description": ":cake: Per sequence functional classification and taxonomic assignments",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PNNL-CompBio/perseq",
    "linting": "\u0027data\u0027 dir with FASTQs has not been set; pass --config data=/path\nSystemExit in line 73 of /tmp/tmpztyucfzh/Snakefile:\n1\n  File \"/tmp/tmpztyucfzh/Snakefile\", line 184, in \u003cmodule\u003e\n  File \"/tmp/tmpztyucfzh/Snakefile\", line 73, in get_samples_from_dir\n",
    "stargazers_count": 5,
    "subscribers_count": 2,
    "topics": [
      "metagenomics",
      "metatranscriptomics"
    ],
    "updated_at": 1602644548.0
  },
  {
    "data_format": 1,
    "description": "a de novo prediction tool of chromatin accessible regions for plant genomes",
    "formatting": "[ERROR] In file \"/tmp/tmpof4ddpw6/Snakefile\":  \u0027EmptyContextError: L3: rule has no keywords attached to it.\u0027\n[INFO] In file \"/tmp/tmpof4ddpw6/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "Yin-Shen/CharPlant",
    "linting": "Lints for rule data_preprocessing (line 11, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule model_training (line 34, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule motif (line 62, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule de_novo_prediction (line 83, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule batch_submit_job (line 108, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cat_predict (line 128, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_positive_sample (line 142, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bowtie2_index (line 161, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bowtie2 (line 187, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule macs2 (line 210, /tmp/tmpof4ddpw6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 5,
    "subscribers_count": 1,
    "topics": [
      "deep-learning",
      "plant",
      "chromatin-accessibiity",
      "de-novo-prediction",
      "cnn-model",
      "open-chromatin-regions"
    ],
    "updated_at": 1616072550.0
  },
  {
    "data_format": 1,
    "description": "Mutational antigenic profiling of SARS-CoV-2 RBD against human convalescent sera from the HAARVI cohort",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_MAP_HAARVI_sera",
    "linting": "ModuleNotFoundError in line 10 of /tmp/tmpm0m1g6nf/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpm0m1g6nf/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 5,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1618275849.0
  },
  {
    "data_format": 1,
    "description": "Mutational antigenic profiling of SARS-CoV-2 RBD against human convalescent sera from the HAARVI cohort",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_MAP_HAARVI_sera",
    "linting": "ModuleNotFoundError in line 10 of /tmp/tmpm0m1g6nf/Snakefile:\nNo module named \u0027Bio\u0027\n  File \"/tmp/tmpm0m1g6nf/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 5,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1618275849.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "seandavi/SimpleSnakemakeTutorial",
    "linting": "Lints for snakefile /tmp/tmpnz9wk_fx/bampost.snk:\n    * Absolute path \"/usr/local/apps/picard/1.94\" in line 1:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/path/to/fastq\" in line 2:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule indexbam (line 4, /tmp/tmpnz9wk_fx/bampost.snk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule picardMarkDuplicates (line 28, /tmp/tmpnz9wk_fx/bampost.snk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule picardAlignmentSummaryMetrics (line 60, /tmp/tmpnz9wk_fx/bampost.snk):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cleanup (line 22, /tmp/tmpnz9wk_fx/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 6,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618476009.0
  },
  {
    "data_format": 1,
    "description": "Amplicon sequence processing workflow using QIIME 2 and Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "aomlomics/tourmaline",
    "linting": "ModuleNotFoundError in line 3 of /tmp/tmpa0flvmyc/Snakefile:\nNo module named \u0027qiime2\u0027\n  File \"/tmp/tmpa0flvmyc/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 6,
    "subscribers_count": 1,
    "topics": [
      "noaa-omics-software",
      "amplicon-sequence-variants",
      "dada2",
      "deblur",
      "snakemake"
    ],
    "updated_at": 1617040892.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake workflow to process single samples or cohorts of paired-end sequencing data (WGS or WES) using bwa/GATK4.",
    "formatting": "[INFO] 16 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ESR-NZ/human_genomics_pipeline",
    "linting": "KeyError in line 21 of /tmp/tmpulhpbdr6/workflow/Snakefile:\n\u0027DATA\u0027\n  File \"/tmp/tmpulhpbdr6/workflow/Snakefile\", line 21, in \u003cmodule\u003e\n",
    "stargazers_count": 6,
    "subscribers_count": 5,
    "topics": [
      "pipeline",
      "genomics",
      "nvidia-clara-parabricks",
      "nvidia-gpu",
      "snakemake-pipeline",
      "gatk4",
      "gatk-bestpractices",
      "illumina",
      "exome-pipeline",
      "cohort-analysis"
    ],
    "updated_at": 1611060698.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for metagenomic community analysis using DNA isolated from virus-like particles",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "wclose/viralMetagenomicsPipeline",
    "linting": "Lints for snakefile /tmp/tmpw6bdiclr/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule trimReadPair (line 49, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule buildContaminantIndices (line 82, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule decontaminateTrimmedReads (line 112, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assembleReads (line 150, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineContigs (line 176, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dedupeContigs (line 199, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule filterContigs (line 225, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getVirsorterDatabase (line 264, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule predictContigsVirsorter (line 285, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getVirfinderModel (line 312, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule predictContigsVirfinder (line 333, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule curateViralContigs (line 365, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule indexContigs (line 398, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mapContigs (line 425, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule binContigs (line 454, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getMarkerSets (line 493, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule detectBinContamination (line 518, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule curateBinLibrary (line 545, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule indexBins (line 578, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mapBins (line 605, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule indexBinBam (line 633, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calcBinRecruitment (line 659, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineBinStats (line 686, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule makeBinShared (line 713, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule splitBinShared (line 739, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule normalizeBinShared (line 770, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule subsampleBinShared (line 797, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule getClassificationDatabases (line 838, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule classifyBins (line 864, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calcBinAlphaDiversity (line 901, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calcBinBetaDiversity (line 934, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calcBinPCoA (line 967, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calcBinNMDS (line 994, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule configureContaminantReadMap (line 1030, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rawContaminantReadMap (line 1057, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rawReadStats (line 1086, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineRawReports (line 1114, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule trimmedReadStats (line 1143, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineTrimmedReports (line 1170, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deconContaminantReadMap (line 1197, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule deconReadStats (line 1226, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineDeconReports (line 1254, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule assemblyStats (line 1283, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule combineAssemblyReports (line 1310, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean (line 1345, /tmp/tmpw6bdiclr/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 6,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613686427.0
  },
  {
    "data_format": 1,
    "description": " Integrated workflow for SV calling from single-cell Strand-seq data ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "friendsofstrandseq/mosaicatcher-pipeline",
    "linting": "IndexError in line 362 of /tmp/tmpoedvl6bc/Snakefile:\nlist index out of range\n  File \"/tmp/tmpoedvl6bc/Snakefile\", line 362, in \u003cmodule\u003e\n",
    "stargazers_count": 6,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1615081852.0
  },
  {
    "data_format": 1,
    "description": "An example engineering analysis workflow for reproducible results.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nickjstevens/analysis-workflow",
    "linting": "Lints for rule run_analysis (line 16, /tmp/tmpmq3j3dc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule post_analysis (line 42, /tmp/tmpmq3j3dc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dag (line 67, /tmp/tmpmq3j3dc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 82, /tmp/tmpmq3j3dc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule report (line 97, /tmp/tmpmq3j3dc6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 6,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1568886493.0
  },
  {
    "data_format": 1,
    "description": "Automated analysis and result reporting for targeted sequencing data",
    "formatting": "[ERROR] In file \"/tmp/tmpt_rg6959/Snakefile\":  \u0027InvalidParameterSyntax: 167samflag=513 if( config[\"parameters\"][\"paired_end_reads\"]==\"yes\") else512\n\u0027\n[INFO] In file \"/tmp/tmpt_rg6959/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "kircherlab/hemoMIPs",
    "linting": "KeyError in line 8 of /tmp/tmpt_rg6959/Snakefile:\n\u0027GATK\u0027\n  File \"/tmp/tmpt_rg6959/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 6,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1602780827.0
  },
  {
    "data_format": 1,
    "description": "Workflow for identifying and classifying homologous gene/protein sequences",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "laelbarlow/amoebae",
    "linting": "Lints for snakefile /tmp/tmprgxl5hst/workflow/Snakefile:\n    * Path composition with \u0027+\u0027 in line 171:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 207:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 78:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 171:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule help (line 97, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_dbs (line 111, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_queries (line 140, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_dbs (line 175, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_dbs_local (line 244, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_query_seqs (line 313, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_query_seqs_local (line 360, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule align_fasta (line 407, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_query_ali (line 450, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule add_to_query_ali_local (line 497, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule list_dbs_and_queries (line 543, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_ref_seqs (line 580, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule setup_and_run_fwd_srch (line 647, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sum_fwd_srch (line 708, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule setup_and_run_rev_srch (line 759, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sum_rev_srch (line 823, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule interp_srchs (line 879, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule find_redun_seqs (line 922, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_results (line 985, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_positive_hit_fastas (line 1034, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_fwd_hit_seqs (line 1080, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule align_positive_hit_fastas (line 1128, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_doc_pdf (line 1181, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plot_workflow (line 1207, /tmp/tmprgxl5hst/workflow/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 6,
    "subscribers_count": 2,
    "topics": [
      "bioinformatics",
      "hmmer3",
      "biopython",
      "python3",
      "blast",
      "genome-annotation",
      "similarity-search",
      "coulson-plot",
      "transcriptome-annotation",
      "snakemake-workflow",
      "comparative-genomics",
      "homology",
      "molecular-evolution",
      "gene-families",
      "protein-families",
      "orthology"
    ],
    "updated_at": 1619181835.0
  },
  {
    "data_format": 1,
    "description": "Accompanying analysis code for the FRASER manuscript",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gagneurlab/FRASER-analysis",
    "linting": "ModuleNotFoundError in line 3 of /tmp/tmpffyf4arc/Snakefile:\nNo module named \u0027wbuild\u0027\n  File \"/tmp/tmpffyf4arc/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 6,
    "subscribers_count": 2,
    "topics": [
      "rna-seq",
      "outlier-detection",
      "rare-disease",
      "r",
      "snakemake",
      "workflow"
    ],
    "updated_at": 1616419511.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipeline for amplicon sequencing using dada2",
    "formatting": "[ERROR] In file \"/tmp/tmpop4_qrqu/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 10:20:     logger.critical(\"Couldn\u0027t find sampletable!\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpop4_qrqu/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmpop4_qrqu/Snakefile\":  2 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "SilasK/16S-dada2",
    "linting": "SyntaxError in line 10 of /tmp/tmpop4_qrqu/Snakefile:\nEOL while scanning string literal\n",
    "stargazers_count": 7,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618233573.0
  },
  {
    "data_format": 1,
    "description": "RNA-seq workflow for Snakemake based on STAR and featureCounts.",
    "formatting": "[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jrderuiter/snakemake-rnaseq",
    "linting": "Lints for snakefile /tmp/tmpogjdvmj8/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpogjdvmj8/rules/input.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpogjdvmj8/rules/alignment.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 2:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpogjdvmj8/rules/counts.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpogjdvmj8/rules/qc.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule copy_input (line 56, /tmp/tmpogjdvmj8/rules/input.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sambamba_sort (line 289, /tmp/tmpogjdvmj8/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_merge (line 327, /tmp/tmpogjdvmj8/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_index (line 355, /tmp/tmpogjdvmj8/rules/alignment.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_counts (line 91, /tmp/tmpogjdvmj8/rules/counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule normalize_counts (line 117, /tmp/tmpogjdvmj8/rules/counts.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule fastqc (line 55, /tmp/tmpogjdvmj8/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_stats (line 80, /tmp/tmpogjdvmj8/rules/qc.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 7,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1608563197.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 13 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nanoporetech/DTR-phage-pipeline",
    "linting": "No kaiju DB found in /media/sfodata/databases/kaiju/nr_euk.\nSkipping Kaiju\nNo kaiju DB found in /media/sfodata/databases/kaiju/nr_euk.\nSkipping Kaiju\nLints for snakefile /tmp/tmpcvjhy_1w/Snakefile:\n    * Path composition with \u0027+\u0027 in line 194:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpcvjhy_1w/rules/kmer_bins.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpcvjhy_1w/rules/align_clusters.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpcvjhy_1w/rules/qc_genomes.smk:\n    * Path composition with \u0027+\u0027 in line 42:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule summary_stats (line 5, /tmp/tmpcvjhy_1w/rules/summary.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule find_all_DTR_reads (line 5, /tmp/tmpcvjhy_1w/rules/dtr_reads.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule virsorter_identify_phage (line 17, /tmp/tmpcvjhy_1w/rules/virsorter.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable VIRSORTER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule virsorter_collect_phage (line 57, /tmp/tmpcvjhy_1w/rules/virsorter.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule kaiju (line 5, /tmp/tmpcvjhy_1w/rules/kaiju.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule kaiju_taxa (line 35, /tmp/tmpcvjhy_1w/rules/kaiju.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule kaiju_attach_taxa (line 57, /tmp/tmpcvjhy_1w/rules/kaiju.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule kaiju_to_krona (line 82, /tmp/tmpcvjhy_1w/rules/kaiju.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule calc_kmer_freq (line 5, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule add_qscore_to_kmer_freq (line 35, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule create_umap_kmer_freq_map (line 57, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule label_umap_freq_map_with_qscore (line 91, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule label_umap_freq_map_with_gc_content (line 118, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule label_umap_freq_map_with_readlength (line 147, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule call_umap_freq_map_bins (line 178, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule list_kmer_bin_ids (line 204, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_umap_freq_map_bins (line 225, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule label_umap_freq_map_with_tax (line 252, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule generate_bins_and_stats (line 287, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule kmer_binned_fasta (line 348, /tmp/tmpcvjhy_1w/rules/kmer_bins.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bin_ava_minimap2 (line 5, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bin_ava_clustering (line 32, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule combine_bin_cluster_read_info (line 62, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule create_bin_cluster_read_lists (line 85, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule get_cluster_reads_fasta (line 127, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule select_read_with_longest_DTR (line 149, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule create_bin_cluster_ref_fasta (line 170, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule create_bin_cluster_pol_fastqs (line 192, /tmp/tmpcvjhy_1w/rules/align_clusters.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule racon_polishing (line 5, /tmp/tmpcvjhy_1w/rules/polish.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_medaka (line 43, /tmp/tmpcvjhy_1w/rules/polish.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rename_polished_ref_reads (line 77, /tmp/tmpcvjhy_1w/rules/polish.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule prodigal (line 5, /tmp/tmpcvjhy_1w/rules/annotate.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_prodigal_result_statistics (line 30, /tmp/tmpcvjhy_1w/rules/annotate.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_pol_reads_to_polished_ref (line 58, /tmp/tmpcvjhy_1w/rules/annotate.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule parse_align_clust_strands_paf (line 89, /tmp/tmpcvjhy_1w/rules/annotate.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule combine_all_polished_ref_reads (line 5, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule remove_adapters_from_ref_reads (line 22, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule find_all_DTR_genomes (line 49, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule aggregate_prodigal_statistics (line 81, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable MEDAKA_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule build_bin_cluster_summary_table (line 106, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule combine_bin_cluster_strand_counts_into_table (line 136, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule combine_dtr_aligns (line 161, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule combine_all_draft_stats (line 196, /tmp/tmpcvjhy_1w/rules/qc_genomes.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule nucmer_for_duplicate_sequences (line 5, /tmp/tmpcvjhy_1w/rules/dedup.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule show_coords_for_duplicate_sequences (line 29, /tmp/tmpcvjhy_1w/rules/dedup.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule rewrite_pol_fasta_without_seq_duplicates (line 51, /tmp/tmpcvjhy_1w/rules/dedup.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_all_prodigal_stats (line 85, /tmp/tmpcvjhy_1w/rules/dedup.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule align_dtr_seqs_to_polished_refs (line 5, /tmp/tmpcvjhy_1w/rules/dtr_align.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Param prefix is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule find_concatemer_read_ids (line 5, /tmp/tmpcvjhy_1w/rules/linear_concats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_concatemer_copy_length_contours (line 40, /tmp/tmpcvjhy_1w/rules/linear_concats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule grep_concatemer_seqs (line 64, /tmp/tmpcvjhy_1w/rules/linear_concats.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule all_kaiju (line 319, /tmp/tmpcvjhy_1w/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 7,
    "subscribers_count": 25,
    "topics": [],
    "updated_at": 1612221733.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for the analysis of PE ChIP-seq data",
    "formatting": "[WARNING] In file \"/tmp/tmp7pwwqwjn/Snakefile\":  Keyword \"input\" at line 102 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 5 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "KoesGroup/Snakemake_ChIPseq_PE",
    "linting": "AttributeError in line 93 of /tmp/tmp7pwwqwjn/rules/pre_processing.smk:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmp7pwwqwjn/Snakefile\", line 113, in \u003cmodule\u003e\n  File \"/tmp/tmp7pwwqwjn/rules/pre_processing.smk\", line 93, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 2,
    "topics": [
      "chip-seq-pipelines",
      "snakemake"
    ],
    "updated_at": 1607442221.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[ERROR] In file \"/tmp/tmp44da2_n5/Snakefile\":  \u0027SyntaxError: L62: Unrecognised keyword \u0027expand\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmp44da2_n5/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "HelenaLC/muscat-comparison",
    "linting": "FileNotFoundError in line 27 of /tmp/tmp44da2_n5/Snakefile:\n[Errno 2] No such file or directory: \u0027meta/sim_ids.json\u0027\n  File \"/tmp/tmp44da2_n5/Snakefile\", line 27, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1618489801.0
  },
  {
    "data_format": 1,
    "description": "A straightforward bioinformatic pipeline for detecting a bacterial strain in one or more metagenome(s).",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tamburinif/StrainSifter",
    "linting": "Lints for snakefile /tmp/tmpnqmj4qbc/Snakefile:\n    * Absolute path \"/|\\.\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule bwa_index (line 24, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bwa_align (line 52, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param ref is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule genomecov (line 90, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calc_coverage (line 118, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_samples (line 150, /tmp/tmpnqmj4qbc/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule faidx (line 183, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pileup (line 206, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_snps (line 235, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snp_consensus (line 267, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine (line 293, /tmp/tmpnqmj4qbc/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule core_snps (line 321, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule core_snps_to_fasta (line 347, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multi_align (line 373, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_tree (line 399, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_tree (line 425, /tmp/tmpnqmj4qbc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pairwise_snvs (line 451, /tmp/tmpnqmj4qbc/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 7,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1603769942.0
  },
  {
    "data_format": 1,
    "description": "Automated pipeline for amplicon sequence analysis ",
    "formatting": "[WARNING] In file \"/tmp/tmpundctq52/Snakefile\":  Keyword \"input\" at line 26 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpundctq52/Snakefile\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] In file \"/tmp/tmpundctq52/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "AlejandroAb/CASCABEL",
    "linting": "KeyError in line 7 of /tmp/tmpundctq52/Snakefile:\n\u0027RUN\u0027\n  File \"/tmp/tmpundctq52/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 1,
    "topics": [
      "amplicon-sequence-analysis",
      "snakemake",
      "illumina-sequencing",
      "pipeline",
      "otu-table"
    ],
    "updated_at": 1613031919.0
  },
  {
    "data_format": 1,
    "description": "JAMIRA is a reproducible Workflow for Integrative Exploration of Genomic Features of prokariotes",
    "formatting": "[WARNING] In file \"/tmp/tmpd1b1gu5f/Snakefile\":  Keyword \"output\" at line 81 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "icaromsc/WGCA",
    "linting": "Lints for snakefile /tmp/tmpd1b1gu5f/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule virulence_identification (line 39, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summarize_virulence (line 63, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule plasmid_prediction (line 86, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summarize_plasmid_prediction (line 110, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule resistome_prediction (line 133, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule generate_resistome_heatmap (line 160, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summary_results (line 185, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genome_annotation (line 225, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule genomic_island_prediction (line 248, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_gbk_to_seed (line 272, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prophage_prediction (line 295, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule summarize_prophage_prediction (line 322, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule summarize_genomic_islands (line 354, /tmp/tmpd1b1gu5f/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 7,
    "subscribers_count": 2,
    "topics": [
      "genomic-islands",
      "prophages",
      "virulence-genes",
      "resistome",
      "comparative-genomics",
      "genomic-data-analysis",
      "bioinformatics-workflow",
      "snakemake"
    ],
    "updated_at": 1605011524.0
  },
  {
    "data_format": 1,
    "description": "We present HRIBO (High-throughput annotation by Ribo-seq), a workflow to enable reproducible and high-throughput analysis of bacterial Ribo-seq data. The workflow performs all required pre-processing steps and quality control.  Importantly, HRIBO outputs annotation-independent ORF predictions based on two complementary prokaryotic-focused tools, and integrates them with additional computed features. This facilitates both the rapid discovery of ORFs and their prioritization for functional characterization.",
    "formatting": "[WARNING] In file \"/tmp/tmpzv9a2x_e/rules/report.smk\":  Keyword \"output\" at line 16 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpzv9a2x_e/rules/qcauxiliary.smk\":  Keyword \"output\" at line 6 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpzv9a2x_e/rules/qcauxiliary.smk\":  Keyword \"output\" at line 21 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpzv9a2x_e/rules/qcauxiliary.smk\":  Keyword \"output\" at line 36 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpzv9a2x_e/rules/mapping.smk\":  Keyword \"shell\" at line 41 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 20 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "RickGelhausen/HRIBO",
    "linting": "KeyError in line 8 of /tmp/tmpzv9a2x_e/Snakefile:\n\u0027adapter\u0027\n  File \"/tmp/tmpzv9a2x_e/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 3,
    "topics": [
      "ribosome-profiling",
      "prokaryotes",
      "snakemake",
      "bioinformatics"
    ],
    "updated_at": 1607961287.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpzn34553u/rules/fgwas.smk\":  Keyword \"shell\" at line 13 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 4 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NCBI-Hackathons/Kipoi-GWAS",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpzn34553u/rules/ukbb.smk:\nNo module named \u0027matplotlib\u0027\n  File \"/tmp/tmpzn34553u/Snakefile\", line 58, in \u003cmodule\u003e\n  File \"/tmp/tmpzn34553u/rules/ukbb.smk\", line 4, in \u003cmodule\u003e\n  File \"/tmp/tmpzn34553u/kipoi_gwas/prepare.py\", line 5, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 10,
    "topics": [],
    "updated_at": 1588170995.0
  },
  {
    "data_format": 1,
    "description": "A tool for generating bacterial genomes from metagenomes with nanopore long read sequencing",
    "formatting": "[WARNING] In file \"/tmp/tmphyaoqvz7/Snakefile\":  Keyword \"output\" at line 95 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmphyaoqvz7/Snakefile\":  Keyword \"shell\" at line 572 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "elimoss/lathe",
    "linting": "KeyError in line 19 of /tmp/tmphyaoqvz7/Snakefile:\n\u0027sample_name\u0027\n  File \"/tmp/tmphyaoqvz7/Snakefile\", line 19, in \u003cmodule\u003e\n",
    "stargazers_count": 7,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1619176494.0
  },
  {
    "data_format": 1,
    "description": "uORF-Tools are a workflow and a collection of tools for the analysis of \u0027Upstream Open Reading Frames\u0027 (short uORFs)",
    "formatting": "[WARNING] In file \"/tmp/tmpqf_qbx2l/rules/mapping.smk\":  Keyword \"output\" at line 5 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Biochemistry1-FFM/uORF-Tools",
    "linting": "KeyError in line 7 of /tmp/tmpqf_qbx2l/Snakefile:\n\u0027uorfannotationpath\u0027\n  File \"/tmp/tmpqf_qbx2l/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 2,
    "topics": [
      "bioinformatics",
      "high-throughput-sequencing",
      "snakemake",
      "ribosome-profiling"
    ],
    "updated_at": 1616562924.0
  },
  {
    "data_format": 1,
    "description": "Modular snakemake workflow for assembly-based metagenomics",
    "formatting": "[ERROR] In file \"/tmp/tmpibcl7hl6/Snakefile\":  \u0027SyntaxError: L42: Unrecognised keyword \u0027expand\u0027 in rule definition\u0027\n[INFO] In file \"/tmp/tmpibcl7hl6/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "tanaes/snakemake_assemble",
    "linting": "WorkflowError in line 19 of /tmp/tmpibcl7hl6/Snakefile:\nFailed to open source file /home/jgsanders/git_sw/snakemake_assemble/bin/snakefiles/simplify_fasta.py\nFileNotFoundError: [Errno 2] No such file or directory: \u0027/home/jgsanders/git_sw/snakemake_assemble/bin/snakefiles/simplify_fasta.py\u0027\n  File \"/tmp/tmpibcl7hl6/Snakefile\", line 19, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1610829273.0
  },
  {
    "data_format": 1,
    "description": "Clin-mNGS: Automated pipeline for pathogen detection from clinical metagenomic data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AkshathaPrasanna/Clin-mNGS",
    "linting": "AttributeError in line 105 of /tmp/tmpzgbou9o5/Snakefile:\ninvalid name for input, output, wildcard, params or log: reverse is reserved for internal use\n  File \"/tmp/tmpzgbou9o5/Snakefile\", line 105, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1614576023.0
  },
  {
    "data_format": 1,
    "description": "Bioinformatics pipeline for nanopore sequencing data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "zhaoc1/nanoflow",
    "linting": "KeyError in line 10 of /tmp/tmprnwi5vi1/Snakefile:\n\u0027project_dir\u0027\n  File \"/tmp/tmprnwi5vi1/Snakefile\", line 10, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 3,
    "topics": [
      "snakemake-workflows",
      "nanopore-sequencing",
      "bioinformatics-pipeline",
      "bacterial-genomes"
    ],
    "updated_at": 1616082261.0
  },
  {
    "data_format": 1,
    "description": "Estimation of solar and wind power generation potentials in Europe.",
    "formatting": "[ERROR] In file \"/tmp/tmpg2c9owt7/rules/data-preprocessing.smk\":  \u0027InvalidParameterSyntax: 165[\"build/srtm_{x:02d}_{y:02d}.tif\". format( x= x, y= y)\n for x in range( SRTM_X_MIN, SRTM_X_MAX+1)\n for y in range( SRTM_Y_MIN, SRTM_Y_MAX+1)\n if not( x is34 and y in[3,4,5,6])]\n\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 6 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "calliope-project/solar-and-wind-potentials",
    "linting": "/tmp/tmpg2c9owt7/rules/data-preprocessing.smk:367: SyntaxWarning: \"is\" with a literal. Did you mean \"==\"?\nModuleNotFoundError in line 2 of /tmp/tmpg2c9owt7/rules/data-preprocessing.smk:\nNo module named \u0027pycountry\u0027\n  File \"/tmp/tmpg2c9owt7/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmpg2c9owt7/rules/data-preprocessing.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 1,
    "topics": [
      "renewable-energy",
      "electricity",
      "snakemake",
      "europe",
      "analysis",
      "reproducible-research"
    ],
    "updated_at": 1618484294.0
  },
  {
    "data_format": 1,
    "description": "Example of a snakemake workflow implementation of the mothur MiSEQ SOP",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "AAFC-BICoE/snakemake-mothur",
    "linting": "AttributeError in line 100 of /tmp/tmpa4tgvk4n/Snakefile:\ninvalid name for input, output, wildcard, params or log: count is reserved for internal use\n  File \"/tmp/tmpa4tgvk4n/Snakefile\", line 100, in \u003cmodule\u003e\n",
    "stargazers_count": 8,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1612983315.0
  },
  {
    "data_format": 1,
    "description": "Code for extracting parallel corpora from pmindia",
    "formatting": "[WARNING] In file \"/tmp/tmp5kc_umtu/Snakefile\":  Keyword \"shell\" at line 313 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bhaddow/pmindia-crawler",
    "linting": "Lints for snakefile /tmp/tmp5kc_umtu/Snakefile:\n    * Absolute path \"/home/bhaddow/moses.new/scripts\" in line 8:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/bhaddow/code/indic_nlp_library/bhaddow\" in line 9:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/bhaddow/tools/hunalign/src/hunalign/hunalign\" in line 10:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/bhaddow/code/vecalign\" in line 11:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/bhaddow/code/LASER\" in line 12:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/home/bhaddow/data/pavlick-dicts\" in line 13:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/filter.py -f {wildcards[language]} \u003c {input} \u003e {output} 2\u003e {output}.err\" in line 173:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*\" in line 194:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/*tsv\" in line 212:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/^\u003cP\u003e$/d\" in line 318:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/^$/d\" in line 321:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/extract.py -f {input[0]} -i {input[1]} -t {output}\" in line 331:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/make_index.py -l en -d {input} \u003e {output}\" in line 341:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/get_urls.py -l {wildcards.language} -m {MAX_URLS} \u003e {output}\" in line 349:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 34:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 35:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 70:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 84:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule release_intersect (line 107, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule release_hunalign (line 134, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule release_mono (line 168, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule intersect_align (line 199, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule filter (line 252, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule crawl (line 280, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule collate_hunalign (line 301, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule collate_vecalign (line 326, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule run_hunalign (line 352, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable HUNALIGN from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule get_dict (line 391, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable PAVLICK_DICTS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable sys from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable PAVLICK_DICTS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_vecalign (line 407, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VECALIGN from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable sys from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule embed_overlap (line 451, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable LASER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable LASER from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_overlap (line 485, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable VECALIGN from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule split (line 519, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable MOSES_SCRIPTS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extract (line 553, /tmp/tmp5kc_umtu/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_index (line 581, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule urls (line 609, /tmp/tmp5kc_umtu/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable MAX_URLS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 8,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1597583801.0
  },
  {
    "data_format": 1,
    "description": "Using RNA-Seq data to improve microRNA target prediction accuracy in animals",
    "formatting": "[ERROR] In file \"/tmp/tmpqaygeodh/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:0: else:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpqaygeodh/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "TBradley27/FilTar",
    "linting": "Lints for snakefile /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 33:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 27:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqaygeodh/modules/with_reannotation/map_reads/hisat2/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 32:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqaygeodh/modules/get_utr_and_cds/without_conservation/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpqaygeodh/modules/data_download/Snakefile:\n    * Absolute path \"/^\\t/d\" in line 38:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqaygeodh/modules/trim_reads/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for snakefile /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 21:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 18:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpqaygeodh/modules/mirna/Snakefile:\n    * Absolute path \"/^$/d\" in line 26:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule get_bed6_file (line 43, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_bed6_file (line 63, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_bed6_file (line 79, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule convert_gtf_to_genepred (line 97, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule convert_genepred_to_bed12 (line 119, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_bedgraph (line 145, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule split_bedgraph (line 175, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bedgraphs_by_run (line 199, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule avg_merged_bedgraph_by_run (line 224, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bedgraphs_by_sample (line 247, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule avg_merged_bedgraph_by_sample (line 272, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_bed12 (line 295, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_bed12 (line 311, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule reannotate_3utrs (line 327, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule get_extended_bed_file (line 347, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregrate_extended_bed_files (line 369, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule identify_APA_sites (line 387, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule aggregate_APA_sites (line 408, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_utr_lengths_chr (line 424, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_utr_lengths (line 440, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_AIR_file (line 456, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gtf_to_bed_CDS (line 474, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_CDS (line 494, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_CDS (line 510, /tmp/tmpqaygeodh/modules/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule create_index (line 130, /tmp/tmpqaygeodh/modules/with_reannotation/map_reads/hisat2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_reads (line 157, /tmp/tmpqaygeodh/modules/with_reannotation/map_reads/hisat2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule sam_to_bam (line 187, /tmp/tmpqaygeodh/modules/with_reannotation/map_reads/hisat2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule samtools_sort (line 210, /tmp/tmpqaygeodh/modules/with_reannotation/map_reads/hisat2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule reformat_bed_file_for_bedtools (line 26, /tmp/tmpqaygeodh/modules/get_utr_and_cds/without_conservation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_single_species_seq_fa (line 42, /tmp/tmpqaygeodh/modules/get_utr_and_cds/without_conservation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule merge_fasta (line 63, /tmp/tmpqaygeodh/modules/get_utr_and_cds/without_conservation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule convert_fasta_to_tsv (line 79, /tmp/tmpqaygeodh/modules/get_utr_and_cds/without_conservation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_single_end_fastq_file (line 20, /tmp/tmpqaygeodh/modules/data_download/ENA/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_paired_end_fastq_files (line 37, /tmp/tmpqaygeodh/modules/data_download/ENA/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prep_mirna_seeds (line 22, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prep_mirna_family_file (line 42, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prep_mirna_mature_alt (line 61, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prep_mir_for_context_scores (line 83, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_mir_for_context_scores (line 103, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_mir_families (line 119, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule targetscan_sites (line 139, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule targetscan_branch_lengths (line 160, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule targetscan_PCTs (line 184, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule targetscan_eightmers (line 208, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule targetscan_context_scores (line 230, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregate_contextpp_scores (line 264, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregate_targetscan_sites (line 280, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_all_transcripts (line 298, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_targetscan_data1 (line 314, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_targetscan_data2 (line 334, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_targetscan_data3 (line 356, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule patch_bin_lengths_script (line 377, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule patch_PCT_script (line 393, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule patch_contextpp_script (line 409, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule expression_filter_contextpp_scores_chr (line 1, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule expression_filter_contextpp_scores (line 23, /tmp/tmpqaygeodh/modules/target_prediction/targetscan/with_reannotation/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_gtf_file (line 30, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule modify_gtf_file (line 46, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule get_transcript_ids (line 62, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_toplevel_dna (line 81, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_primary_assembly_dna (line 97, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule download_chromsomal_dna (line 113, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule decompress_dna_file (line 132, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule download_maf_files_human (line 154, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_maf_files_mouse (line 172, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_cdna_file (line 190, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule decompress_cdna_file (line 206, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_first_APAtrap_script (line 222, /tmp/tmpqaygeodh/modules/data_download/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_single_end_reads (line 28, /tmp/tmpqaygeodh/modules/trim_reads/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule trim_paired_end_reads (line 51, /tmp/tmpqaygeodh/modules/trim_reads/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule salmon_index (line 23, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_index_for_lib_types (line 48, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_quant_lib_type (line 86, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_quant (line 116, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_get_lib_type (line 146, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule salmon_quantmerge_by_run (line 174, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_average_quantmerge_runs (line 193, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule salmon_quantmerge_by_sample (line 221, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule salmon_average_quantmerge_samples (line 240, /tmp/tmpqaygeodh/modules/quant_reads/salmon/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mirna_species_fasta (line 23, /tmp/tmpqaygeodh/modules/mirna/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_mirna_species_fasta (line 39, /tmp/tmpqaygeodh/modules/mirna/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule get_genome_postions (line 3, /tmp/tmpqaygeodh/modules/get_target_coordinates/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 8,
    "subscribers_count": 2,
    "topics": [
      "mirna-targets",
      "mirna",
      "mirna-mrna-interaction"
    ],
    "updated_at": 1617024504.0
  },
  {
    "data_format": 1,
    "description": "snakemake rules for running anvio",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "tanaes/snakemake_anvio",
    "linting": "FileNotFoundError in line 3 of /tmp/tmpch4subc6/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpch4subc6/Snakefile\", line 3, in \u003cmodule\u003e\n",
    "stargazers_count": 9,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1610656786.0
  },
  {
    "data_format": 1,
    "description": "Is your European region able to provide itself with 100% renewable electricity?",
    "formatting": "[ERROR] In file \"/tmp/tmpeawjp2at/rules/data-preprocessing.smk\":  \u0027InvalidParameterSyntax: 225[\"build/srtm_{x:02d}_{y:02d}.tif\". format( x= x, y= y)\n for x in range( SRTM_X_MIN, SRTM_X_MAX+1)\n for y in range( SRTM_Y_MIN, SRTM_Y_MAX+1)\n if not( x is34 and y in[3,4,5,6])]\n\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "timtroendle/possibility-for-electricity-autarky",
    "linting": "/tmp/tmpeawjp2at/rules/data-preprocessing.smk:475: SyntaxWarning: \"is\" with a literal. Did you mean \"==\"?\nModuleNotFoundError in line 2 of /tmp/tmpeawjp2at/rules/data-preprocessing.smk:\nNo module named \u0027pycountry\u0027\n  File \"/tmp/tmpeawjp2at/Snakefile\", line 9, in \u003cmodule\u003e\n  File \"/tmp/tmpeawjp2at/rules/data-preprocessing.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 9,
    "subscribers_count": 3,
    "topics": [
      "research",
      "renewable-energy",
      "paper",
      "europe",
      "analysis",
      "electricity-autarky",
      "reproducible-research"
    ],
    "updated_at": 1602775310.0
  },
  {
    "data_format": 1,
    "description": "Repo to analyze population genetic data with many different methods",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NovembreLab/eems-around-the-world",
    "linting": "FileNotFoundError in line 1 of /tmp/tmpk534p281/Snakefile:\n[Errno 2] No such file or directory: \u0027config/config.yaml\u0027\n  File \"/tmp/tmpk534p281/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 9,
    "subscribers_count": 5,
    "topics": [
      "population-genetics",
      "data-visualization",
      "principle-component-analysis",
      "structure",
      "admixture",
      "snakemake-workflow"
    ],
    "updated_at": 1603765326.0
  },
  {
    "data_format": 1,
    "description": "D\u00e9monstration d\u2019outils de bioinfo dans le cadre d\u2019un projet",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "thomasdenecker/FAIR_Bioinfo",
    "linting": "PermissionError in line 29 of /tmp/tmpjzrx9s6z/Snakefile:\n[Errno 13] Permission denied: \u0027/home/rstudio\u0027\n  File \"/tmp/tmpjzrx9s6z/Snakefile\", line 29, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 215, in makedirs\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/os.py\", line 225, in makedirs\n",
    "stargazers_count": 9,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1608129932.0
  },
  {
    "data_format": 1,
    "description": "Recovery of prokaryotic genomes from shotgun metagenomic sequencing data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Finn-Lab/MAG_Snakemake_wf",
    "linting": "Lints for snakefile /tmp/tmpym3mfr7d/modules/sra_download.Snakefile:\n    * Deprecated singularity directive used for container definition in line 27.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 189:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 180:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 28.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 45.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 63.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 83.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 112.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 145.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 161.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 179.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/coas.Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 50:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 49.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/assembly.Snakefile:\n    * Deprecated singularity directive used for container definition in line 32.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 56.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/binning.Snakefile:\n    * Deprecated singularity directive used for container definition in line 60.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 91.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/refine.Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Deprecated singularity directive used for container definition in line 24.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 99.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 139.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile:\n    * Deprecated singularity directive used for container definition in line 26.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 63.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 92.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 108.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/framework.Snakefile:\n    * Path composition with \u0027+\u0027 in line 53:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 178:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 201:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 31.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 52.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 90.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 106.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 126.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 147.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 164.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 177.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 237.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Deprecated singularity directive used for container definition in line 24.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 103.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/cmseq.Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 203:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 228:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 166:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 203:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 59.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 101.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 130.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 165.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 262.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for snakefile /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile:\n    * Path composition with \u0027+\u0027 in line 109:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Deprecated singularity directive used for container definition in line 26.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 41.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 59.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Deprecated singularity directive used for container definition in line 160.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sra_download (line 19, /tmp/tmpym3mfr7d/modules/sra_download.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule raw_fastqc_fwd (line 19, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule raw_fastqc_rev (line 55, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule raw_multiqc (line 92, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param indir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kneaddata_download_database (line 130, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule kneaddata_bowtie (line 162, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param indx is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule postpreprocessing_fastqc_fwd (line 242, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule postpreprocessing_fastqc_rev (line 277, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule postpreprocessing_multiqc (line 314, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param indir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule readcount_fq (line 355, /tmp/tmpym3mfr7d/modules/preprocessing.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule coas (line 29, /tmp/tmpym3mfr7d/modules/coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort (line 56, /tmp/tmpym3mfr7d/modules/coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule zip_file (line 90, /tmp/tmpym3mfr7d/modules/coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule readcounts_coas (line 137, /tmp/tmpym3mfr7d/modules/coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule spades (line 19, /tmp/tmpym3mfr7d/modules/assembly.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule spades_coas (line 67, /tmp/tmpym3mfr7d/modules/assembly.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule metawrap_binning (line 53, /tmp/tmpym3mfr7d/modules/binning.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule metawrap_binning_coas (line 104, /tmp/tmpym3mfr7d/modules/binning.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param assembly is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule bin_refinement (line 19, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule refine_bins_init (line 63, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param refined_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule copy_bins (line 102, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param dir1 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param dir2 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param out1 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param out2 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule copy_refined (line 140, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule checkm (line 164, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_checkm (line 202, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_checkm (line 241, /tmp/tmpym3mfr7d/modules/refine.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dRep (line 19, /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule combine_checkm (line 60, /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dRep_coas (line 93, /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule GTDB_TK (line 143, /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule plot_GTDB (line 187, /tmp/tmpym3mfr7d/modules/dRep_GTDB.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mapreads_scaffold (line 20, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_mapreads_scaffold (line 72, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule aggregate_mapreads_scaffold (line 100, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule cat_MAGs (line 132, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule cat_MAGs_coas (line 166, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule readmap (line 200, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule readmap_coassembly (line 242, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule parse_readmap (line 284, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule parse_readmap_coas (line 312, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule write_scaffold (line 340, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule write_scaffold_coas (line 368, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule plot_framework (line 397, /tmp/tmpym3mfr7d/modules/framework.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule bin_refinement_coas (line 19, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule refine_bins_init_coas (line 65, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param refined_folder is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule copy_bins_coas (line 103, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param dir1 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param dir2 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param out1 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param out2 is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule copy_refined_coas (line 141, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule checkm_coas (line 165, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_checkm_coas (line 203, /tmp/tmpym3mfr7d/modules/refine_coas.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rename_fasta (line 29, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prokka (line 61, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prokka is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule rename_fasta_coas (line 104, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prokka_coas (line 136, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param out_prokka is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cmseq (line 179, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n    * Param dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule cmseq_coas (line 226, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param name is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule append_cmseq (line 294, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule append_cmseq_coas (line 328, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule aggregate_cmseq (line 362, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregate_cmseq_coas (line 386, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_cmseq (line 410, /tmp/tmpym3mfr7d/modules/cmseq.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule mash_dist (line 19, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule best_mash (line 48, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dnadiff (line 79, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param outdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule parse_dnadiff (line 123, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule aggregate_dnadiff (line 171, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule aggregate_dnadiff_coas (line 195, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule summarize_dnadiff (line 219, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_dnadiff (line 250, /tmp/tmpym3mfr7d/modules/dnadiff.Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 10,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1618875925.0
  },
  {
    "data_format": 1,
    "description": "Use an ensemble of variant callers to call variants from ATAC-seq data",
    "formatting": "[ERROR] In file \"/tmp/tmprp3pb0cp/rules/prepare.smk\":  \u0027IndentationError: unindent does not match any outer indentation level (\u003ctokenize\u003e, line 21)\u0027\n[ERROR] In file \"/tmp/tmprp3pb0cp/rules/classify.smk\":  \u0027InvalidParameterSyntax: 109balance=int( config[\u0027balance\u0027]) if\u0027balance\u0027 in config else0\n\u0027\n[INFO] In file \"/tmp/tmprp3pb0cp/rules/classify.smk\":  2 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmprp3pb0cp/rules/classify.smk\":  1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "aryarm/varCA",
    "linting": "FileNotFoundError in line 26 of /tmp/tmprp3pb0cp/Snakefile:\n[Errno 2] No such file or directory: \u0027data/samples.tsv\u0027\n  File \"/tmp/tmprp3pb0cp/Snakefile\", line 37, in \u003cmodule\u003e\n  File \"/tmp/tmprp3pb0cp/Snakefile\", line 26, in read_samples\n",
    "stargazers_count": 10,
    "subscribers_count": 2,
    "topics": [
      "random-forest",
      "machine-learning",
      "snakemake",
      "atac-seq-data",
      "variant-calling"
    ],
    "updated_at": 1618536503.0
  },
  {
    "data_format": 1,
    "description": "Sparse Signaling Pathway Sampling: MCMC for signaling pathway inference",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "gitter-lab/ssps",
    "linting": "Lints for snakefile /tmp/tmps7t4hlvf/Snakefile:\n    * Absolute path \"/simulation_study/heatmaps/aucpr-{style}.png\" in line 136:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dream_scores.tsv\" in line 138:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/v={v}_r={r}_a={a}_t={t}_replicate={rep}.json\" in line 143:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sim_scores.tsv\" in line 147:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={d}_lstd={lstd}/cl={cell_line}_stim={stimulus}_replicate={rep}.json\" in line 153:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/cl={cell_line}_stim={stimulus}.json\" in line 155:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dream_scores.tsv\" in line 158:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/simulate_data.jl\" in line 165:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/v={v}_r={r}_a={a}_t={t}_replicate={rep}.csv\" in line 167:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/v={v}_r={r}_a={a}_t={t}_replicate={rep}.csv\" in line 168:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/v={v}_r={r}_a={a}_t={t}_replicate={rep}.csv\" in line 169:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/scoring.jl\" in line 179:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 180:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/{replicate}.json\" in line 181:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/{replicate}.json\" in line 183:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/sim_scores.tsv\" in line 196:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/simulation_study/heatmaps/{score}-mean.png\" in line 198:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/simulation_study/heatmaps/{score}-t.png\" in line 199:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={{d}}_lstd={{lstd}}/{{dataset}}_replicate={rep}.json\" in line 205:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/simulation_study/convergence/mcmc_d={d}_lstd={lstd}/{dataset}.png\" in line 208:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/convergence_viz.py\" in line 210:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/postprocess_samples.jl\" in line 219:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{{mcmc_settings}}/{{dataset}}/{chain}.json\" in line 220:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{mcmc_settings}/{dataset}.json\" in line 222:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{dataset}.csv\" in line 234:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{dataset}.csv\" in line 235:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={d}_lstd={lstd}/{dataset}/{chain}.json\" in line 237:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/postprocess_samples.jl\" in line 250:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_{{mcmc_settings}}/{{replicate}}/{chain}.json\" in line 251:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_{mcmc_settings}/{replicate}.json\" in line 254:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 265:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 266:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={d}/{replicate}/{chain}.json\" in line 268:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/postprocess_samples.jl\" in line 284:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/uniform/{{replicate}}/{chain}.json\" in line 285:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/uniform/{replicate}.json\" in line 288:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 300:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 301:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/uniform/{replicate}/{chain}.json\" in line 303:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/funchisq_wrapper.R\" in line 322:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 323:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/funchisq/{replicate}.json\" in line 325:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 345:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 346:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/hill/{replicate}.json\" in line 348:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 359:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 360:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/preds/hill_deg={deg}_mode={mode}/{replicate}.json\" in line 362:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/preds/hill_deg=\" in line 373:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/v=\" in line 373:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/hill_method_timetest.csv\" in line 375:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lasso.jl\" in line 387:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 388:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 389:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lasso/{replicate}.json\" in line 391:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/prior_baseline.jl\" in line 406:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 407:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/prior_baseline/{replicate}.json\" in line 409:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={{d}}_lstd={{lstd}}/{{dataset}}_replicate={rep}.json\" in line 426:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/dream/convergence/mcmc_d={d}_lstd={lstd}/{dataset}.png\" in line 429:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/convergence_viz.py\" in line 431:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/score_dream.py\" in line 435:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/TrueVec_{cell_line}_{stim}.csv\" in line 436:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/cl={cell_line}_stim={stim}_replicate={rep}.json\" in line 437:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_antibodies.json\" in line 438:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/cl={cell_line}_stim={stim}_replicate={rep}.json\" in line 440:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/postprocess_samples.jl\" in line 450:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_{{mcmc_settings}}/{{replicate}}/chain={chain}.json\" in line 451:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_{mcmc_settings}/{replicate}.json\" in line 454:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_stim={stimulus}.csv\" in line 467:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}.csv\" in line 468:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/mcmc_d={d}_lstd={lstd}/cl={cell_line}_stim={stimulus}_replicate={replicate}/chain={chain}.json\" in line 470:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/score_dream.py\" in line 483:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/TrueVec_{cell_line}_{stim}.csv\" in line 484:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/cl={cell_line}_stim={stim}.json\" in line 485:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_antibodies.json\" in line 486:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{method}/cl={cell_line}_stim={stim}.json\" in line 488:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/funchisq_wrapper.R\" in line 498:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{replicate}.csv\" in line 499:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/funchisq/{replicate}.json\" in line 501:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_stim={stim}.csv\" in line 512:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}.csv\" in line 513:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/hill/cl={cell_line}_stim={stim}.json\" in line 515:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lasso.jl\" in line 526:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}.csv\" in line 527:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_stim={stim}.csv\" in line 528:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/lasso/cl={cell_line}_stim={stim}.json\" in line 530:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/prior_baseline.jl\" in line 541:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}.csv\" in line 542:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/prior_baseline/cl={cell_line}_stim={stim}.json\" in line 544:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{cell_line}_main.csv\" in line 555:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={{cell_line}}_stim={stimulus}.csv\" in line 557:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/{cell_line}.eda\" in line 564:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}.csv\" in line 566:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/cl={cell_line}_antibodies.json\" in line 567:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 136:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 138:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 147:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 153:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 155:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 158:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 165:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 168:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 179:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 181:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 183:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 196:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 199:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 205:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 208:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 210:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 220:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 222:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 234:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 237:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 250:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 254:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 265:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 268:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 284:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 288:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 300:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 303:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 322:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 325:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 345:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 348:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 359:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 362:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 373:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 375:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 387:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 389:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 406:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 409:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 426:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 429:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 431:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 436:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 438:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 450:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 454:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 467:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 470:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 483:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 485:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 486:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 498:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 501:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 512:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 515:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 526:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 528:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 541:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 544:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 555:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 564:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 567:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 20:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 373:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule tabulate_sim_scores (line 148, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule tabulate_dream_scores (line 169, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule simulate_data (line 192, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_M from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable POLY_DEG from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule score_sim_predictions (line 219, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule sim_heatmaps (line 249, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SIM_METHODS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule convergence_viz (line 269, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule postprocess_conv_mcmc_sim (line 296, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONV_BURNIN from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONV_STOPPOINTS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_conv_mcmc_sim (line 325, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONV_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONV_MAX_SAMPLES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule postprocess_sim_mcmc (line 357, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_mcmc (line 384, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_MAX_SAMPLES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule postprocess_sim_uniform_mcmc (line 419, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_uniform_mcmc (line 447, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_MAX_SAMPLES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_funchisq (line 485, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable FUNCH_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_hill (line 521, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable HILL_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_timetest_hill (line 548, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable HILL_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable HILL_TIME_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tabulate_timetest_hill (line 575, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SCRIPT_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_lasso (line 600, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_sim_prior_baseline (line 632, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule dream_convergence_viz (line 665, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule score_dream_mcmc (line 687, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule postprocess_dream_mcmc (line 715, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DREAM_STOPPOINTS from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_dream_mcmc (line 745, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable DREAM_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONV_MAX_SAMPLES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable MCMC_INDEG from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule score_dream_predictions (line 777, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_dream_funchisq (line 805, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule run_dream_hill (line 832, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable HILL_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable SIM_TIMEOUT from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_dream_lasso (line 859, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule run_dream_prior_baseline (line 887, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable JULIA_PROJ_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule preprocess_dream_timeseries (line 914, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable DREAM_PREP_TS_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule preprocess_dream_prior (line 934, /tmp/tmps7t4hlvf/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 10,
    "subscribers_count": 3,
    "topics": [
      "dynamic-bayesian-network",
      "gen",
      "probabilistic-programming",
      "signaling-pathway"
    ],
    "updated_at": 1615072293.0
  },
  {
    "data_format": 1,
    "description": "ChIP-seq peak-calling, QC and differential analysis pipeline (Snakemake port of the nextflow pipeline at https://nf-co.re/chipseq).",
    "formatting": "[WARNING] In file \"/tmp/tmpj1vt_8ma/workflow/rules/peak_analysis.smk\":  Keyword \"params\" at line 41 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpj1vt_8ma/workflow/rules/filtering.smk\":  Keyword \"params\" at line 6 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmpj1vt_8ma/workflow/rules/filtering.smk\":  Inline-formatted keyword \"threads\" at line 56 had its comments relocated above it.\nPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 11 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "snakemake-workflows/chipseq",
    "linting": null,
    "stargazers_count": 11,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1615454934.0
  },
  {
    "data_format": 1,
    "description": "Harmonization of AMR predictor tool outputs",
    "formatting": "[WARNING] In file \"/tmp/tmp19bf5ygk/Snakefile\":  Keyword \"input\" at line 55 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[WARNING] In file \"/tmp/tmp19bf5ygk/rules/staramr.smk\":  Keyword \"shell\" at line 17 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmp19bf5ygk/rules/srax.smk\":  \u0027SyntaxError: L32: Colon (not \u0027srax\u0027) expected after \u0027rule hamronize\u0027\u0027\n[INFO] In file \"/tmp/tmp19bf5ygk/rules/srax.smk\":  1 file(s) raised parsing errors \ud83e\udd15\n[INFO] In file \"/tmp/tmp19bf5ygk/rules/srax.smk\":  17 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pha4ge/hAMRonization_workflow",
    "linting": "KeyError in line 7 of /tmp/tmp19bf5ygk/Snakefile:\n\u0027samples\u0027\n  File \"/tmp/tmp19bf5ygk/Snakefile\", line 7, in \u003cmodule\u003e\n",
    "stargazers_count": 11,
    "subscribers_count": 9,
    "topics": [
      "antimicrobial-resistance",
      "standardization",
      "file-format"
    ],
    "updated_at": 1619215097.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[WARNING] In file \"/tmp/tmpzqe9kb4t/rules/mapping.smk\":  Keyword \"shell\" at line 23 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 3 file(s) would be changed \ud83d\ude2c\n[INFO] 3 file(s) would be left unchanged \ud83c\udf89\n",
    "full_name": "nanoporetech/Pore-C-Snakemake",
    "linting": "ModuleNotFoundError in line 1 of /tmp/tmpzqe9kb4t/rules/common.smk:\nNo module named \u0027box\u0027\n  File \"/tmp/tmpzqe9kb4t/Snakefile\", line 25, in \u003cmodule\u003e\n  File \"/tmp/tmpzqe9kb4t/rules/common.smk\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 11,
    "subscribers_count": 22,
    "topics": [],
    "updated_at": 1614354595.0
  },
  {
    "data_format": 1,
    "description": "RNAseq pipeline based on snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-RNAseq",
    "linting": "FileNotFoundError in line 6 of /tmp/tmps8vcjqwv/Snakefile:\n[Errno 2] No such file or directory: \u0027./samples.json\u0027\n  File \"/tmp/tmps8vcjqwv/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 11,
    "subscribers_count": 2,
    "topics": [
      "rnaseq",
      "snakemake"
    ],
    "updated_at": 1607821432.0
  },
  {
    "data_format": 1,
    "description": "snakemake workflow for post-processing scATACseq data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-scATACseq",
    "linting": "FileNotFoundError in line 18 of /tmp/tmpwop4146g/Snakefile:\n[Errno 2] No such file or directory: \u0027/path/to/cluster_sample1.csv\u0027\n  File \"/tmp/tmpwop4146g/Snakefile\", line 37, in \u003cmodule\u003e\n  File \"/tmp/tmpwop4146g/Snakefile\", line 18, in get_cluster_id\n",
    "stargazers_count": 11,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1597229833.0
  },
  {
    "data_format": 1,
    "description": "shotgun pipeline",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "biocore/oecophylla",
    "linting": "KeyError in line 6 of /tmp/tmp0nzvoo8c/Snakefile:\n\u0027tmp_dir_root\u0027\n  File \"/tmp/tmp0nzvoo8c/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 12,
    "subscribers_count": 16,
    "topics": [],
    "updated_at": 1588351642.0
  },
  {
    "data_format": 1,
    "description": "Class materials for the NIH HPC snakemake class",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "NIH-HPC/snakemake-class",
    "linting": "Lints for snakefile /tmp/tmpj_m1z56k/Snakefile:\n    * Absolute path \"/^\u003e/ {{NF=1}} {{print}}\" in line 157:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Deprecated singularity directive used for container definition in line 259.:\n      Use the container directive instead (it is agnostic of the underlying\n      container runtime).\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule setup (line 19, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean (line 57, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fetch_container (line 84, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CONTAINER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONTAINER_NAME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONTAINER_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule wrapper (line 106, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CONTAINER_DIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CONTAINER_NAME from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fetch_sample_desc (line 135, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sample_table (line 165, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable SAMPLES from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fetch_fastq (line 196, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule config_yml (line 228, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fetch_genome (line 259, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ENSEMBL_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fetch_transcriptome (line 281, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ENSEMBL_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ENSEMBL_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule fetch_gtf (line 315, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ENSEMBL_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ENSEMBL_RELEASE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ENSEMBL_RELEASE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable ENSEMBL_RELEASE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule gtf2bed12 (line 337, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule make_transcript_gene_map (line 380, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule chroms (line 415, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ENSEMBL_URL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ref_yaml (line 435, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable ENSEMBL_RELEASE from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule make_hisat_index (line 464, /tmp/tmpj_m1z56k/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 12,
    "subscribers_count": 6,
    "topics": [],
    "updated_at": 1613596422.0
  },
  {
    "data_format": 1,
    "description": "Bioinformatic pipeline for identifying dsDNA breaks by marker based incorporation, such as breaks induced by designer nucleases like Cas9.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "cnobles/iGUIDE",
    "linting": "ModuleNotFoundError in line 12 of /tmp/tmp3idbg161/Snakefile:\nNo module named \u0027iguidelib\u0027\n  File \"/tmp/tmp3idbg161/Snakefile\", line 12, in \u003cmodule\u003e\n",
    "stargazers_count": 12,
    "subscribers_count": 1,
    "topics": [],
    "updated_at": 1613699838.0
  },
  {
    "data_format": 1,
    "description": "SigMa is a probabilistic model for the sequential dependencies of mutation signatures",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lrgr/sigma",
    "linting": "FileNotFoundError in line 21 of /tmp/tmpyypyhlaw/Snakefile:\n[Errno 2] No such file or directory: \u0027data/mutations/ICGC-BRCA-EU.RELEASE_22.SBS.renamed.sigma.json\u0027\n  File \"/tmp/tmpyypyhlaw/Snakefile\", line 21, in \u003cmodule\u003e\n",
    "stargazers_count": 12,
    "subscribers_count": 7,
    "topics": [
      "mutational-signatures",
      "hidden-markov-models",
      "cancer",
      "breast-cancer"
    ],
    "updated_at": 1596079024.0
  },
  {
    "data_format": 1,
    "description": "Reproducibility workflow for Gigante et al., 2018: Using long-read sequencing to detect imprinted DNA methylation",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "scottgigante/haplotyped-methylome",
    "linting": "The flag \u0027directory\u0027 used in rule untar_cleanup is only valid for outputs, not inputs.\nLints for rule suppdb_bam (line 1, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_methylation_by_alignment (line 22, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_methylation_frequency (line 43, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_variant_proportion (line 62, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_by_read (line 90, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sort_by_site (line 116, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule read_summary (line 142, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule read_haplotype (line 161, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fit_reads (line 180, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule split_methylation_by_haplotype (line 206, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calculate_allele_meth_freq (line 227, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule paired_dss (line 246, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compare_haplotype_methylation (line 272, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_dmrlist (line 293, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_methylation_df (line 313, /tmp/tmpf_fia2ds/scripts/allelic_methylation.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_variant_proportion_threeway (line 1, /tmp/tmpf_fia2ds/scripts/fvb_resolution.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fvb_resolution (line 26, /tmp/tmpf_fia2ds/scripts/fvb_resolution.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_threeway_variants (line 52, /tmp/tmpf_fia2ds/scripts/fvb_resolution.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_b6xcast_0503 (line 1, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_b6xcast_0512 (line 20, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_b6xcast_0525 (line 39, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_castxb6 (line 58, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_b6 (line 77, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_fast5_cast (line 96, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fast5_md5 (line 115, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule untar (line 135, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule untar_cleanup (line 157, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_b6xcast (line 186, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule albacore_minion (line 215, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule albacore_promethion (line 245, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule albacore_merge_output (line 275, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bwa_mem_align (line 294, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule nanopolish_index (line 325, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule nanopolish_methylation (line 349, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule nanopolish_phase (line 376, /tmp/tmpf_fia2ds/scripts/nanopolish.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule b6_haplotype_analysis (line 1, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cast_haplotype_analysis (line 27, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule b6xcast_haplotype_analysis (line 53, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule castxb6_haplotype_analysis (line 79, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule haplotype_auc (line 105, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dmr_comparison (line 130, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule visualise_dmrs (line 158, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule differential_methylation_heatmaps (line 195, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_level_methylation_summary (line 222, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule nanopolish_methylation_validation (line 250, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rnaseq_analysis_notebook (line 276, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_visualisation (line 321, /tmp/tmpf_fia2ds/scripts/notebooks.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_B6_genome (line 1, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_gtf (line 20, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gunzip_fa (line 39, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gunzip_vcf (line 58, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gunzip_gtf (line 77, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bwa_index (line 96, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_index (line 124, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_vcf (line 143, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule mask_with_cast_genome (line 163, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule ensembl_gtf_to_tsv (line 183, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule icr_plot_region_string (line 202, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome_cpg_coordinates (line 221, /tmp/tmpf_fia2ds/scripts/misc.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_B6Cast2 (line 6, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_B6Cast3 (line 26, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_B6Cast4 (line 46, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_B6Cast5 (line 66, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_CastB62 (line 86, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_CastB63 (line 106, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_CastB64 (line 126, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_rnaseq_CastB65 (line 146, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule fastq_md5 (line 166, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule trim_galore (line 186, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snpsplit_create_path (line 215, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snpsplit_prepare_genome (line 234, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_up_snpsplit_genome (line 284, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param tempdir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule build_hisat2 (line 334, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule map_hisat2 (line 360, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param base is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule snp_split_hisat2 (line 396, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snp_split_sort (line 422, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule samtools_depth (line 451, /tmp/tmpf_fia2ds/scripts/rna_seq.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_bisulfite_B6Cast1 (line 5, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_bisulfite_B6Cast2 (line 25, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_bisulfite_B6Cast5 (line 45, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_bisulfite_B6Cast6 (line 65, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_create_path (line 85, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_prepare_genome (line 104, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_align (line 132, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param genome is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule sort_bisulfite (line 168, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snpsplit_bismark (line 195, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bismark_extract (line 221, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bisulfite_genome1 (line 255, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_bisulfite_genome2 (line 277, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule merge_matched_bisulfite (line 299, /tmp/tmpf_fia2ds/scripts/rrbs.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_nanopolish_b6xcast (line 1, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_nanopolish_castxb6 (line 20, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_nanopolish_cast (line 39, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_nanopolish_b6 (line 58, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule intermediate_md5 (line 77, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule intermediate_gunzip (line 97, /tmp/tmpf_fia2ds/scripts/intermediate.snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 14,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1614375266.0
  },
  {
    "data_format": 1,
    "description": "A model of the European power system built using Calliope.",
    "formatting": "[INFO] 3 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "calliope-project/euro-calliope",
    "linting": "ModuleNotFoundError in line 2 of /tmp/tmpmki557_j/rules/shapes.smk:\nNo module named \u0027pycountry\u0027\n  File \"/tmp/tmpmki557_j/Snakefile\", line 21, in \u003cmodule\u003e\n  File \"/tmp/tmpmki557_j/rules/shapes.smk\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 14,
    "subscribers_count": 5,
    "topics": [
      "calliope-models",
      "conda",
      "europe",
      "research",
      "energy",
      "renewable-energy"
    ],
    "updated_at": 1619190520.0
  },
  {
    "data_format": 1,
    "description": "Deep mutational scanning of the receptor-binding domain of SARS-CoV-2 Spike",
    "formatting": "[ERROR] In file \"/tmp/tmp10wnqrmy/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 1:0: elif config[\u0027seqdata_source\u0027] == \u0027SRA\u0027:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmp10wnqrmy/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "jbloomlab/SARS-CoV-2-RBD_DMS",
    "linting": "Lints for snakefile /tmp/tmp10wnqrmy/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule make_summary (line 34, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule make_dag (line 141, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule interactive_heatmap (line 161, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule sarbecovirus_diversity (line 185, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule antibody_epitopes (line 224, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule circulating_variants (line 263, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule logoplots_of_muteffects (line 301, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule structure_function (line 326, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule single_mut_effects (line 365, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule global_epistasis_binding (line 408, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule global_epistasis_expression (line 432, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compute_Titeseq_Kds (line 456, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule compute_expression_meanFs (line 496, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Additionally specify a conda environment or container for each rule, environment modules are not enough:\n      While environment modules allow to document and deploy the required\n      software on a certain platform, they lock your workflow in there,\n      disabling easy reproducibility on other machines that don\u0027t have exactly\n      the same environment modules. Hence env modules (which might be beneficial\n      in certain cluster environments), should allways be complemented with\n      equivalent conda environments.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule analyze_counts (line 536, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule count_variants (line 561, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_variants (line 588, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule process_ccs (line 614, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_ccs (line 648, /tmp/tmp10wnqrmy/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 14,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1619105803.0
  },
  {
    "data_format": 1,
    "description": "A straightforward bioinformatic pipeline for detecting a bacterial strain in one or more metagenome(s).",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bhattlab/StrainSifter",
    "linting": "Lints for snakefile /tmp/tmpg0agu4p6/Snakefile:\n    * Absolute path \"/|\\.\" in line 16:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule bwa_index (line 24, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule bwa_align (line 52, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Param ref is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule genomecov (line 90, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule calc_coverage (line 118, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filter_samples (line 150, /tmp/tmpg0agu4p6/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule faidx (line 183, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pileup (line 206, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule call_snps (line 235, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule snp_consensus (line 267, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule combine (line 293, /tmp/tmpg0agu4p6/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule core_snps (line 321, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule core_snps_to_fasta (line 347, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule multi_align (line 373, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_tree (line 399, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_tree (line 425, /tmp/tmpg0agu4p6/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule pairwise_snvs (line 451, /tmp/tmpg0agu4p6/Snakefile):\n    * The dynamic flag is deprecated:\n      Use checkpoints instead, which are more powerful and less error-prone.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#data-dependent-conditional-execution\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 14,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1610657508.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for de novo clustering of long transcriptomic reads",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "nanoporetech/pipeline-nanopore-denovo-isoforms",
    "linting": "find: \u2018/tmp/tmp865aaiyk/evaluation/data/SIRV_PSC109_phmm_fl.fq\u2019: No such file or directory\nCalledProcessError in line 154 of /tmp/tmp865aaiyk/Snakefile:\nCommand \u0027set -euo pipefail;  find /tmp/tmp865aaiyk/evaluation/data/SIRV_PSC109_phmm_fl.fq  -regextype posix-extended -regex \u0027.*\\.(fastq|fq)$\u0027 -exec cat {} \\; \u003e processed_reads/input_reads.fq\u0027 returned non-zero exit status 1.\n  File \"/tmp/tmp865aaiyk/Snakefile\", line 171, in \u003cmodule\u003e\n  File \"/tmp/tmp865aaiyk/Snakefile\", line 154, in preprocess_reads\n",
    "stargazers_count": 14,
    "subscribers_count": 22,
    "topics": [
      "cdna",
      "rna-seq",
      "rna",
      "transcriptomics"
    ],
    "updated_at": 1615883381.0
  },
  {
    "data_format": 1,
    "description": "Snakemake skeleton - Build workflows with Snakemake",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jlanga/smsk",
    "linting": "Lints for snakefile /tmp/tmpamv9goy3/src/snakefiles/raw.smk:\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 40:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 43:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 44:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpamv9goy3/src/snakefiles/map.smk:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 4:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpamv9goy3/src/snakefiles/call.smk:\n    * Path composition with \u0027+\u0027 in line 6:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule clean_map (line 1, /tmp/tmpamv9goy3/src/snakefiles/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable MAP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_call (line 15, /tmp/tmpamv9goy3/src/snakefiles/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable CALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean_report (line 29, /tmp/tmpamv9goy3/src/snakefiles/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable REPORT_CALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule clean (line 43, /tmp/tmpamv9goy3/src/snakefiles/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable MAP from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable CALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable REPORT_CALL from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule raw_extract_genome (line 34, /tmp/tmpamv9goy3/src/snakefiles/raw.smk):\n    * Shell command directly uses variable RAW from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule raw_extract_samples (line 83, /tmp/tmpamv9goy3/src/snakefiles/raw.smk):\n    * Shell command directly uses variable RAW from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule call_report (line 58, /tmp/tmpamv9goy3/src/snakefiles/call.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\n",
    "stargazers_count": 15,
    "subscribers_count": 5,
    "topics": [
      "snakemake",
      "tutorial",
      "workflow",
      "skeleton"
    ],
    "updated_at": 1616787025.0
  },
  {
    "data_format": 1,
    "description": "StaG Metagenomic Workflow Collaboration",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "ctmrbio/stag-mwc",
    "linting": "WorkflowError in line 43 of /tmp/tmp10k9svyj/Snakefile:\nFound no samples! Check input file pattern and path in config.yaml\n  File \"/tmp/tmp10k9svyj/Snakefile\", line 43, in \u003cmodule\u003e\n",
    "stargazers_count": 16,
    "subscribers_count": 6,
    "topics": [
      "snakemake-workflows",
      "metagenomic-analysis",
      "metagenomics",
      "research",
      "microbiome"
    ],
    "updated_at": 1618328487.0
  },
  {
    "data_format": 1,
    "description": null,
    "formatting": "[INFO] 35 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "yesimon/metax_bakeoff_2019",
    "linting": "DefaultCredentialsError in line 2 of /tmp/tmpjxbi9wh8/rules/download.smk:\nCould not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started\n  File \"/tmp/tmpjxbi9wh8/Snakefile\", line 102, in \u003cmodule\u003e\n  File \"/tmp/tmpjxbi9wh8/rules/download.smk\", line 2, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/snakemake/remote/GS.py\", line 124, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/storage/client.py\", line 121, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 249, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 201, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/client.py\", line 216, in _determine_default\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/cloud/_helpers.py\", line 186, in _determine_default_project\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/google/auth/_default.py\", line 483, in default\n",
    "stargazers_count": 16,
    "subscribers_count": 0,
    "topics": [],
    "updated_at": 1611274385.0
  },
  {
    "data_format": 1,
    "description": "Reproducible reanalysis of a combined ChIP-Seq \u0026 RNA-Seq data set",
    "formatting": "[ERROR] In file \"/tmp/tmpomgw29lu/Snakefile\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] In file \"/tmp/tmpomgw29lu/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "DarwinAwardWinner/CD4-csaw",
    "linting": "ModuleNotFoundError in line 4 of /tmp/tmpomgw29lu/Snakefile:\nNo module named \u0027rpy2\u0027\n  File \"/tmp/tmpomgw29lu/Snakefile\", line 4, in \u003cmodule\u003e\n",
    "stargazers_count": 16,
    "subscribers_count": 2,
    "topics": [
      "r",
      "bioinformatics-pipeline",
      "reproducible-research",
      "rna-seq",
      "chip-seq",
      "bioconductor"
    ],
    "updated_at": 1590666469.0
  },
  {
    "data_format": 1,
    "description": "A workflow for creating small NGS test data sets, useful for continuous integration.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "snakemake-workflows/ngs-test-data",
    "linting": "Lints for snakefile /tmp/tmp0xnsw9xc/Snakefile:\n    * Absolute path \"/\u003e\u003cAttribute name = \" in line 39:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n\nLints for rule annotation (line 25, /tmp/tmp0xnsw9xc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule genome (line 45, /tmp/tmp0xnsw9xc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule transcriptome (line 64, /tmp/tmp0xnsw9xc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule reads (line 80, /tmp/tmp0xnsw9xc/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\n",
    "stargazers_count": 17,
    "subscribers_count": 1,
    "topics": [
      "snakemake",
      "snakemake-workflows",
      "ngs"
    ],
    "updated_at": 1615315913.0
  },
  {
    "data_format": 1,
    "description": "Files and methodology pertaining to the sequencing and analysis of SARS-CoV-2, causative agent of COVID-19.",
    "formatting": "[ERROR] In file \"/tmp/tmpj1dngxpu/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 2:0:     else:\n```\nwhile formatting code with black.\u0027\n[INFO] In file \"/tmp/tmpj1dngxpu/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "jaleezyy/covid-19-signal",
    "linting": "FileNotFoundError in line 45 of /tmp/tmpj1dngxpu/Snakefile:\n[Errno 2] No such file or directory: \u0027sample_table.csv\u0027\n  File \"/tmp/tmpj1dngxpu/Snakefile\", line 45, in \u003cmodule\u003e\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 689, in read_table\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 462, in _read\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 819, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1050, in _make_engine\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1867, in __init__\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/parsers.py\", line 1362, in _open_handles\n  File \"/home/runner/micromamba/envs/snakemake-workflow-catalog/lib/python3.9/site-packages/pandas/io/common.py\", line 642, in get_handle\n",
    "stargazers_count": 17,
    "subscribers_count": 12,
    "topics": [],
    "updated_at": 1618955789.0
  },
  {
    "data_format": 1,
    "description": "Open-source bioinformatics pipeline for the preprocessing of raw sequencing data.",
    "formatting": "[INFO] 9 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "MW55/Natrix",
    "linting": "KeyError in line 6 of /tmp/tmps60ckm_a/Snakefile:\n\u0027general\u0027\n  File \"/tmp/tmps60ckm_a/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 19,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1615829706.0
  },
  {
    "data_format": 1,
    "description": "Small collection of tools for performing quality control on coronavirus sequencing data and genomes",
    "formatting": "[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "jts/ncov-tools",
    "linting": "FileNotFoundError in line 6 of /tmp/tmp3i3jn2f5/workflow/rules/common.smk:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmp3i3jn2f5/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n  File \"/tmp/tmp3i3jn2f5/workflow/rules/common.smk\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 21,
    "subscribers_count": 11,
    "topics": [],
    "updated_at": 1617656404.0
  },
  {
    "data_format": 1,
    "description": "reproducible research project template for R using Renv and snakemake with an econ application",
    "formatting": "[ERROR] In file \"/tmp/tmpxi5106io/rules/slides.smk\":  \u0027AttributeError: \u0027NoneType\u0027 object has no attribute \u0027group\u0027\u0027\n[INFO] 1 file(s) raised parsing errors \ud83e\udd15\n[INFO] 10 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "lachlandeer/snakemake-econ-r",
    "linting": "Lints for snakefile /tmp/tmpxi5106io/Snakefile:\n    * Path composition with \u0027+\u0027 in line 30:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpxi5106io/rules/paper.smk:\n    * Path composition with \u0027+\u0027 in line 23:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for snakefile /tmp/tmpxi5106io/rules/slides.smk:\n    * Path composition with \u0027+\u0027 in line 11:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule clean_all (line 42, /tmp/tmpxi5106io/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_main (line 57, /tmp/tmpxi5106io/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule gen_regression_vars (line 7, /tmp/tmpxi5106io/rules/data_mgt.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule rename_vars (line 40, /tmp/tmpxi5106io/rules/data_mgt.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule models (line 21, /tmp/tmpxi5106io/rules/analysis.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule figures (line 18, /tmp/tmpxi5106io/rules/figures.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule tables (line 20, /tmp/tmpxi5106io/rules/tables.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule paper2root (line 19, /tmp/tmpxi5106io/rules/paper.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_paper (line 39, /tmp/tmpxi5106io/rules/paper.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule slides2root (line 7, /tmp/tmpxi5106io/rules/slides.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_slides (line 27, /tmp/tmpxi5106io/rules/slides.smk):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable logAll from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule renv_install (line 10, /tmp/tmpxi5106io/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable runR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule renv_consent (line 22, /tmp/tmpxi5106io/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_init (line 34, /tmp/tmpxi5106io/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_snap (line 46, /tmp/tmpxi5106io/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule renv_restore (line 58, /tmp/tmpxi5106io/rules/renv.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_output (line 6, /tmp/tmpxi5106io/rules/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_log (line 18, /tmp/tmpxi5106io/rules/clean.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule dag (line 8, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule filegraph (line 29, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rulegraph (line 49, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule rulegraph_to_png (line 69, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule graphviz (line 92, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule graphviz_mac (line 104, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule install_poppler (line 116, /tmp/tmpxi5106io/rules/dag.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_analysis (line 6, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_data_mgt (line 21, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_figures (line 36, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_paper (line 51, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_slides (line 66, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule help_tables (line 81, /tmp/tmpxi5106io/rules/help.smk):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 22,
    "subscribers_count": 2,
    "topics": [
      "snakemake",
      "snakemake-workflow",
      "economics",
      "workflow",
      "renv",
      "reproducible-research",
      "reproducibility"
    ],
    "updated_at": 1617505128.0
  },
  {
    "data_format": 1,
    "description": "PyPSA-Eur-Sec: A Sector-Coupled Open Optimisation Model of the European Energy System",
    "formatting": "[WARNING] In file \"/tmp/tmpon3o568t/Snakefile\":  Keyword \"input\" at line 371 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PyPSA/pypsa-eur-sec",
    "linting": "FileNotFoundError in line 2 of /tmp/tmpon3o568t/Snakefile:\n[Errno 2] No such file or directory: \u0027config.yaml\u0027\n  File \"/tmp/tmpon3o568t/Snakefile\", line 2, in \u003cmodule\u003e\n",
    "stargazers_count": 22,
    "subscribers_count": 11,
    "topics": [],
    "updated_at": 1619341195.0
  },
  {
    "data_format": 1,
    "description": "Ley Lab MetaGenome Profiler DataBase generator",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "leylabmpi/Struo",
    "linting": "/tmp/tmpft_h_rbp/Snakefile:28: FutureWarning: The default value of regex will change from True to False in a future version.\n  config[\u0027samples\u0027] = config[\u0027samples\u0027].set_index(config[\u0027samples\u0027][config[\u0027samples_col\u0027]])\nSamples table (Row 1): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_002009845.1/GCA_002009845.1_ASM200984v1_genomic.fna.gz; Skipping\nSamples table (Row 2): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000214495.2/GCA_000214495.2_ASM21449v1_genomic.fna.gz; Skipping\nSamples table (Row 3): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000221545.2/GCA_000221545.2_ASM22154v1_genomic.fna.gz; Skipping\nSamples table (Row 4): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000764165.1/GCA_000764165.1_ASM76416v1_genomic.fna.gz; Skipping\nSamples table (Row 5): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_900143645.1/GCA_900143645.1_IMG-taxon_2582580727_annotated_assembly_genomic.fna.gz; Skipping\nSamples table (Row 6): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_900205925.1/GCA_900205925.1_Clostridium_peptidoveorans_DPC7177_genomic.fna.gz; Skipping\nSamples table (Row 7): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_900176335.1/GCA_900176335.1_IMG-taxon_2588254267_annotated_assembly_genomic.fna.gz; Skipping\nSamples table (Row 8): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000755365.1/GCA_000755365.1_ASM75536v1_genomic.fna.gz; Skipping\nSamples table (Row 9): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000422585.1/GCA_000422585.1_ASM42258v1_genomic.fna.gz; Skipping\nSamples table (Row 10): Cannot find file: /ebio/abt3_projects/databases_no-backup/GTDB/release86/genbank/bacteria/GCA_000298175.1/GCA_000298175.1_ASM29817v1_genomic.fna.gz; Skipping\nTotal number of skipped rows: 10\nValueError in line 44 of /tmp/tmpft_h_rbp/Snakefile:\nNo genomes remaining after filtering!\n  File \"/tmp/tmpft_h_rbp/Snakefile\", line 44, in \u003cmodule\u003e\n",
    "stargazers_count": 24,
    "subscribers_count": 5,
    "topics": [],
    "updated_at": 1616748173.0
  },
  {
    "data_format": 1,
    "description": "Extract lineage CSVs from NCBI for use with sourmash lca.",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "dib-lab/2018-ncbi-lineages",
    "linting": "Lints for rule download_wgs_acc2taxid (line 11, /tmp/tmpx8ojgmd2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_gb_acc2taxid (line 26, /tmp/tmpx8ojgmd2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule download_taxdump (line 41, /tmp/tmpx8ojgmd2/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_taxid (line 57, /tmp/tmpx8ojgmd2/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_lineage_csv (line 78, /tmp/tmpx8ojgmd2/Snakefile):\n    * Do not access input and output files individually by index in shell commands:\n      When individual access to input or output files is needed (i.e., just\n      writing \u0027{input}\u0027 is impossible), use names (\u0027{input.somename}\u0027) instead\n      of index based access.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#rules\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 25,
    "subscribers_count": 2,
    "topics": [],
    "updated_at": 1592359897.0
  },
  {
    "data_format": 1,
    "description": "Snakemake workflow for metagenomic classification with Kraken2",
    "formatting": "[WARNING] In file \"/tmp/tmpmqz_pzje/Snakefile\":  Keyword \"input\" at line 131 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bhattlab/kraken2_classification",
    "linting": "KeyError in line 8 of /tmp/tmpmqz_pzje/Snakefile:\n\u0027outdir\u0027\n  File \"/tmp/tmpmqz_pzje/Snakefile\", line 8, in \u003cmodule\u003e\n",
    "stargazers_count": 26,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1618981942.0
  },
  {
    "data_format": 1,
    "description": "A Snakemake pipeline for the generation of MAGs, reconstruction of GEMs, and simulation of cross-feeding interactions within microbial communities from lab cultures, human gut, ocean, plant-associated, and bulk soil microbiomes",
    "formatting": "[ERROR] In file \"/tmp/tmpz9x85izc/Snakefile\":  \u0027NoParametersError: L567: In output definition.\u0027\n[INFO] In file \"/tmp/tmpz9x85izc/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "franciscozorrilla/metaGEM",
    "linting": "Lints for snakefile /tmp/tmpz9x85izc/Snakefile:\n    * Absolute path \"/^$/d\" in line 48:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/other_related_references(genome_id,species_name,radius,ANI,AF)/d\" in line 1095:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/other_related_references(genome_id,species_name,radius,ANI,AF)/d\" in line 1152:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/-/d\" in line 1276:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/N\\/A/d\" in line 1276:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Absolute path \"/community/d\" in line 1349:\n      Do not define absolute paths inside of the workflow, since this renders\n      your workflow irreproducible on other machines. Use path relative to the\n      working directory instead, or make the path configurable via a config\n      file.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/configuration.html#configuration\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule all (line 21, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule createFolders (line 48, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule downloadToy (line 101, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule organizeData (line 191, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule qfilter (line 262, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule qfilterVis (line 368, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule megahit (line 449, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule assemblyVis (line 567, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule crossMap (line 647, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule kallistoIndex (line 823, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule crossMap3 (line 889, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule gatherCrossMap3 (line 954, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule kallisto2concoctTable (line 973, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule concoct (line 1013, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule metabat (line 1094, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule metabatCross (line 1219, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule maxbin (line 1264, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule maxbinCross (line 1333, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule binRefine (line 1390, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule binReassemble (line 1468, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule binningVis (line 1536, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule abundance (line 1807, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule GTDBtk (line 2050, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule GTDBtkVis (line 2105, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule compositionVis (line 2188, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extractProteinBins (line 2277, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule carveme (line 2317, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule modelVis (line 2387, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule ECvis (line 2456, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule organizeGEMs (line 2549, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule smetana (line 2590, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule interactionVis (line 2640, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable print from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule memote (line 2670, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule grid (line 2724, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule extractDnaBins (line 2786, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule prokka (line 2830, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule roary (line 2877, /tmp/tmpz9x85izc/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 26,
    "subscribers_count": 2,
    "topics": [
      "metagenomics",
      "computational-biology",
      "metabolic-models",
      "gut-microbiome",
      "snakemake",
      "community-simulations",
      "metagenome-assembled-genomes"
    ],
    "updated_at": 1619269671.0
  },
  {
    "data_format": 1,
    "description": "Variant detection with Delly and Freebayes, followed by statistical assessment with Varlociraptor",
    "formatting": "[WARNING] In file \"/tmp/tmpck72c0nc/workflow/rules/common.smk\":  Keyword \"wildcard_constraints\" at line 291 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 16 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "snakemake-workflows/dna-seq-varlociraptor",
    "linting": null,
    "stargazers_count": 32,
    "subscribers_count": 4,
    "topics": [],
    "updated_at": 1617977162.0,
    "standardized": true,
    "mandatory_flags": { "flags": "", "desc": ""},
    "report": true,
    "software_stack_deployment": ["conda"],
    "latest_release": "v1.0.0",
    "config_readme": "<html><head></head><body>dummy config readme</body></html>"
  },
  {
    "data_format": 1,
    "description": "CADD scripts release for offline scoring. For more information about CADD, please visit our website",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "kircherlab/CADD-scripts",
    "linting": "Lints for rule decompress (line 4, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare (line 25, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule prescore (line 55, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule annotation (line 111, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule imputation (line 151, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule score (line 179, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule join (line 225, /tmp/tmp88v2wlbi/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable config from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 34,
    "subscribers_count": 6,
    "topics": [],
    "updated_at": 1616588433.0
  },
  {
    "data_format": 1,
    "description": "A workflow for RNA-seq differential expression analysis based on Kallisto and Sleuth.",
    "formatting": null,
    "full_name": "snakemake-workflows/rna-seq-kallisto-sleuth",
    "linting": null,
    "stargazers_count": 36,
    "subscribers_count": 3,
    "topics": [],
    "updated_at": 1618821327.0
  },
  {
    "data_format": 1,
    "description": "A tool for generating bacterial genomes from metagenomes with nanopore long read sequencing",
    "formatting": "[WARNING] In file \"/tmp/tmpxd_47i4z/Snakefile\":  Keyword \"output\" at line 133 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bhattlab/lathe",
    "linting": "KeyError in line 22 of /tmp/tmpxd_47i4z/Snakefile:\n\u0027file_names_txt\u0027\n  File \"/tmp/tmpxd_47i4z/Snakefile\", line 22, in \u003cmodule\u003e\n",
    "stargazers_count": 43,
    "subscribers_count": 6,
    "topics": [],
    "updated_at": 1619176295.0
  },
  {
    "data_format": 1,
    "description": "ATAC-seq snakemake pipeline",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-ATACseq",
    "linting": "FileNotFoundError in line 11 of /tmp/tmp0xp024_1/Snakefile:\n[Errno 2] No such file or directory: \u0027./samples.json\u0027\n  File \"/tmp/tmp0xp024_1/Snakefile\", line 11, in \u003cmodule\u003e\n",
    "stargazers_count": 48,
    "subscribers_count": 5,
    "topics": [
      "atac-seq",
      "snakemake"
    ],
    "updated_at": 1618792184.0
  },
  {
    "data_format": 1,
    "description": "Meta-pipeline to identify transposable element insertions using next generation sequencing data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bergmanlab/mcclintock",
    "linting": "KeyError in line 1 of /tmp/tmpl6osh3a7/Snakefile:\n\u0027args\u0027\n  File \"/tmp/tmpl6osh3a7/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 51,
    "subscribers_count": 14,
    "topics": [],
    "updated_at": 1618848506.0
  },
  {
    "data_format": 1,
    "description": "Meta-pipeline to identify transposable element insertions using next generation sequencing data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bergmanlab/mcclintock",
    "linting": "KeyError in line 1 of /tmp/tmpl6osh3a7/Snakefile:\n\u0027args\u0027\n  File \"/tmp/tmpl6osh3a7/Snakefile\", line 1, in \u003cmodule\u003e\n",
    "stargazers_count": 51,
    "subscribers_count": 14,
    "topics": [],
    "updated_at": 1618848506.0
  },
  {
    "data_format": 1,
    "description": "Plotting tools for nanopore methylation data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "wdecoster/methplotlib",
    "linting": "KeyError in line 23 of /tmp/tmpaskgg0hm/Snakefile:\n\u0027fastq\u0027\n  File \"/tmp/tmpaskgg0hm/Snakefile\", line 23, in \u003cmodule\u003e\n",
    "stargazers_count": 52,
    "subscribers_count": 8,
    "topics": [],
    "updated_at": 1616492882.0
  },
  {
    "data_format": 1,
    "description": "A single cell RNA-seq workflow, including highly variable gene analysis, cell type assignment and differential expression analysis.",
    "formatting": null,
    "full_name": "snakemake-workflows/single-cell-rna-seq",
    "linting": null,
    "stargazers_count": 63,
    "subscribers_count": 7,
    "topics": [],
    "updated_at": 1618360767.0
  },
  {
    "data_format": 1,
    "description": "Pipeline for differential gene expression (DGE) and differential transcript usage (DTU) analysis using long reads",
    "formatting": "[ERROR] In file \"/tmp/tmp_3z31mjw/Snakefile\":  \u0027NoParametersError: L92: In input definition.\u0027\n[INFO] In file \"/tmp/tmp_3z31mjw/Snakefile\":  1 file(s) raised parsing errors \ud83e\udd15\n",
    "full_name": "nanoporetech/pipeline-transcriptome-de",
    "linting": "Lints for snakefile /tmp/tmp_3z31mjw/snakelib/utils.snake:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\nLints for rule help (line 12, /tmp/tmp_3z31mjw/snakelib/utils.snake):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule clean_workdir (line 25, /tmp/tmp_3z31mjw/snakelib/utils.snake):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule clean_resdir (line 39, /tmp/tmp_3z31mjw/snakelib/utils.snake):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule info (line 53, /tmp/tmp_3z31mjw/snakelib/utils.snake):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule dump_versions (line 27, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule build_minimap_index (line 48, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule map_reads (line 80, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n\nLints for rule count_reads (line 122, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Param tsv_dir is a prefix of input or output file but hardcoded:\n      If this is meant to represent a file path prefix, it will fail when\n      running workflow in environments without a shared filesystem. Instead,\n      provide a function that infers the appropriate prefix from the input or\n      output file, e.g.: lambda w, input: os.path.splitext(input[0])[0]\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n      https://snakemake.readthedocs.io/en/stable/tutorial/advanced.html#tutorial-input-functions\n\nLints for rule merge_counts (line 156, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SNAKEDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule write_coldata (line 181, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule write_de_params (line 206, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Migrate long run directives into scripts or notebooks:\n      Long run directives hamper workflow readability. Use the script or\n      notebook direcive instead. Note that the script or notebook directive does\n      not involve boilerplate. Similar to run, you will have direct access to\n      params, input, output, and wildcards.Only use the run direcive for a\n      handful of lines.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#external-scripts\n      https://snakemake.readthedocs.io/en/latest/snakefiles/rules.html#jupyter-notebook-integration\n\nLints for rule de_analysis (line 228, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SNAKEDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\nLints for rule plot_dtu_res (line 261, /tmp/tmp_3z31mjw/Snakefile):\n    * No log directive defined:\n      Without a log directive, all output will be printed to the terminal. In\n      distributed environments, this means that errors are harder to discover.\n      In local environments, output of concurrent jobs will be mixed and become\n      unreadable.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#log-files\n    * Shell command directly uses variable SNAKEDIR from outside of the rule:\n      It is recommended to pass all files as input and output, and non-file\n      parameters via the params directive. Otherwise, provenance tracking is\n      less accurate.\n      Also see:\n      https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules\n\n",
    "stargazers_count": 64,
    "subscribers_count": 21,
    "topics": [
      "rna-seq",
      "cdna",
      "differential-gene-expression",
      "differential-transcript-usage",
      "nanopore"
    ],
    "updated_at": 1619176241.0
  },
  {
    "data_format": 1,
    "description": "Snakemake pipelines for nanopore sequencing data archiving and processing",
    "formatting": "[ERROR] In file \"/tmp/tmpxr2vflgb/rules/alignment.smk\":  \u0027InvalidParameterSyntax: 164config. get(\u0027threads_samtools\u0027) or1\n\u0027\n[ERROR] In file \"/tmp/tmpxr2vflgb/rules/methylation.smk\":  \u0027InvalidParameterSyntax: L111:Invalid key lambda wildcards, config\u0027\n[ERROR] In file \"/tmp/tmpxr2vflgb/rules/asm.smk\":  \u0027InvalidParameterSyntax: 46config. get(\u0027threads_asm\u0027) or1\n\u0027\n[ERROR] In file \"/tmp/tmpxr2vflgb/rules/install.smk\":  \u0027InvalidParameterSyntax: L431:Invalid key lambda wildcards, config\u0027\n[WARNING] In file \"/tmp/tmpxr2vflgb/rules/transcript.smk\":  Keyword \"input\" at line 37 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpxr2vflgb/Snakefile\":  \u0027InvalidPython: Got error:\n```\nCannot parse: 4:23:         print(* args , ** kwargs)\n```\nwhile formatting code with black.\u0027\n[INFO] 5 file(s) raised parsing errors \ud83e\udd15\n[INFO] 7 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "giesselmann/nanopype",
    "linting": "SyntaxError in line 208 of /tmp/tmpxr2vflgb/Snakefile:\nExpected name after module keyword. (Snakefile, line 208)\n",
    "stargazers_count": 70,
    "subscribers_count": 6,
    "topics": [
      "nanopore",
      "workflow",
      "snakemake-pipelines",
      "docker",
      "pipeline",
      "singularity"
    ],
    "updated_at": 1618899800.0
  },
  {
    "data_format": 1,
    "description": "a snakemake pipeline to process ChIP-seq files from GEO or in-house ",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "crazyhottommy/pyflow-ChIPseq",
    "linting": "SyntaxError:\nNot all output, log and benchmark files of rule make_inputSubtract_bigwigs contain the same wildcards. This is crucial though, in order to avoid that two or more jobs write to the same file.\n  File \"/tmp/tmpxkauul0e/Snakefile\", line 224, in \u003cmodule\u003e\n",
    "stargazers_count": 76,
    "subscribers_count": 8,
    "topics": [
      "chip-seq",
      "snakemake",
      "python"
    ],
    "updated_at": 1619402930.0
  },
  {
    "data_format": 1,
    "description": "PyPSA-Eur: An Open Optimisation Model of the European Transmission System",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "PyPSA/pypsa-eur",
    "linting": "Lints for snakefile /tmp/tmpkqg8_ye5/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n    * Path composition with \u0027+\u0027 in line 8:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 186:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n    * Path composition with \u0027+\u0027 in line 201:\n      This becomes quickly unreadable. Usually, it is better to endure some\n      redundancy against having a more readable workflow. Hence, just repeat\n      common prefixes. If path composition is unavoidable, use pathlib or\n      (python \u003e= 3.6) string formatting with f\"...\".\n      Also see:\n\n\nLints for rule retrieve_databundle (line 102, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_load_data (line 119, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_powerplants (line 136, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule base_network (line 164, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_shapes (line 204, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_bus_regions (line 241, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule retrieve_cutout (line 300, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule retrieve_natura_raster (line 341, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_renewable_profiles (line 358, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_hydro_profile (line 401, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_electricity (line 427, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule simplify_network (line 466, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule cluster_network (line 503, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule add_extra_components (line 544, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule prepare_network (line 575, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_network (line 622, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule solve_operations_network (line 657, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_network (line 694, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule make_summary (line 734, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_summary (line 754, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule plot_p_nom_max (line 780, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\nLints for rule build_country_flh (line 800, /tmp/tmpkqg8_ye5/Snakefile):\n    * Specify a conda environment or container for each rule.:\n      This way, the used software for each specific step is documented, and the\n      workflow can be executed on any machine without prerequisites.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#integrated-package-management\n      https://snakemake.readthedocs.io/en/latest/snakefiles/deployment.html#running-jobs-in-containers\n\n",
    "stargazers_count": 82,
    "subscribers_count": 10,
    "topics": [
      "snakemake"
    ],
    "updated_at": 1619341206.0
  },
  {
    "data_format": 1,
    "description": "Benchmarking analysis of data integration tools",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "theislab/scib",
    "linting": "KeyError in line 5 of /tmp/tmpu2olrejz/Snakefile:\n\u0027ROOT\u0027\n  File \"/tmp/tmpu2olrejz/Snakefile\", line 5, in \u003cmodule\u003e\n  File \"/tmp/tmpu2olrejz/scripts/snakemake_parse.py\", line 29, in __init__\n",
    "stargazers_count": 84,
    "subscribers_count": 11,
    "topics": [],
    "updated_at": 1619183818.0
  },
  {
    "data_format": 1,
    "description": "Light-weight Snakemake workflow for preprocessing and statistical analysis of RNA-seq data",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "csoneson/ARMOR",
    "linting": "Lints for snakefile /tmp/tmp6i8szdgy/Snakefile:\n    * Mixed rules and functions in same snakefile.:\n      Small one-liner functions used only once should be defined as lambda\n      expressions. Other functions should be collected in a common module, e.g.\n      \u0027rules/common.smk\u0027. This makes the workflow steps more readable.\n      Also see:\n      https://snakemake.readthedocs.io/en/latest/snakefiles/modularization.html#includes\n\n",
    "stargazers_count": 114,
    "subscribers_count": 11,
    "topics": [
      "rna-seq",
      "workflow"
    ],
    "updated_at": 1618651285.0
  },
  {
    "data_format": 1,
    "description": "A SingleCell RNASeq pre-processing snakemake workflow",
    "formatting": "[INFO] 14 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "Hoohm/dropSeqPipe",
    "linting": "WorkflowError in line 20 of /tmp/tmpr_k9eyen/Snakefile:\nConfig file config.yaml not found.\n  File \"/tmp/tmpr_k9eyen/Snakefile\", line 20, in \u003cmodule\u003e\n",
    "stargazers_count": 127,
    "subscribers_count": 7,
    "topics": [
      "drop-seq",
      "snakemake",
      "pipeline",
      "dropseq",
      "yaml",
      "star",
      "picard",
      "reference-genome",
      "umi",
      "scrbseq",
      "scrb-seq",
      "multiqc",
      "conda",
      "plot",
      "dropseqtools"
    ],
    "updated_at": 1614184764.0
  },
  {
    "data_format": 1,
    "description": "This Snakemake pipeline implements the GATK best-practices workflow",
    "formatting": null,
    "full_name": "snakemake-workflows/dna-seq-gatk-variant-calling",
    "linting": null,
    "stargazers_count": 138,
    "subscribers_count": 3,
    "topics": [
      "reproducibility",
      "snakemake",
      "sciworkflows"
    ],
    "updated_at": 1619098863.0
  },
  {
    "data_format": 1,
    "description": "RNA-seq workflow using STAR and DESeq2",
    "formatting": null,
    "full_name": "snakemake-workflows/rna-seq-star-deseq2",
    "linting": null,
    "stargazers_count": 160,
    "subscribers_count": 5,
    "topics": [
      "snakemake",
      "snakemake-workflows",
      "ngs",
      "rna-seq",
      "reproducible-science"
    ],
    "updated_at": 1618590929.0
  },
  {
    "data_format": 1,
    "description": "Bitextor generates translation memories from multilingual websites",
    "formatting": "[ERROR] In file \"/tmp/tmpj7y0ya6g/workflow/rules/dictdocsegalign.smk\":  \u0027InvalidParameterSyntax: 159c1=1 if DOCALIGN==\"DIC\" else2\n\u0027\n[WARNING] In file \"/tmp/tmpj7y0ya6g/workflow/Snakefile\":  Keyword \"input\" at line 1129 has comments under a value.\n\tPEP8 recommends block comments appear before what they describe\n(see https://www.python.org/dev/peps/pep-0008/#id30)\n[ERROR] In file \"/tmp/tmpj7y0ya6g/workflow/Snakefile\":  \u0027InvalidParameterSyntax: 1185lambda wildcards:2 if BICLEANER and ELRC else1\n\u0027\n[INFO] 2 file(s) raised parsing errors \ud83e\udd15\n[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "bitextor/bitextor",
    "linting": "KeyError in line 6 of /tmp/tmpj7y0ya6g/workflow/Snakefile:\n\u0027bitextor\u0027\n  File \"/tmp/tmpj7y0ya6g/workflow/Snakefile\", line 6, in \u003cmodule\u003e\n",
    "stargazers_count": 175,
    "subscribers_count": 25,
    "topics": [
      "document-aligner",
      "httrack",
      "apertium",
      "dictionaries",
      "crawler",
      "wget",
      "creepy",
      "hunalign",
      "sentence-segmentation",
      "tokenizer",
      "bicleaner",
      "tmx",
      "warc",
      "corpus-tools",
      "corpus-processing",
      "corpus-generator",
      "parallel-corpora",
      "machine-translation",
      "neural-machine-translation",
      "statistical-machine-translation"
    ],
    "updated_at": 1619039047.0
  },
  {
    "data_format": 1,
    "description": "A benchmark dataset for data-driven weather forecasting",
    "formatting": "[INFO] 1 file(s) would be changed \ud83d\ude2c\n",
    "full_name": "pangeo-data/WeatherBench",
    "linting": "KeyError in line 9 of /tmp/tmpmhnusvhs/Snakefile:\n\u0027datadir\u0027\n  File \"/tmp/tmpmhnusvhs/Snakefile\", line 9, in \u003cmodule\u003e\n",
    "stargazers_count": 238,
    "subscribers_count": 24,
    "topics": [
      "dataset",
      "benchmark",
      "weather-forecast",
      "deep-learning"
    ],
    "updated_at": 1619262920.0
  }
]
